
-- --------------------------------------------------------

--
-- Table structure for table `sample_annotation`
--

CREATE TABLE `sample_annotation` (
  `Sample_channel_count` int(1) DEFAULT NULL,
  `Sample_characteristics_ch1` varchar(215) DEFAULT NULL,
  `Sample_contact_address` varchar(78) DEFAULT NULL,
  `Sample_contact_city` varchar(14) DEFAULT NULL,
  `Sample_contact_country` varchar(14) DEFAULT NULL,
  `Sample_contact_department` varchar(60) DEFAULT NULL,
  `Sample_contact_email` varchar(32) DEFAULT NULL,
  `Sample_contact_institute` varchar(56) DEFAULT NULL,
  `Sample_contact_laboratory` varchar(36) DEFAULT NULL,
  `Sample_contact_name` varchar(23) DEFAULT NULL,
  `Sample_contact_phone` varchar(14) DEFAULT NULL,
  `Sample_contact_zip-postal_code` varchar(10) DEFAULT NULL,
  `Sample_data_processing` varchar(5172) DEFAULT NULL,
  `Sample_data_row_count` int(1) DEFAULT NULL,
  `Sample_description` varchar(208) DEFAULT NULL,
  `Sample_extract_protocol_ch1` varchar(5618) DEFAULT NULL,
  `Sample_geo_accession` varchar(10) DEFAULT NULL,
  `Sample_instrument_model` varchar(20) DEFAULT NULL,
  `Sample_last_update_date` varchar(11) DEFAULT NULL,
  `Sample_library_selection` varchar(5) DEFAULT NULL,
  `Sample_library_source` varchar(14) DEFAULT NULL,
  `Sample_library_strategy` varchar(7) DEFAULT NULL,
  `Sample_molecule_ch1` varchar(9) DEFAULT NULL,
  `Sample_organism_ch1` varchar(12) DEFAULT NULL,
  `Sample_platform_id` varchar(8) DEFAULT NULL,
  `Sample_relation` varchar(150) DEFAULT NULL,
  `Sample_series_id` varchar(23) DEFAULT NULL,
  `Sample_source_name_ch1` varchar(73) DEFAULT NULL,
  `Sample_status` varchar(21) DEFAULT NULL,
  `Sample_submission_date` varchar(11) DEFAULT NULL,
  `Sample_supplementary_file_1` varchar(129) DEFAULT NULL,
  `Sample_supplementary_file_2` varchar(132) DEFAULT NULL,
  `Sample_taxid_ch1` int(4) DEFAULT NULL,
  `Sample_title` varchar(90) DEFAULT NULL,
  `Sample_type` varchar(3) DEFAULT NULL,
  `reads_aligned` int(9) DEFAULT NULL,
  `reads_total` int(9) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

--
-- Dumping data for table `sample_annotation`
--

INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: HCC2157Xx-xXcell type: breast cancer cellsXx-xXtransfection: control siRNA', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113310', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259565Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997749', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113310/suppl/GSM1113310_HCC2157-siControl.read_cnt.txt.gz', '', 9606, 'HCC2157_RNAseq_Control', 'SRA', 35038107, 70066556),
(1, 'treatment: VehicleXx-xXreplicate: 2', '929 E 57th St W325D', 'Chicago', 'USA', '', 'lethien@uchicago.edu', 'University of Chicago', 'Geoffrey Greene', 'Thien,,Le', '', '60637', 'Alignment: Short-reads were aligned to the human reference genome (hg18, NCBI36), including autosomes, X-chromosome, and one complete copy of an rDNA repeat (GenBank ID: U13369.1). The SOAP2 software package was used to align reads with the following options: (1) all non-unique mappings were removed (-r 0), (2) three mismatches were allowed in each mapped read (-v 3), (3) low-quality reads with more than 10 ambiguous bases were removed (-N 10), and (4) for reads failing to align over the entire length of the read, the first 32 bp was used (-l 32). SOAP2 output was processed using custom Perl scripts, and imported into R for most the analysis.  In order to identify E2-regulated genes, we focused on RefSeq-annotated genes, counting reads in a fixed window between +1 kb and +13 kb relative to the transcription start site of each gene, so as to avoid possible complications introduced by paused polymerases, and to allow easy side-by-side comparison among samples. The normalized expression value of 1e-5 (normalized against the sample total read counts) was used as the threshold to select genes for further analyses. Comparing E2 treated samples versus vehicle treated samples, genes with FC>1.2 were considered upregulated; those with FC<0.8 were considered downregulated.Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: txt: read locations, count reads', 0, 'GROseq - Global run-on followed by sequencing', 'After 1 hour of E2 or Vehicle treatment, nuclei from C4-12/Flag-ERβ cells were extracted and processed with nuclear run-on assay.Xx-xXRun-on RNA was ligated with adapters and reverse transcribed to generate cDNA library.', 'GSM1171525', 'Illumina HiSeq 2000', 'Jun 21 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02209909Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX312097', 'GSE48159Xx-xXGSE48161', 'Immortalized human breast cancer cells', 'Public on Jun 21 2013', 'Jun 20 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1171nnn/GSM1171525/suppl/GSM1171525_GEOarchive_TL_GROseq_Veh_2.txt.gz', '', 9606, 'Veh_2', 'SRA', 16092350, 145225992),
(1, 'treatment: 10nM E2Xx-xXreplicate: 2', '929 E 57th St W325D', 'Chicago', 'USA', '', 'lethien@uchicago.edu', 'University of Chicago', 'Geoffrey Greene', 'Thien,,Le', '', '60637', 'Alignment: Short-reads were aligned to the human reference genome (hg18, NCBI36), including autosomes, X-chromosome, and one complete copy of an rDNA repeat (GenBank ID: U13369.1). The SOAP2 software package was used to align reads with the following options: (1) all non-unique mappings were removed (-r 0), (2) three mismatches were allowed in each mapped read (-v 3), (3) low-quality reads with more than 10 ambiguous bases were removed (-N 10), and (4) for reads failing to align over the entire length of the read, the first 32 bp was used (-l 32). SOAP2 output was processed using custom Perl scripts, and imported into R for most the analysis.  In order to identify E2-regulated genes, we focused on RefSeq-annotated genes, counting reads in a fixed window between +1 kb and +13 kb relative to the transcription start site of each gene, so as to avoid possible complications introduced by paused polymerases, and to allow easy side-by-side comparison among samples. The normalized expression value of 1e-5 (normalized against the sample total read counts) was used as the threshold to select genes for further analyses. Comparing E2 treated samples versus vehicle treated samples, genes with FC>1.2 were considered upregulated; those with FC<0.8 were considered downregulated.Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: txt: read locations, count reads', 0, 'GROseq - Global run-on followed by sequencing', 'After 1 hour of E2 or Vehicle treatment, nuclei from C4-12/Flag-ERβ cells were extracted and processed with nuclear run-on assay.Xx-xXRun-on RNA was ligated with adapters and reverse transcribed to generate cDNA library.', 'GSM1171527', 'Illumina HiSeq 2000', 'Jun 21 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02209911Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX312099', 'GSE48159Xx-xXGSE48161', 'Immortalized human breast cancer cells', 'Public on Jun 21 2013', 'Jun 20 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1171nnn/GSM1171527/suppl/GSM1171527_GEOarchive_TL_GROseq_E2_2.txt.gz', '', 9606, 'E2_2', 'SRA', 6378468, 138728263),
(1, 'treatment: 8 hour 100 nM bisphenol AXx-xXreplicate: 1Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937160', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149607Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998512', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937160/suppl/GSM937160_SL11542_refseq_rpkms.txt.gz', '', 9606, 'T47D_BPA_R1', 'SRA', 32395590, 51567448),
(1, 'cell line: MDA-MB-231Xx-xXcell type: breast cancer cellsXx-xXtransfection: control siRNA', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113308', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259563Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997747', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113308/suppl/GSM1113308_MDA231-siControl.read_cnt.txt.gz', '', 9606, 'MDAMB231_RNAseq_Control', 'SRA', 47132160, 74606138),
(1, 'cell line: T-47DXx-xXcell type: breast cancer cellsXx-xXtransfection: control siRNA', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113301', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259556Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997740', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113301/suppl/GSM1113301_T47D-siControl.read_cnt.txt.gz', '', 9606, 'T47D_RNAseq_Control', 'SRA', 62662384, 97821632),
(1, 'cell line: SUM159PTXx-xXcell type: breast cancer cellsXx-xXtransfection: siJARID1B', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113307', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259562Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997746', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113307/suppl/GSM1113307_SUM159-siJarid1b.read_cnt.txt.gz', '', 9606, 'SUM159_RNAseq_siJARID1B', 'SRA', 23808160, 84016705),
(1, 'cell line: T-47DXx-xXcell type: breast cancer cellsXx-xXtransfection: siJARID1B', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113302', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259557Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997741', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113302/suppl/GSM1113302_T47D-siJarid1b.read_cnt.txt.gz', '', 9606, 'T47D_RNAseq_siJARID1B', 'SRA', 30717467, 87518387),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cellsXx-xXtransfection: control siRNA', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113298', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259553Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997737', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113298/suppl/GSM1113298_MCF7-siControl.read_cnt.txt.gz', '', 9606, 'MCF7_RNAseq_Control', 'SRA', 47065363, 59892656),
(1, 'cell line: HCC2157Xx-xXcell type: breast cancer cellsXx-xXtransfection: siJARID1B', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113311', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259566Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997750', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113311/suppl/GSM1113311_HCC2157-siJarid1b.read_cnt.txt.gz', '', 9606, 'HCC2157_RNAseq_siJARID1B', 'SRA', 42153386, 74040354),
(1, 'cell line: SUM185PEXx-xXcell type: breast cancer cellsXx-xXtransfection: siCTCF', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113305', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259560Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997744', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113305/suppl/GSM1113305_SUM185-siCTCF.read_cnt.txt.gz', '', 9606, 'SUM185_RNAseq_siCTCF', 'SRA', 39808509, 106309600),
(1, 'treatment: 8 hour 10 nM 17β-estradiolXx-xXreplicate: 2Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937168', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149613Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998518', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937168/suppl/GSM937168_SL11548_refseq_rpkms.txt.gz', '', 9606, 'T47D_E2_R2', 'SRA', 31461552, 49858355),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cellsXx-xXtransfection: siJARID1B', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113299', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259554Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997738', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113299/suppl/GSM1113299_MCF7-siJarid1b.read_cnt.txt.gz', '', 9606, 'MCF7_RNAseq_siJARID1B', 'SRA', 41843670, 53738938),
(1, 'cell line: MDA-MB-231Xx-xXcell type: breast cancer cellsXx-xXtransfection: siJARID1B', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113309', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259564Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997748', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113309/suppl/GSM1113309_MDA231-siJarid1b.read_cnt.txt.gz', '', 9606, 'MDAMB231_RNAseq_siJARID1B', 'SRA', 46490503, 83925359),
(1, 'treatment: 8 hour 100 nM genisteinXx-xXreplicate: 1Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937164', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149610Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998515', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937164/suppl/GSM937164_SL11545_refseq_rpkms.txt.gz', '', 9606, 'T47D_GEN_R1', 'SRA', 31913762, 53479085),
(1, 'treatment: 8 hour 10 nM 17β-estradiolXx-xXreplicate: 1Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937163', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149609Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998514', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937163/suppl/GSM937163_SL11544_refseq_rpkms.txt.gz', '', 9606, 'T47D_E2_R1', 'SRA', 36405890, 62959511),
(1, 'treatment: 8 hour 0.02% DMSOXx-xXreplicate: 1Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937162', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149608Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998513', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937162/suppl/GSM937162_SL11543_refseq_rpkms.txt.gz', '', 9606, 'T47D_DMSO_R1', 'SRA', 39445878, 60338557),
(1, 'cell line: SUM185PEXx-xXcell type: breast cancer cellsXx-xXtransfection: siJARID1B', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113304', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259559Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997743', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113304/suppl/GSM1113304_SUM185-siJarid1b.read_cnt.txt.gz', '', 9606, 'SUM185_RNAseq_siJARID1B', 'SRA', 31017699, 84437646),
(1, 'treatment: no treatXx-xXexperiment: RNA-SeqXx-xXantibody: noXx-xXrestriction enzyme: no', '10900 Euclid Avenue', 'Cleveland', 'USA', '', 'fxj45@case.edu', 'Case Western Reserve University', 'Fulai Jin', 'Fulai,,Jin', '2163681811', '44106', 'The paired-end Hi-C reads were mapped to human genome hg18 using BOWTIE. Only first 36 bases were used for mapping when reads is longer. The two reads were mapped independently and then merged into pairs using in-house script. Duplicated read pairs from the same biological library were removed.Xx-xXFor GRO-seq, the first round of mapping used the first 36 bases; the unmapped reads were then mapped in two additional iterations using first 30 or 25 bases.Xx-xXFor ChIP-Seq, first 36 bases of each read were used for mapping.Xx-xXRNA-seq data were analyzed using TOPHAT and CUFFLINK packages.Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: Mapped non-redundant reads for RNA-seq, GRO-seq and ChIP-seq are in BED format. Mapped non-redundant read pairs for Hi-C are in txt format.', 0, '', 'Hi-C experiments were conducted in biological replicates using HindIII according to previous publication (Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science 326, 289-93 (2009).).  ChIP-seq protocol can be found from http://bioinformatics-renlab.ucsd.edu/RenLabLibraryProtocolV1.pdf. GRO-seq experiments were performed according to previous publication (Wang, D. et al. Reprogramming transcription by distinct classes of enhancers functionally defined by eRNA. Nature 474, 390-4 (2011).).Xx-xXLibraries were prepared according to Illumina\'s instruction or cited literatures.', 'GSM1154039', 'Illumina HiSeq 2000', 'Oct 20 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02191469Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX294966', 'GSE43070', 'breast cancer MCF7 cells', 'Public on Oct 20 2013', 'Jun 05 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1154nnn/GSM1154039/suppl/GSM1154039_rnaseq.MCF7.monoclonal.bed.gz', '', 9606, 'RNA-Seq MCF7', 'SRA', 29148252, 59060152),
(1, 'treatment: 8 hour 0.02% DMSOXx-xXreplicate: 2Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937167', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149612Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998517', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937167/suppl/GSM937167_SL11547_refseq_rpkms.txt.gz', '', 9606, 'T47D_DMSO_R2', 'SRA', 28086027, 45443844),
(1, 'treatment: 8 hour 100 nM bisphenol AXx-xXreplicate: 2Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937166', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149611Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998516', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937166/suppl/GSM937166_SL11546_refseq_rpkms.txt.gz', '', 9606, 'T47D_BPA_R2', 'SRA', 27328604, 45916920),
(1, 'treatment: 8 hour 100 nM genisteinXx-xXreplicate: 2Xx-xXcell line: T-47D', '601 Genome Way', 'Huntsville', 'USA', '', 'jgertz@hudsonalpha.com', 'HudsonAlpha Institute', '', 'Jason,,Gertz', '', '35806', 'To calculate expression levels in each library, sequence reads were aligned to a database of all spliced RefSeq transcripts using Bowtie with the following parameters:-n 2 -a -m 10 -X 3000. The number of reads aligning to each transcript was multiplied by 1 million, then divided by the length of the transcript times the total number of aligned reads to calculate RPKM values.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the official gene symbol, refseq id and RPKM value', 0, '', 'mRNA was harvested using an mRNA-direct kit (Invitrogen), followed by DNase treatment.  Non-directional libraries were then constructed as described in Gertz J, Varley KE, Davis NS, Baas BJ et al. Transposase mediated construction of RNA-seq libraries. Genome Res 2012 Jan;22(1):134-41.', 'GSM937170', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX149614Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00998519', 'GSE38234', 'Breast cancer cell line', 'Public on Jun 13 2012', 'May 24 2012', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM937nnn/GSM937170/suppl/GSM937170_SL11549_refseq_rpkms.txt.gz', '', 9606, 'T47D_GEN_R2', 'SRA', 28068298, 46203349),
(1, 'cell line: SUM159PTXx-xXcell type: breast cancer cellsXx-xXtransfection: control siRNA', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113306', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259561Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997745', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113306/suppl/GSM1113306_SUM159-siControl.read_cnt.txt.gz', '', 9606, 'SUM159_RNAseq_Control', 'SRA', 25037776, 80556921),
(1, 'cell line: SUM185PEXx-xXcell type: breast cancer cellsXx-xXtransfection: control siRNA', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113303', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259558Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997742', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113303/suppl/GSM1113303_SUM185-siControl.read_cnt.txt.gz', '', 9606, 'SUM185_RNAseq_Control', 'SRA', 33773185, 79419014),
(1, 'treatment: β-estradiol (100nM) 160minXx-xXexperiment: RNA-SeqXx-xXantibody: noXx-xXrestriction enzyme: no', '10900 Euclid Avenue', 'Cleveland', 'USA', '', 'fxj45@case.edu', 'Case Western Reserve University', 'Fulai Jin', 'Fulai,,Jin', '2163681811', '44106', 'The paired-end Hi-C reads were mapped to human genome hg18 using BOWTIE. Only first 36 bases were used for mapping when reads is longer. The two reads were mapped independently and then merged into pairs using in-house script. Duplicated read pairs from the same biological library were removed.Xx-xXFor GRO-seq, the first round of mapping used the first 36 bases; the unmapped reads were then mapped in two additional iterations using first 30 or 25 bases.Xx-xXFor ChIP-Seq, first 36 bases of each read were used for mapping.Xx-xXRNA-seq data were analyzed using TOPHAT and CUFFLINK packages.Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: Mapped non-redundant reads for RNA-seq, GRO-seq and ChIP-seq are in BED format. Mapped non-redundant read pairs for Hi-C are in txt format.', 0, '', 'Hi-C experiments were conducted in biological replicates using HindIII according to previous publication (Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science 326, 289-93 (2009).).  ChIP-seq protocol can be found from http://bioinformatics-renlab.ucsd.edu/RenLabLibraryProtocolV1.pdf. GRO-seq experiments were performed according to previous publication (Wang, D. et al. Reprogramming transcription by distinct classes of enhancers functionally defined by eRNA. Nature 474, 390-4 (2011).).Xx-xXLibraries were prepared according to Illumina\'s instruction or cited literatures.', 'GSM1154040', 'Illumina HiSeq 2000', 'Oct 20 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02191470Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX294967', 'GSE43070', 'breast cancer MCF7 cells', 'Public on Oct 20 2013', 'Jun 05 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1154nnn/GSM1154040/suppl/GSM1154040_rnaseq.MCF7_E2.monoclonal.bed.gz', '', 9606, 'RNA-Seq MCF7 + E2', 'SRA', 22008459, 45669530),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cellsXx-xXtransfection: siCTCF', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXRead counting and differential gene expression analysis were done using GFold  using significance cutoffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Text file (.cnt) of mapped read counts was generated using Gfold.', 0, '', 'Total RNA was isolated from cells using RNeasy Mini Kit (Qiagen) following the manufacturer’s protocol. Then polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).Xx-xXPolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification for 18 cycles, the final library underwent gel purification.', 'GSM1113300', 'Illumina HiSeq 2000', 'Jun 17 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX259555Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN01997739', 'GSE45732Xx-xXGSE46073', 'breast cancer cells', 'Public on Jun 17 2014', 'Apr 03 2013', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1113nnn/GSM1113300/suppl/GSM1113300_MCF7-siCTCF.read_cnt.txt.gz', '', 9606, 'MCF7_RNAseq_siCTCF', 'SRA', 45999214, 98371590);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401701', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821142Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559766', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401701/suppl/GSM1401701_SL10806_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401701/suppl/GSM1401701_SL10806_gene.fpkm_tracking.gz', 9606, '07-10-A110a', 'SRA', 33709536, 68486496),
(1, 'subtype: basalXx-xXcell line: MDAMB468Xx-xXagent: PKC412', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556181', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246799Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790668', 'GSE63721', 'Breast cancer cell line (basal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556181/suppl/GSM1556181_MDAMB468_PKC412_R3.csv.gz', '', 9606, 'MDAMB468_PKC412_R3', 'SRA', 27010043, 29866048),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401724', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821108Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559789', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401724/suppl/GSM1401724_SL7727_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401724/suppl/GSM1401724_SL7727_gene.fpkm_tracking.gz', 9606, '04-07-A416b', 'SRA', 57989899, 76376053),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401678', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821122Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559743', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401678/suppl/GSM1401678_SL10637_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401678/suppl/GSM1401678_SL10637_gene.fpkm_tracking.gz', 9606, '01-02-A056a', 'SRA', 42132947, 65248351),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 12h, ZNA, 30 uM', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431879', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905576Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648513', 'GSE59251', 'MCF-7 breast cancer cell line_12hZNA30uM-rep1', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '12hZNA30uM-rep1', 'SRA', 25143403, 27673110),
(1, 'cell line: SKBR5Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SKBR5_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897337', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293350Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122156', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897337/suppl/GSM1897337_SKBR5_FPKM.txt.gz', '', 9606, 'SKBR5', 'SRA', 38638235, 47627196),
(1, 'cell line: DU4475Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'DU4475_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897288', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293301Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122167', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897288/suppl/GSM1897288_DU4475_FPKM.txt.gz', '', 9606, 'DU4475', 'SRA', 33352778, 48086589),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401710', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821093Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559775', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401710/suppl/GSM1401710_SL10856_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401710/suppl/GSM1401710_SL10856_gene.fpkm_tracking.gz', 9606, '09-05-A135a', 'SRA', 39145383, 55568976),
(1, 'gender: FemaleXx-xXcell line: MDAMB436Xx-xXcell type: Basal-Like Carcinoma', '26, Rue d\'Ulm', 'Paris', 'France', '', 'Ronan.Chaligne@curie.fr', 'Institut Curie', 'Edith Heard\'s lab', 'Ronan,,Chaligné', '', '75248', 'Supplementary_files_format_and_content: Each sample contains three bedgraph files.Xx-xXSupplementary_files_format_and_content: 1. ChrX full bedgraph displayXx-xXSupplementary_files_format_and_content: 2. ChrX allele1 specific display of bedgraph only on SNP positionsXx-xXSupplementary_files_format_and_content: 3. ChrX allele2 specific display of bedgraph only on SNP positionsXx-xXSupplementary_files_format_and_content: Expression results: fpkm_tracking for genes standard output from cufflinksXx-xXSupplementary_files_format_and_content: Final SNP list: List of final SNPs derived from ChIP-seq, mRNA-seq and Exome-seq used for allele specific analysisXx-xXDatasets were subjected to two types of quality control. FASTQC-0.10.1 (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) was used to assess the quality of sequencing and potential adapter or cross contaminants. Average sequencing quality (phred score) per base was above 30 (Q ≥ 30) for all datasets. In addition, aligned datasets were then subjected toNGS-QC ((Mendoza-Parra et al., 2013); www.ngs-qc.org) to assess the robustness of enrichment. The majority of data sets were of «triple A» quality, no data set was below «triple B». For exome-seq and ChIP-seq, alignment was performed using BWA-MEM-0.7.7 (Li and Durbin, 2009) with default parameters, which simultaneously checks for both global and local alignment for reads. Alignment was followed by three sets of filters to prevent bias in the analysis. 1) duplicate reads (PCR clonal reads) were filtered out using Picard tools-1.86 (http://picard.sourceforge.net) 2) reads with mapping quality less than 10 were filtered out using Bamtools-2.2.3 (https://github.com/pezmaster31/bamtools) and 3) reads with more than one alignment reported were filtered out using in-house scripts. Further analysis was carried out on processed alignment file which is around 10-100M reads after these filters.Xx-xXTo prepare the allele information for each cell-line for allele-specific analysis down the line, SNP analysis was carried out along with SNP6 data. To identify novel variation (apart from known SNPs from SNP6 data), SNP analysis was carried out on all three ChIP-seq, Exome-seq and RNA-seq data individually. ChIP-seq data of different marks (H3K4me3, H3K27me3, RNA Pol II and Input) were merged for each cell line to increase the depth and confidence for variation calling. Variation calling was performed for each cell line separately for ChIP-seq and Exome-seq following the best practice GATK-2.6.5 pipeline by filtering reads with Mapping quality ≥ 1 (Van der Auwera GA et al., 2013; http://www.broadinstitute.org/gatk/guide/best-practices?bpm=DNAseq). Variation calling for RNA-seq was carried out following the methods of Piskol, Robert et al., 2013 to avoid artifacts specific to RNA- seq data (Piskol et al., 2013). A final list of allele information was generated by combining the SNP information from the different data sets for each cell line. To increase the allele-specific sensitivity for the alignment, reads were additionally realigned in an allele-specific manner following the method of Satya et al. (Satya et al., 2012). Read counts for each allele and SNP position were extracted for each mark using in-house scripts. SNP positions with at least three reads from both alleles were considered as heterozygous positions.Xx-xXPeak calling was performed using HOMER ((Heinz et al., 2010); http://homer.salk.edu/homer/index.html) with default parameters. For H3K27me3 and RNA Pol II, the\'style\' parameter was chosen as \'histone\' due to the broad patterns for this mark, whereas for H3K4me3, which generally give sharp peaks, the parameter \'factor\' was chosen. Genomic context annotation over identified peaks were carried out using the HOMER annotation module but with basic annotation by excluding references other than coding genes and non-coding RNA.Xx-xXA gene-based analysis of annotation integration was carried out using in-house scripts to integrate all annotation from peak and variation calling (informative SNP counts, read depths, homo/heterozygous SNP count and weighted allelic imbalance). To include annotations from regulatory regions 1Kb sequences upstream from the TSS and downstream of the TES were considered. A weighted arithmetic average was calculated for each gene by calculating average Allelic Imbalance (AI) where each SNP\'s AI was weighted by its read depth.Xx-xXTo illustrate the comparisons across cell lines, ChIP-seq data were normalized using an in-house developed tool called \'Epimetheus\', which is based on quantile normalization (manuscript under preparation). Read Count Intensity (RCI) was calculated for a window of 100bp bin size across chromosomes and then these intensities were normalized using quantile normalization from the limma package. The impact of normalization was assessed using MA plots before and after normalization. Specified genomic feature based normalized RCI was constructed, which are illustrated in Figure 6. For TSS-centered plots and heatmaps, a separate TSS-based normalization was carried out with 20bp bin size to obtain higher resolution.Xx-xXGenome_build: hg19', 0, '', 'Total RNA was extracted using the Trizol (Invitrogen) procedure.Xx-xXStandard Illumina protocols are followed.Xx-xXmRNA: total RNA was enriched for polyadenylated RNA (polyA mRNA).Xx-xXmRNA : RNA-sequencing correspond of paired-reads lane 2x100 bp sequencing on poly-A RNA purified  and were prepared according to the standard Illumina protocol. Sequencing has been performed on high-throughput Illumina HiSeq sequencer.', 'GSM1537306', 'Illumina HiSeq 2000', 'Jan 19 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03160885Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX750681', 'GSE62907Xx-xXGSE62966', 'Human Breast Cancer Cells', 'Public on Jan 19 2015', 'Nov 04 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537306/suppl/GSM1537306_MDAMB436-mRNA.chrX.SNPs.allele1.bedgraph.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537306/suppl/GSM1537306_MDAMB436-mRNA.chrX.SNPs.allele2.bedgraph.gz', 9606, 'MDAMB436_mRNA', 'SRA', 145828154, 167201050),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401743', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821069Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559808', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401743/suppl/GSM1401743_SL7931_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401743/suppl/GSM1401743_SL7931_gene.fpkm_tracking.gz', 9606, '02-03-A078a', 'SRA', 88739467, 112442692),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401726', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821117Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559791', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401726/suppl/GSM1401726_SL7730_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401726/suppl/GSM1401726_SL7730_gene.fpkm_tracking.gz', 9606, '01-09-A215b', 'SRA', 63168612, 84444477),
(1, 'cell line: EFM192AXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'EFM192A_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897290', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293303Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122171', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897290/suppl/GSM1897290_EFM19_FPKM.txt.gz', '', 9606, 'EFM192A', 'SRA', 32047032, 35388424),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401759', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821082Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559824', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401759/suppl/GSM1401759_SL9349_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401759/suppl/GSM1401759_SL9349_gene.fpkm_tracking.gz', 9606, '10-11-A138B', 'SRA', 33733899, 52390616),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 160 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533420', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145668Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745851', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533420/suppl/GSM1533420_MCF7_mRNA_t0160.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533420/suppl/GSM1533420_MCF7_premRNA_t0160min.wig.txt.gz', 9606, 'MCF7_E2_160min_RNASeq', 'SRA', 47913293, 98180531),
(1, 'cell line: 600MPEXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, '600MPE_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897277', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293290Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122216', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897277/suppl/GSM1897277_600MPE_FPKM.txt.gz', '', 9606, '600MPE', 'SRA', 29885371, 38017848);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 12h, ZNA, 30 uM', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431880', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905577Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648514', 'GSE59251', 'MCF-7 breast cancer cell line_12hZNA30uM-rep2', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '12hZNA30uM-rep2', 'SRA', 26953670, 29436486),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401742', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821053Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559807', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401742/suppl/GSM1401742_SL7928_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401742/suppl/GSM1401742_SL7928_gene.fpkm_tracking.gz', 9606, '05-09-A223a', 'SRA', 58500847, 87923089),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401755', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821078Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559820', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401755/suppl/GSM1401755_SL8362_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401755/suppl/GSM1401755_SL8362_gene.fpkm_tracking.gz', 9606, '04-07-A389b', 'SRA', 45110328, 56571123),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401757', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821083Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559822', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401757/suppl/GSM1401757_SL9347_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401757/suppl/GSM1401757_SL9347_gene.fpkm_tracking.gz', 9606, '04-08-A019a', 'SRA', 32175233, 44672494),
(1, 'cell line: SUM52Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM52_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897348', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293361Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122335', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897348/suppl/GSM1897348_SUM52_FPKM.txt.gz', '', 9606, 'SUM52', 'SRA', 30975071, 34581609),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401730', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821111Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559795', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401730/suppl/GSM1401730_SL7913_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401730/suppl/GSM1401730_SL7913_gene.fpkm_tracking.gz', 9606, '10-07-A138', 'SRA', 60179115, 85512430),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348545', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689294Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490456', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'BRD4 siRNA Veh rep1', 'SRA', 25802714, 31259132),
(1, 'cell line: MDAMB415Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB415_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897329', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293342Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122148', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897329/suppl/GSM1897329_MDAMB415_FPKM.txt.gz', '', 9606, 'MDAMB415', 'SRA', 31052510, 39822671),
(1, 'cell line: 1833Xx-xXsubtype: Triple negativeXx-xXmetastasis: Bone metastasisXx-xXknockdown: Both Abl and Arg', '308 Research Drive LSRC C231', 'Durham', 'USA', 'Pharmacology and Cancer Biology', '', 'Duke University', 'Pendergast lab', 'Jun,,Wang', '', '27710', 'Sequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg18 whole genome using tophatXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using cuffdiff based on the protocol Cole Trapnell et al. Nature Biotechnology 31, 46-53 (2013)Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: diff files include RPKM values for each Sample', 0, '', 'RNA was harvested using Qiagen Rneasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1693179', 'Illumina HiSeq 2000', 'Mar 02 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03704041Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1036355', 'GSE69125', 'breast cancer cells', 'Public on Mar 02 2016', 'May 21 2015', 'NONE', '', 9606, 'miAA', 'SRA', 35016839, 51651330),
(1, 'cell line: 1833Xx-xXsubtype: Triple negativeXx-xXmetastasis: Bone metastasisXx-xXknockdown: Arg', '308 Research Drive LSRC C231', 'Durham', 'USA', 'Pharmacology and Cancer Biology', '', 'Duke University', 'Pendergast lab', 'Jun,,Wang', '', '27710', 'Sequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg18 whole genome using tophatXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using cuffdiff based on the protocol Cole Trapnell et al. Nature Biotechnology 31, 46-53 (2013)Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: diff files include RPKM values for each Sample', 0, '', 'RNA was harvested using Qiagen Rneasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1693178', 'Illumina HiSeq 2000', 'Mar 02 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03704040Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1036354', 'GSE69125', 'breast cancer cells', 'Public on Mar 02 2016', 'May 21 2015', 'NONE', '', 9606, 'miArg', 'SRA', 32588017, 49486334),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401708', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821148Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559773', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401708/suppl/GSM1401708_SL10853_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401708/suppl/GSM1401708_SL10853_gene.fpkm_tracking.gz', 9606, '09-01-A214e', 'SRA', 35161130, 50253467),
(1, 'cell line: MFM223Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MFM223_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897333', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293346Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122152', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897333/suppl/GSM1897333_MFM223_FPKM.txt.gz', '', 9606, 'MFM223', 'SRA', 34032208, 37600580),
(1, 'cell line: 1833Xx-xXsubtype: Triple negativeXx-xXmetastasis: Bone metastasisXx-xXknockdown: Scramble', '308 Research Drive LSRC C231', 'Durham', 'USA', 'Pharmacology and Cancer Biology', '', 'Duke University', 'Pendergast lab', 'Jun,,Wang', '', '27710', 'Sequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg18 whole genome using tophatXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using cuffdiff based on the protocol Cole Trapnell et al. Nature Biotechnology 31, 46-53 (2013)Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: diff files include RPKM values for each Sample', 0, '', 'RNA was harvested using Qiagen Rneasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1693176', 'Illumina HiSeq 2000', 'Mar 02 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03704038Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1036352', 'GSE69125', 'breast cancer cells', 'Public on Mar 02 2016', 'May 21 2015', 'NONE', '', 9606, 'Scr', 'SRA', 42383450, 53923590),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: 200 nM INK128 for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of total RNA from cells treated with 200 nM INK128 for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503445', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067413Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699970', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'totRNA_ink128', 'SRA', 31148139, 405648954),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401738', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821060Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559803', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401738/suppl/GSM1401738_SL7923_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401738/suppl/GSM1401738_SL7923_gene.fpkm_tracking.gz', 9606, '05-05-A201a', 'SRA', 50397775, 73330508),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 5 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT10) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300629', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566884Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404079', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 5 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT10-dox-5', 'SRA', 104993933, 120971396),
(1, 'cell line: HCC1419Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1419_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897297', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293310Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122176', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897297/suppl/GSM1897297_HCC1419_FPKM.txt.gz', '', 9606, 'HCC1419', 'SRA', 38175337, 42319888),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 40 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533418', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145662Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745849', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533418/suppl/GSM1533418_MCF7_mRNA_t0040.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533418/suppl/GSM1533418_MCF7_premRNA_t0040min.wig.txt.gz', 9606, 'MCF7_E2_40min_RNASeq', 'SRA', 45628762, 93037228),
(1, 'cell line: MDA-MB-231 breast cancer cell lineXx-xXtreatment: PBS control-12hr', '3181 S.W. Sam Jackson Park Rd', 'Portland', 'USA', 'MMG', 'searsr@ohsu.edu', 'OHSU', '', 'Rosalie,,Sears', '5034946885', '97239-3098', 'Samples were sequenced on an Illumina HiSeq 2000 and assembled into fastq files using the CASAVA software at the OHSU Massively Parallel Sequencing Shared Resource.Xx-xXSequences were aligned to the human hg19 genome using Bowtie version 1.0.0 and seting requiring unique alignments with no more than 3 mismatches.Xx-xXTrim settings for Bowtie were selected to discard the first 4 bases, align to the next 44, and discard remaining bases from each sequence.Xx-xXAligned sequences that fell in exonic regions of RefSeq gene models were counted and normalized for sequencing depth.Xx-xXGenes that showed low normalized sequence counts for all samples were discarded from further analysis.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Tab-delimited .txt file containing normalized sequence counts for gene models before filtering out low expression genes.', 0, '', 'RNA was extracted from cells by Trozol reagent.Xx-xXTotal RNA was purified and reverse transcribed with oligo dT. Second strand DNA was synthesized and ligated to proprietary indexed adapters. Ligated material was amplified using Illumina genomic primers and sequenced on an Illumina GAII sequencer.', 'GSM1399413', 'Illumina HiSeq 2000', 'May 28 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02800680Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX554355', 'GSE58008', 'MDA-MB-231 breast cancer cell line', 'Public on May 28 2014', 'May 27 2014', 'NONE', '', 9606, 'cont-12hr', 'SRA', 14375062, 15757774),
(1, 'subtype: luminalXx-xXcell line: ZR75-1Xx-xXagent: DMSO', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556182', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246800Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790669', 'GSE63721', 'Breast cancer cell line (luminal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556182/suppl/GSM1556182_ZR751_DMSO_R1.csv.gz', '', 9606, 'ZR751_DMSO_R1', 'SRA', 37276739, 43200996);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MCF-7Xx-xXcell type: weakly invasiveXx-xXtransfected with: si-RESTXx-xXgenotype/variation: Downregulation of REST', '1501 San Pablo St.', 'Los Angeles', 'USA', 'Biomedical Engineering', 'nanlee@usc.edu', 'University of Southern California', '', 'Nan,Sook,Lee', '6268243058', '90033', 'Bcl conversion: Illumina bcl2Fastq ver 1.8.4 to convert from bcl format to fastq.gz raw reads fileXx-xXAlignment and read counting done with GTFAR ver 0.1 using gencode19 annotation.Xx-xXA second alignment was done with Tophat ver 1.4.0 using hg19_ucsc reference and parameter -G hg19_ucsc.gtf (optional reference annotation)Xx-xXTranscript abundance estimation on the second alignment done with Cufflinks ver 1.0.1 using parameters -u (for multi-read-correct), -b hg19_ucsc.fa (for bias detection and correction) and -G hg19_ucsc.gtf (for optional reference annotation)Xx-xXGenome_build: hg19_ucscXx-xXSupplementary_files_format_and_content: tab-delimited text files from GT-FAR runs and Cufflinks runs which include RPKM values for each sample and isoform counts for each sample; more details in the \'README_1.txt\'', 0, '', 'Total RNAs were isolated from cell lines using Trizol (Invitrogen) and digested with Turbo DNase (Ambion) to remove genomic DNA, according to the manufacturer’s instructions.Xx-xXLibraries from rRNA-depleted samples were prepared using a TruSeq RNA Sample Preparation kit v2 (Illumina) following the recommended protocol starting from the RNA fragmentation stage. Purification of polyadenylated RNA was omitted. Libraries were pooled (4 samples per pool), clustered on cBOT (Illumina), and sequenced on HiSeq2000, each pool in one lane. We performed single-end 100 bp reads. Reads were mapped using TopHat followed by data analysis using Cufflinks and Cuffdiff software, as well as our own pipeline.', 'GSM1553457', 'Illumina HiSeq 2000', 'Dec 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03220238Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX767448', 'GSE63608Xx-xXGSE63610', 'Breast cancer cell line', 'Public on Dec 01 2014', 'Nov 25 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553457/suppl/GSM1553457_GTFAR-MCF-7-si-2_geneRPKMS.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553457/suppl/GSM1553457_GTFAR-MCF-7-si-2_rawRestIsoformCnts.txt.gz', 9606, 'si-REST_2', 'SRA', 41406522, 99177630),
(1, 'cell line: HCC1599Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1599_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897301', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293314Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122180', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897301/suppl/GSM1897301_HCC1599_FPKM.txt.gz', '', 9606, 'HCC1599', 'SRA', 34242837, 41854224),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401741', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821063Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559806', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401741/suppl/GSM1401741_SL7927_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401741/suppl/GSM1401741_SL7927_gene.fpkm_tracking.gz', 9606, '09-06-A139A', 'SRA', 50173044, 78640642),
(1, 'cell line: EFM19Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'EFM19_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897289', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293302Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122168', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897289/suppl/GSM1897289_EFM192A_FPKM.txt.gz', '', 9606, 'EFM19', 'SRA', 20409572, 22477872),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401653', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820989Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559718', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401653/suppl/GSM1401653_SL5821_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401653/suppl/GSM1401653_SL5821_gene.fpkm_tracking.gz', 9606, 'MCF-7', 'SRA', 39176589, 53818524),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348551', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689303Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490462', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'JQ1 E2 rep1', 'SRA', 36151819, 40324890),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401671', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821006Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559736', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401671/suppl/GSM1401671_SL7544_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401671/suppl/GSM1401671_SL7544_gene.fpkm_tracking.gz', 9606, 'MDA-MB-157', 'SRA', 70509564, 85980268),
(1, 'cell line: MCF10AXx-xXcell type: Human normal breast epithelial cellsXx-xXpassages: 5', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized count values for each Sample.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1874955', 'Illumina HiSeq 2000', 'Oct 06 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04046657Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1224642', 'GSE72932Xx-xXGSE72933', 'MCF10A normal breast epithelial cells breast cancer cells', 'Public on Oct 06 2016', 'Sep 11 2015', 'NONE', '', 9606, 'BRD4 siRNA rep2', 'SRA', 29282061, 33532479),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401721', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821107Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559786', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401721/suppl/GSM1401721_SL7723_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401721/suppl/GSM1401721_SL7723_gene.fpkm_tracking.gz', 9606, '03-10-A451c', 'SRA', 65613938, 107108392),
(1, 'cell line: MCF-7Xx-xXer status: ER+Xx-xXconfluency: LowXx-xXmi63 treatment: Yes', '8001 Redwood Blvd', 'Novato', 'USA', '', 'cyau@buckinstitute.org', 'Buck Institute of Age Research', 'Benz Lab', 'Christina,,Yau', '415-209-2248', '94945', 'Resulting reads were aligned to the transcriptome defined by UCSC hg19 Known Genes using the RNA-Seq pipeline RSEM v1.2.9 and Bowtie v1.0.0Xx-xXRSEM estimated Fragments Per Kilobase Mapped (FPKMs) and Transcripts Per Million (TPMs) per transcript and isoform using an expectation maximization algorithm.Xx-xXTPMs for individual read groups were used for downstream differential expression analysesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include TPM values of UCSC known genes for each sample from 3 read groups', 0, '', 'Total RNA was extracted using the RNeasy mini kit (Qiagen, CA) and assayed using a 2100 Bioanalyzer (Agilent, CA).Xx-xX~ 0.5-1.0 µg cDNA from each was first sheared to a size ranging between 200-300 bp using the Covaris-S2 sonicator (Covaris, Woburn, MA) according to the manufacturer’s recommended protocols, and the fragmented cDNA samples were then prepared for whole transcriptome library generation using the TruSeq v1 Multiplex Sample Preparation kit (Illumina, San Diego, CA). Briefly, cDNA fragments were end-repaired, dA-tailed and ligated to multiplex adapters according to manufacturer\'s instructions. After ligation, DNA fragments smaller than 150 bp were removed with AmPure XP beads (Beckman Coulter Genomics, Danvers, MA). The purified adapter ligated products were enriched using polymerase chain reaction (14 cycles). The final amplified libraries were resolved on 2.0% agarose gel and manually size-selected in the range of 350–380 bp. The final RNA-Seq libraries were quantitated using the Agilent 2100 Bioanalyzer and pooled together in equal concentration for sequencing.', 'GSM1891972', 'Illumina HiSeq 2000', 'Oct 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04108563Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1281666', 'GSE73370Xx-xXGSE73371', 'Breast Cancer Cell Line', 'Public on Oct 26 2015', 'Sep 23 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1891nnn/GSM1891972/suppl/GSM1891972_MI63_treated_MCF7_geneCount.txt.gz', '', 9606, 'MI63_treated_MCF7', 'SRA', 47127674, 157839288),
(1, 'cell line: SUM229Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM229_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897346', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293359Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122333', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897346/suppl/GSM1897346_SUM229_FPKM.txt.gz', '', 9606, 'SUM229', 'SRA', 41844157, 46910219),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401682', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821127Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559747', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401682/suppl/GSM1401682_SL10644_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401682/suppl/GSM1401682_SL10644_gene.fpkm_tracking.gz', 9606, '01-05-A181a', 'SRA', 34695524, 68140610),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348549', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689298Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490460', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'JQ1 Veh rep1', 'SRA', 35388725, 42327174),
(1, 'cell line: T47DXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'T47D_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897350', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293363Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122337', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897350/suppl/GSM1897350_T47D_FPKM.txt.gz', '', 9606, 'T47D', 'SRA', 32107213, 35081205),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401663', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821016Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559728', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401663/suppl/GSM1401663_SL6577_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401663/suppl/GSM1401663_SL6577_gene.fpkm_tracking.gz', 9606, 'DY36T2', 'SRA', 57576778, 138323261),
(1, 'cell line: HCC1187Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1187_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897295', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293308Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122170', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897295/suppl/GSM1897295_HCC1187_FPKM.txt.gz', '', 9606, 'HCC1187', 'SRA', 22054609, 24237432),
(1, 'cell line: CAL851Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'CAL851_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897286', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293299Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122165', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897286/suppl/GSM1897286_CAL851_FPKM.txt.gz', '', 9606, 'CAL851', 'SRA', 31435860, 39389606),
(1, 'cell line: MCF12AXx-xXcell type: breast cancer cancer cell lineXx-xXstable expression: empty vector', '12801 E. 17th Ave.', 'Aurora', 'USA', 'Medicine', 'Jihye.Kim@UCDenver.edu', 'UCDenver', 'Tan', 'Jihye,,Kim', '303-724-8229', '80016', 'Alignment: TopHat v1.4.1 using default settings. hg19 (UCSC).Xx-xXExpression levels: Cufflinks v1.3.0 and cuffdiff using default settings.Xx-xXGSEA anlalysis with GeneSigDB v4.Xx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Tab-delimited text file with FPKM values.', 0, 'mRNA', 'Total RNA was extracted and mRNA isolated using the miRNeasy Kit (Qiagen) according to manufacturer\'s instructions.Xx-xXRNA libraries were prepared for sequencing using TruSEQ RNA Sample Prep v2 using standard Illumina protocols.', 'GSM1317636', 'Illumina HiSeq 2000', 'Feb 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02598926Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX451744', 'GSE54505', 'breast cancer cancer cell line, control', 'Public on Feb 26 2015', 'Jan 29 2014', 'NONE', '', 9606, 'EV1', 'SRA', 51958872, 59643960);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401686', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821128Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559751', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401686/suppl/GSM1401686_SL10653_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401686/suppl/GSM1401686_SL10653_gene.fpkm_tracking.gz', 9606, '02-04-A362a', 'SRA', 53733823, 66181050),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401732', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821120Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559797', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401732/suppl/GSM1401732_SL7915_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401732/suppl/GSM1401732_SL7915_gene.fpkm_tracking.gz', 9606, '10-10-A177A', 'SRA', 41078576, 69295769),
(1, 'cell line: MDAMB436Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB436_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897330', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293343Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122149', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897330/suppl/GSM1897330_MDAMB436_FPKM.txt.gz', '', 9606, 'MDAMB436', 'SRA', 44430322, 49004332),
(1, 'cell line: SKBR7Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SKBR7_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897338', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293351Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122344', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897338/suppl/GSM1897338_SKBR7_FPKM.txt.gz', '', 9606, 'SKBR7', 'SRA', 27687609, 34716812),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47-I281fsXx-xXdoxycycline concentration: 0 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (MUT3) of the MDA231 breast cancer cell line transduced with doxycycline-inducible RBM47-I281fs mutant.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300631', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566885Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404081', 'GSE53779', 'breast cancer cell line clone, RBM47-I281fs mutant, 0 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'MUT3-dox-0', 'SRA', 35444098, 42804012),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401685', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821129Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559750', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401685/suppl/GSM1401685_SL10650_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401685/suppl/GSM1401685_SL10650_gene.fpkm_tracking.gz', 9606, '02-01-A013a', 'SRA', 37151715, 63122749),
(1, 'cell line: HCC2218Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC2218_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897307', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293320Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122186', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897307/suppl/GSM1897307_HCC2218_FPKM.txt.gz', '', 9606, 'HCC2218', 'SRA', 36334821, 40930419),
(1, 'cell line: MDAMB361Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB361_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897328', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293341Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122147', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897328/suppl/GSM1897328_MDAMB361_FPKM.txt.gz', '', 9606, 'MDAMB361', 'SRA', 34132753, 37513048),
(1, 'cell line: MCF12AXx-xXcell type: breast cancer cancer cell lineXx-xXstable expression: miR-424 over-expressed', '12801 E. 17th Ave.', 'Aurora', 'USA', 'Medicine', 'Jihye.Kim@UCDenver.edu', 'UCDenver', 'Tan', 'Jihye,,Kim', '303-724-8229', '80016', 'Alignment: TopHat v1.4.1 using default settings. hg19 (UCSC).Xx-xXExpression levels: Cufflinks v1.3.0 and cuffdiff using default settings.Xx-xXGSEA anlalysis with GeneSigDB v4.Xx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Tab-delimited text file with FPKM values.', 0, 'mRNA', 'Total RNA was extracted and mRNA isolated using the miRNeasy Kit (Qiagen) according to manufacturer\'s instructions.Xx-xXRNA libraries were prepared for sequencing using TruSEQ RNA Sample Prep v2 using standard Illumina protocols.', 'GSM1317641', 'Illumina HiSeq 2000', 'Feb 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02598925Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX451749', 'GSE54505', 'breast cancer cancer cell line, over-expressing miR-424', 'Public on Feb 26 2015', 'Jan 29 2014', 'NONE', '', 9606, 'miR424_3', 'SRA', 52073028, 58535043),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401695', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821139Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559760', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401695/suppl/GSM1401695_SL10793_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401695/suppl/GSM1401695_SL10793_gene.fpkm_tracking.gz', 9606, '04-08-A204c', 'SRA', 40619637, 63762671),
(1, 'cell line: HCC518Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1008_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897293', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293306Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122174', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897293/suppl/GSM1897293_HCC1008_FPKM.txt.gz', '', 9606, 'HCC1008', 'SRA', 38896144, 49519344),
(1, 'cell line: MCF7+BORIS (Clone2, single-cell clone of MCF7 cell, stably expressing BORIS)', '5640 Fishers Lane', 'Rockville', 'USA', 'NIAID', 'epugacheva@niaid.nih.gov', 'NIH', 'LIG', 'Elena,,Pugacheva', '', '20852', 'Sequences generated by the Illumina genome analyzer (36bp and 50bp reads) were aligned against either human (build hg19) or mouse (build mm9) genome using Bowtie program (http://bowtie-bio.sourceforge.net).  The alignment was performed with default parameters except the sequence tags that mapped to more than one location in the genome were excluded from the analysis using –m1 option.Xx-xXPeaks were called using Model-based Analysis for ChIP-Seq (MACS) (http://liulab.dfci.harvard.edu/MACS) using default parameters. After MACS, we applied Peak Splitter algorithm (part of MACS) to call sub-peaks, summit of peaks and improve peak resolution.Xx-xXFor the Illumina RNA-seq library, fastq.gz files were mapped to UCSC Human reference (build hg19) using TopHat2 with the default parameter setting of 20 alignments per read and up to 2 mismatches per alignment.Xx-xXFor Ion Torrent RNA-seq sequencing, fastq.gz files were mapped to UCSC Human reference (build hg19) using two-step alignments. First, the reads were aligned with TopHat2. Second, the unmapped reads from the first step are then extracted and aligned with Bowtie2 with –local mode and --very-sensitive-local option.Xx-xXFor both Illumina and Ion Torrent RNA-seq, the resulting aligned reads (BAM files) were then analyzed with Cufflinks 2.0.0 to estimate the transcripts relative abundance using the UCSC reference annotated transcripts (build hg19). The expression of each transcript was quantified as the number of reads mapping to a transcript divided by the transcript length in kilobases and the total number of mapped reads in millions (FPKM).Xx-xXGenome_build: hg19, mm9Xx-xXSupplementary_files_format_and_content: Cuffdiff output.', 0, '', 'After overnight incubation with DiaMag magnetic beads (Diagenode, Inc.) and CTCF or BORIS monoclonal or polyclonal antibodies (characterized and described by PMID:15731119; PMID: 21659515; PMID:20231363), precipitated chromatin was then washed, decrosslinked, and digested with proteinase K. The resulting DNA was purified using phenol/chloroform/isoamyl alcohol. DNA concentration was assessed with Quant‐it PicoGreen dsDNA kit (Invitrogen) and 5–10 ng was used to generate sequencing libraries.Xx-xXTruSeq ChIP Sample Preparation Kit (Illumina, Inc., USA). RNA-seq libraries were prepared using either Ion Total RNA-seq v2 kit or TruSeq Stranded RNA LT Kit (Illumina).', 'GSM1817677', 'Illumina HiSeq 2000', 'Jul 24 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03854463Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1091836', 'GSE70764', 'Human breast cancer cell line', 'Public on Jul 23 2015', 'Jul 10 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1817nnn/GSM1817677/suppl/GSM1817677_RNA_Seq_MCF7+EV_VS_MCF7+BORIS_Clone2.txt.gz', '', 9606, 'MCF7+BORIS_Clone2_RNA_Seq', 'SRA', 7715962, 13068223),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401665', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821013Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559730', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401665/suppl/GSM1401665_SL6579_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401665/suppl/GSM1401665_SL6579_gene.fpkm_tracking.gz', 9606, 'HCC-1954', 'SRA', 45690749, 72012932),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401705', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821092Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559770', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401705/suppl/GSM1401705_SL10811_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401705/suppl/GSM1401705_SL10811_gene.fpkm_tracking.gz', 9606, '08-02-A355a', 'SRA', 37905639, 49934090),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401706', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821091Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559771', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401706/suppl/GSM1401706_SL10813_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401706/suppl/GSM1401706_SL10813_gene.fpkm_tracking.gz', 9606, '08-08-A142a', 'SRA', 37690280, 70393294),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: DMSO for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with DMSO for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503442', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067410Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699967', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_dmso', 'SRA', 26544153, 387551359),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: 25 nM Silvestrol for 1 hour', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with 25 nM Silvestrol for 1 hour', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503435', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067403Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699960', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_silvestrol_1hr', 'SRA', 33713779, 219904127);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401694', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821137Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559759', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401694/suppl/GSM1401694_SL10791_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401694/suppl/GSM1401694_SL10791_gene.fpkm_tracking.gz', 9606, '04-07-A432a', 'SRA', 28942079, 54712464),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 2.5 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT10) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300628', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566882Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404078', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 2.5 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT10-dox-2.5', 'SRA', 42127097, 47739967),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401667', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820997Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559732', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401667/suppl/GSM1401667_SL6582_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401667/suppl/GSM1401667_SL6582_gene.fpkm_tracking.gz', 9606, 'MDA-MB-453', 'SRA', 28248282, 103253446),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348550', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689301Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490461', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'JQ1 Veh rep2', 'SRA', 36979313, 43269576),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 20 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533417', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145655Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745848', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533417/suppl/GSM1533417_MCF7_mRNA_t0020.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533417/suppl/GSM1533417_MCF7_premRNA_t0020min.wig.txt.gz', 9606, 'MCF7_E2_20min_RNASeq', 'SRA', 44463591, 90513713),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401722', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821097Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559787', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401722/suppl/GSM1401722_SL7725_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401722/suppl/GSM1401722_SL7725_gene.fpkm_tracking.gz', 9606, '02-06-A195a', 'SRA', 31799480, 52025029),
(1, 'cell line: 184B5Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, '184B5_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897276', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293289Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122215', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897276/suppl/GSM1897276_184B5_FPKM.txt.gz', '', 9606, '184B5', 'SRA', 31983791, 41217242),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401718', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821105Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559783', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401718/suppl/GSM1401718_SL13094_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401718/suppl/GSM1401718_SL13094_gene.fpkm_tracking.gz', 9606, '03-09-A268c', 'SRA', 38706533, 74026808),
(1, 'transfection: oct4 plasmidXx-xXcell line: MDA-MB-231Xx-xXcell type: epithelial cellXx-xXdisease: adenocarcinomaXx-xXethnicity: caucasian', '94 Weijin Road', 'Tianjin', 'China', '', 'sxsl818@163.com', 'Medical School of Nankai University', '', 'Long,,Shen', '15202209080', '300071', 'Base-calling: OLB-1.9.4Xx-xXData filtering: R&D Perl Remove reads with adapters; Remove reads in which unknown bases are more than 5%;Remove low quality reads (the percentage of low quality bases is over 50% in a read, we define the low quality base to be the base whose sequencing quality is no more than 10).Xx-xXRead alignment: SOAP2.20 -m 0 -x 1000 -s 40 -l 32 -v 5 -r 2 -p 3Xx-xXDEGs detection: R&D Perl FDR ≤ 0.001 and the absolute value of Log2Ratio ≥ 1Xx-xXCluster Analysis: Cluster 3.0  -e 7 -g 7 -m aXx-xXGenome_build: Build Genome: hg19Xx-xXSupplementary_files_format_and_content: RPKM', 0, '', 'Total RNA was harvested using Trizol reagent.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1314288', 'Illumina HiSeq 2000', 'Jan 27 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02596094Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX449619', 'GSE54392', 'Breast cancer cells', 'Public on Jan 25 2014', 'Jan 24 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1314nnn/GSM1314288/suppl/GSM1314288_231-oct4A.Gene.rpkm.txt.gz', '', 9606, '231-oct4A', 'SRA', 4854865, 6114218),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401697', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821141Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559762', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401697/suppl/GSM1401697_SL10797_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401697/suppl/GSM1401697_SL10797_gene.fpkm_tracking.gz', 9606, '05-01-A256a', 'SRA', 33008437, 51196435),
(1, 'cell line: T47DXx-xXsmall interfering knock-down: si-CBX3Xx-xXtime after r5020: 0 hours', 'Dr. Aiguader, 88', 'Barcelona', 'Spain', '', 'francois.ledily@crg.eu', 'CRG-Centre for Genomic Regulation', '', 'François,,Le Dily', '', '8003', 'RNA-seq reads were trimmed using Trimmomatic to remove sequencing adapters with parameters: sequencing type=PE, seedMismatches=2, palindromeClipThreshold=30, simpleClipThreshold=12, minAdapterLength=1, keepBothReads=True, minQual=3, minLength=36 (all other parameters as default)Xx-xXRNA-seq reads were aligned to hg19 using STAR with parameters: outFilterType \"BySJout\", outFilterMultimapNmax 20, alignSJoverhangMin 8, alignSJDBoverhangMin 1, outFilterMismatchNmax 999, outFilterMismatchNoverLmax 0.04, alignIntronMin 20, alignIntronMax 1000000, alignMatesGapMax 1000000 (all other parameters as default)Xx-xXGene expression was measured as per-gene read counts using featureCounts treating reads as reversely stranded (all other parameters as default)Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: txt files with gene raw read counts (featureCounts standard output format)', 0, 'small interfering (si) knock-down of CBX3 before treatment with R5020 (synthetic analog of the progesterone) (replicate #1)', 'Extraction protocol not provided.Xx-xXLibrary construction protocol not provided.', 'GSM2218825', 'Illumina HiSeq 2000', 'Jul 11 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05296737Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1880872', 'GSE83785', 'Breast cancer cells', 'Public on Jul 11 2016', 'Jun 27 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2218nnn/GSM2218825/suppl/GSM2218825.txt.gz', '', 9606, 'RNA-seq si-CBX3 T0 rep1', 'SRA', 71538995, 76480320),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 1 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT10) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300627', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566881Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404077', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 1 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT10-dox-1', 'SRA', 43891180, 50443009),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348546', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689293Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490457', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'BRD4 siRNA Veh rep2', 'SRA', 24963629, 30249570),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401674', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821007Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559739', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401674/suppl/GSM1401674_SL7547_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401674/suppl/GSM1401674_SL7547_gene.fpkm_tracking.gz', 9606, 'HCC-1569', 'SRA', 60085301, 99143391),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47-I281fsXx-xXdoxycycline concentration: 1 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (MUT3) of the MDA231 breast cancer cell line transduced with doxycycline-inducible RBM47-I281fs mutant.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300632', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566889Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404082', 'GSE53779', 'breast cancer cell line clone, RBM47-I281fs mutant, 1 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'MUT3-dox-1', 'SRA', 39852587, 45942689),
(1, 'cell line: MCF7', '9 Divinity Ave', 'Cambridge', 'USA', 'Stem Cell and Regenerative Biology', '', 'Harvard University', '', 'John,,Rinn', '', '2138', 'RNA-seq: For  RNA-Seq analysis, first, the adapter sequences were removed from the RNA-seq reads.  Ribosomal RNA reads were then filtered out using Bowtie (Song et al. 2014).  After these filtration and adapter removal steps, the reads were aligned to a transciptome and quantified using RSEM v1.2.7 (Li and Dewey 2011). Differential gene expression was calculated by using the Deseq2 package in R 3.1.0 by using the mean value of gene-wise dispersion estimates (Love et al. 2014).Xx-xXChIP-seq: ~1x107 parental MCF-7 cells were crosslinked with formaldehyde at room temperature for 10 minutes.  Then, the cells were lysed using lysis buffer A (50mM HEPES, 140mM NaCl, 1mM EDTA pH=8, 10% Glycerol, 0.5% NP-40, 0.25% Triton X-100), and the residual cytoplasmic protein was removed using lysis buffer B (10mM Tris-HCl pH=8, 200mM NaCl, 1mM EDTA, 1mM EGTA).  The nuclear fraction was released using lysis buffer C (10mM Tris-Hcl pH=8, 100mM NaCl, 1mM EDTA, 1mM EGTA, 0.1% Sodium Deoxycholate, 0.5% N-lauroylsarcosine).  The chromatin was then sheared by using a Covaris S2 instrument with 10% duty cycle, 5 intensity, 200 cycles per burst, frequency sweeping mode, 60 second process time and for 4 cycles. The pull-down was performed by using the RUNX1 antibody (Cell Signaling #4334).  Samples were washed three times with RIPA buffer (Tris-HCl pH=8, 150mM NaCl, 1mM EDTA, 1% NP-40, 0.25% Sodium deoxycholate, 0.1% SDS) and were eluted.  The pull-down and input control sequencing libraries were generated by using the NEXTflex Rapid DNA Sequencing Kit (Bioo Scientific #5144-02) and were sequenced by using SE100 reads with a HiSeq 2000 instrument.Xx-xXdata processing step: The Hi-C data were mapped with Bowtie and  binned at 6.5Mb, 1Mb, 250kb, 100kb and 40kb non-overlapping genomic intervals.  Iterative mapping and correction of Hi-C data were performed as previously described (Imakaev et al. 2012).  Biological replicates showed high reproducibility (Pearson\'s correlation coefficient > 0.9 for 1Mb resolution data).  Similarly, the first eigenvector comparison of the replicates showed high reproducibility.  For the downstream analyses, sequences obtained from both biological replicates were pooled and ICE-corrected to serve as a combined dataset.Xx-xXCompartment analysis: First, the z-scores of the interaction matrices at 250kb resolution were generated as described previously (Lieberman, 2009).  Then, Pearson Correlation on the Z-score matrices was calculated.  By performing principal component analysis (Lieberman-Aiden et al. 2009; Zhang et al. 2012), the first eigenvector typically represents the compartment profile(Lieberman-Aiden et al. 2009), where the positive and negative 1st eigenvalues represent different compartments.   Gene density for each compartment was calculated to call the “A” and “B” compartmentalization.Xx-xXTAD Calling : TAD calling was performed as calculating the “insulation” score of each bin using the 40kb resolution combined Hi-C data (as previously described.  The mean of the interactions across each bin were calculated.  By sliding a 1Mb x 1Mb (25bins x 25bins) square along the diagonal of the interaction matrix for every chromosome, we obtained the “insulation score” of the interaction matrix.  Valleys in the insulation score indicated the depletion of Hi-C interactions occurring across a bin.  These 40kb valleys represent the TAD boundaries.  Based on the variation of boundaries between replicates, we chose to add a total of 160kb (80kb to each side) to the boundary to account for replicate variation.  The final boundaries span a 200kb region. All boundaries with a boundary strength < 0.15 were excluded as they were considered weak and non-reproducible.  The insulation plots for the biological replicates showed high reproducibility, suggesting the robustness of the method.  Similarly, the overlap of detected boundaries also showed high reproducibility between the biological replicates.  Therefore, we used the combined Hi-C replicates for the TAD analyses.Xx-xXZ-Score Calculation: We modeled the overall Hi-C decay with distance using a modified LOWESS method (alphaXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: HiC: Interaction matrices at 2.5Mb, 1Mb, 250kb and 40kb resolution; RNA-seq: Expected count file for all genes and a txt file showing the shRUNX1/shNS log2 Fold Change and pvalues for all genes; ChIP-seq: BedGeapgh UCSC visualization files and the MCF7 RUNX1 peak bed fileXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_processed_HiC_files - all normalized/corrected Hi-C data matrices binned at 40kb/250kb/1Mb/2.5MbXx-xXSupplementary_files_format_and_content: MCF7_RUNX1_ChIPseq_peaks.bed - BED file of RUNX1 ChIP-seq peaksXx-xXSupplementary_files_format_and_content: MCF7_shNS_shRUNX1_genes_expression_expected_count.tsv - TSV file showing the RNA-seq counts for each geneXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_RNAseq_log2_foldchange.txt - TSV file showing the log2 fold change of all  genes between shRUNX1 and shNS samples', 0, '', 'Hi-C was sequenced using PE100 reads; RNA-seq and ChIP-seq were sequenced using SE100 reads by using a HiSeq 2000 instrument', 'GSM1942105', 'Illumina HiSeq 2000', 'Aug 23 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04272126Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1434846', 'GSE75070', 'Human breast cancer cell line', 'Public on Aug 23 2016', 'Nov 16 2015', 'NONE', '', 9606, 'MCF-7 shNS  RNA-seq Replicate2', 'SRA', 13245530, 25028273);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MCF-7Xx-xXer status: ER+Xx-xXconfluency: LowXx-xXmi63 treatment: No', '8001 Redwood Blvd', 'Novato', 'USA', '', 'cyau@buckinstitute.org', 'Buck Institute of Age Research', 'Benz Lab', 'Christina,,Yau', '415-209-2248', '94945', 'Resulting reads were aligned to the transcriptome defined by UCSC hg19 Known Genes using the RNA-Seq pipeline RSEM v1.2.9 and Bowtie v1.0.0Xx-xXRSEM estimated Fragments Per Kilobase Mapped (FPKMs) and Transcripts Per Million (TPMs) per transcript and isoform using an expectation maximization algorithm.Xx-xXTPMs for individual read groups were used for downstream differential expression analysesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include TPM values of UCSC known genes for each sample from 3 read groups', 0, '', 'Total RNA was extracted using the RNeasy mini kit (Qiagen, CA) and assayed using a 2100 Bioanalyzer (Agilent, CA).Xx-xX~ 0.5-1.0 µg cDNA from each was first sheared to a size ranging between 200-300 bp using the Covaris-S2 sonicator (Covaris, Woburn, MA) according to the manufacturer’s recommended protocols, and the fragmented cDNA samples were then prepared for whole transcriptome library generation using the TruSeq v1 Multiplex Sample Preparation kit (Illumina, San Diego, CA). Briefly, cDNA fragments were end-repaired, dA-tailed and ligated to multiplex adapters according to manufacturer\'s instructions. After ligation, DNA fragments smaller than 150 bp were removed with AmPure XP beads (Beckman Coulter Genomics, Danvers, MA). The purified adapter ligated products were enriched using polymerase chain reaction (14 cycles). The final amplified libraries were resolved on 2.0% agarose gel and manually size-selected in the range of 350–380 bp. The final RNA-Seq libraries were quantitated using the Agilent 2100 Bioanalyzer and pooled together in equal concentration for sequencing.', 'GSM1891970', 'Illumina HiSeq 2000', 'Oct 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04108561Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1281664', 'GSE73370Xx-xXGSE73371', 'Breast Cancer Cell Line', 'Public on Oct 26 2015', 'Sep 23 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1891nnn/GSM1891970/suppl/GSM1891970_Proliferating_MCF7_geneCount.txt.gz', '', 9606, 'Proliferating_MCF7', 'SRA', 38140376, 175888717),
(1, 'cell line: HCC1395Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1395_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897296', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293309Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122175', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897296/suppl/GSM1897296_HCC1395_FPKM.txt.gz', '', 9606, 'HCC1395', 'SRA', 88485418, 97276607),
(1, 'cell line: HCC712Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC712_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897312', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293325Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122131', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897312/suppl/GSM1897312_HCC712_FPKM.txt.gz', '', 9606, 'HCC712', 'SRA', 46079536, 57465828),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401734', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821112Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559799', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401734/suppl/GSM1401734_SL7917_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401734/suppl/GSM1401734_SL7917_gene.fpkm_tracking.gz', 9606, '03-02-A158a', 'SRA', 53760358, 84242373),
(1, 'cell line: MCF-7Xx-xXer status: ER+Xx-xXconfluency: HighXx-xXmi63 treatment: No', '8001 Redwood Blvd', 'Novato', 'USA', '', 'cyau@buckinstitute.org', 'Buck Institute of Age Research', 'Benz Lab', 'Christina,,Yau', '415-209-2248', '94945', 'Resulting reads were aligned to the transcriptome defined by UCSC hg19 Known Genes using the RNA-Seq pipeline RSEM v1.2.9 and Bowtie v1.0.0Xx-xXRSEM estimated Fragments Per Kilobase Mapped (FPKMs) and Transcripts Per Million (TPMs) per transcript and isoform using an expectation maximization algorithm.Xx-xXTPMs for individual read groups were used for downstream differential expression analysesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include TPM values of UCSC known genes for each sample from 3 read groups', 0, '', 'Total RNA was extracted using the RNeasy mini kit (Qiagen, CA) and assayed using a 2100 Bioanalyzer (Agilent, CA).Xx-xX~ 0.5-1.0 µg cDNA from each was first sheared to a size ranging between 200-300 bp using the Covaris-S2 sonicator (Covaris, Woburn, MA) according to the manufacturer’s recommended protocols, and the fragmented cDNA samples were then prepared for whole transcriptome library generation using the TruSeq v1 Multiplex Sample Preparation kit (Illumina, San Diego, CA). Briefly, cDNA fragments were end-repaired, dA-tailed and ligated to multiplex adapters according to manufacturer\'s instructions. After ligation, DNA fragments smaller than 150 bp were removed with AmPure XP beads (Beckman Coulter Genomics, Danvers, MA). The purified adapter ligated products were enriched using polymerase chain reaction (14 cycles). The final amplified libraries were resolved on 2.0% agarose gel and manually size-selected in the range of 350–380 bp. The final RNA-Seq libraries were quantitated using the Agilent 2100 Bioanalyzer and pooled together in equal concentration for sequencing.', 'GSM1891971', 'Illumina HiSeq 2000', 'Oct 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04108562Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1281665', 'GSE73370Xx-xXGSE73371', 'Breast Cancer Cell Line', 'Public on Oct 26 2015', 'Sep 23 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1891nnn/GSM1891971/suppl/GSM1891971_Confluent_MCF7_geneCount.txt.gz', '', 9606, 'Confluent_MCF7', 'SRA', 44343613, 168960677),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401699', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821145Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559764', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401699/suppl/GSM1401699_SL10801_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401699/suppl/GSM1401699_SL10801_gene.fpkm_tracking.gz', 9606, '05-09-A159a', 'SRA', 49036793, 70863611),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401691', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821136Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559756', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401691/suppl/GSM1401691_SL10663_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401691/suppl/GSM1401691_SL10663_gene.fpkm_tracking.gz', 9606, '03-03-A069a', 'SRA', 41350379, 63875262),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401657', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820994Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559722', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401657/suppl/GSM1401657_SL6571_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401657/suppl/GSM1401657_SL6571_gene.fpkm_tracking.gz', 9606, 'BT-549', 'SRA', 58685697, 85493460),
(1, 'cell line: MCF10AXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MCF10A_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897320', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293333Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122140', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897320/suppl/GSM1897320_MCF10A_FPKM.txt.gz', '', 9606, 'MCF10A', 'SRA', 29383541, 33678606),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 1280 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533423', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145663Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745854', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533423/suppl/GSM1533423_MCF7_mRNA_t1280.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533423/suppl/GSM1533423_MCF7_premRNA_t1280min.wig.txt.gz', 9606, 'MCF7_E2_1280min_RNASeq', 'SRA', 44210716, 96814669),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 1 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT6) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300622', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566878Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404072', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 1 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT6-dox-1', 'SRA', 47541405, 54359973),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401662', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820990Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559727', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401662/suppl/GSM1401662_SL6576_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401662/suppl/GSM1401662_SL6576_gene.fpkm_tracking.gz', 9606, 'MDA-MB-134', 'SRA', 58665371, 108462283),
(1, 'cell line: MX1Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MX1_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897334', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293347Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122153', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897334/suppl/GSM1897334_MX1_FPKM.txt.gz', '', 9606, 'MX1', 'SRA', 30213813, 37700989),
(1, 'tissue: Normal tissue adjacent to breast cancer', 'Shanghai Medical College, Fudan University 1207 Rm., 2# Bldg., 270 Dong An Rd.', 'Shanghai', 'China', 'Fudan University Shanghai Cancer Center', 'cybao.cpu@163.com', 'Fudan University', '', 'Chunyang,,Bao', '86-21-34777580', '200032', 'Illumina Casava1.7 software used for basecalling.Xx-xXSequenced reads were trimmed for adaptor sequence then mapped to hg19 whole genome using TopHat v2.1.0 with default parameters.Xx-xXBack-spliced reads were calculated using a protocol from Memczak et al., Nature, 2013. Briefly, the unmapped reads were processed to 20-nucleotide anchors from both ends of the read. Anchors that aligned in the reversed orientation (head-to-tail) indicated a back-spliced junction. Anchor alignments were extended such that the complete read aligns and the breakpoints were flanked by GT/AG splice sites. To estimate the relative expression of a circular RNA, denoted as spliced reads per billion mapped reads (SRPBM), we counted the total number of reads mapped to the human reference genome in each sample and normalized the number of reads spanning the back-spliced junction to the total number of mapped reads (units in billion) and read length. To  estimate the relative expression of mRNA, fragments per kilobase of transcript per million fragments mapped (FPKM) were calculated using Cufflinks with default parameters.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include SRPBM values for each Sample and tab-delimited text files include FPKM values for each Sample', 0, '', 'Tissues were removed, flash frozen on liquid nitrogen, and RNA was harvested using Trizol reagent. NEBNext® Ultra™ Directional RNA Library Prep Kit for Illumina (Cat#E7420L) was used with 3 ug of total RNA for the construction of sequencing libraries.Xx-xXStrand-specific RNA-seq libraries were prepared using the NEBNext® Ultra™ Directional RNA Library Prep Kit for Illumina (Cat#E7420L) following the manufacturer’s instructions.', 'GSM2055792', 'Illumina HiSeq 2000', 'Mar 04 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04479411Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1564799', 'GSE77661', 'Normal tissue adjacent to breast cancer', 'Public on Mar 04 2016', 'Feb 08 2016', 'NONE', '', 9606, 'BRCA_NT', 'SRA', 9105425, 31378028),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401680', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821123Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559745', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401680/suppl/GSM1401680_SL10641_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401680/suppl/GSM1401680_SL10641_gene.fpkm_tracking.gz', 9606, '01-03-A207a', 'SRA', 41423355, 68145320),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 25 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT10) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300630', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566883Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404080', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 25 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT10-dox-25', 'SRA', 40913609, 48342113),
(1, 'cell line: SUM190Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM190_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897344', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293357Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122331', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897344/suppl/GSM1897344_SUM190_FPKM.txt.gz', '', 9606, 'SUM190', 'SRA', 45558613, 56320107),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 12h, DMSO', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431878', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905575Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648512', 'GSE59251', 'MCF-7 breast cancer cell line_12h Control-rep3', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '12h Control-rep3', 'SRA', 19507244, 21395239);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348548', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689299Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490459', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'BRD4 siRNA E2 rep2', 'SRA', 29547588, 36000000),
(1, 'cell line: ZR751Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'ZR751_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897354', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293367Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122341', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897354/suppl/GSM1897354_ZR751_FPKM.txt.gz', '', 9606, 'ZR751', 'SRA', 43719031, 48747904),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348547', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689295Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490458', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'BRD4 siRNA E2 rep1', 'SRA', 29525446, 36000000),
(1, 'cell line: HCC1806Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1806_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897302', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293315Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122181', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897302/suppl/GSM1897302_HCC1806_FPKM.txt.gz', '', 9606, 'HCC1806', 'SRA', 53890292, 59623900),
(1, 'cell line: MCF10AXx-xXcell type: Human normal breast epithelial cellsXx-xXpassages: 5', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized count values for each Sample.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1874957', 'Illumina HiSeq 2000', 'Oct 06 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04046659Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1224644', 'GSE72932Xx-xXGSE72933', 'MCF10A normal breast epithelial cells breast cancer cells', 'Public on Oct 06 2016', 'Sep 11 2015', 'NONE', '', 9606, 'JQ1 rep2', 'SRA', 32387433, 37454291),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401740', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821068Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559805', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401740/suppl/GSM1401740_SL7926_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401740/suppl/GSM1401740_SL7926_gene.fpkm_tracking.gz', 9606, '10-06-A120c', 'SRA', 40711465, 76108512),
(1, 'cell line: MCF12AXx-xXcell type: breast cancer cancer cell lineXx-xXstable expression: miR-424 over-expressed', '12801 E. 17th Ave.', 'Aurora', 'USA', 'Medicine', 'Jihye.Kim@UCDenver.edu', 'UCDenver', 'Tan', 'Jihye,,Kim', '303-724-8229', '80016', 'Alignment: TopHat v1.4.1 using default settings. hg19 (UCSC).Xx-xXExpression levels: Cufflinks v1.3.0 and cuffdiff using default settings.Xx-xXGSEA anlalysis with GeneSigDB v4.Xx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Tab-delimited text file with FPKM values.', 0, 'mRNA', 'Total RNA was extracted and mRNA isolated using the miRNeasy Kit (Qiagen) according to manufacturer\'s instructions.Xx-xXRNA libraries were prepared for sequencing using TruSEQ RNA Sample Prep v2 using standard Illumina protocols.', 'GSM1317639', 'Illumina HiSeq 2000', 'Feb 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02598927Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX451747', 'GSE54505', 'breast cancer cancer cell line, over-expressing miR-424', 'Public on Feb 26 2015', 'Jan 29 2014', 'NONE', '', 9606, 'miR424_1', 'SRA', 43321921, 49343661),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401737', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821071Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559802', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401737/suppl/GSM1401737_SL7922_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401737/suppl/GSM1401737_SL7922_gene.fpkm_tracking.gz', 9606, '08-10-A297d', 'SRA', 48781303, 78615682),
(1, 'cell line: MCF10AXx-xXcell type: Human normal breast epithelial cellsXx-xXpassages: 5', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized count values for each Sample.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1874953', 'Illumina HiSeq 2000', 'Oct 06 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04046655Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1224640', 'GSE72932Xx-xXGSE72933', 'MCF10A normal breast epithelial cells breast cancer cells', 'Public on Oct 06 2016', 'Sep 11 2015', 'NONE', '', 9606, 'Control siRNA rep2', 'SRA', 31767138, 35619705),
(1, 'transfection: controlXx-xXcell line: MDA-MB-231Xx-xXcell type: epithelial cellXx-xXdisease: adenocarcinomaXx-xXethnicity: caucasian', '94 Weijin Road', 'Tianjin', 'China', '', 'sxsl818@163.com', 'Medical School of Nankai University', '', 'Long,,Shen', '15202209080', '300071', 'Base-calling: OLB-1.9.4Xx-xXData filtering: R&D Perl Remove reads with adapters; Remove reads in which unknown bases are more than 5%;Remove low quality reads (the percentage of low quality bases is over 50% in a read, we define the low quality base to be the base whose sequencing quality is no more than 10).Xx-xXRead alignment: SOAP2.20 -m 0 -x 1000 -s 40 -l 32 -v 5 -r 2 -p 3Xx-xXDEGs detection: R&D Perl FDR ≤ 0.001 and the absolute value of Log2Ratio ≥ 1Xx-xXCluster Analysis: Cluster 3.0  -e 7 -g 7 -m aXx-xXGenome_build: Build Genome: hg19Xx-xXSupplementary_files_format_and_content: RPKM', 0, '', 'Total RNA was harvested using Trizol reagent.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1314287', 'Illumina HiSeq 2000', 'Jan 27 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02596093Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX449618', 'GSE54392', 'Breast cancer cells', 'Public on Jan 25 2014', 'Jan 24 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1314nnn/GSM1314287/suppl/GSM1314287_231-mcsA.Gene.rpkm.txt.gz', '', 9606, '231-mcsA', 'SRA', 4668616, 5912977),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401729', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821118Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559794', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401729/suppl/GSM1401729_SL7912_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401729/suppl/GSM1401729_SL7912_gene.fpkm_tracking.gz', 9606, '10-09-A101A', 'SRA', 44913841, 94843173),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401693', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821134Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559758', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401693/suppl/GSM1401693_SL10679_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401693/suppl/GSM1401693_SL10679_gene.fpkm_tracking.gz', 9606, '04-07-A414a', 'SRA', 51673703, 86947568),
(1, 'cell line: HCC1500Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1500_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897299', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293312Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122178', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897299/suppl/GSM1897299_HCC1500_FPKM.txt.gz', '', 9606, 'HCC1500', 'SRA', 38423883, 44306171),
(1, 'cell line: MCF7 breast tumor cell line', 'Via P. Castellino, 111', 'Naples', 'Italy', 'Institute of Genetics and Biophysics', 'valerio.costa@igb.cnr.it', 'National Research Council', '', 'Valerio,,Costa', '3.90816E+11', '80131', 'Sequenced reads were trimmed for adaptor sequence, and for low-quality sequence, then mapped to hg19 whole genome using TopHat2 v2.0.10 with parameters: -p 8 -N 2 -g 10 -r 200 -a 15 -m 1 -i 100 --library-type fr-unstranded --segment-mismatches 3 --read-edit-dist 2 -GXx-xXGene expression and alternative splicing were analyzed using Cufflinks v2.1.1 and a computational procedure described in Scarpato et al., 2014Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Processed data are provided in tab delimited text format and consist of normalized counts (FPKM);', 0, 'Replicate 2', 'Total RNA was extracted using TRIzol. The integrity and quantity of RNAs were assessed by Experion.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocol. Illumina TruSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used  for the construction of sequencing libraries.Xx-xXRNA-Seq paired-end (75+75 bp)', 'GSM1666033', 'Illumina HiSeq 2000', 'Jun 01 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03566779Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1008005', 'GSE68228', 'MCF7 breast-cancer cell line', 'Public on Jun 01 2015', 'Apr 24 2015', 'NONE', '', 9606, 'MCF7_Sample 2', 'SRA', 36974114, 42395683),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 12h, DMSO', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431876', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905573Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648510', 'GSE59251', 'MCF-7 breast cancer cell line_12h Control-rep1', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '12h Control-rep1', 'SRA', 18224115, 19902085),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401669', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821004Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559734', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401669/suppl/GSM1401669_SL6585_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401669/suppl/GSM1401669_SL6585_gene.fpkm_tracking.gz', 9606, 'SUM-149', 'SRA', 29941134, 118991091),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: untreated', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from untreated cells', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503437', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067405Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699962', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_untreated', 'SRA', 32429397, 220919185),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401654', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820986Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559719', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401654/suppl/GSM1401654_SL5822_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401654/suppl/GSM1401654_SL5822_gene.fpkm_tracking.gz', 9606, 'T-47-D', 'SRA', 30086763, 53522634),
(1, 'cell line: MDAMB330Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB330_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897327', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293340Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122146', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897327/suppl/GSM1897327_MDAMB330_FPKM.txt.gz', '', 9606, 'MDAMB330', 'SRA', 33434902, 42280875),
(1, 'cell line: MCF10AXx-xXcell type: Human normal breast epithelial cellsXx-xXpassages: 5', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized count values for each Sample.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1874956', 'Illumina HiSeq 2000', 'Oct 06 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04046658Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1224643', 'GSE72932Xx-xXGSE72933', 'MCF10A normal breast epithelial cells breast cancer cells', 'Public on Oct 06 2016', 'Sep 11 2015', 'NONE', '', 9606, 'JQ1 rep1', 'SRA', 29300948, 33682531);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401703', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821149Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559768', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401703/suppl/GSM1401703_SL10809_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401703/suppl/GSM1401703_SL10809_gene.fpkm_tracking.gz', 9606, '07-12-A271b', 'SRA', 38772253, 58438278),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401700', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821144Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559765', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401700/suppl/GSM1401700_SL10803_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401700/suppl/GSM1401700_SL10803_gene.fpkm_tracking.gz', 9606, '05-11-A240b', 'SRA', 53464031, 70252436),
(1, 'cell line: MDA-MB-231Xx-xXcell type: highly invasiveXx-xXtransfected with: wt-RESTXx-xXgenotype/variation: overexpression of wild type REST', '1501 San Pablo St.', 'Los Angeles', 'USA', 'Biomedical Engineering', 'nanlee@usc.edu', 'University of Southern California', '', 'Nan,Sook,Lee', '6268243058', '90033', 'Bcl conversion: Illumina bcl2Fastq ver 1.8.4 to convert from bcl format to fastq.gz raw reads fileXx-xXAlignment and read counting done with GTFAR ver 0.1 using gencode19 annotation.Xx-xXA second alignment was done with Tophat ver 1.4.0 using hg19_ucsc reference and parameter -G hg19_ucsc.gtf (optional reference annotation)Xx-xXTranscript abundance estimation on the second alignment done with Cufflinks ver 1.0.1 using parameters -u (for multi-read-correct), -b hg19_ucsc.fa (for bias detection and correction) and -G hg19_ucsc.gtf (for optional reference annotation)Xx-xXGenome_build: hg19_ucscXx-xXSupplementary_files_format_and_content: tab-delimited text files from GT-FAR runs and Cufflinks runs which include RPKM values for each sample and isoform counts for each sample; more details in the \'README_1.txt\'', 0, '', 'Total RNAs were isolated from cell lines using Trizol (Invitrogen) and digested with Turbo DNase (Ambion) to remove genomic DNA, according to the manufacturer’s instructions.Xx-xXLibraries from rRNA-depleted samples were prepared using a TruSeq RNA Sample Preparation kit v2 (Illumina) following the recommended protocol starting from the RNA fragmentation stage. Purification of polyadenylated RNA was omitted. Libraries were pooled (4 samples per pool), clustered on cBOT (Illumina), and sequenced on HiSeq2000, each pool in one lane. We performed single-end 100 bp reads. Reads were mapped using TopHat followed by data analysis using Cufflinks and Cuffdiff software, as well as our own pipeline.', 'GSM1553460', 'Illumina HiSeq 2000', 'Dec 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03220246Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX767451', 'GSE63608Xx-xXGSE63610', 'Breast cancer cell line', 'Public on Dec 01 2014', 'Nov 25 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553460/suppl/GSM1553460_GTFAR-MB-231-wt-136_geneRPKMS.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553460/suppl/GSM1553460_GTFAR-MB-231-wt-136_rawRestIsoformCnts.txt.gz', 9606, 'wt-REST', 'SRA', 45341076, 95841612),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401652', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820983Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559717', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401652/suppl/GSM1401652_SL5820_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401652/suppl/GSM1401652_SL5820_gene.fpkm_tracking.gz', 9606, 'SUM-159', 'SRA', 24843351, 33235081),
(1, 'cell line: BT483Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'BT483_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897281', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293294Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122160', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897281/suppl/GSM1897281_BT483_FPKM.txt.gz', '', 9606, 'BT483', 'SRA', 32164114, 36837668),
(1, 'gender: FemaleXx-xXcell line: SKBR3Xx-xXcell type: HER2+', '26, Rue d\'Ulm', 'Paris', 'France', '', 'Ronan.Chaligne@curie.fr', 'Institut Curie', 'Edith Heard\'s lab', 'Ronan,,Chaligné', '', '75248', 'Supplementary_files_format_and_content: Each sample contains three bedgraph files.Xx-xXSupplementary_files_format_and_content: 1. ChrX full bedgraph displayXx-xXSupplementary_files_format_and_content: 2. ChrX allele1 specific display of bedgraph only on SNP positionsXx-xXSupplementary_files_format_and_content: 3. ChrX allele2 specific display of bedgraph only on SNP positionsXx-xXSupplementary_files_format_and_content: Expression results: fpkm_tracking for genes standard output from cufflinksXx-xXSupplementary_files_format_and_content: Final SNP list: List of final SNPs derived from ChIP-seq, mRNA-seq and Exome-seq used for allele specific analysisXx-xXDatasets were subjected to two types of quality control. FASTQC-0.10.1 (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) was used to assess the quality of sequencing and potential adapter or cross contaminants. Average sequencing quality (phred score) per base was above 30 (Q ≥ 30) for all datasets. In addition, aligned datasets were then subjected toNGS-QC ((Mendoza-Parra et al., 2013); www.ngs-qc.org) to assess the robustness of enrichment. The majority of data sets were of «triple A» quality, no data set was below «triple B». For exome-seq and ChIP-seq, alignment was performed using BWA-MEM-0.7.7 (Li and Durbin, 2009) with default parameters, which simultaneously checks for both global and local alignment for reads. Alignment was followed by three sets of filters to prevent bias in the analysis. 1) duplicate reads (PCR clonal reads) were filtered out using Picard tools-1.86 (http://picard.sourceforge.net) 2) reads with mapping quality less than 10 were filtered out using Bamtools-2.2.3 (https://github.com/pezmaster31/bamtools) and 3) reads with more than one alignment reported were filtered out using in-house scripts. Further analysis was carried out on processed alignment file which is around 10-100M reads after these filters.Xx-xXTo prepare the allele information for each cell-line for allele-specific analysis down the line, SNP analysis was carried out along with SNP6 data. To identify novel variation (apart from known SNPs from SNP6 data), SNP analysis was carried out on all three ChIP-seq, Exome-seq and RNA-seq data individually. ChIP-seq data of different marks (H3K4me3, H3K27me3, RNA Pol II and Input) were merged for each cell line to increase the depth and confidence for variation calling. Variation calling was performed for each cell line separately for ChIP-seq and Exome-seq following the best practice GATK-2.6.5 pipeline by filtering reads with Mapping quality ≥ 1 (Van der Auwera GA et al., 2013; http://www.broadinstitute.org/gatk/guide/best-practices?bpm=DNAseq). Variation calling for RNA-seq was carried out following the methods of Piskol, Robert et al., 2013 to avoid artifacts specific to RNA- seq data (Piskol et al., 2013). A final list of allele information was generated by combining the SNP information from the different data sets for each cell line. To increase the allele-specific sensitivity for the alignment, reads were additionally realigned in an allele-specific manner following the method of Satya et al. (Satya et al., 2012). Read counts for each allele and SNP position were extracted for each mark using in-house scripts. SNP positions with at least three reads from both alleles were considered as heterozygous positions.Xx-xXPeak calling was performed using HOMER ((Heinz et al., 2010); http://homer.salk.edu/homer/index.html) with default parameters. For H3K27me3 and RNA Pol II, the\'style\' parameter was chosen as \'histone\' due to the broad patterns for this mark, whereas for H3K4me3, which generally give sharp peaks, the parameter \'factor\' was chosen. Genomic context annotation over identified peaks were carried out using the HOMER annotation module but with basic annotation by excluding references other than coding genes and non-coding RNA.Xx-xXA gene-based analysis of annotation integration was carried out using in-house scripts to integrate all annotation from peak and variation calling (informative SNP counts, read depths, homo/heterozygous SNP count and weighted allelic imbalance). To include annotations from regulatory regions 1Kb sequences upstream from the TSS and downstream of the TES were considered. A weighted arithmetic average was calculated for each gene by calculating average Allelic Imbalance (AI) where each SNP\'s AI was weighted by its read depth.Xx-xXTo illustrate the comparisons across cell lines, ChIP-seq data were normalized using an in-house developed tool called \'Epimetheus\', which is based on quantile normalization (manuscript under preparation). Read Count Intensity (RCI) was calculated for a window of 100bp bin size across chromosomes and then these intensities were normalized using quantile normalization from the limma package. The impact of normalization was assessed using MA plots before and after normalization. Specified genomic feature based normalized RCI was constructed, which are illustrated in Figure 6. For TSS-centered plots and heatmaps, a separate TSS-based normalization was carried out with 20bp bin size to obtain higher resolution.Xx-xXGenome_build: hg19', 0, '', 'Total RNA was extracted using the Trizol (Invitrogen) procedure.Xx-xXStandard Illumina protocols are followed.Xx-xXmRNA: total RNA was enriched for polyadenylated RNA (polyA mRNA).Xx-xXmRNA : RNA-sequencing correspond of paired-reads lane 2x100 bp sequencing on poly-A RNA purified  and were prepared according to the standard Illumina protocol. Sequencing has been performed on high-throughput Illumina HiSeq sequencer.', 'GSM1537305', 'Illumina HiSeq 2000', 'Jan 19 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03160891Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX750680', 'GSE62907Xx-xXGSE62966', 'Human Breast Cancer Cells', 'Public on Jan 19 2015', 'Nov 04 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537305/suppl/GSM1537305_SKBR3-mRNA.chrX.SNPs.allele1.bedgraph.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537305/suppl/GSM1537305_SKBR3-mRNA.chrX.SNPs.allele2.bedgraph.gz', 9606, 'SKBR3_mRNA', 'SRA', 92406546, 104732765),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401670', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821003Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559735', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401670/suppl/GSM1401670_SL6586_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401670/suppl/GSM1401670_SL6586_gene.fpkm_tracking.gz', 9606, '2-LMP', 'SRA', 41788969, 59613594),
(1, 'cell line: HCC1569Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1569_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897300', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293313Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122179', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897300/suppl/GSM1897300_HCC1569_FPKM.txt.gz', '', 9606, 'HCC1569', 'SRA', 144763973, 158822674),
(1, 'cell line: 1833Xx-xXsubtype: Triple negativeXx-xXmetastasis: Bone metastasisXx-xXknockdown: Abl', '308 Research Drive LSRC C231', 'Durham', 'USA', 'Pharmacology and Cancer Biology', '', 'Duke University', 'Pendergast lab', 'Jun,,Wang', '', '27710', 'Sequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg18 whole genome using tophatXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using cuffdiff based on the protocol Cole Trapnell et al. Nature Biotechnology 31, 46-53 (2013)Xx-xXGenome_build: hg18Xx-xXSupplementary_files_format_and_content: diff files include RPKM values for each Sample', 0, '', 'RNA was harvested using Qiagen Rneasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1693177', 'Illumina HiSeq 2000', 'Mar 02 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03704039Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1036353', 'GSE69125', 'breast cancer cells', 'Public on Mar 02 2016', 'May 21 2015', 'NONE', '', 9606, 'miAbl', 'SRA', 38114056, 50355410),
(1, 'cell line: MDAMB231Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB231_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897326', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293339Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122145', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897326/suppl/GSM1897326_MDAMB231_FPKM.txt.gz', '', 9606, 'MDAMB231', 'SRA', 31436483, 34763501),
(1, 'cell line: LY2Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'LY2_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897317', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293330Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122137', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897317/suppl/GSM1897317_LY2_FPKM.txt.gz', '', 9606, 'LY2', 'SRA', 31402022, 38883311),
(1, 'cell line: KPL1Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'KPL1_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897316', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293329Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122136', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897316/suppl/GSM1897316_KPL1_FPKM.txt.gz', '', 9606, 'KPL1', 'SRA', 54423742, 59889772),
(1, 'cell line: MCF7', '9 Divinity Ave', 'Cambridge', 'USA', 'Stem Cell and Regenerative Biology', '', 'Harvard University', '', 'John,,Rinn', '', '2138', 'RNA-seq: For  RNA-Seq analysis, first, the adapter sequences were removed from the RNA-seq reads.  Ribosomal RNA reads were then filtered out using Bowtie (Song et al. 2014).  After these filtration and adapter removal steps, the reads were aligned to a transciptome and quantified using RSEM v1.2.7 (Li and Dewey 2011). Differential gene expression was calculated by using the Deseq2 package in R 3.1.0 by using the mean value of gene-wise dispersion estimates (Love et al. 2014).Xx-xXChIP-seq: ~1x107 parental MCF-7 cells were crosslinked with formaldehyde at room temperature for 10 minutes.  Then, the cells were lysed using lysis buffer A (50mM HEPES, 140mM NaCl, 1mM EDTA pH=8, 10% Glycerol, 0.5% NP-40, 0.25% Triton X-100), and the residual cytoplasmic protein was removed using lysis buffer B (10mM Tris-HCl pH=8, 200mM NaCl, 1mM EDTA, 1mM EGTA).  The nuclear fraction was released using lysis buffer C (10mM Tris-Hcl pH=8, 100mM NaCl, 1mM EDTA, 1mM EGTA, 0.1% Sodium Deoxycholate, 0.5% N-lauroylsarcosine).  The chromatin was then sheared by using a Covaris S2 instrument with 10% duty cycle, 5 intensity, 200 cycles per burst, frequency sweeping mode, 60 second process time and for 4 cycles. The pull-down was performed by using the RUNX1 antibody (Cell Signaling #4334).  Samples were washed three times with RIPA buffer (Tris-HCl pH=8, 150mM NaCl, 1mM EDTA, 1% NP-40, 0.25% Sodium deoxycholate, 0.1% SDS) and were eluted.  The pull-down and input control sequencing libraries were generated by using the NEXTflex Rapid DNA Sequencing Kit (Bioo Scientific #5144-02) and were sequenced by using SE100 reads with a HiSeq 2000 instrument.Xx-xXdata processing step: The Hi-C data were mapped with Bowtie and  binned at 6.5Mb, 1Mb, 250kb, 100kb and 40kb non-overlapping genomic intervals.  Iterative mapping and correction of Hi-C data were performed as previously described (Imakaev et al. 2012).  Biological replicates showed high reproducibility (Pearson\'s correlation coefficient > 0.9 for 1Mb resolution data).  Similarly, the first eigenvector comparison of the replicates showed high reproducibility.  For the downstream analyses, sequences obtained from both biological replicates were pooled and ICE-corrected to serve as a combined dataset.Xx-xXCompartment analysis: First, the z-scores of the interaction matrices at 250kb resolution were generated as described previously (Lieberman, 2009).  Then, Pearson Correlation on the Z-score matrices was calculated.  By performing principal component analysis (Lieberman-Aiden et al. 2009; Zhang et al. 2012), the first eigenvector typically represents the compartment profile(Lieberman-Aiden et al. 2009), where the positive and negative 1st eigenvalues represent different compartments.   Gene density for each compartment was calculated to call the “A” and “B” compartmentalization.Xx-xXTAD Calling : TAD calling was performed as calculating the “insulation” score of each bin using the 40kb resolution combined Hi-C data (as previously described.  The mean of the interactions across each bin were calculated.  By sliding a 1Mb x 1Mb (25bins x 25bins) square along the diagonal of the interaction matrix for every chromosome, we obtained the “insulation score” of the interaction matrix.  Valleys in the insulation score indicated the depletion of Hi-C interactions occurring across a bin.  These 40kb valleys represent the TAD boundaries.  Based on the variation of boundaries between replicates, we chose to add a total of 160kb (80kb to each side) to the boundary to account for replicate variation.  The final boundaries span a 200kb region. All boundaries with a boundary strength < 0.15 were excluded as they were considered weak and non-reproducible.  The insulation plots for the biological replicates showed high reproducibility, suggesting the robustness of the method.  Similarly, the overlap of detected boundaries also showed high reproducibility between the biological replicates.  Therefore, we used the combined Hi-C replicates for the TAD analyses.Xx-xXZ-Score Calculation: We modeled the overall Hi-C decay with distance using a modified LOWESS method (alphaXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: HiC: Interaction matrices at 2.5Mb, 1Mb, 250kb and 40kb resolution; RNA-seq: Expected count file for all genes and a txt file showing the shRUNX1/shNS log2 Fold Change and pvalues for all genes; ChIP-seq: BedGeapgh UCSC visualization files and the MCF7 RUNX1 peak bed fileXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_processed_HiC_files - all normalized/corrected Hi-C data matrices binned at 40kb/250kb/1Mb/2.5MbXx-xXSupplementary_files_format_and_content: MCF7_RUNX1_ChIPseq_peaks.bed - BED file of RUNX1 ChIP-seq peaksXx-xXSupplementary_files_format_and_content: MCF7_shNS_shRUNX1_genes_expression_expected_count.tsv - TSV file showing the RNA-seq counts for each geneXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_RNAseq_log2_foldchange.txt - TSV file showing the log2 fold change of all  genes between shRUNX1 and shNS samples', 0, '', 'Hi-C was sequenced using PE100 reads; RNA-seq and ChIP-seq were sequenced using SE100 reads by using a HiSeq 2000 instrument', 'GSM1942107', 'Illumina HiSeq 2000', 'Aug 23 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04272128Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1434848', 'GSE75070', 'Human breast cancer cell line', 'Public on Aug 23 2016', 'Nov 16 2015', 'NONE', '', 9606, 'MCF-7 shRUNX1  RNA-seq Replicate1', 'SRA', 11857974, 25020673),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 80 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533419', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145661Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745850', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533419/suppl/GSM1533419_MCF7_mRNA_t0080.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533419/suppl/GSM1533419_MCF7_premRNA_t0080min.wig.txt.gz', 9606, 'MCF7_E2_80min_RNASeq', 'SRA', 45378740, 97915156),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401684', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821132Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559749', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401684/suppl/GSM1401684_SL10648_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401684/suppl/GSM1401684_SL10648_gene.fpkm_tracking.gz', 9606, '01-07-A281b', 'SRA', 34615306, 53970617);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348541', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689292Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490452', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'Control siRNA Veh rep1', 'SRA', 30277273, 35405319),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 2.5 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT6) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300623', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566879Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404073', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 2.5 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT6-dox-2.5', 'SRA', 56046853, 63764926),
(1, 'cell line: SKBR3Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SKBR3_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897336', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293349Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122155', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897336/suppl/GSM1897336_SKBR3_FPKM.txt.gz', '', 9606, 'SKBR3', 'SRA', 29420674, 33099393),
(1, 'cell line: CAL148Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'CAL148_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897284', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293297Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122163', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897284/suppl/GSM1897284_CAL148_FPKM.txt.gz', '', 9606, 'CAL148', 'SRA', 31967695, 38429178),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401704', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821089Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559769', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401704/suppl/GSM1401704_SL10810_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401704/suppl/GSM1401704_SL10810_gene.fpkm_tracking.gz', 9606, '08-01-A301a', 'SRA', 55744507, 73052086),
(1, 'cell line: MCF7', '9 Divinity Ave', 'Cambridge', 'USA', 'Stem Cell and Regenerative Biology', '', 'Harvard University', '', 'John,,Rinn', '', '2138', 'RNA-seq: For  RNA-Seq analysis, first, the adapter sequences were removed from the RNA-seq reads.  Ribosomal RNA reads were then filtered out using Bowtie (Song et al. 2014).  After these filtration and adapter removal steps, the reads were aligned to a transciptome and quantified using RSEM v1.2.7 (Li and Dewey 2011). Differential gene expression was calculated by using the Deseq2 package in R 3.1.0 by using the mean value of gene-wise dispersion estimates (Love et al. 2014).Xx-xXChIP-seq: ~1x107 parental MCF-7 cells were crosslinked with formaldehyde at room temperature for 10 minutes.  Then, the cells were lysed using lysis buffer A (50mM HEPES, 140mM NaCl, 1mM EDTA pH=8, 10% Glycerol, 0.5% NP-40, 0.25% Triton X-100), and the residual cytoplasmic protein was removed using lysis buffer B (10mM Tris-HCl pH=8, 200mM NaCl, 1mM EDTA, 1mM EGTA).  The nuclear fraction was released using lysis buffer C (10mM Tris-Hcl pH=8, 100mM NaCl, 1mM EDTA, 1mM EGTA, 0.1% Sodium Deoxycholate, 0.5% N-lauroylsarcosine).  The chromatin was then sheared by using a Covaris S2 instrument with 10% duty cycle, 5 intensity, 200 cycles per burst, frequency sweeping mode, 60 second process time and for 4 cycles. The pull-down was performed by using the RUNX1 antibody (Cell Signaling #4334).  Samples were washed three times with RIPA buffer (Tris-HCl pH=8, 150mM NaCl, 1mM EDTA, 1% NP-40, 0.25% Sodium deoxycholate, 0.1% SDS) and were eluted.  The pull-down and input control sequencing libraries were generated by using the NEXTflex Rapid DNA Sequencing Kit (Bioo Scientific #5144-02) and were sequenced by using SE100 reads with a HiSeq 2000 instrument.Xx-xXdata processing step: The Hi-C data were mapped with Bowtie and  binned at 6.5Mb, 1Mb, 250kb, 100kb and 40kb non-overlapping genomic intervals.  Iterative mapping and correction of Hi-C data were performed as previously described (Imakaev et al. 2012).  Biological replicates showed high reproducibility (Pearson\'s correlation coefficient > 0.9 for 1Mb resolution data).  Similarly, the first eigenvector comparison of the replicates showed high reproducibility.  For the downstream analyses, sequences obtained from both biological replicates were pooled and ICE-corrected to serve as a combined dataset.Xx-xXCompartment analysis: First, the z-scores of the interaction matrices at 250kb resolution were generated as described previously (Lieberman, 2009).  Then, Pearson Correlation on the Z-score matrices was calculated.  By performing principal component analysis (Lieberman-Aiden et al. 2009; Zhang et al. 2012), the first eigenvector typically represents the compartment profile(Lieberman-Aiden et al. 2009), where the positive and negative 1st eigenvalues represent different compartments.   Gene density for each compartment was calculated to call the “A” and “B” compartmentalization.Xx-xXTAD Calling : TAD calling was performed as calculating the “insulation” score of each bin using the 40kb resolution combined Hi-C data (as previously described.  The mean of the interactions across each bin were calculated.  By sliding a 1Mb x 1Mb (25bins x 25bins) square along the diagonal of the interaction matrix for every chromosome, we obtained the “insulation score” of the interaction matrix.  Valleys in the insulation score indicated the depletion of Hi-C interactions occurring across a bin.  These 40kb valleys represent the TAD boundaries.  Based on the variation of boundaries between replicates, we chose to add a total of 160kb (80kb to each side) to the boundary to account for replicate variation.  The final boundaries span a 200kb region. All boundaries with a boundary strength < 0.15 were excluded as they were considered weak and non-reproducible.  The insulation plots for the biological replicates showed high reproducibility, suggesting the robustness of the method.  Similarly, the overlap of detected boundaries also showed high reproducibility between the biological replicates.  Therefore, we used the combined Hi-C replicates for the TAD analyses.Xx-xXZ-Score Calculation: We modeled the overall Hi-C decay with distance using a modified LOWESS method (alphaXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: HiC: Interaction matrices at 2.5Mb, 1Mb, 250kb and 40kb resolution; RNA-seq: Expected count file for all genes and a txt file showing the shRUNX1/shNS log2 Fold Change and pvalues for all genes; ChIP-seq: BedGeapgh UCSC visualization files and the MCF7 RUNX1 peak bed fileXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_processed_HiC_files - all normalized/corrected Hi-C data matrices binned at 40kb/250kb/1Mb/2.5MbXx-xXSupplementary_files_format_and_content: MCF7_RUNX1_ChIPseq_peaks.bed - BED file of RUNX1 ChIP-seq peaksXx-xXSupplementary_files_format_and_content: MCF7_shNS_shRUNX1_genes_expression_expected_count.tsv - TSV file showing the RNA-seq counts for each geneXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_RNAseq_log2_foldchange.txt - TSV file showing the log2 fold change of all  genes between shRUNX1 and shNS samples', 0, '', 'Hi-C was sequenced using PE100 reads; RNA-seq and ChIP-seq were sequenced using SE100 reads by using a HiSeq 2000 instrument', 'GSM1942104', 'Illumina HiSeq 2000', 'Aug 23 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04272125Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1434845', 'GSE75070', 'Human breast cancer cell line', 'Public on Aug 23 2016', 'Nov 16 2015', 'NONE', '', 9606, 'MCF-7 shNS  RNA-seq Replicate1', 'SRA', 9185834, 21297178),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401690', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821133Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559755', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401690/suppl/GSM1401690_SL10661_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401690/suppl/GSM1401690_SL10661_gene.fpkm_tracking.gz', 9606, '03-02-A013a', 'SRA', 39773482, 54089491),
(1, 'cell line: MDA-MB-468Xx-xXdisease state: breast cancer', '4500 San Pablo Road South', 'Jacksonville', 'USA', '', 'asmann.yan@mayo.edu', 'Mayo Clinic', '', 'Yan,W.,Asmann', '', '32224', 'Illumina\'s Genome Analyzer system software programs for base-callingXx-xXthe 50bp paired end reads were aligned using BWA 0.6.2Xx-xXthe input files were the fastq files of the two ends, and the out put files were the SnowShoes-FTD called fusion transcriptsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The .txt file reports fusion transcripts identified using SnowShoes-FTD version 2.0', 0, 'Sample 6', 'RNA was extracted by Qiagen Mini RNeasy Kit (Qiagen)Xx-xXTruSeq RNA sample preparation kit (Illumina)', 'GSM1273497', 'Illumina HiSeq 2000', 'Nov 22 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02419119Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX381537', 'GSE52643', 'breast cancer cell line', 'Public on Nov 22 2013', 'Nov 21 2013', 'NONE', '', 9606, 'MDA-MB-468', 'SRA', 28057428, 32629020),
(1, 'cell line: SUM159Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM159_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897342', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293355Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122348', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897342/suppl/GSM1897342_SUM159_FPKM.txt.gz', '', 9606, 'SUM159', 'SRA', 86066227, 95533983),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348544', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689300Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490455', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'Control siRNA E2 rep2', 'SRA', 30872256, 36380540),
(1, 'cell line: HS578TXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HS578T_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897314', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293327Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122133', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897314/suppl/GSM1897314_HS578T_FPKM.txt.gz', '', 9606, 'HS578T', 'SRA', 43479410, 46902920),
(1, 'cell line: MCF7', '9 Divinity Ave', 'Cambridge', 'USA', 'Stem Cell and Regenerative Biology', '', 'Harvard University', '', 'John,,Rinn', '', '2138', 'RNA-seq: For  RNA-Seq analysis, first, the adapter sequences were removed from the RNA-seq reads.  Ribosomal RNA reads were then filtered out using Bowtie (Song et al. 2014).  After these filtration and adapter removal steps, the reads were aligned to a transciptome and quantified using RSEM v1.2.7 (Li and Dewey 2011). Differential gene expression was calculated by using the Deseq2 package in R 3.1.0 by using the mean value of gene-wise dispersion estimates (Love et al. 2014).Xx-xXChIP-seq: ~1x107 parental MCF-7 cells were crosslinked with formaldehyde at room temperature for 10 minutes.  Then, the cells were lysed using lysis buffer A (50mM HEPES, 140mM NaCl, 1mM EDTA pH=8, 10% Glycerol, 0.5% NP-40, 0.25% Triton X-100), and the residual cytoplasmic protein was removed using lysis buffer B (10mM Tris-HCl pH=8, 200mM NaCl, 1mM EDTA, 1mM EGTA).  The nuclear fraction was released using lysis buffer C (10mM Tris-Hcl pH=8, 100mM NaCl, 1mM EDTA, 1mM EGTA, 0.1% Sodium Deoxycholate, 0.5% N-lauroylsarcosine).  The chromatin was then sheared by using a Covaris S2 instrument with 10% duty cycle, 5 intensity, 200 cycles per burst, frequency sweeping mode, 60 second process time and for 4 cycles. The pull-down was performed by using the RUNX1 antibody (Cell Signaling #4334).  Samples were washed three times with RIPA buffer (Tris-HCl pH=8, 150mM NaCl, 1mM EDTA, 1% NP-40, 0.25% Sodium deoxycholate, 0.1% SDS) and were eluted.  The pull-down and input control sequencing libraries were generated by using the NEXTflex Rapid DNA Sequencing Kit (Bioo Scientific #5144-02) and were sequenced by using SE100 reads with a HiSeq 2000 instrument.Xx-xXdata processing step: The Hi-C data were mapped with Bowtie and  binned at 6.5Mb, 1Mb, 250kb, 100kb and 40kb non-overlapping genomic intervals.  Iterative mapping and correction of Hi-C data were performed as previously described (Imakaev et al. 2012).  Biological replicates showed high reproducibility (Pearson\'s correlation coefficient > 0.9 for 1Mb resolution data).  Similarly, the first eigenvector comparison of the replicates showed high reproducibility.  For the downstream analyses, sequences obtained from both biological replicates were pooled and ICE-corrected to serve as a combined dataset.Xx-xXCompartment analysis: First, the z-scores of the interaction matrices at 250kb resolution were generated as described previously (Lieberman, 2009).  Then, Pearson Correlation on the Z-score matrices was calculated.  By performing principal component analysis (Lieberman-Aiden et al. 2009; Zhang et al. 2012), the first eigenvector typically represents the compartment profile(Lieberman-Aiden et al. 2009), where the positive and negative 1st eigenvalues represent different compartments.   Gene density for each compartment was calculated to call the “A” and “B” compartmentalization.Xx-xXTAD Calling : TAD calling was performed as calculating the “insulation” score of each bin using the 40kb resolution combined Hi-C data (as previously described.  The mean of the interactions across each bin were calculated.  By sliding a 1Mb x 1Mb (25bins x 25bins) square along the diagonal of the interaction matrix for every chromosome, we obtained the “insulation score” of the interaction matrix.  Valleys in the insulation score indicated the depletion of Hi-C interactions occurring across a bin.  These 40kb valleys represent the TAD boundaries.  Based on the variation of boundaries between replicates, we chose to add a total of 160kb (80kb to each side) to the boundary to account for replicate variation.  The final boundaries span a 200kb region. All boundaries with a boundary strength < 0.15 were excluded as they were considered weak and non-reproducible.  The insulation plots for the biological replicates showed high reproducibility, suggesting the robustness of the method.  Similarly, the overlap of detected boundaries also showed high reproducibility between the biological replicates.  Therefore, we used the combined Hi-C replicates for the TAD analyses.Xx-xXZ-Score Calculation: We modeled the overall Hi-C decay with distance using a modified LOWESS method (alphaXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: HiC: Interaction matrices at 2.5Mb, 1Mb, 250kb and 40kb resolution; RNA-seq: Expected count file for all genes and a txt file showing the shRUNX1/shNS log2 Fold Change and pvalues for all genes; ChIP-seq: BedGeapgh UCSC visualization files and the MCF7 RUNX1 peak bed fileXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_processed_HiC_files - all normalized/corrected Hi-C data matrices binned at 40kb/250kb/1Mb/2.5MbXx-xXSupplementary_files_format_and_content: MCF7_RUNX1_ChIPseq_peaks.bed - BED file of RUNX1 ChIP-seq peaksXx-xXSupplementary_files_format_and_content: MCF7_shNS_shRUNX1_genes_expression_expected_count.tsv - TSV file showing the RNA-seq counts for each geneXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_RNAseq_log2_foldchange.txt - TSV file showing the log2 fold change of all  genes between shRUNX1 and shNS samples', 0, '', 'Hi-C was sequenced using PE100 reads; RNA-seq and ChIP-seq were sequenced using SE100 reads by using a HiSeq 2000 instrument', 'GSM1942109', 'Illumina HiSeq 2000', 'Aug 23 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04272130Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1434850', 'GSE75070', 'Human breast cancer cell line', 'Public on Aug 23 2016', 'Nov 16 2015', 'NONE', '', 9606, 'MCF-7 shRUNX1  RNA-seq Replicate3', 'SRA', 11558048, 32568047),
(1, 'cell line: Breast cancer cell line T47D-MTVLXx-xXtreatment: EstradiolXx-xXtime (minutes): 360', 'Aiguader 88', 'Barcelona', 'Spain', '', 'miguel.beato@crg.eu', 'CRG', '', 'Miguel,,Beato', '', '8003', 'Basecalls performed using Illumina pipeline version 1.4.0Xx-xXChIP-Seq Single-end reads were processed by aligning to the reference human genome (GRCh37/hg19) using Bowtie v1.0.0. ChIP-seq signals were summed in windows of 100 Kb. Summed reads were normalized for sequencing depth and divided by the corresponding signal obtained for an input DNA of T47D cells to determine the normalized signal over input enrichment/depletion. Homer (Hypergeometric Optimization of Motif EnRichment) v4.3 was used to find regions of significance enrichment using an input as a control.Xx-xXHiC Paired-end reads were processed by first aligning to the reference human genome (GRCh37/hg19) using BWA. Reads filtered from analysis include those that were not uniquely mapped, mapped more than 500 bp from relevant restriction sites, had bad sequence quality (e.g. 40% or more of the bases with Sanger PHRED quality ≤ 2) or bad BWA mapping quality (≤ 20), or were located in regions classified to have exceptionally high sequence depth (top 0.1%) by the 1000 Genomes project’s data. To offset bias introduced by PCR amplification of the sequencing library, only one of the duplicated pairs were used for subsequent analyses. Datasets were used to generate contact matrices at 100 Kb.Xx-xXRNA-Seq Paired-end reads were mapped using GEM mRNA Mapping Pipeline (v1.7) using last Gencode Annotation version 18, which followed specific steps: 1) Map reads to genome reference index (including chromosme 1 to X) with maximum mismatch ratio of 0.06 and base quality threshold of 26. 2) Map reads to a transcriptome index generated from previous reference index using same options as step 1. 3) Create de-novo trasncriptome index and map again the reads to this index using maximum mismatch ratio of 0.04, minimmum split size of 15 bp and maximmum of 500000 bp. 4) Map reads again to de-novo transcriptome index created using same options as step 1. 5) Merge mappings created and pair alignments using with base quality threshold of 26 and maximmum insertion size of 500000 bp. As a result BAM alignment files were obtained and used to generate strand-specific genome-wide normalized profiles using RSeQC3 software. Exon quantifications summarized by per-genes for expression level determination were obtained either as raw read counts and reads per kilobase per milion mapped reads (RPKM) using Flux Capacitor.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: ChIP-seq data files are peak results “TSV” files with genomic coordinates of peaks. HiC data files are whole genome “TSV” matrices. RNA-Seq data file is a “TSV” table which summarizes read counts per genes.', 0, 'Sample 12Xx-xXmRNA-seqXx-xXprocessed data file: rna-seq.readCounts.txt', 'Total RNA was extracted using Trizol.Xx-xXmRNA-Seq sample prep kit (ref. RS-930-1001, Illumina)', 'GSM1294045', 'Illumina HiSeq 2000', 'Oct 06 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02464034Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX395607', 'GSE53463', 'Breast cancer cell line', 'Public on Oct 06 2014', 'Dec 18 2013', 'NONE', '', 9606, 'RNA-Seq E6', 'SRA', 39866651, 137914437),
(1, 'cell line: BT20Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'BT20_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897279', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293292Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122158', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897279/suppl/GSM1897279_BT20_FPKM.txt.gz', '', 9606, 'BT20', 'SRA', 17100131, 18415906),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401713', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821094Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559778', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401713/suppl/GSM1401713_SL10860_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401713/suppl/GSM1401713_SL10860_gene.fpkm_tracking.gz', 9606, '09-10-A057a', 'SRA', 43641015, 63794541),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401696', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821143Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559761', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401696/suppl/GSM1401696_SL10795_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401696/suppl/GSM1401696_SL10795_gene.fpkm_tracking.gz', 9606, '04-09-A066b', 'SRA', 54078309, 68195611);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401676', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821119Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559741', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401676/suppl/GSM1401676_SL10633_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401676/suppl/GSM1401676_SL10633_gene.fpkm_tracking.gz', 9606, '00-11-A037b', 'SRA', 34812227, 53336273),
(1, 'cell line: Breast cancer cell line T47D-MTVLXx-xXtreatment: ProgesteroneXx-xXtime (minutes): 360', 'Aiguader 88', 'Barcelona', 'Spain', '', 'miguel.beato@crg.eu', 'CRG', '', 'Miguel,,Beato', '', '8003', 'Basecalls performed using Illumina pipeline version 1.4.0Xx-xXChIP-Seq Single-end reads were processed by aligning to the reference human genome (GRCh37/hg19) using Bowtie v1.0.0. ChIP-seq signals were summed in windows of 100 Kb. Summed reads were normalized for sequencing depth and divided by the corresponding signal obtained for an input DNA of T47D cells to determine the normalized signal over input enrichment/depletion. Homer (Hypergeometric Optimization of Motif EnRichment) v4.3 was used to find regions of significance enrichment using an input as a control.Xx-xXHiC Paired-end reads were processed by first aligning to the reference human genome (GRCh37/hg19) using BWA. Reads filtered from analysis include those that were not uniquely mapped, mapped more than 500 bp from relevant restriction sites, had bad sequence quality (e.g. 40% or more of the bases with Sanger PHRED quality ≤ 2) or bad BWA mapping quality (≤ 20), or were located in regions classified to have exceptionally high sequence depth (top 0.1%) by the 1000 Genomes project’s data. To offset bias introduced by PCR amplification of the sequencing library, only one of the duplicated pairs were used for subsequent analyses. Datasets were used to generate contact matrices at 100 Kb.Xx-xXRNA-Seq Paired-end reads were mapped using GEM mRNA Mapping Pipeline (v1.7) using last Gencode Annotation version 18, which followed specific steps: 1) Map reads to genome reference index (including chromosme 1 to X) with maximum mismatch ratio of 0.06 and base quality threshold of 26. 2) Map reads to a transcriptome index generated from previous reference index using same options as step 1. 3) Create de-novo trasncriptome index and map again the reads to this index using maximum mismatch ratio of 0.04, minimmum split size of 15 bp and maximmum of 500000 bp. 4) Map reads again to de-novo transcriptome index created using same options as step 1. 5) Merge mappings created and pair alignments using with base quality threshold of 26 and maximmum insertion size of 500000 bp. As a result BAM alignment files were obtained and used to generate strand-specific genome-wide normalized profiles using RSeQC3 software. Exon quantifications summarized by per-genes for expression level determination were obtained either as raw read counts and reads per kilobase per milion mapped reads (RPKM) using Flux Capacitor.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: ChIP-seq data files are peak results “TSV” files with genomic coordinates of peaks. HiC data files are whole genome “TSV” matrices. RNA-Seq data file is a “TSV” table which summarizes read counts per genes.', 0, 'Sample 11Xx-xXmRNA-seqXx-xXprocessed data file: rna-seq.readCounts.txt', 'Total RNA was extracted using Trizol.Xx-xXmRNA-Seq sample prep kit (ref. RS-930-1001, Illumina)', 'GSM1294044', 'Illumina HiSeq 2000', 'Oct 06 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02464031Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX395606', 'GSE53463', 'Breast cancer cell line', 'Public on Oct 06 2014', 'Dec 18 2013', 'NONE', '', 9606, 'RNA-Seq R6', 'SRA', 51549371, 142986449),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 320 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533421', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145669Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745852', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533421/suppl/GSM1533421_MCF7_mRNA_t0320.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533421/suppl/GSM1533421_MCF7_premRNA_t0320min.wig.txt.gz', 9606, 'MCF7_E2_320min_RNASeq', 'SRA', 41970909, 92579731),
(1, 'cell line: HCC2688Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC2688_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897308', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293321Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122127', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897308/suppl/GSM1897308_HCC2688_FPKM.txt.gz', '', 9606, 'HCC2688', 'SRA', 31395056, 39691769),
(1, 'cell line: MCF7 breast tumor cell line', 'Via P. Castellino, 111', 'Naples', 'Italy', 'Institute of Genetics and Biophysics', 'valerio.costa@igb.cnr.it', 'National Research Council', '', 'Valerio,,Costa', '3.90816E+11', '80131', 'Sequenced reads were trimmed for adaptor sequence, and for low-quality sequence, then mapped to hg19 whole genome using TopHat2 v2.0.10 with parameters: -p 8 -N 2 -g 10 -r 200 -a 15 -m 1 -i 100 --library-type fr-unstranded --segment-mismatches 3 --read-edit-dist 2 -GXx-xXGene expression and alternative splicing were analyzed using Cufflinks v2.1.1 and a computational procedure described in Scarpato et al., 2014Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Processed data are provided in tab delimited text format and consist of normalized counts (FPKM);', 0, 'Replicate 1', 'Total RNA was extracted using TRIzol. The integrity and quantity of RNAs were assessed by Experion.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocol. Illumina TruSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used  for the construction of sequencing libraries.Xx-xXRNA-Seq paired-end (75+75 bp)', 'GSM1666032', 'Illumina HiSeq 2000', 'Jun 01 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03566778Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1008004', 'GSE68228', 'MCF7 breast-cancer cell line', 'Public on Jun 01 2015', 'Apr 24 2015', 'NONE', '', 9606, 'MCF7_Sample 1', 'SRA', 28646575, 32879534),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401672', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821011Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559737', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401672/suppl/GSM1401672_SL7545_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401672/suppl/GSM1401672_SL7545_gene.fpkm_tracking.gz', 9606, 'HCC-1187', 'SRA', 56836757, 88001843),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47-I281fsXx-xXdoxycycline concentration: 2.5 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (MUT3) of the MDA231 breast cancer cell line transduced with doxycycline-inducible RBM47-I281fs mutant.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300633', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566887Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404083', 'GSE53779', 'breast cancer cell line clone, RBM47-I281fs mutant, 2.5 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'MUT3-dox-2.5', 'SRA', 44560274, 49949493),
(1, 'cell line: HCC2185Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC2185_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897306', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293319Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122185', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897306/suppl/GSM1897306_HCC2185_FPKM.txt.gz', '', 9606, 'HCC2185', 'SRA', 30530404, 37757916),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 5 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533415', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145657Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745846', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533415/suppl/GSM1533415_MCF7_mRNA_t0005.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533415/suppl/GSM1533415_MCF7_premRNA_t0005min.wig.txt.gz', 9606, 'MCF7_E2_5min_RNASeq', 'SRA', 41039007, 83716641),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401698', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821140Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559763', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401698/suppl/GSM1401698_SL10799_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401698/suppl/GSM1401698_SL10799_gene.fpkm_tracking.gz', 9606, '05-07-A186a', 'SRA', 40275569, 54794196),
(1, 'cell line: MDA-MB-231 breast cancer cell lineXx-xXtreatment: CIP2A siRNA', '3181 S.W. Sam Jackson Park Rd', 'Portland', 'USA', 'MMG', 'searsr@ohsu.edu', 'OHSU', '', 'Rosalie,,Sears', '5034946885', '97239-3098', 'Samples were sequenced on an Illumina HiSeq 2000 and assembled into fastq files using the CASAVA software at the OHSU Massively Parallel Sequencing Shared Resource.Xx-xXSequences were aligned to the human hg19 genome using Bowtie version 1.0.0 and seting requiring unique alignments with no more than 3 mismatches.Xx-xXTrim settings for Bowtie were selected to discard the first 4 bases, align to the next 44, and discard remaining bases from each sequence.Xx-xXAligned sequences that fell in exonic regions of RefSeq gene models were counted and normalized for sequencing depth.Xx-xXGenes that showed low normalized sequence counts for all samples were discarded from further analysis.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Tab-delimited .txt file containing normalized sequence counts for gene models before filtering out low expression genes.', 0, '', 'RNA was extracted from cells by Trozol reagent.Xx-xXTotal RNA was purified and reverse transcribed with oligo dT. Second strand DNA was synthesized and ligated to proprietary indexed adapters. Ligated material was amplified using Illumina genomic primers and sequenced on an Illumina GAII sequencer.', 'GSM1399412', 'Illumina HiSeq 2000', 'May 28 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02800676Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX554354', 'GSE58008', 'MDA-MB-231 breast cancer cell line', 'Public on May 28 2014', 'May 27 2014', 'NONE', '', 9606, 'siCIP2A', 'SRA', 14848279, 16501529),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 4Xx-xXtreatment: 25 nM Silvestrol for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with 25 nM Silvestrol for 2 hours; biological replicate', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503441', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067409Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699966', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_silvestrol_2hr_br', 'SRA', 3348652, 74824980),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 5 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT6) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300624', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566880Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404074', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 5 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT6-dox-5', 'SRA', 41030861, 45600785),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401758', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821081Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559823', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401758/suppl/GSM1401758_SL9348_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401758/suppl/GSM1401758_SL9348_gene.fpkm_tracking.gz', 9606, '06-04-A286b', 'SRA', 31948283, 50208208),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 640 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533422', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145670Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745853', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533422/suppl/GSM1533422_MCF7_mRNA_t0640.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533422/suppl/GSM1533422_MCF7_premRNA_t0640min.wig.txt.gz', 9606, 'MCF7_E2_640min_RNASeq', 'SRA', 43618189, 91156939),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401745', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821067Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559810', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401745/suppl/GSM1401745_SL7933_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401745/suppl/GSM1401745_SL7933_gene.fpkm_tracking.gz', 9606, '03-06-A283b', 'SRA', 64412886, 107816296),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401735', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821121Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559800', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401735/suppl/GSM1401735_SL7918_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401735/suppl/GSM1401735_SL7918_gene.fpkm_tracking.gz', 9606, '04-06-A164a', 'SRA', 40385667, 68649552);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401715', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821104Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559780', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401715/suppl/GSM1401715_SL10864_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401715/suppl/GSM1401715_SL10864_gene.fpkm_tracking.gz', 9606, '10-01-A051a', 'SRA', 35356213, 50742277),
(1, 'cell line: SUM1315Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM1315_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897340', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293353Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122346', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897340/suppl/GSM1897340_SUM1315_FPKM.txt.gz', '', 9606, 'SUM1315', 'SRA', 39796240, 43515924),
(1, 'cell line: MDA-MB-231 breast cancer cell lineXx-xXtreatment: 1μMOP449-12hr', '3181 S.W. Sam Jackson Park Rd', 'Portland', 'USA', 'MMG', 'searsr@ohsu.edu', 'OHSU', '', 'Rosalie,,Sears', '5034946885', '97239-3098', 'Samples were sequenced on an Illumina HiSeq 2000 and assembled into fastq files using the CASAVA software at the OHSU Massively Parallel Sequencing Shared Resource.Xx-xXSequences were aligned to the human hg19 genome using Bowtie version 1.0.0 and seting requiring unique alignments with no more than 3 mismatches.Xx-xXTrim settings for Bowtie were selected to discard the first 4 bases, align to the next 44, and discard remaining bases from each sequence.Xx-xXAligned sequences that fell in exonic regions of RefSeq gene models were counted and normalized for sequencing depth.Xx-xXGenes that showed low normalized sequence counts for all samples were discarded from further analysis.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Tab-delimited .txt file containing normalized sequence counts for gene models before filtering out low expression genes.', 0, '', 'RNA was extracted from cells by Trozol reagent.Xx-xXTotal RNA was purified and reverse transcribed with oligo dT. Second strand DNA was synthesized and ligated to proprietary indexed adapters. Ligated material was amplified using Illumina genomic primers and sequenced on an Illumina GAII sequencer.', 'GSM1399414', 'Illumina HiSeq 2000', 'May 28 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02800678Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX554356', 'GSE58008', 'MDA-MB-231 breast cancer cell line', 'Public on May 28 2014', 'May 27 2014', 'NONE', '', 9606, '1μMOP449-12hr', 'SRA', 12439224, 13732685),
(1, 'cell line: MCF12AXx-xXcell type: breast cancer cancer cell lineXx-xXstable expression: empty vector', '12801 E. 17th Ave.', 'Aurora', 'USA', 'Medicine', 'Jihye.Kim@UCDenver.edu', 'UCDenver', 'Tan', 'Jihye,,Kim', '303-724-8229', '80016', 'Alignment: TopHat v1.4.1 using default settings. hg19 (UCSC).Xx-xXExpression levels: Cufflinks v1.3.0 and cuffdiff using default settings.Xx-xXGSEA anlalysis with GeneSigDB v4.Xx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Tab-delimited text file with FPKM values.', 0, 'mRNA', 'Total RNA was extracted and mRNA isolated using the miRNeasy Kit (Qiagen) according to manufacturer\'s instructions.Xx-xXRNA libraries were prepared for sequencing using TruSEQ RNA Sample Prep v2 using standard Illumina protocols.', 'GSM1317637', 'Illumina HiSeq 2000', 'Feb 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02598923Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX451745', 'GSE54505', 'breast cancer cancer cell line, control', 'Public on Feb 26 2015', 'Jan 29 2014', 'NONE', '', 9606, 'EV2', 'SRA', 54831173, 63003053),
(1, 'subtype: luminalXx-xXcell line: ZR75-1Xx-xXagent: DMSO', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556183', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246802Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790670', 'GSE63721', 'Breast cancer cell line (luminal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556183/suppl/GSM1556183_ZR751_DMSO_R2.csv.gz', '', 9606, 'ZR751_DMSO_R2', 'SRA', 32130393, 35521620),
(1, 'cell line: HCC38Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC38_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897310', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293323Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122129', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897310/suppl/GSM1897310_HCC38_FPKM.txt.gz', '', 9606, 'HCC38', 'SRA', 61041120, 67164499),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: 200 nM INK128 for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with 200 nM INK128 for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503443', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067411Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699968', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_ink128', 'SRA', 20347767, 387554714),
(1, 'cell line: SUM225Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM225_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897345', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293358Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122332', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897345/suppl/GSM1897345_SUM225_FPKM.txt.gz', '', 9606, 'SUM225', 'SRA', 51092401, 56223287),
(1, 'cell line: ZR7530Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'ZR7530_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897355', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293368Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122342', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897355/suppl/GSM1897355_ZR7530_FPKM.txt.gz', '', 9606, 'ZR7530', 'SRA', 33257373, 41052587),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 12h, DMSO', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431877', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905574Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648511', 'GSE59251', 'MCF-7 breast cancer cell line_12h Control-rep2', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '12h Control-rep2', 'SRA', 19658603, 21249046),
(1, 'cell line: MDA-MB-231Xx-xXcell type: highly invasiveXx-xXtransfected with: EGFP (control)Xx-xXgenotype/variation: EGFP control', '1501 San Pablo St.', 'Los Angeles', 'USA', 'Biomedical Engineering', 'nanlee@usc.edu', 'University of Southern California', '', 'Nan,Sook,Lee', '6268243058', '90033', 'Bcl conversion: Illumina bcl2Fastq ver 1.8.4 to convert from bcl format to fastq.gz raw reads fileXx-xXAlignment and read counting done with GTFAR ver 0.1 using gencode19 annotation.Xx-xXA second alignment was done with Tophat ver 1.4.0 using hg19_ucsc reference and parameter -G hg19_ucsc.gtf (optional reference annotation)Xx-xXTranscript abundance estimation on the second alignment done with Cufflinks ver 1.0.1 using parameters -u (for multi-read-correct), -b hg19_ucsc.fa (for bias detection and correction) and -G hg19_ucsc.gtf (for optional reference annotation)Xx-xXGenome_build: hg19_ucscXx-xXSupplementary_files_format_and_content: tab-delimited text files from GT-FAR runs and Cufflinks runs which include RPKM values for each sample and isoform counts for each sample; more details in the \'README_1.txt\'', 0, '', 'Total RNAs were isolated from cell lines using Trizol (Invitrogen) and digested with Turbo DNase (Ambion) to remove genomic DNA, according to the manufacturer’s instructions.Xx-xXLibraries from rRNA-depleted samples were prepared using a TruSeq RNA Sample Preparation kit v2 (Illumina) following the recommended protocol starting from the RNA fragmentation stage. Purification of polyadenylated RNA was omitted. Libraries were pooled (4 samples per pool), clustered on cBOT (Illumina), and sequenced on HiSeq2000, each pool in one lane. We performed single-end 100 bp reads. Reads were mapped using TopHat followed by data analysis using Cufflinks and Cuffdiff software, as well as our own pipeline.', 'GSM1553458', 'Illumina HiSeq 2000', 'Dec 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03220239Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX767449', 'GSE63608Xx-xXGSE63610', 'Breast cancer cell line', 'Public on Dec 01 2014', 'Nov 25 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553458/suppl/GSM1553458_GTFAR-MB-231-EGFP-CONTROL_geneRPKMS.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553458/suppl/GSM1553458_GTFAR-MB-231-EGFP-CONTROL_rawRestIsoformCnts.txt.gz', 9606, 'EGFP', 'SRA', 61550833, 166946670),
(1, 'cell line: UACC3199Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'UACC3199_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897351', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293364Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122338', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897351/suppl/GSM1897351_UACC3199_FPKM.txt.gz', '', 9606, 'UACC3199', 'SRA', 35918931, 44111127),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401649', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820982Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559714', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401649/suppl/GSM1401649_SL5422_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401649/suppl/GSM1401649_SL5422_gene.fpkm_tracking.gz', 9606, 'BT-474', 'SRA', 21429004, 31045901),
(1, 'cell line: SUM44Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM44_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897347', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293360Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122334', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897347/suppl/GSM1897347_SUM44_FPKM.txt.gz', '', 9606, 'SUM44', 'SRA', 35824554, 40763107),
(1, 'cell line: BT474Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'BT474_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897280', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293293Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122159', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897280/suppl/GSM1897280_BT474_FPKM.txt.gz', '', 9606, 'BT474', 'SRA', 32675978, 36222251),
(1, 'cell line: T47DXx-xXdisease state: breast cancer', '4500 San Pablo Road South', 'Jacksonville', 'USA', '', 'asmann.yan@mayo.edu', 'Mayo Clinic', '', 'Yan,W.,Asmann', '', '32224', 'Illumina\'s Genome Analyzer system software programs for base-callingXx-xXthe 50bp paired end reads were aligned using BWA 0.6.2Xx-xXthe input files were the fastq files of the two ends, and the out put files were the SnowShoes-FTD called fusion transcriptsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The .txt file reports fusion transcripts identified using SnowShoes-FTD version 2.0', 0, 'Sample 7', 'RNA was extracted by Qiagen Mini RNeasy Kit (Qiagen)Xx-xXTruSeq RNA sample preparation kit (Illumina)', 'GSM1273498', 'Illumina HiSeq 2000', 'Nov 22 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02419120Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX381538', 'GSE52643', 'breast cancer cell line', 'Public on Nov 22 2013', 'Nov 21 2013', 'NONE', '', 9606, 'T47D', 'SRA', 24628553, 27834336),
(1, 'cell line: MDAMB134VIXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB134VI_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897323', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293336Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122143', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897323/suppl/GSM1897323_MDAMB134VI_FPKM.txt.gz', '', 9606, 'MDAMB134VI', 'SRA', 40353576, 44843510),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401754', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821076Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559819', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401754/suppl/GSM1401754_SL8361_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401754/suppl/GSM1401754_SL8361_gene.fpkm_tracking.gz', 9606, '03-12-A154a', 'SRA', 35605063, 49719373),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401739', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821049Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559804', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401739/suppl/GSM1401739_SL7924_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401739/suppl/GSM1401739_SL7924_gene.fpkm_tracking.gz', 9606, '03-12-A386a', 'SRA', 47363852, 82632530),
(1, 'cell line: MCF-7Xx-xXcell type: weakly invasiveXx-xXtransfected with: si-GAPDH (control)Xx-xXgenotype/variation: control', '1501 San Pablo St.', 'Los Angeles', 'USA', 'Biomedical Engineering', 'nanlee@usc.edu', 'University of Southern California', '', 'Nan,Sook,Lee', '6268243058', '90033', 'Bcl conversion: Illumina bcl2Fastq ver 1.8.4 to convert from bcl format to fastq.gz raw reads fileXx-xXAlignment and read counting done with GTFAR ver 0.1 using gencode19 annotation.Xx-xXA second alignment was done with Tophat ver 1.4.0 using hg19_ucsc reference and parameter -G hg19_ucsc.gtf (optional reference annotation)Xx-xXTranscript abundance estimation on the second alignment done with Cufflinks ver 1.0.1 using parameters -u (for multi-read-correct), -b hg19_ucsc.fa (for bias detection and correction) and -G hg19_ucsc.gtf (for optional reference annotation)Xx-xXGenome_build: hg19_ucscXx-xXSupplementary_files_format_and_content: tab-delimited text files from GT-FAR runs and Cufflinks runs which include RPKM values for each sample and isoform counts for each sample; more details in the \'README_1.txt\'', 0, '', 'Total RNAs were isolated from cell lines using Trizol (Invitrogen) and digested with Turbo DNase (Ambion) to remove genomic DNA, according to the manufacturer’s instructions.Xx-xXLibraries from rRNA-depleted samples were prepared using a TruSeq RNA Sample Preparation kit v2 (Illumina) following the recommended protocol starting from the RNA fragmentation stage. Purification of polyadenylated RNA was omitted. Libraries were pooled (4 samples per pool), clustered on cBOT (Illumina), and sequenced on HiSeq2000, each pool in one lane. We performed single-end 100 bp reads. Reads were mapped using TopHat followed by data analysis using Cufflinks and Cuffdiff software, as well as our own pipeline.', 'GSM1553455', 'Illumina HiSeq 2000', 'Dec 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03220237Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX767446', 'GSE63608Xx-xXGSE63610', 'Breast cancer cell line', 'Public on Dec 01 2014', 'Nov 25 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553455/suppl/GSM1553455_GTFAR-MCF-7-si-GAPDH-CONTROL_geneRPKMS.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553455/suppl/GSM1553455_GTFAR-MCF-7-si-GAPDH-CONTROL_rawRestIsoformCnts.txt.gz', 9606, 'si-Gapdh', 'SRA', 45346633, 96919901),
(1, 'cell line: SUM149Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM149_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897341', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293354Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122347', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897341/suppl/GSM1897341_SUM149_FPKM.txt.gz', '', 9606, 'SUM149', 'SRA', 87514604, 100534884);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401753', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821079Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559818', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401753/suppl/GSM1401753_SL8233_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401753/suppl/GSM1401753_SL8233_gene.fpkm_tracking.gz', 9606, '03-09-A087c', 'SRA', 59696154, 75220747),
(1, 'cell line: MB157Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MB157_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897319', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293332Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122139', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897319/suppl/GSM1897319_MB157_FPKM.txt.gz', '', 9606, 'MB157', 'SRA', 38823603, 47017345),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401707', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821102Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559772', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401707/suppl/GSM1401707_SL10814_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401707/suppl/GSM1401707_SL10814_gene.fpkm_tracking.gz', 9606, '08-09-A192a', 'SRA', 49284754, 62350776),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401683', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821125Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559748', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401683/suppl/GSM1401683_SL10646_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401683/suppl/GSM1401683_SL10646_gene.fpkm_tracking.gz', 9606, '01-05-A359a', 'SRA', 43589567, 71612889),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401751', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821075Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559816', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401751/suppl/GSM1401751_SL8229_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401751/suppl/GSM1401751_SL8229_gene.fpkm_tracking.gz', 9606, '03-07-A523B', 'SRA', 65954069, 82674106),
(1, 'cell line: ZR-75-1Xx-xXdisease state: breast cancer', '4500 San Pablo Road South', 'Jacksonville', 'USA', '', 'asmann.yan@mayo.edu', 'Mayo Clinic', '', 'Yan,W.,Asmann', '', '32224', 'Illumina\'s Genome Analyzer system software programs for base-callingXx-xXthe 50bp paired end reads were aligned using BWA 0.6.2Xx-xXthe input files were the fastq files of the two ends, and the out put files were the SnowShoes-FTD called fusion transcriptsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The .txt file reports fusion transcripts identified using SnowShoes-FTD version 2.0', 0, 'Sample 8', 'RNA was extracted by Qiagen Mini RNeasy Kit (Qiagen)Xx-xXTruSeq RNA sample preparation kit (Illumina)', 'GSM1273499', 'Illumina HiSeq 2000', 'Nov 22 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02419116Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX381539', 'GSE52643', 'breast cancer cell line', 'Public on Nov 22 2013', 'Nov 21 2013', 'NONE', '', 9606, 'ZR-75-1', 'SRA', 22632813, 28279001),
(1, 'cell line: CAMA1Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'CAMA1_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897287', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293300Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122166', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897287/suppl/GSM1897287_CAMA1_FPKM.txt.gz', '', 9606, 'CAMA1', 'SRA', 24791177, 27251600),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401658', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821015Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559723', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401658/suppl/GSM1401658_SL6572_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401658/suppl/GSM1401658_SL6572_gene.fpkm_tracking.gz', 9606, 'MDA-MB-231', 'SRA', 84167337, 108951546),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401749', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821070Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559814', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401749/suppl/GSM1401749_SL8214_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401749/suppl/GSM1401749_SL8214_gene.fpkm_tracking.gz', 9606, '04-01-A168c', 'SRA', 42341081, 60766284),
(1, 'cell line: MDAMB468Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB468_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897332', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293345Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122151', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897332/suppl/GSM1897332_MDAMB468_FPKM.txt.gz', '', 9606, 'MDAMB468', 'SRA', 35250611, 39370809),
(1, 'cell line: AU565Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'AU565_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897278', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293291Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122157', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897278/suppl/GSM1897278_AU565_FPKM.txt.gz', '', 9606, 'AU565', 'SRA', 42678446, 47751786),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401720', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821113Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559785', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401720/suppl/GSM1401720_SL13096_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401720/suppl/GSM1401720_SL13096_gene.fpkm_tracking.gz', 9606, '06-06-A277a', 'SRA', 37844478, 63007173),
(1, 'cell line: ZR75BXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'ZR75B_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897356', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293369Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122343', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897356/suppl/GSM1897356_ZR75B_FPKM.txt.gz', '', 9606, 'ZR75B', 'SRA', 33117783, 39761975),
(1, 'gender: FemaleXx-xXcell line: ZR751Xx-xXcell type: Luminal', '26, Rue d\'Ulm', 'Paris', 'France', '', 'Ronan.Chaligne@curie.fr', 'Institut Curie', 'Edith Heard\'s lab', 'Ronan,,Chaligné', '', '75248', 'Supplementary_files_format_and_content: Each sample contains three bedgraph files.Xx-xXSupplementary_files_format_and_content: 1. ChrX full bedgraph displayXx-xXSupplementary_files_format_and_content: 2. ChrX allele1 specific display of bedgraph only on SNP positionsXx-xXSupplementary_files_format_and_content: 3. ChrX allele2 specific display of bedgraph only on SNP positionsXx-xXSupplementary_files_format_and_content: Expression results: fpkm_tracking for genes standard output from cufflinksXx-xXSupplementary_files_format_and_content: Final SNP list: List of final SNPs derived from ChIP-seq, mRNA-seq and Exome-seq used for allele specific analysisXx-xXDatasets were subjected to two types of quality control. FASTQC-0.10.1 (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) was used to assess the quality of sequencing and potential adapter or cross contaminants. Average sequencing quality (phred score) per base was above 30 (Q ≥ 30) for all datasets. In addition, aligned datasets were then subjected toNGS-QC ((Mendoza-Parra et al., 2013); www.ngs-qc.org) to assess the robustness of enrichment. The majority of data sets were of «triple A» quality, no data set was below «triple B». For exome-seq and ChIP-seq, alignment was performed using BWA-MEM-0.7.7 (Li and Durbin, 2009) with default parameters, which simultaneously checks for both global and local alignment for reads. Alignment was followed by three sets of filters to prevent bias in the analysis. 1) duplicate reads (PCR clonal reads) were filtered out using Picard tools-1.86 (http://picard.sourceforge.net) 2) reads with mapping quality less than 10 were filtered out using Bamtools-2.2.3 (https://github.com/pezmaster31/bamtools) and 3) reads with more than one alignment reported were filtered out using in-house scripts. Further analysis was carried out on processed alignment file which is around 10-100M reads after these filters.Xx-xXTo prepare the allele information for each cell-line for allele-specific analysis down the line, SNP analysis was carried out along with SNP6 data. To identify novel variation (apart from known SNPs from SNP6 data), SNP analysis was carried out on all three ChIP-seq, Exome-seq and RNA-seq data individually. ChIP-seq data of different marks (H3K4me3, H3K27me3, RNA Pol II and Input) were merged for each cell line to increase the depth and confidence for variation calling. Variation calling was performed for each cell line separately for ChIP-seq and Exome-seq following the best practice GATK-2.6.5 pipeline by filtering reads with Mapping quality ≥ 1 (Van der Auwera GA et al., 2013; http://www.broadinstitute.org/gatk/guide/best-practices?bpm=DNAseq). Variation calling for RNA-seq was carried out following the methods of Piskol, Robert et al., 2013 to avoid artifacts specific to RNA- seq data (Piskol et al., 2013). A final list of allele information was generated by combining the SNP information from the different data sets for each cell line. To increase the allele-specific sensitivity for the alignment, reads were additionally realigned in an allele-specific manner following the method of Satya et al. (Satya et al., 2012). Read counts for each allele and SNP position were extracted for each mark using in-house scripts. SNP positions with at least three reads from both alleles were considered as heterozygous positions.Xx-xXPeak calling was performed using HOMER ((Heinz et al., 2010); http://homer.salk.edu/homer/index.html) with default parameters. For H3K27me3 and RNA Pol II, the\'style\' parameter was chosen as \'histone\' due to the broad patterns for this mark, whereas for H3K4me3, which generally give sharp peaks, the parameter \'factor\' was chosen. Genomic context annotation over identified peaks were carried out using the HOMER annotation module but with basic annotation by excluding references other than coding genes and non-coding RNA.Xx-xXA gene-based analysis of annotation integration was carried out using in-house scripts to integrate all annotation from peak and variation calling (informative SNP counts, read depths, homo/heterozygous SNP count and weighted allelic imbalance). To include annotations from regulatory regions 1Kb sequences upstream from the TSS and downstream of the TES were considered. A weighted arithmetic average was calculated for each gene by calculating average Allelic Imbalance (AI) where each SNP\'s AI was weighted by its read depth.Xx-xXTo illustrate the comparisons across cell lines, ChIP-seq data were normalized using an in-house developed tool called \'Epimetheus\', which is based on quantile normalization (manuscript under preparation). Read Count Intensity (RCI) was calculated for a window of 100bp bin size across chromosomes and then these intensities were normalized using quantile normalization from the limma package. The impact of normalization was assessed using MA plots before and after normalization. Specified genomic feature based normalized RCI was constructed, which are illustrated in Figure 6. For TSS-centered plots and heatmaps, a separate TSS-based normalization was carried out with 20bp bin size to obtain higher resolution.Xx-xXGenome_build: hg19', 0, '', 'Total RNA was extracted using the Trizol (Invitrogen) procedure.Xx-xXStandard Illumina protocols are followed.Xx-xXmRNA: total RNA was enriched for polyadenylated RNA (polyA mRNA).Xx-xXmRNA : RNA-sequencing correspond of paired-reads lane 2x100 bp sequencing on poly-A RNA purified  and were prepared according to the standard Illumina protocol. Sequencing has been performed on high-throughput Illumina HiSeq sequencer.', 'GSM1537307', 'Illumina HiSeq 2000', 'Jan 19 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03160894Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX750682', 'GSE62907Xx-xXGSE62966', 'Human Breast Cancer Cells', 'Public on Jan 19 2015', 'Nov 04 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537307/suppl/GSM1537307_ZR751-mRNA.chrX.SNPs.allele1.bedgraph.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537307/suppl/GSM1537307_ZR751-mRNA.chrX.SNPs.allele2.bedgraph.gz', 9606, 'ZR751_mRNA', 'SRA', 43945125, 50487800);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401675', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821009Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559740', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401675/suppl/GSM1401675_SL8450_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401675/suppl/GSM1401675_SL8450_gene.fpkm_tracking.gz', 9606, 'SUM-102', 'SRA', 73238903, 98973518),
(1, 'cell line: MDA-MB-231 breast cancer cell lineXx-xXtreatment: SET siRNA-48hr', '3181 S.W. Sam Jackson Park Rd', 'Portland', 'USA', 'MMG', 'searsr@ohsu.edu', 'OHSU', '', 'Rosalie,,Sears', '5034946885', '97239-3098', 'Samples were sequenced on an Illumina HiSeq 2000 and assembled into fastq files using the CASAVA software at the OHSU Massively Parallel Sequencing Shared Resource.Xx-xXSequences were aligned to the human hg19 genome using Bowtie version 1.0.0 and seting requiring unique alignments with no more than 3 mismatches.Xx-xXTrim settings for Bowtie were selected to discard the first 4 bases, align to the next 44, and discard remaining bases from each sequence.Xx-xXAligned sequences that fell in exonic regions of RefSeq gene models were counted and normalized for sequencing depth.Xx-xXGenes that showed low normalized sequence counts for all samples were discarded from further analysis.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Tab-delimited .txt file containing normalized sequence counts for gene models before filtering out low expression genes.', 0, '', 'RNA was extracted from cells by Trozol reagent.Xx-xXTotal RNA was purified and reverse transcribed with oligo dT. Second strand DNA was synthesized and ligated to proprietary indexed adapters. Ligated material was amplified using Illumina genomic primers and sequenced on an Illumina GAII sequencer.', 'GSM1399411', 'Illumina HiSeq 2000', 'May 28 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02800677Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX554353', 'GSE58008', 'MDA-MB-231 breast cancer cell line', 'Public on May 28 2014', 'May 27 2014', 'NONE', '', 9606, 'siSET', 'SRA', 9622440, 10570272),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401716', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821095Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559781', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401716/suppl/GSM1401716_SL12607_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401716/suppl/GSM1401716_SL12607_gene.fpkm_tracking.gz', 9606, '02-01-A055b', 'SRA', 60180122, 88509866),
(1, 'cell line: MCF12AXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MCF12A_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897321', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293334Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122141', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897321/suppl/GSM1897321_MCF12A_FPKM.txt.gz', '', 9606, 'MCF12A', 'SRA', 33697042, 41611204),
(1, 'cell line: MDAMB453Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB453_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897331', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293344Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122150', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897331/suppl/GSM1897331_MDAMB453_FPKM.txt.gz', '', 9606, 'MDAMB453', 'SRA', 37461506, 42323781),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401688', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821130Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559753', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401688/suppl/GSM1401688_SL10658_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401688/suppl/GSM1401688_SL10658_gene.fpkm_tracking.gz', 9606, '02-06-A431a', 'SRA', 53491666, 71437487),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MCF7_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897322', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293335Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122142', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897322/suppl/GSM1897322_MCF7_FPKM.txt.gz', '', 9606, 'MCF7', 'SRA', 41801704, 47425419),
(1, 'cell line: MCF7+EV (Empty Vector)', '5640 Fishers Lane', 'Rockville', 'USA', 'NIAID', 'epugacheva@niaid.nih.gov', 'NIH', 'LIG', 'Elena,,Pugacheva', '', '20852', 'Sequences generated by the Illumina genome analyzer (36bp and 50bp reads) were aligned against either human (build hg19) or mouse (build mm9) genome using Bowtie program (http://bowtie-bio.sourceforge.net).  The alignment was performed with default parameters except the sequence tags that mapped to more than one location in the genome were excluded from the analysis using –m1 option.Xx-xXPeaks were called using Model-based Analysis for ChIP-Seq (MACS) (http://liulab.dfci.harvard.edu/MACS) using default parameters. After MACS, we applied Peak Splitter algorithm (part of MACS) to call sub-peaks, summit of peaks and improve peak resolution.Xx-xXFor the Illumina RNA-seq library, fastq.gz files were mapped to UCSC Human reference (build hg19) using TopHat2 with the default parameter setting of 20 alignments per read and up to 2 mismatches per alignment.Xx-xXFor Ion Torrent RNA-seq sequencing, fastq.gz files were mapped to UCSC Human reference (build hg19) using two-step alignments. First, the reads were aligned with TopHat2. Second, the unmapped reads from the first step are then extracted and aligned with Bowtie2 with –local mode and --very-sensitive-local option.Xx-xXFor both Illumina and Ion Torrent RNA-seq, the resulting aligned reads (BAM files) were then analyzed with Cufflinks 2.0.0 to estimate the transcripts relative abundance using the UCSC reference annotated transcripts (build hg19). The expression of each transcript was quantified as the number of reads mapping to a transcript divided by the transcript length in kilobases and the total number of mapped reads in millions (FPKM).Xx-xXGenome_build: hg19, mm9Xx-xXSupplementary_files_format_and_content: Cuffdiff output.', 0, 'Processed data for this sample is included in the files RNA_Seq_MCF7+EV_VS_MCF7+BORIS_Clone1.txt.gz and RNA_Seq_MCF7+EV_VS_MCF7+BORIS_Clone2.txt.gz available on the sample pages for GSM1817676 and GSM1817677.', 'After overnight incubation with DiaMag magnetic beads (Diagenode, Inc.) and CTCF or BORIS monoclonal or polyclonal antibodies (characterized and described by PMID:15731119; PMID: 21659515; PMID:20231363), precipitated chromatin was then washed, decrosslinked, and digested with proteinase K. The resulting DNA was purified using phenol/chloroform/isoamyl alcohol. DNA concentration was assessed with Quant‐it PicoGreen dsDNA kit (Invitrogen) and 5–10 ng was used to generate sequencing libraries.Xx-xXTruSeq ChIP Sample Preparation Kit (Illumina, Inc., USA). RNA-seq libraries were prepared using either Ion Total RNA-seq v2 kit or TruSeq Stranded RNA LT Kit (Illumina).', 'GSM1817678', 'Illumina HiSeq 2000', 'Jul 24 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03854464Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1091837', 'GSE70764', 'Human breast cancer cell line', 'Public on Jul 23 2015', 'Jul 10 2015', 'NONE', '', 9606, 'MCF7+EV_RNA_Seq', 'SRA', 8982029, 13625480),
(1, 'cell line: SUM185Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM185_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897343', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293356Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122349', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897343/suppl/GSM1897343_SUM185_FPKM.txt.gz', '', 9606, 'SUM185', 'SRA', 37079716, 40893876),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 12h, ZNA, 30 uM', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431881', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905578Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648515', 'GSE59251', 'MCF-7 breast cancer cell line_12hZNA30uM-rep3', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '12hZNA30uM-rep3', 'SRA', 21134928, 22970213),
(1, 'subtype: basalXx-xXcell line: MDAMB468Xx-xXagent: DMSO', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556177', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246801Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790664', 'GSE63721', 'Breast cancer cell line (basal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556177/suppl/GSM1556177_MDAMB468_DMSO_R1.csv.gz', '', 9606, 'MDAMB468_DMSO_R1', 'SRA', 35297092, 39461338),
(1, 'cell line: HCC3153Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC3153_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897309', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293322Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122128', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897309/suppl/GSM1897309_HCC3153_FPKM.txt.gz', '', 9606, 'HCC3153', 'SRA', 39162282, 48872744),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401728', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821101Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559793', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401728/suppl/GSM1401728_SL7732_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401728/suppl/GSM1401728_SL7732_gene.fpkm_tracking.gz', 9606, '09-06-A120A', 'SRA', 59282713, 89132245),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401650', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820992Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559715', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401650/suppl/GSM1401650_SL5485_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401650/suppl/GSM1401650_SL5485_gene.fpkm_tracking.gz', 9606, 'MDA-MB-468', 'SRA', 20014324, 45060440),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401655', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820985Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559720', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401655/suppl/GSM1401655_SL6569_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401655/suppl/GSM1401655_SL6569_gene.fpkm_tracking.gz', 9606, 'HCC-1599', 'SRA', 77600651, 157648179),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401673', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821001Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559738', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401673/suppl/GSM1401673_SL7546_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401673/suppl/GSM1401673_SL7546_gene.fpkm_tracking.gz', 9606, 'BT-20', 'SRA', 71836882, 89273042);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401717', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821109Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559782', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401717/suppl/GSM1401717_SL13098_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401717/suppl/GSM1401717_SL13098_gene.fpkm_tracking.gz', 9606, '06-01-A009b', 'SRA', 49008999, 78822143),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: DMSO for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with DMSO for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503434', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067402Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699959', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_dmso_2hr', 'SRA', 36425360, 224915719),
(1, 'cell line: HCC70Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC70_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897311', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293324Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122130', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897311/suppl/GSM1897311_HCC70_FPKM.txt.gz', '', 9606, 'HCC70', 'SRA', 36420147, 45191511),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 3h, ZNA, 30 uM', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431874', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905571Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648508', 'GSE59251', 'MCF-7 breast cancer cell line_3hZNA30uM-rep2', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '3hZNA30uM-rep2', 'SRA', 23110234, 25316375),
(1, 'subtype: luminalXx-xXcell line: ZR75-1Xx-xXagent: PKC412', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556186', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246804Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790673', 'GSE63721', 'Breast cancer cell line (luminal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556186/suppl/GSM1556186_ZR751_PKC412_R3.csv.gz', '', 9606, 'ZR751_PKC412_R3', 'SRA', 39272960, 43682142),
(1, 'cell line: HCC1937Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1937_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897303', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293316Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122182', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897303/suppl/GSM1897303_HCC1937_FPKM.txt.gz', '', 9606, 'HCC1937', 'SRA', 25559776, 28402795),
(1, 'subtype: luminalXx-xXcell line: ZR75-1Xx-xXagent: PKC412', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556185', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246803Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790672', 'GSE63721', 'Breast cancer cell line (luminal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556185/suppl/GSM1556185_ZR751_PKC412_R2.csv.gz', '', 9606, 'ZR751_PKC412_R2', 'SRA', 30841412, 34212221),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401659', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820987Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559724', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401659/suppl/GSM1401659_SL6573_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401659/suppl/GSM1401659_SL6573_gene.fpkm_tracking.gz', 9606, 'SK-BR-3', 'SRA', 95501883, 135567944),
(1, 'cell line: SW527Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SW527_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897349', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293362Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122336', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897349/suppl/GSM1897349_SW527_FPKM.txt.gz', '', 9606, 'SW527', 'SRA', 36754538, 40693676),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401711', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821099Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559776', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401711/suppl/GSM1401711_SL10857_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401711/suppl/GSM1401711_SL10857_gene.fpkm_tracking.gz', 9606, '09-05-A149a', 'SRA', 39681901, 55830719),
(1, 'cell line: MDAMB157Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB157_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897324', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293337Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122135', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897324/suppl/GSM1897324_MDAMB157_FPKM.txt.gz', '', 9606, 'MDAMB157', 'SRA', 133052851, 147772248),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401648', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820988Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559713', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401648/suppl/GSM1401648_SL5421_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401648/suppl/GSM1401648_SL5421_gene.fpkm_tracking.gz', 9606, 'ZR-75-1', 'SRA', 21132463, 27960342),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401733', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821116Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559798', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401733/suppl/GSM1401733_SL7916_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401733/suppl/GSM1401733_SL7916_gene.fpkm_tracking.gz', 9606, '01-04-A048C', 'SRA', 67467873, 87975129),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 3h, DMSO', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431870', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905567Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648504', 'GSE59251', 'MCF-7 breast cancer cell line_3hControl-rep1', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '3hControl-rep1', 'SRA', 16622168, 18037476),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 4Xx-xXtreatment: DMSO for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with DMSO for 2 hours; biological replicate', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503440', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067408Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699965', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_dmso_2hr_br', 'SRA', 6193119, 68204426),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401651', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820984Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559716', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401651/suppl/GSM1401651_SL5492_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401651/suppl/GSM1401651_SL5492_gene.fpkm_tracking.gz', 9606, 'HCC-38', 'SRA', 17116959, 48306721),
(1, 'cell line: MCF12AXx-xXcell type: breast cancer cancer cell lineXx-xXstable expression: empty vector', '12801 E. 17th Ave.', 'Aurora', 'USA', 'Medicine', 'Jihye.Kim@UCDenver.edu', 'UCDenver', 'Tan', 'Jihye,,Kim', '303-724-8229', '80016', 'Alignment: TopHat v1.4.1 using default settings. hg19 (UCSC).Xx-xXExpression levels: Cufflinks v1.3.0 and cuffdiff using default settings.Xx-xXGSEA anlalysis with GeneSigDB v4.Xx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Tab-delimited text file with FPKM values.', 0, 'mRNA', 'Total RNA was extracted and mRNA isolated using the miRNeasy Kit (Qiagen) according to manufacturer\'s instructions.Xx-xXRNA libraries were prepared for sequencing using TruSEQ RNA Sample Prep v2 using standard Illumina protocols.', 'GSM1317638', 'Illumina HiSeq 2000', 'Feb 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02598922Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX451746', 'GSE54505', 'breast cancer cancer cell line, control', 'Public on Feb 26 2015', 'Jan 29 2014', 'NONE', '', 9606, 'EV3', 'SRA', 51460793, 59133317),
(1, 'cell line: UACC893Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'UACC893_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897353', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293366Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122340', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897353/suppl/GSM1897353_UACC893_FPKM.txt.gz', '', 9606, 'UACC893', 'SRA', 33629382, 37112429),
(1, 'cell line: HDQP1Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HDQP1_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897313', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293326Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122132', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897313/suppl/GSM1897313_HDQP1_FPKM.txt.gz', '', 9606, 'HDQP1', 'SRA', 29547038, 38277666),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 3h, DMSO', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431871', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905568Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648505', 'GSE59251', 'MCF-7 breast cancer cell line_3hControl-rep2', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '3hControl-rep2', 'SRA', 19544199, 21619285);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: Breast cancer cell line T47D-MTVLXx-xXtreatment: No TreatmentXx-xXtime (minutes): 0', 'Aiguader 88', 'Barcelona', 'Spain', '', 'miguel.beato@crg.eu', 'CRG', '', 'Miguel,,Beato', '', '8003', 'Basecalls performed using Illumina pipeline version 1.4.0Xx-xXChIP-Seq Single-end reads were processed by aligning to the reference human genome (GRCh37/hg19) using Bowtie v1.0.0. ChIP-seq signals were summed in windows of 100 Kb. Summed reads were normalized for sequencing depth and divided by the corresponding signal obtained for an input DNA of T47D cells to determine the normalized signal over input enrichment/depletion. Homer (Hypergeometric Optimization of Motif EnRichment) v4.3 was used to find regions of significance enrichment using an input as a control.Xx-xXHiC Paired-end reads were processed by first aligning to the reference human genome (GRCh37/hg19) using BWA. Reads filtered from analysis include those that were not uniquely mapped, mapped more than 500 bp from relevant restriction sites, had bad sequence quality (e.g. 40% or more of the bases with Sanger PHRED quality ≤ 2) or bad BWA mapping quality (≤ 20), or were located in regions classified to have exceptionally high sequence depth (top 0.1%) by the 1000 Genomes project’s data. To offset bias introduced by PCR amplification of the sequencing library, only one of the duplicated pairs were used for subsequent analyses. Datasets were used to generate contact matrices at 100 Kb.Xx-xXRNA-Seq Paired-end reads were mapped using GEM mRNA Mapping Pipeline (v1.7) using last Gencode Annotation version 18, which followed specific steps: 1) Map reads to genome reference index (including chromosme 1 to X) with maximum mismatch ratio of 0.06 and base quality threshold of 26. 2) Map reads to a transcriptome index generated from previous reference index using same options as step 1. 3) Create de-novo trasncriptome index and map again the reads to this index using maximum mismatch ratio of 0.04, minimmum split size of 15 bp and maximmum of 500000 bp. 4) Map reads again to de-novo transcriptome index created using same options as step 1. 5) Merge mappings created and pair alignments using with base quality threshold of 26 and maximmum insertion size of 500000 bp. As a result BAM alignment files were obtained and used to generate strand-specific genome-wide normalized profiles using RSeQC3 software. Exon quantifications summarized by per-genes for expression level determination were obtained either as raw read counts and reads per kilobase per milion mapped reads (RPKM) using Flux Capacitor.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: ChIP-seq data files are peak results “TSV” files with genomic coordinates of peaks. HiC data files are whole genome “TSV” matrices. RNA-Seq data file is a “TSV” table which summarizes read counts per genes.', 0, 'Sample 9Xx-xXmRNA-seqXx-xXprocessed data file: rna-seq.readCounts.txt', 'Total RNA was extracted using Trizol.Xx-xXmRNA-Seq sample prep kit (ref. RS-930-1001, Illumina)', 'GSM1294042', 'Illumina HiSeq 2000', 'Oct 06 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02464029Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX395604', 'GSE53463', 'Breast cancer cell line', 'Public on Oct 06 2014', 'Dec 18 2013', 'NONE', '', 9606, 'RNA-Seq T0', 'SRA', 35174941, 136008311),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401709', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821147Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559774', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401709/suppl/GSM1401709_SL10854_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401709/suppl/GSM1401709_SL10854_gene.fpkm_tracking.gz', 9606, '09-03-A118a', 'SRA', 40024747, 57122404),
(1, 'cell line: BT-20Xx-xXdisease state: breast cancer', '4500 San Pablo Road South', 'Jacksonville', 'USA', '', 'asmann.yan@mayo.edu', 'Mayo Clinic', '', 'Yan,W.,Asmann', '', '32224', 'Illumina\'s Genome Analyzer system software programs for base-callingXx-xXthe 50bp paired end reads were aligned using BWA 0.6.2Xx-xXthe input files were the fastq files of the two ends, and the out put files were the SnowShoes-FTD called fusion transcriptsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The .txt file reports fusion transcripts identified using SnowShoes-FTD version 2.0', 0, 'Sample 3', 'RNA was extracted by Qiagen Mini RNeasy Kit (Qiagen)Xx-xXTruSeq RNA sample preparation kit (Illumina)', 'GSM1273494', 'Illumina HiSeq 2000', 'Nov 22 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02419105Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX381534', 'GSE52643', 'breast cancer cell line', 'Public on Nov 22 2013', 'Nov 21 2013', 'NONE', '', 9606, 'BT-20', 'SRA', 29623422, 33004454),
(1, 'subtype: luminalXx-xXcell line: ZR75-1Xx-xXagent: PKC412', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556184', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246805Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790671', 'GSE63721', 'Breast cancer cell line (luminal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556184/suppl/GSM1556184_ZR751_PKC412_R1.csv.gz', '', 9606, 'ZR751_PKC412_R1', 'SRA', 33327092, 37456508),
(1, 'cell line: HBL51Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HBL100_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897292', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293305Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122173', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897292/suppl/GSM1897292_HBL100_FPKM.txt.gz', '', 9606, 'HBL100', 'SRA', 28967016, 35955415),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401756', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821080Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559821', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401756/suppl/GSM1401756_SL8370_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401756/suppl/GSM1401756_SL8370_gene.fpkm_tracking.gz', 9606, '02-10-A335a', 'SRA', 30646862, 48199607),
(1, 'cell line: UACC812Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'UACC812_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897352', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293365Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122339', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897352/suppl/GSM1897352_UACC812_FPKM.txt.gz', '', 9606, 'UACC812', 'SRA', 34695701, 45503495),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401727', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821110Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559792', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401727/suppl/GSM1401727_SL7731_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401727/suppl/GSM1401727_SL7731_gene.fpkm_tracking.gz', 9606, '09-03-A053A', 'SRA', 43085514, 90135225),
(1, 'cell line: SUM102Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'SUM102_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897339', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293352Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122345', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897339/suppl/GSM1897339_SUM102_FPKM.txt.gz', '', 9606, 'SUM102', 'SRA', 38131446, 47544391),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401677', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821114Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559742', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401677/suppl/GSM1401677_SL10635_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401677/suppl/GSM1401677_SL10635_gene.fpkm_tracking.gz', 9606, '01-01-A061a', 'SRA', 43248449, 65595965),
(1, 'cell line: MCF12AXx-xXcell type: breast cancer cancer cell lineXx-xXstable expression: miR-424 over-expressed', '12801 E. 17th Ave.', 'Aurora', 'USA', 'Medicine', 'Jihye.Kim@UCDenver.edu', 'UCDenver', 'Tan', 'Jihye,,Kim', '303-724-8229', '80016', 'Alignment: TopHat v1.4.1 using default settings. hg19 (UCSC).Xx-xXExpression levels: Cufflinks v1.3.0 and cuffdiff using default settings.Xx-xXGSEA anlalysis with GeneSigDB v4.Xx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Tab-delimited text file with FPKM values.', 0, 'mRNA', 'Total RNA was extracted and mRNA isolated using the miRNeasy Kit (Qiagen) according to manufacturer\'s instructions.Xx-xXRNA libraries were prepared for sequencing using TruSEQ RNA Sample Prep v2 using standard Illumina protocols.', 'GSM1317640', 'Illumina HiSeq 2000', 'Feb 26 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02598924Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX451748', 'GSE54505', 'breast cancer cancer cell line, over-expressing miR-424', 'Public on Feb 26 2015', 'Jan 29 2014', 'NONE', '', 9606, 'miR424_2', 'SRA', 40007186, 45539477),
(1, 'cell line: HCC1143Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1143_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897294', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293307Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122169', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897294/suppl/GSM1897294_HCC1143_FPKM.txt.gz', '', 9606, 'HCC1143', 'SRA', 18073105, 19894781),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401687', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821135Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559752', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401687/suppl/GSM1401687_SL10655_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401687/suppl/GSM1401687_SL10655_gene.fpkm_tracking.gz', 9606, '02-05-A203b', 'SRA', 46889930, 58942926),
(1, 'cell line: MCF7+BORIS (Clone1, single-cell clone of MCF7 cell, stably expressing BORIS)', '5640 Fishers Lane', 'Rockville', 'USA', 'NIAID', 'epugacheva@niaid.nih.gov', 'NIH', 'LIG', 'Elena,,Pugacheva', '', '20852', 'Sequences generated by the Illumina genome analyzer (36bp and 50bp reads) were aligned against either human (build hg19) or mouse (build mm9) genome using Bowtie program (http://bowtie-bio.sourceforge.net).  The alignment was performed with default parameters except the sequence tags that mapped to more than one location in the genome were excluded from the analysis using –m1 option.Xx-xXPeaks were called using Model-based Analysis for ChIP-Seq (MACS) (http://liulab.dfci.harvard.edu/MACS) using default parameters. After MACS, we applied Peak Splitter algorithm (part of MACS) to call sub-peaks, summit of peaks and improve peak resolution.Xx-xXFor the Illumina RNA-seq library, fastq.gz files were mapped to UCSC Human reference (build hg19) using TopHat2 with the default parameter setting of 20 alignments per read and up to 2 mismatches per alignment.Xx-xXFor Ion Torrent RNA-seq sequencing, fastq.gz files were mapped to UCSC Human reference (build hg19) using two-step alignments. First, the reads were aligned with TopHat2. Second, the unmapped reads from the first step are then extracted and aligned with Bowtie2 with –local mode and --very-sensitive-local option.Xx-xXFor both Illumina and Ion Torrent RNA-seq, the resulting aligned reads (BAM files) were then analyzed with Cufflinks 2.0.0 to estimate the transcripts relative abundance using the UCSC reference annotated transcripts (build hg19). The expression of each transcript was quantified as the number of reads mapping to a transcript divided by the transcript length in kilobases and the total number of mapped reads in millions (FPKM).Xx-xXGenome_build: hg19, mm9Xx-xXSupplementary_files_format_and_content: Cuffdiff output.', 0, '', 'After overnight incubation with DiaMag magnetic beads (Diagenode, Inc.) and CTCF or BORIS monoclonal or polyclonal antibodies (characterized and described by PMID:15731119; PMID: 21659515; PMID:20231363), precipitated chromatin was then washed, decrosslinked, and digested with proteinase K. The resulting DNA was purified using phenol/chloroform/isoamyl alcohol. DNA concentration was assessed with Quant‐it PicoGreen dsDNA kit (Invitrogen) and 5–10 ng was used to generate sequencing libraries.Xx-xXTruSeq ChIP Sample Preparation Kit (Illumina, Inc., USA). RNA-seq libraries were prepared using either Ion Total RNA-seq v2 kit or TruSeq Stranded RNA LT Kit (Illumina).', 'GSM1817676', 'Illumina HiSeq 2000', 'Jul 24 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03854462Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1091835', 'GSE70764', 'Human breast cancer cell line', 'Public on Jul 23 2015', 'Jul 10 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1817nnn/GSM1817676/suppl/GSM1817676_RNA_Seq_MCF7+EV_VS_MCF7+BORIS_Clone1.txt.gz', '', 9606, 'MCF7+BORIS_Clone1_RNA_Seq', 'SRA', 8177576, 13845891),
(1, 'cell line: HCC1428Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1428_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897298', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293311Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122177', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897298/suppl/GSM1897298_HCC1428_FPKM.txt.gz', '', 9606, 'HCC1428', 'SRA', 58138715, 64230330),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401712', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821103Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559777', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401712/suppl/GSM1401712_SL10858_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401712/suppl/GSM1401712_SL10858_gene.fpkm_tracking.gz', 9606, '09-09-A106a', 'SRA', 41076521, 55759480),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: DMSO for 1 hour', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with DMSO for 1 hour', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503433', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067401Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699958', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_dmso_1hr', 'SRA', 35721574, 216597503),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 0 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT10) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300626', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566888Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404076', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 0 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT10-dox-0', 'SRA', 39880604, 45056155),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47-I281fsXx-xXdoxycycline concentration: 5 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (MUT3) of the MDA231 breast cancer cell line transduced with doxycycline-inducible RBM47-I281fs mutant.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300634', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566890Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404084', 'GSE53779', 'breast cancer cell line clone, RBM47-I281fs mutant, 5 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'MUT3-dox-5', 'SRA', 47797867, 53736922);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'subtype: basalXx-xXcell line: MDAMB468Xx-xXagent: PKC412', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556179', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246797Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790666', 'GSE63721', 'Breast cancer cell line (basal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556179/suppl/GSM1556179_MDAMB468_PKC412_R1.csv.gz', '', 9606, 'MDAMB468_PKC412_R1', 'SRA', 25225050, 27828715),
(1, 'tissue: Breast cancer', 'Shanghai Medical College, Fudan University 1207 Rm., 2# Bldg., 270 Dong An Rd.', 'Shanghai', 'China', 'Fudan University Shanghai Cancer Center', 'cybao.cpu@163.com', 'Fudan University', '', 'Chunyang,,Bao', '86-21-34777580', '200032', 'Illumina Casava1.7 software used for basecalling.Xx-xXSequenced reads were trimmed for adaptor sequence then mapped to hg19 whole genome using TopHat v2.1.0 with default parameters.Xx-xXBack-spliced reads were calculated using a protocol from Memczak et al., Nature, 2013. Briefly, the unmapped reads were processed to 20-nucleotide anchors from both ends of the read. Anchors that aligned in the reversed orientation (head-to-tail) indicated a back-spliced junction. Anchor alignments were extended such that the complete read aligns and the breakpoints were flanked by GT/AG splice sites. To estimate the relative expression of a circular RNA, denoted as spliced reads per billion mapped reads (SRPBM), we counted the total number of reads mapped to the human reference genome in each sample and normalized the number of reads spanning the back-spliced junction to the total number of mapped reads (units in billion) and read length. To  estimate the relative expression of mRNA, fragments per kilobase of transcript per million fragments mapped (FPKM) were calculated using Cufflinks with default parameters.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include SRPBM values for each Sample and tab-delimited text files include FPKM values for each Sample', 0, '', 'Tissues were removed, flash frozen on liquid nitrogen, and RNA was harvested using Trizol reagent. NEBNext® Ultra™ Directional RNA Library Prep Kit for Illumina (Cat#E7420L) was used with 3 ug of total RNA for the construction of sequencing libraries.Xx-xXStrand-specific RNA-seq libraries were prepared using the NEBNext® Ultra™ Directional RNA Library Prep Kit for Illumina (Cat#E7420L) following the manufacturer’s instructions.', 'GSM2055799', 'Illumina HiSeq 2000', 'Mar 04 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04479418Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1564806', 'GSE77661', 'Breast cancer', 'Public on Mar 04 2016', 'Feb 08 2016', 'NONE', '', 9606, 'BRCA', 'SRA', 17794506, 35233593),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 3h, ZNA, 30 uM', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431873', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905570Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648507', 'GSE59251', 'MCF-7 breast cancer cell line_3hZNA30uM-rep1', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '3hZNA30uM-rep1', 'SRA', 29453382, 32285441),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401746', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821066Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559811', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401746/suppl/GSM1401746_SL8211_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401746/suppl/GSM1401746_SL8211_gene.fpkm_tracking.gz', 9606, '01-08-A054a', 'SRA', 52186049, 66153963),
(1, 'cell line: MCF7Xx-xXtreatment: E2Xx-xXtime: 10 minXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533416', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145660Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745847', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533416/suppl/GSM1533416_MCF7_mRNA_t0010.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533416/suppl/GSM1533416_MCF7_premRNA_t0010min.wig.txt.gz', 9606, 'MCF7_E2_10min_RNASeq', 'SRA', 42883094, 83847382),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401664', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820995Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559729', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401664/suppl/GSM1401664_SL6578_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401664/suppl/GSM1401664_SL6578_gene.fpkm_tracking.gz', 9606, 'MDA-MB-361', 'SRA', 63458951, 150701119),
(1, 'cell line: MCF10AXx-xXcell type: Human normal breast epithelial cellsXx-xXpassages: 5', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized count values for each Sample.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1874952', 'Illumina HiSeq 2000', 'Oct 06 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04046654Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1224639', 'GSE72932Xx-xXGSE72933', 'MCF10A normal breast epithelial cells breast cancer cells', 'Public on Oct 06 2016', 'Sep 11 2015', 'NONE', '', 9606, 'Control siRNA rep1', 'SRA', 37606871, 41784091),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401731', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821106Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559796', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401731/suppl/GSM1401731_SL7914_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401731/suppl/GSM1401731_SL7914_gene.fpkm_tracking.gz', 9606, '09-12-A158B', 'SRA', 40479477, 81620552),
(1, 'cell line: MCF-7Xx-xXcell type: weakly invasiveXx-xXtransfected with: si-RESTXx-xXgenotype/variation: Downregulation of REST', '1501 San Pablo St.', 'Los Angeles', 'USA', 'Biomedical Engineering', 'nanlee@usc.edu', 'University of Southern California', '', 'Nan,Sook,Lee', '6268243058', '90033', 'Bcl conversion: Illumina bcl2Fastq ver 1.8.4 to convert from bcl format to fastq.gz raw reads fileXx-xXAlignment and read counting done with GTFAR ver 0.1 using gencode19 annotation.Xx-xXA second alignment was done with Tophat ver 1.4.0 using hg19_ucsc reference and parameter -G hg19_ucsc.gtf (optional reference annotation)Xx-xXTranscript abundance estimation on the second alignment done with Cufflinks ver 1.0.1 using parameters -u (for multi-read-correct), -b hg19_ucsc.fa (for bias detection and correction) and -G hg19_ucsc.gtf (for optional reference annotation)Xx-xXGenome_build: hg19_ucscXx-xXSupplementary_files_format_and_content: tab-delimited text files from GT-FAR runs and Cufflinks runs which include RPKM values for each sample and isoform counts for each sample; more details in the \'README_1.txt\'', 0, '', 'Total RNAs were isolated from cell lines using Trizol (Invitrogen) and digested with Turbo DNase (Ambion) to remove genomic DNA, according to the manufacturer’s instructions.Xx-xXLibraries from rRNA-depleted samples were prepared using a TruSeq RNA Sample Preparation kit v2 (Illumina) following the recommended protocol starting from the RNA fragmentation stage. Purification of polyadenylated RNA was omitted. Libraries were pooled (4 samples per pool), clustered on cBOT (Illumina), and sequenced on HiSeq2000, each pool in one lane. We performed single-end 100 bp reads. Reads were mapped using TopHat followed by data analysis using Cufflinks and Cuffdiff software, as well as our own pipeline.', 'GSM1553456', 'Illumina HiSeq 2000', 'Dec 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03220240Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX767447', 'GSE63608Xx-xXGSE63610', 'Breast cancer cell line', 'Public on Dec 01 2014', 'Nov 25 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553456/suppl/GSM1553456_GTFAR-MCF-7-si-1_geneRPKMS.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553456/suppl/GSM1553456_GTFAR-MCF-7-si-1_rawRestIsoformCnts.txt.gz', 9606, 'si-REST_1', 'SRA', 27635350, 98452786),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 3h, DMSO', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431872', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905569Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648506', 'GSE59251', 'MCF-7 breast cancer cell line_3hControl-rep3', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '3hControl-rep3', 'SRA', 22231801, 24308901),
(1, 'cell line: OCUBMXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'OCUBM_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897335', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293348Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122154', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897335/suppl/GSM1897335_OCUBM_FPKM.txt.gz', '', 9606, 'OCUBM', 'SRA', 44705689, 55238821),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401679', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821124Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559744', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401679/suppl/GSM1401679_SL10639_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401679/suppl/GSM1401679_SL10639_gene.fpkm_tracking.gz', 9606, '01-02-A325a', 'SRA', 28307606, 51654470),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348552', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689302Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490463', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'JQ1 E2 rep2', 'SRA', 33582963, 39583834),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401668', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821020Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559733', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401668/suppl/GSM1401668_SL6584_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401668/suppl/GSM1401668_SL6584_gene.fpkm_tracking.gz', 9606, 'HCC-1937', 'SRA', 20617569, 106629533),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47-I281fsXx-xXdoxycycline concentration: 25 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (MUT3) of the MDA231 breast cancer cell line transduced with doxycycline-inducible RBM47-I281fs mutant.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300635', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566891Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404085', 'GSE53779', 'breast cancer cell line clone, RBM47-I281fs mutant, 25 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'MUT3-dox-25', 'SRA', 42329913, 47674662),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401750', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821073Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559815', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401750/suppl/GSM1401750_SL8220_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401750/suppl/GSM1401750_SL8220_gene.fpkm_tracking.gz', 9606, '04-08-A020B', 'SRA', 32977379, 52226357),
(1, 'cell line: MCF7Xx-xXdisease state: breast cancer', '4500 San Pablo Road South', 'Jacksonville', 'USA', '', 'asmann.yan@mayo.edu', 'Mayo Clinic', '', 'Yan,W.,Asmann', '', '32224', 'Illumina\'s Genome Analyzer system software programs for base-callingXx-xXthe 50bp paired end reads were aligned using BWA 0.6.2Xx-xXthe input files were the fastq files of the two ends, and the out put files were the SnowShoes-FTD called fusion transcriptsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The .txt file reports fusion transcripts identified using SnowShoes-FTD version 2.0', 0, 'Sample 4', 'RNA was extracted by Qiagen Mini RNeasy Kit (Qiagen)Xx-xXTruSeq RNA sample preparation kit (Illumina)', 'GSM1273495', 'Illumina HiSeq 2000', 'Nov 22 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02419107Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX381535', 'GSE52643', 'breast cancer cell line', 'Public on Nov 22 2013', 'Nov 21 2013', 'NONE', '', 9606, 'MCF7', 'SRA', 25989390, 29777246);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MCF7', '9 Divinity Ave', 'Cambridge', 'USA', 'Stem Cell and Regenerative Biology', '', 'Harvard University', '', 'John,,Rinn', '', '2138', 'RNA-seq: For  RNA-Seq analysis, first, the adapter sequences were removed from the RNA-seq reads.  Ribosomal RNA reads were then filtered out using Bowtie (Song et al. 2014).  After these filtration and adapter removal steps, the reads were aligned to a transciptome and quantified using RSEM v1.2.7 (Li and Dewey 2011). Differential gene expression was calculated by using the Deseq2 package in R 3.1.0 by using the mean value of gene-wise dispersion estimates (Love et al. 2014).Xx-xXChIP-seq: ~1x107 parental MCF-7 cells were crosslinked with formaldehyde at room temperature for 10 minutes.  Then, the cells were lysed using lysis buffer A (50mM HEPES, 140mM NaCl, 1mM EDTA pH=8, 10% Glycerol, 0.5% NP-40, 0.25% Triton X-100), and the residual cytoplasmic protein was removed using lysis buffer B (10mM Tris-HCl pH=8, 200mM NaCl, 1mM EDTA, 1mM EGTA).  The nuclear fraction was released using lysis buffer C (10mM Tris-Hcl pH=8, 100mM NaCl, 1mM EDTA, 1mM EGTA, 0.1% Sodium Deoxycholate, 0.5% N-lauroylsarcosine).  The chromatin was then sheared by using a Covaris S2 instrument with 10% duty cycle, 5 intensity, 200 cycles per burst, frequency sweeping mode, 60 second process time and for 4 cycles. The pull-down was performed by using the RUNX1 antibody (Cell Signaling #4334).  Samples were washed three times with RIPA buffer (Tris-HCl pH=8, 150mM NaCl, 1mM EDTA, 1% NP-40, 0.25% Sodium deoxycholate, 0.1% SDS) and were eluted.  The pull-down and input control sequencing libraries were generated by using the NEXTflex Rapid DNA Sequencing Kit (Bioo Scientific #5144-02) and were sequenced by using SE100 reads with a HiSeq 2000 instrument.Xx-xXdata processing step: The Hi-C data were mapped with Bowtie and  binned at 6.5Mb, 1Mb, 250kb, 100kb and 40kb non-overlapping genomic intervals.  Iterative mapping and correction of Hi-C data were performed as previously described (Imakaev et al. 2012).  Biological replicates showed high reproducibility (Pearson\'s correlation coefficient > 0.9 for 1Mb resolution data).  Similarly, the first eigenvector comparison of the replicates showed high reproducibility.  For the downstream analyses, sequences obtained from both biological replicates were pooled and ICE-corrected to serve as a combined dataset.Xx-xXCompartment analysis: First, the z-scores of the interaction matrices at 250kb resolution were generated as described previously (Lieberman, 2009).  Then, Pearson Correlation on the Z-score matrices was calculated.  By performing principal component analysis (Lieberman-Aiden et al. 2009; Zhang et al. 2012), the first eigenvector typically represents the compartment profile(Lieberman-Aiden et al. 2009), where the positive and negative 1st eigenvalues represent different compartments.   Gene density for each compartment was calculated to call the “A” and “B” compartmentalization.Xx-xXTAD Calling : TAD calling was performed as calculating the “insulation” score of each bin using the 40kb resolution combined Hi-C data (as previously described.  The mean of the interactions across each bin were calculated.  By sliding a 1Mb x 1Mb (25bins x 25bins) square along the diagonal of the interaction matrix for every chromosome, we obtained the “insulation score” of the interaction matrix.  Valleys in the insulation score indicated the depletion of Hi-C interactions occurring across a bin.  These 40kb valleys represent the TAD boundaries.  Based on the variation of boundaries between replicates, we chose to add a total of 160kb (80kb to each side) to the boundary to account for replicate variation.  The final boundaries span a 200kb region. All boundaries with a boundary strength < 0.15 were excluded as they were considered weak and non-reproducible.  The insulation plots for the biological replicates showed high reproducibility, suggesting the robustness of the method.  Similarly, the overlap of detected boundaries also showed high reproducibility between the biological replicates.  Therefore, we used the combined Hi-C replicates for the TAD analyses.Xx-xXZ-Score Calculation: We modeled the overall Hi-C decay with distance using a modified LOWESS method (alphaXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: HiC: Interaction matrices at 2.5Mb, 1Mb, 250kb and 40kb resolution; RNA-seq: Expected count file for all genes and a txt file showing the shRUNX1/shNS log2 Fold Change and pvalues for all genes; ChIP-seq: BedGeapgh UCSC visualization files and the MCF7 RUNX1 peak bed fileXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_processed_HiC_files - all normalized/corrected Hi-C data matrices binned at 40kb/250kb/1Mb/2.5MbXx-xXSupplementary_files_format_and_content: MCF7_RUNX1_ChIPseq_peaks.bed - BED file of RUNX1 ChIP-seq peaksXx-xXSupplementary_files_format_and_content: MCF7_shNS_shRUNX1_genes_expression_expected_count.tsv - TSV file showing the RNA-seq counts for each geneXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_RNAseq_log2_foldchange.txt - TSV file showing the log2 fold change of all  genes between shRUNX1 and shNS samples', 0, '', 'Hi-C was sequenced using PE100 reads; RNA-seq and ChIP-seq were sequenced using SE100 reads by using a HiSeq 2000 instrument', 'GSM1942108', 'Illumina HiSeq 2000', 'Aug 23 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04272129Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1434849', 'GSE75070', 'Human breast cancer cell line', 'Public on Aug 23 2016', 'Nov 16 2015', 'NONE', '', 9606, 'MCF-7 shRUNX1  RNA-seq Replicate2', 'SRA', 11449282, 35788724),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401714', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821100Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559779', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401714/suppl/GSM1401714_SL10862_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401714/suppl/GSM1401714_SL10862_gene.fpkm_tracking.gz', 9606, '09-11-A112a', 'SRA', 31268219, 62620988),
(1, 'subtype: basalXx-xXcell line: MDAMB468Xx-xXagent: DMSO', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556178', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246796Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790665', 'GSE63721', 'Breast cancer cell line (basal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556178/suppl/GSM1556178_MDAMB468_DMSO_R2.csv.gz', '', 9606, 'MDAMB468_DMSO_R2', 'SRA', 37988658, 42218422),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401747', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821072Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559812', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401747/suppl/GSM1401747_SL8212_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401747/suppl/GSM1401747_SL8212_gene.fpkm_tracking.gz', 9606, '02-06-A263c', 'SRA', 56066479, 71243158),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401725', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821098Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559790', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401725/suppl/GSM1401725_SL7728_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401725/suppl/GSM1401725_SL7728_gene.fpkm_tracking.gz', 9606, '05-12-A078b', 'SRA', 61056204, 74434889),
(1, 'cell line: BT-474Xx-xXdisease state: breast cancer', '4500 San Pablo Road South', 'Jacksonville', 'USA', '', 'asmann.yan@mayo.edu', 'Mayo Clinic', '', 'Yan,W.,Asmann', '', '32224', 'Illumina\'s Genome Analyzer system software programs for base-callingXx-xXthe 50bp paired end reads were aligned using BWA 0.6.2Xx-xXthe input files were the fastq files of the two ends, and the out put files were the SnowShoes-FTD called fusion transcriptsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The .txt file reports fusion transcripts identified using SnowShoes-FTD version 2.0', 0, 'Sample 1', 'RNA was extracted by Qiagen Mini RNeasy Kit (Qiagen)Xx-xXTruSeq RNA sample preparation kit (Illumina)', 'GSM1273492', 'Illumina HiSeq 2000', 'Nov 22 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02419108Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX381532', 'GSE52643', 'breast cancer cell line', 'Public on Nov 22 2013', 'Nov 21 2013', 'NONE', '', 9606, 'BT-474', 'SRA', 25902626, 33108579),
(1, 'cell line: MDA-MB-231Xx-xXcell type: highly invasiveXx-xXtransfected with: mt-RESTXx-xXgenotype/variation: overexpression of mutant REST', '1501 San Pablo St.', 'Los Angeles', 'USA', 'Biomedical Engineering', 'nanlee@usc.edu', 'University of Southern California', '', 'Nan,Sook,Lee', '6268243058', '90033', 'Bcl conversion: Illumina bcl2Fastq ver 1.8.4 to convert from bcl format to fastq.gz raw reads fileXx-xXAlignment and read counting done with GTFAR ver 0.1 using gencode19 annotation.Xx-xXA second alignment was done with Tophat ver 1.4.0 using hg19_ucsc reference and parameter -G hg19_ucsc.gtf (optional reference annotation)Xx-xXTranscript abundance estimation on the second alignment done with Cufflinks ver 1.0.1 using parameters -u (for multi-read-correct), -b hg19_ucsc.fa (for bias detection and correction) and -G hg19_ucsc.gtf (for optional reference annotation)Xx-xXGenome_build: hg19_ucscXx-xXSupplementary_files_format_and_content: tab-delimited text files from GT-FAR runs and Cufflinks runs which include RPKM values for each sample and isoform counts for each sample; more details in the \'README_1.txt\'', 0, '', 'Total RNAs were isolated from cell lines using Trizol (Invitrogen) and digested with Turbo DNase (Ambion) to remove genomic DNA, according to the manufacturer’s instructions.Xx-xXLibraries from rRNA-depleted samples were prepared using a TruSeq RNA Sample Preparation kit v2 (Illumina) following the recommended protocol starting from the RNA fragmentation stage. Purification of polyadenylated RNA was omitted. Libraries were pooled (4 samples per pool), clustered on cBOT (Illumina), and sequenced on HiSeq2000, each pool in one lane. We performed single-end 100 bp reads. Reads were mapped using TopHat followed by data analysis using Cufflinks and Cuffdiff software, as well as our own pipeline.', 'GSM1553459', 'Illumina HiSeq 2000', 'Dec 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03220245Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX767450', 'GSE63608Xx-xXGSE63610', 'Breast cancer cell line', 'Public on Dec 01 2014', 'Nov 25 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553459/suppl/GSM1553459_GTFAR-MB-231-mt-139_geneRPKMS.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1553nnn/GSM1553459/suppl/GSM1553459_GTFAR-MB-231-mt-139_rawRestIsoformCnts.txt.gz', 9606, 'mt-REST', 'SRA', 41176603, 91656885),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: 25 nM Silvestrol for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of mRNA from cells treated with 25 nM Silvestrol for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503439', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067407Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699964', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'mRNA_silvestrol_2hr', 'SRA', 29905169, 91843210),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401666', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821017Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559731', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401666/suppl/GSM1401666_SL6581_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401666/suppl/GSM1401666_SL6581_gene.fpkm_tracking.gz', 9606, 'ZR-75-30', 'SRA', 20993952, 70204993),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401661', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820993Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559726', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401661/suppl/GSM1401661_SL6575_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401661/suppl/GSM1401661_SL6575_gene.fpkm_tracking.gz', 9606, 'HCC-1143', 'SRA', 100541920, 161802224),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401692', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821138Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559757', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401692/suppl/GSM1401692_SL10665_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401692/suppl/GSM1401692_SL10665_gene.fpkm_tracking.gz', 9606, '03-03-A117b', 'SRA', 46138024, 65425784),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401689', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821131Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559754', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401689/suppl/GSM1401689_SL10659_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401689/suppl/GSM1401689_SL10659_gene.fpkm_tracking.gz', 9606, '03-01-A341c', 'SRA', 45162624, 71552653),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348542', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689296Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490453', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'Control siRNA Veh rep2', 'SRA', 27390173, 32544619),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 25 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT6) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300625', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566886Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404075', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 25 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT6-dox-25', 'SRA', 41598891, 49095317),
(1, 'cell line: CAL51Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'CAL51_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897285', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293298Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122164', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897285/suppl/GSM1897285_CAL51_FPKM.txt.gz', '', 9606, 'CAL51', 'SRA', 80183166, 86598261);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401752', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821077Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559817', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401752/suppl/GSM1401752_SL8232_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401752/suppl/GSM1401752_SL8232_gene.fpkm_tracking.gz', 9606, '04-08-A101c', 'SRA', 39304742, 71060803),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: untreated', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of mRNA from cells treated with DMSO for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503438', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067406Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699963', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'mRNA_dmso_2hr', 'SRA', 69146249, 210008365),
(1, 'cell line: MCF7', '9 Divinity Ave', 'Cambridge', 'USA', 'Stem Cell and Regenerative Biology', '', 'Harvard University', '', 'John,,Rinn', '', '2138', 'RNA-seq: For  RNA-Seq analysis, first, the adapter sequences were removed from the RNA-seq reads.  Ribosomal RNA reads were then filtered out using Bowtie (Song et al. 2014).  After these filtration and adapter removal steps, the reads were aligned to a transciptome and quantified using RSEM v1.2.7 (Li and Dewey 2011). Differential gene expression was calculated by using the Deseq2 package in R 3.1.0 by using the mean value of gene-wise dispersion estimates (Love et al. 2014).Xx-xXChIP-seq: ~1x107 parental MCF-7 cells were crosslinked with formaldehyde at room temperature for 10 minutes.  Then, the cells were lysed using lysis buffer A (50mM HEPES, 140mM NaCl, 1mM EDTA pH=8, 10% Glycerol, 0.5% NP-40, 0.25% Triton X-100), and the residual cytoplasmic protein was removed using lysis buffer B (10mM Tris-HCl pH=8, 200mM NaCl, 1mM EDTA, 1mM EGTA).  The nuclear fraction was released using lysis buffer C (10mM Tris-Hcl pH=8, 100mM NaCl, 1mM EDTA, 1mM EGTA, 0.1% Sodium Deoxycholate, 0.5% N-lauroylsarcosine).  The chromatin was then sheared by using a Covaris S2 instrument with 10% duty cycle, 5 intensity, 200 cycles per burst, frequency sweeping mode, 60 second process time and for 4 cycles. The pull-down was performed by using the RUNX1 antibody (Cell Signaling #4334).  Samples were washed three times with RIPA buffer (Tris-HCl pH=8, 150mM NaCl, 1mM EDTA, 1% NP-40, 0.25% Sodium deoxycholate, 0.1% SDS) and were eluted.  The pull-down and input control sequencing libraries were generated by using the NEXTflex Rapid DNA Sequencing Kit (Bioo Scientific #5144-02) and were sequenced by using SE100 reads with a HiSeq 2000 instrument.Xx-xXdata processing step: The Hi-C data were mapped with Bowtie and  binned at 6.5Mb, 1Mb, 250kb, 100kb and 40kb non-overlapping genomic intervals.  Iterative mapping and correction of Hi-C data were performed as previously described (Imakaev et al. 2012).  Biological replicates showed high reproducibility (Pearson\'s correlation coefficient > 0.9 for 1Mb resolution data).  Similarly, the first eigenvector comparison of the replicates showed high reproducibility.  For the downstream analyses, sequences obtained from both biological replicates were pooled and ICE-corrected to serve as a combined dataset.Xx-xXCompartment analysis: First, the z-scores of the interaction matrices at 250kb resolution were generated as described previously (Lieberman, 2009).  Then, Pearson Correlation on the Z-score matrices was calculated.  By performing principal component analysis (Lieberman-Aiden et al. 2009; Zhang et al. 2012), the first eigenvector typically represents the compartment profile(Lieberman-Aiden et al. 2009), where the positive and negative 1st eigenvalues represent different compartments.   Gene density for each compartment was calculated to call the “A” and “B” compartmentalization.Xx-xXTAD Calling : TAD calling was performed as calculating the “insulation” score of each bin using the 40kb resolution combined Hi-C data (as previously described.  The mean of the interactions across each bin were calculated.  By sliding a 1Mb x 1Mb (25bins x 25bins) square along the diagonal of the interaction matrix for every chromosome, we obtained the “insulation score” of the interaction matrix.  Valleys in the insulation score indicated the depletion of Hi-C interactions occurring across a bin.  These 40kb valleys represent the TAD boundaries.  Based on the variation of boundaries between replicates, we chose to add a total of 160kb (80kb to each side) to the boundary to account for replicate variation.  The final boundaries span a 200kb region. All boundaries with a boundary strength < 0.15 were excluded as they were considered weak and non-reproducible.  The insulation plots for the biological replicates showed high reproducibility, suggesting the robustness of the method.  Similarly, the overlap of detected boundaries also showed high reproducibility between the biological replicates.  Therefore, we used the combined Hi-C replicates for the TAD analyses.Xx-xXZ-Score Calculation: We modeled the overall Hi-C decay with distance using a modified LOWESS method (alphaXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: HiC: Interaction matrices at 2.5Mb, 1Mb, 250kb and 40kb resolution; RNA-seq: Expected count file for all genes and a txt file showing the shRUNX1/shNS log2 Fold Change and pvalues for all genes; ChIP-seq: BedGeapgh UCSC visualization files and the MCF7 RUNX1 peak bed fileXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_processed_HiC_files - all normalized/corrected Hi-C data matrices binned at 40kb/250kb/1Mb/2.5MbXx-xXSupplementary_files_format_and_content: MCF7_RUNX1_ChIPseq_peaks.bed - BED file of RUNX1 ChIP-seq peaksXx-xXSupplementary_files_format_and_content: MCF7_shNS_shRUNX1_genes_expression_expected_count.tsv - TSV file showing the RNA-seq counts for each geneXx-xXSupplementary_files_format_and_content: MCF7_shRUNX1_shNS_RNAseq_log2_foldchange.txt - TSV file showing the log2 fold change of all  genes between shRUNX1 and shNS samples', 0, '', 'Hi-C was sequenced using PE100 reads; RNA-seq and ChIP-seq were sequenced using SE100 reads by using a HiSeq 2000 instrument', 'GSM1942106', 'Illumina HiSeq 2000', 'Aug 23 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04272127Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1434847', 'GSE75070', 'Human breast cancer cell line', 'Public on Aug 23 2016', 'Nov 16 2015', 'NONE', '', 9606, 'MCF-7 shNS  RNA-seq Replicate3', 'SRA', 6693377, 20595300),
(1, 'cell line: BT549Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'BT549_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897282', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293295Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122161', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897282/suppl/GSM1897282_BT549_FPKM.txt.gz', '', 9606, 'BT549', 'SRA', 28513743, 31077469),
(1, 'cell line: EVSATXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'EVSAT_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897291', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293304Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122172', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897291/suppl/GSM1897291_EVSAT_FPKM.txt.gz', '', 9606, 'EVSAT', 'SRA', 32976453, 36971841),
(1, 'cell line: MACLS2Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MACLS2_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897318', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293331Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122138', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897318/suppl/GSM1897318_MACLS2_FPKM.txt.gz', '', 9606, 'MACLS2', 'SRA', 26098089, 30904839),
(1, 'cell line: JIMT1Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'JIMT1_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897315', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293328Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122134', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897315/suppl/GSM1897315_JIMT1_FPKM.txt.gz', '', 9606, 'JIMT1', 'SRA', 88361914, 98743983),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: 25 nM Silvestrol for 1 hour', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of ribosome footprints from cells treated with 25 nM Silvestrol for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503436', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067404Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699961', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'FP_silvestrol_2hr', 'SRA', 17706684, 164084745),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401719', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821096Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559784', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401719/suppl/GSM1401719_SL13095_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401719/suppl/GSM1401719_SL13095_gene.fpkm_tracking.gz', 9606, '05-04-A096a', 'SRA', 35423393, 66428030),
(1, 'cell line: HCC202Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC202_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897305', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293318Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122184', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897305/suppl/GSM1897305_HCC202_FPKM.txt.gz', '', 9606, 'HCC202', 'SRA', 42892009, 49994438),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112769', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631347Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688765', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_R5020_12 Hours_RNAseq', 'SRA', 30925991, 37439978),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112723', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631358Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688718', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P2_Vehicle_24 Hours_RNAseq', 'SRA', 18711788, 24016509),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siFOXA1_annotated100kb.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112783', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631286Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688779', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siFOXA1_Vehicle_12 Hours_RNAseq', 'SRA', 17555540, 21349568),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112754', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631332Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688750', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N1_E2+R5020_24 Hours_RNAseq', 'SRA', 18594118, 24180237),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112734', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631369Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688729', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P4_E2+R5020_48 Hours_RNAseq', 'SRA', 9892278, 18833916),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P7.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112743', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631378Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688738', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P7_Vehicle_24 Hours_RNAseq', 'SRA', 17269150, 23490087),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: PR-deficient T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112790', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631293Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688786', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D(PR-deficient)_E2+R5020_12 Hours_RNAseq', 'SRA', 38989500, 44598252),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112764', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631342Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688760', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N4_E2_24 Hours_RNAseq', 'SRA', 24059208, 32600677),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P5.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112735', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631370Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688730', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P5_Vehicle_48 Hours_RNAseq', 'SRA', 24703968, 40676888),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112770', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631348Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688766', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_E2+R5020_12 Hours_RNAseq', 'SRA', 30470097, 36757363);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P8.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112750', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631328Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688746', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P8_E2+R5020_24 Hours_RNAseq', 'SRA', 25465632, 32298514),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: MCF7Xx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: MCF7.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112794', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631297Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688790', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_MCF7_E2+R5020_12 Hours_RNAseq', 'SRA', 10590029, 12846845),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112761', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631339Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688757', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N3_R5020_48 Hours_RNAseq', 'SRA', 68904931, 105086980),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112751', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631329Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688747', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N1_Vehicle_24 Hours_RNAseq', 'SRA', 16323550, 21185736),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P5.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112736', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631371Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688731', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P5_E2_48 Hours_RNAseq', 'SRA', 18682827, 24833370),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112720', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631355Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688715', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P1_E2_24 Hours_RNAseq', 'SRA', 15213336, 21248049),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112725', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631360Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688720', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P2_R5020_24 Hours_RNAseq', 'SRA', 17561498, 22591664),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siControl.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112776', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631279Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688772', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siControl_E2_12 Hours_RNAseq', 'SRA', 14029053, 16944138),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112733', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631368Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688728', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P4_R5020_48 Hours_RNAseq', 'SRA', 10984197, 25871888),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112762', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631340Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688758', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N3_E2+R5020_48 Hours_RNAseq', 'SRA', 11137553, 17116689),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siFOXA1_annotated100kb.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112786', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631289Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688782', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siFOXA1_E2+R5020_12 Hours_RNAseq', 'SRA', 18695940, 22413998),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: ZR75Xx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: ZR75.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112774', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631352Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688770', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_ZR75_E2+R5020_12 Hours_RNAseq', 'SRA', 16748511, 20041392),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P7.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112746', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631324Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688742', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P7_E2+R5020_24 Hours_RNAseq', 'SRA', 15256931, 22267056),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: ZR75Xx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: ZR75.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112772', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631350Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688768', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_ZR75_E2_12 Hours_RNAseq', 'SRA', 17267589, 20691667),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siControl.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112778', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631281Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688774', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siControl_E2+R5020_12 Hours_RNAseq', 'SRA', 16031379, 19256280),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P7.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112744', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631379Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688739', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P7_E2_24 Hours_RNAseq', 'SRA', 20886565, 28752363),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: PR-deficient T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112788', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631291Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688784', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D(PR-deficient)_E2_12 Hours_RNAseq', 'SRA', 29706768, 34497721),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: PR-deficient T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112789', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631292Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688785', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D(PR-deficient)_R5020_12 Hours_RNAseq', 'SRA', 25902726, 29852896),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112753', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631331Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688749', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N1_R5020_24 Hours_RNAseq', 'SRA', 20358239, 26828402),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112724', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631359Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688719', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P2_E2_24 Hours_RNAseq', 'SRA', 17793571, 22221316),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112730', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631365Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688725', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P3_E2+R5020_24 Hours_RNAseq', 'SRA', 17095416, 21997975),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112765', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631343Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688761', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N4_R5020_24 Hours_RNAseq', 'SRA', 17711772, 23829810);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112727', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631362Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688722', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P3_Vehicle_24 Hours_RNAseq', 'SRA', 35550244, 44499558),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P6.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112742', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631377Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688737', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P6_E2+R5020_48 Hours_RNAseq', 'SRA', 23036797, 32913737),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: PR-deficient T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112787', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631290Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688783', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D(PR-deficient)_Vehicle_12 Hours_RNAseq', 'SRA', 46980936, 54937277),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P6.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112741', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631376Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688736', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P6_R5020_48 Hours_RNAseq', 'SRA', 30093193, 42092396),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112726', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631361Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688721', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P2_E2+R5020_24 Hours_RNAseq', 'SRA', 18563038, 23418799),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: MCF7Xx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: MCF7.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112793', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631296Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688789', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_MCF7_R5020_12 Hours_RNAseq', 'SRA', 6531472, 8166039),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P7.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112745', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631380Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688741', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P7_R5020_24 Hours_RNAseq', 'SRA', 16042905, 21799835),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112721', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631356Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688716', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P1_R5020_24 Hours_RNAseq', 'SRA', 17145737, 24910622),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112722', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631357Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688717', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P1_E2+R5020_24 Hours_RNAseq', 'SRA', 17189349, 24790130),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siFOXA1_annotated100kb.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112785', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631288Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688781', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siFOXA1_R5020_12 Hours_RNAseq', 'SRA', 18680688, 22564132),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P8.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112748', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631326Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688744', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P8_E2_24 Hours_RNAseq', 'SRA', 25529556, 32262715),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P8.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112749', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631327Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688745', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P8_R5020_24 Hours_RNAseq', 'SRA', 22439188, 29156580),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siControl.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112775', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631353Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688771', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siControl_Vehicle_12 Hours_RNAseq', 'SRA', 15914114, 21295433),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112768', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631346Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688764', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_E2_12 Hours_RNAseq', 'SRA', 29163787, 35695666),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112729', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631364Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688724', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P3_R5020_24 Hours_RNAseq', 'SRA', 22847760, 29452835),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112752', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631330Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688748', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N1_E2_24 Hours_RNAseq', 'SRA', 20560841, 25960407),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112758', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631336Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688754', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N2_E2+R5020_48 Hours_RNAseq', 'SRA', 9172682, 14382768),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siNF1C.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112782', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631285Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688778', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siNF1C_E2+R5020_12 Hours_RNAseq', 'SRA', 16752121, 20130325),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: wild typeXx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864032', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018861Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174296', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'wt_replicate_1', 'SRA', 45301021, 51680784),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112760', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631338Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688756', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N3_E2_48 Hours_RNAseq', 'SRA', 9691272, 16508198),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112763', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631341Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688759', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N4_Vehicle_24 Hours_RNAseq', 'SRA', 19997850, 27486757),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P6.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112739', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631374Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688734', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P6_Vehicle_48 Hours_RNAseq', 'SRA', 32031384, 45686079),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siNF1C.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112780', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631283Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688776', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siNF1C_E2_12 Hours_RNAseq', 'SRA', 13335221, 16428445);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112757', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631335Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688753', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N2_R5020_48 Hours_RNAseq', 'SRA', 13689183, 29518146),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112759', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631337Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688755', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N3_Vehicle_48 Hours_RNAseq', 'SRA', 8213714, 14453098),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siNF1C.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112781', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631284Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688777', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siNF1C_R5020_12 Hours_RNAseq', 'SRA', 7674354, 9815507),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P1.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112719', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631354Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688714', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P1_Vehicle_24 Hours_RNAseq', 'SRA', 16456758, 23306292),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P5.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112738', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631373Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688733', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P5_E2+R5020_48 Hours_RNAseq', 'SRA', 16724914, 30031277),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112732', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631367Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688727', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P4_E2_48 Hours_RNAseq', 'SRA', 3214745, 17745600),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: ezh2-/-Xx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864035', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018864Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174299', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'ezh2-/-_replicate_2', 'SRA', 40500520, 45847044),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: ZR75Xx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: ZR75.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112771', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631349Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688767', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_ZR75_Vehicle_12 Hours_RNAseq', 'SRA', 17013643, 20509327),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: ZR75Xx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: ZR75.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112773', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631351Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688769', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_ZR75_R5020_12 Hours_RNAseq', 'SRA', 16270575, 19751926),
(1, 'cell line: MDA-MB-231Xx-xXcell type: breast cancerXx-xXgender: female', '1500 Duarte Rd.', 'Duarte', 'USA', '', 'ckuo@coh.org', 'City of Hope/Beckman Research Institute', '', 'Ching-Ying,,Kuo', '', '91010', 'Illumina Casava1.8 software used for basecalling.Xx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using bowtieXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using software PartekXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using Trizol protocol.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1908527', 'Illumina HiSeq 2500', 'May 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166630Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336211', 'GSE74031', 'breast cancer cells', 'Public on May 01 2016', 'Oct 14 2015', 'NONE', '', 9606, '231_rep2', 'SRA', 40015714, 44834540),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112766', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631344Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688762', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N4_E2+R5020_24 Hours_RNAseq', 'SRA', 17513712, 24309246),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P6.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112740', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631375Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688735', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P6_E2_48 Hours_RNAseq', 'SRA', 34570939, 45116015),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: ezh2-/-; hotair overexpressedXx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864039', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018868Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174303', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'ezh2-/-_replicate_2 + Hotair', 'SRA', 39074802, 44303566),
(1, 'cell line: MDA-MB-231Xx-xXcell type: breast cancerXx-xXgender: female', '1500 Duarte Rd.', 'Duarte', 'USA', '', 'ckuo@coh.org', 'City of Hope/Beckman Research Institute', '', 'Ching-Ying,,Kuo', '', '91010', 'Illumina Casava1.8 software used for basecalling.Xx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using bowtieXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using software PartekXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using Trizol protocol.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1908526', 'Illumina HiSeq 2500', 'May 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166629Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336210', 'GSE74031', 'breast cancer cells', 'Public on May 01 2016', 'Oct 14 2015', 'NONE', '', 9606, '231_rep1', 'SRA', 38903420, 43599918),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: T47D.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112767', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631345Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688763', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_Vehicle_12 Hours_RNAseq', 'SRA', 38974117, 48860689),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P3.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112728', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631363Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688723', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P3_E2_24 Hours_RNAseq', 'SRA', 22951119, 29264536),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112756', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631334Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688752', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N2_E2_48 Hours_RNAseq', 'SRA', 20533878, 31396304),
(1, 'cell line: MDA-MB-231Xx-xXcell type: breast cancerXx-xXgender: female', '1500 Duarte Rd.', 'Duarte', 'USA', '', 'ckuo@coh.org', 'City of Hope/Beckman Research Institute', '', 'Ching-Ying,,Kuo', '', '91010', 'Illumina Casava1.8 software used for basecalling.Xx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using bowtieXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using software PartekXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using Trizol protocol.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1908528', 'Illumina HiSeq 2500', 'May 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166631Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336212', 'GSE74031', 'breast cancer cells', 'Public on May 01 2016', 'Oct 14 2015', 'NONE', '', 9606, '231/DKG_rep1', 'SRA', 33273062, 39103949),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: wild type; hotair overexpressedXx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864036', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018865Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174300', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'wt_replicate_1 + Hotair', 'SRA', 38997178, 45109613),
(1, 'tissue type: ER+/PR- Human tumor explantsXx-xXer/pr status: ER+/PR-Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor N2.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112755', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631333Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688751', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_N2_Vehicle_48Hours_RNAseq', 'SRA', 7668615, 22059024),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: MCF7Xx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: MCF7.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112792', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631295Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688788', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_MCF7_E2_12 Hours_RNAseq', 'SRA', 11829408, 14110646),
(1, 'cell line: SK-BR-3Xx-xXcell type: Breast Cancer cell line', 'Uppsalalaan 8', 'Utrecht', 'Netherlands', '', 'l.kester@hubrecht.eu', 'Hubrecht Institute', '', 'Lennart,,Kester', '', '3584CT', 'For bulk gDNA and MALBAC libraries, paired end sequencing reads were aligned to the genome release mm10 for mouse cells (E14) and to the genome release hg19 for human cells (SK-BR-3) using BWA with default parameters. For the single cell gDNA libraries processed by DR-Seq, paired end sequencing reads were aligned to a masked genome mm10 for mouse cells and to a masked genome hg19 for human cells using BWA with default parameters. The masked genomes mm10 and hg19 were created by replacing all the coding sequences within the genome with \"N\". This is because the fraction used to sequence gDNA contains sequences that could originate from the cDNA within coding regions. By masking the coding sequences within the genome, such ambiguous reads that might arise from either gDNA or cDNA are discarded computationally leaving only reads that arise from gDNA. This does not pose a problem for calling copy number variations since the coding region constitutes only approximately 2% of the genome.Xx-xXAll PCR duplicates within mapped reads from the bulk, MALBAC or DR-Seq libraries are removed. As the first step towards quantifying the gDNA data, the genome is divided into bins. To account for the masking of the genome in the DR-Seq data, the start and end coordinates of each bin are chosen such that the length of all bins are the same after excluding coding regions within each bin. Next, to further reduce amplification biases, we developed a coverage-based method to quantify the reads within bins. This coverage-based method significantly reduces bin-to-bin technical noise (see supplementary document). The reads are then corrected for GC bias. The corrected read distribution is then used to identify breakpoints using the circular binary segmentation (CBS) algorithm26. Finally, the median read counts for each segment are used to call copy number variations in single cells.Xx-xXFor bulk mRNA and CEL-Seq libraries, paired end sequencing reads were aligned to the transcriptome using Burrows-Wheeler Aligner (BWA) with default parameters. For single cell mRNA processed using DR-Seq, the Ad-2 adapter sequence was trimmed computationally from the right mate and then aligned to the transcriptome using BWA with default parameters. For the E14 cells, we used the RefSeq gene models based on the mouse genome release mm10. For the SK-BR-3 cells, we used the RefSeq gene models based on the human genome release hg19. For bulk mRNA sequencing both mates of each read were mapped to the transcriptome. For CEL-Seq and DR-Seq, the right mate of each read pair was mapped to the transcriptome and the ERCC spike-ins. The left mate was used to identify the cell from which the transcript came based on the cell-specific barcode. Reads mapping to more than one region were distributed uniformly.Xx-xXFor the bulk mRNA sequencing libraries, PCR duplicates were then removed to obtain the dataset used in all the analysis. The left mate of the CEL-Seq libraries also contained a 4-bp random sequence, introduced during reverse transcription, to count unique cDNA molecules, as previously described. Length-based identifiers were determined for each read in the single-cell mRNA libraries processed by DR-Seq using the first coordinate of the right mate after trimming off adapter Ad-2. The length-based identifiers were used to minimize amplification biases and achieve resolution close to identifying unique cDNA molecules.Xx-xXGenome_build: mm10 and hg19Xx-xXSupplementary_files_format_and_content: details included in the \'processed_data_files_description.txt\' file.', 0, 'RPKM dataXx-xXSKBRbkmRNA', 'Illumina stranded mRNA kitXx-xXTrypsinized single cells were picked using a mouth pipet with a 30μm glass capillary under a stereomicroscope. Picked cells were deposited in the center of the lid of a 0.2ml PCR tube and snap frozen in liquid nitrogen.', 'GSM1537129', 'Illumina HiSeq 2500', 'Dec 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03160700Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX750532', 'GSE62952', 'Breast Cancer cell line', 'Public on Dec 11 2014', 'Nov 04 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537129/suppl/GSM1537129_SKBRbkmRNA.txt.gz', '', 9606, 'Bulk mRNA sequencing data from SKBR3 cells', 'SRA', 21962416, 24127256),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siFOXA1_annotated100kb.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112784', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631287Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688780', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siFOXA1_E2_12 Hours_RNAseq', 'SRA', 14407502, 16960645),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siNF1C.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112779', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631282Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688775', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siNF1C_Vehicle_12 Hours_RNAseq', 'SRA', 16472831, 20693762);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MDA-MB-231Xx-xXcell type: breast cancerXx-xXgender: female', '1500 Duarte Rd.', 'Duarte', 'USA', '', 'ckuo@coh.org', 'City of Hope/Beckman Research Institute', '', 'Ching-Ying,,Kuo', '', '91010', 'Illumina Casava1.8 software used for basecalling.Xx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using bowtieXx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using software PartekXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using Trizol protocol.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1908529', 'Illumina HiSeq 2500', 'May 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166632Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336213', 'GSE74031', 'breast cancer cells', 'Public on May 01 2016', 'Oct 14 2015', 'NONE', '', 9606, '231/DKG_rep2', 'SRA', 38285065, 44954729),
(1, 'tissue type: ER+/PR-deficient cell modelsXx-xXcell line: MCF7Xx-xXer/pr status: ER+/PR-deficientXx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: MCF7.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112791', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631294Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688787', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_MCF7_Vehicle_12 Hours_RNAseq', 'SRA', 16912813, 20649926),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 24 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P8.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112747', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631325Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688743', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P8_Vehicle_24 Hours_RNAseq', 'SRA', 20836852, 27196973),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P5.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112737', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631372Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688732', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P5_R5020_48 Hours_RNAseq', 'SRA', 21688830, 31592911),
(1, 'tissue type: ER+/PR+ Human tumor explantsXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 48 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Processed data file: Tumor P4.csv', 'Hormone-treated explant was homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112731', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631366Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688726', 'GSE80098Xx-xXGSE80365', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Tumor_P4_Vehicle_48 Hours_RNAseq', 'SRA', 11385066, 39457034),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: ezh2-/-Xx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864034', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018863Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174298', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'ezh2-/-_replicate_1', 'SRA', 43906103, 50516106),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: wild type; hotair overexpressedXx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864037', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018866Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174301', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'wt_replicate_2 + Hotair', 'SRA', 32679226, 37309309),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: wild typeXx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864033', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018862Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174297', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'wt_replicate_2', 'SRA', 51078872, 57757953),
(1, 'tissue type: ER+/PR+ cell modelsXx-xXcell line: T47DXx-xXer/pr status: ER+/PR+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'mRNA-seq (knockdown)Xx-xXProcessed data file: T47D_siControl.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2112777', 'Illumina HiSeq 2500', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04631280Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1688773', 'GSE80098Xx-xXGSE80366', 'Breast cancer cells', 'Public on Jun 01 2016', 'Apr 10 2016', 'NONE', '', 9606, 'Cells_T47D_siControl_R5020_12 Hours_RNAseq', 'SRA', 13307170, 15885401),
(1, 'gender: femaleXx-xXtissue: breastXx-xXgenotype: ezh2-/-; hotair overexpressedXx-xXcell line: MDA-MB-231', '26 rue d\'ulm', 'Paris Cedex 05', 'France', '', 'Nicolas.Servant@curie.fr', 'Institut Curie', '', 'Nicolas,,Servant', '', '75248', 'Sequenced reads were trimmed for adaptor sequence, And  mapped to hg19 whole genome using  tophat_2.0.6, Method: Global alignment - No mismatch in seed alignment (of size 22), 3 mismatches in read length, Keep uniques best hits - Library type: Fr-firststrXx-xXreads count estimation with HTSeq-0.6.1Xx-xXdifferential expression with R (R-3.2.0), Limma 3.24.13  packages, EdgeR 3.10.2 packagesXx-xXGenome_build: hg19', 0, '', 'RNA was extracted from cells resuspended in Trizol reagent. 1.5 ug of total RNA was used to prepare RNA library sequeced by Illumina HiSeq 2500.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1864038', 'Illumina HiSeq 2500', 'Aug 12 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04018867Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1174302', 'GSE72524', 'MDA-MB-231 human breast cancer cell lines', 'Public on Aug 12 2016', 'Aug 31 2015', 'NONE', '', 9606, 'ezh2-/-_replicate_1 + Hotair', 'SRA', 39837535, 45988874),
(1, 'cell line: SK-BR-3Xx-xXcell type: Breast Cancer cell line', 'Uppsalalaan 8', 'Utrecht', 'Netherlands', '', 'l.kester@hubrecht.eu', 'Hubrecht Institute', '', 'Lennart,,Kester', '', '3584CT', 'For bulk gDNA and MALBAC libraries, paired end sequencing reads were aligned to the genome release mm10 for mouse cells (E14) and to the genome release hg19 for human cells (SK-BR-3) using BWA with default parameters. For the single cell gDNA libraries processed by DR-Seq, paired end sequencing reads were aligned to a masked genome mm10 for mouse cells and to a masked genome hg19 for human cells using BWA with default parameters. The masked genomes mm10 and hg19 were created by replacing all the coding sequences within the genome with \"N\". This is because the fraction used to sequence gDNA contains sequences that could originate from the cDNA within coding regions. By masking the coding sequences within the genome, such ambiguous reads that might arise from either gDNA or cDNA are discarded computationally leaving only reads that arise from gDNA. This does not pose a problem for calling copy number variations since the coding region constitutes only approximately 2% of the genome.Xx-xXAll PCR duplicates within mapped reads from the bulk, MALBAC or DR-Seq libraries are removed. As the first step towards quantifying the gDNA data, the genome is divided into bins. To account for the masking of the genome in the DR-Seq data, the start and end coordinates of each bin are chosen such that the length of all bins are the same after excluding coding regions within each bin. Next, to further reduce amplification biases, we developed a coverage-based method to quantify the reads within bins. This coverage-based method significantly reduces bin-to-bin technical noise (see supplementary document). The reads are then corrected for GC bias. The corrected read distribution is then used to identify breakpoints using the circular binary segmentation (CBS) algorithm26. Finally, the median read counts for each segment are used to call copy number variations in single cells.Xx-xXFor bulk mRNA and CEL-Seq libraries, paired end sequencing reads were aligned to the transcriptome using Burrows-Wheeler Aligner (BWA) with default parameters. For single cell mRNA processed using DR-Seq, the Ad-2 adapter sequence was trimmed computationally from the right mate and then aligned to the transcriptome using BWA with default parameters. For the E14 cells, we used the RefSeq gene models based on the mouse genome release mm10. For the SK-BR-3 cells, we used the RefSeq gene models based on the human genome release hg19. For bulk mRNA sequencing both mates of each read were mapped to the transcriptome. For CEL-Seq and DR-Seq, the right mate of each read pair was mapped to the transcriptome and the ERCC spike-ins. The left mate was used to identify the cell from which the transcript came based on the cell-specific barcode. Reads mapping to more than one region were distributed uniformly.Xx-xXFor the bulk mRNA sequencing libraries, PCR duplicates were then removed to obtain the dataset used in all the analysis. The left mate of the CEL-Seq libraries also contained a 4-bp random sequence, introduced during reverse transcription, to count unique cDNA molecules, as previously described. Length-based identifiers were determined for each read in the single-cell mRNA libraries processed by DR-Seq using the first coordinate of the right mate after trimming off adapter Ad-2. The length-based identifiers were used to minimize amplification biases and achieve resolution close to identifying unique cDNA molecules.Xx-xXGenome_build: mm10 and hg19Xx-xXSupplementary_files_format_and_content: details included in the \'processed_data_files_description.txt\' file.', 0, 'Read counts for all the single cellsXx-xXSKBRscmRNA', 'DR-Seq. First strand cDNA synthesis was performed by adding 2 μL of reaction mix containing 0.2 μL first strand buffer (MessageAmp II, Life technologies), 0.4 μL of dNTP mix (MessageAmp II, Life technologies), 0.1 μL Arrayscript (MessageAmp II, Life technologies), 0.1 μL RNAse inhibitor (MessageAmp II, Life technologies), 0.2 μL RT primer with cell specific barcode (Ad-1x)13, 0.2 μL 1:500000 diluted ERCC spike-in mix 1 (Life technologies) and 0.05% IGEPAL in water. The first strand cDNA synthesis and lysis reaction mix together with the spike-in molecules were added directly to the drop in the lid of the tube containing a single cell. Samples were incubated in a PCR machine with lid and block set to 42C for 15 minutes after which the samples were spun down and incubated for another 105 minutes. After first strand synthesis, samples were incubated for 10 minutes at 80oC. Quasilinear amplification buffer containing 6.0 μL ThermoPol buffer (NEB) 1.0 μL 10 mM dNTP mix, 26 μL water and 0.15 μL 50 μM primer mix (Ad-2)7 was added to each sample. Samples were incubated for 3 minutes at 94 oC to denature the DNA. Seven cycles of quasilinear amplification was performed (10oC for 45 seconds,15oC for 45 seconds, 20oC for 45 seconds, 30oC for 45 seconds, 40oC for 45 seconds, 50oC for 45 seconds, 65oC for 2 minutes, 95oC for 20 seconds, 58oC for 40 seconds and then immediately quench on ice). Prior to each cycle 0.6 μL polymerase mix containing 2U Bst large fragment (NEB) and 0.8U Pyrophage 3173 exo- (Lucigen) was added. Note that the 58oC step for 40 seconds step, prior to quenching the reaction on ice, is not performed for the first quasilinear amplification round. After 7 sounds of quasilinear amplification, samples were split in two. One half of the sample was processed for gDNA sequencing, the other half was processed for mRNA sequencing. For mRNA sequencing, second strand synthesis of the quasilinear amplified cDNA was performed using the P1 primer (5\' - CGATTGAGGCCGGTAATAC - 3\') in a single cycle of PCR (94oC for 20 sec, 51oC for 20 sec, 72oC for 7 min). After this, samples with non-overlapping barcodes were pooled and cleaned up on a cDNA purification column (MessageAmp II, Life technologies), and eluted twice with 9 μL of  water at 55oC. Next, the volume of the sample was reduced to 6.4 μL using a SpeedVac®. In vitro transcription (IVT) mix containing 1.6 μL 10x IVT buffer, 1.6 μL ATP, 1.6 μL GTP, 1.6 μL CTP, 1.6 μL UTP and 1.6 μL enzyme mix (MessageAmp II, Life technologies) were added to the samples and incubated at 37oC for 13 hours. After IVT, samples were cleaned up using the aRNA clean up columns (MessageAmp II, Life technologies) and aRNA was eluted twice in 12 μL of warm water at 55oC. After clean up, aRNA quality was assessed on a bioanalyzer (Agilent) Eukaryote Total RNA Pico chip. Library preparation was performed as previously described in Hashimshony et al. Cell reports 2012. For DNA sequencing, the other half of the quasilinear amplification product was amplified further by PCR. PCR mix containing 1.0 μL 10mM dNTP, 3 μL Thermopol buffer (10X), 0.2 μL 100 μM primer P2 (5\' - GTGAGTGATGGTTGAGGTAGTGTGGAG - 3\') and 1.0 μL Deep VentR (exo-) polymerase (NEB) was added to each sample for a final volume of 68 μL. PCR was performed as follows, 21 cycles of (94oC for 20 seconds, 59oC for 20 seconds, 65oC for 1 minute, 72oC for 2 minutes), and 72oC for 5 minutes at the end. After PCR, the quality of the products were assessed by agarose gel electrophoresis and the samples were cleaned up using a PCR purification column. Next, to remove adapter Ad-2 from the PCR product prior to preparing Illumina libraries, another PCR was done starting with 80 ng of product from the previous step. PCR mix containing 0.3 μL of 50 μM primer P3 with a 5\' biotinylated end (5\' - GTGAGCTGGAGTTGAGGTAGTGTGGAG - 3\'), 5 μL Thermopol buffer (10X), 1 μL 10mM dNTP and 1 μL Deep VentR (exo-) polymerase (NEB) was added to each sample for a final volume of 50 μL. PCR was performed as follows, 94oC for 2 minutes, then 4 cycles of (94oC for 20 seconds, 46oC for 20 seconds, 65oC for 1 minute and 72oC for 2 minutes) and 9 cycles of (94oC for 20 seconds, 59oC for 20 seconds, 65oC for 1 minute and 72oC for 2 minutes). The PCR product was sheared using a sonicator (Biorupter®) on the low power setting with 15  cycles of 1 minute (30 seconds On, 30 seconds Off) with constant cooling at 4oC. The sheared products was then cleaned up using a PCR purification column (Qiagen) and eluted in 50 μL water. The final product distribution was verified on a bioanalyzer (Agilent) High Sensitivity DNA chip to have an average product size of approximately 300 bp. The DNA products was then added to Dynabeads MyOne Streptavidin C1 beads (Life Technologies) in 50 μL 2X BW buffer (10 mM Tris-HCl, 1mM EDTA and 2mM NaCl). After immobilizing the DNA products on the beads for 15 minutes, the biotinylated DNA was separated using a magnetic stand and the supernatant was stored. The biotinylated DNA was digested on the magnetic beads and the beads were washed twice with 50 μL  1X BW buffer. These two washes were then combined with the first supernatant and purified using a PCR purification column (Qiagen). Finally, Illumina libraries were prepared with different index primers for each single cell using the NEBNext Ultra DNA Library Prep Kit for Illumina® (NEB).Xx-xXTrypsinized single cells were picked using a mouth pipet with a 30μm glass capillary under a stereomicroscope. Picked cells were deposited in the center of the lid of a 0.2ml PCR tube and snap frozen in liquid nitrogen.', 'GSM1537130', 'Illumina HiSeq 2500', 'Dec 11 2014', 'other', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03160693Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX750533', 'GSE62952', 'Breast Cancer cell line', 'Public on Dec 11 2014', 'Nov 04 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1537nnn/GSM1537130/suppl/GSM1537130_SKBRscmRNA.txt.gz', '', 9606, 'single cell mRNA sequencing data from single SKBR3 cells obtained with the DR-seq protocol', 'SRA', 9007352, 29792816),
(1, 'cell line: MDA-MB-231Xx-xXdisease state: breast cancer', '4500 San Pablo Road South', 'Jacksonville', 'USA', '', 'asmann.yan@mayo.edu', 'Mayo Clinic', '', 'Yan,W.,Asmann', '', '32224', 'Illumina\'s Genome Analyzer system software programs for base-callingXx-xXthe 50bp paired end reads were aligned using BWA 0.6.2Xx-xXthe input files were the fastq files of the two ends, and the out put files were the SnowShoes-FTD called fusion transcriptsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The .txt file reports fusion transcripts identified using SnowShoes-FTD version 2.0', 0, 'Sample 5', 'RNA was extracted by Qiagen Mini RNeasy Kit (Qiagen)Xx-xXTruSeq RNA sample preparation kit (Illumina)', 'GSM1273496', 'Illumina HiSeq 2000', 'Nov 22 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02419118Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX381536', 'GSE52643', 'breast cancer cell line', 'Public on Nov 22 2013', 'Nov 21 2013', 'NONE', '', 9606, 'MDA-MB-231', 'SRA', 28257690, 31393951),
(1, 'cell line: Breast cancer cell line T47D-MTVLXx-xXtreatment: ProgesteroneXx-xXtime (minutes): 60', 'Aiguader 88', 'Barcelona', 'Spain', '', 'miguel.beato@crg.eu', 'CRG', '', 'Miguel,,Beato', '', '8003', 'Basecalls performed using Illumina pipeline version 1.4.0Xx-xXChIP-Seq Single-end reads were processed by aligning to the reference human genome (GRCh37/hg19) using Bowtie v1.0.0. ChIP-seq signals were summed in windows of 100 Kb. Summed reads were normalized for sequencing depth and divided by the corresponding signal obtained for an input DNA of T47D cells to determine the normalized signal over input enrichment/depletion. Homer (Hypergeometric Optimization of Motif EnRichment) v4.3 was used to find regions of significance enrichment using an input as a control.Xx-xXHiC Paired-end reads were processed by first aligning to the reference human genome (GRCh37/hg19) using BWA. Reads filtered from analysis include those that were not uniquely mapped, mapped more than 500 bp from relevant restriction sites, had bad sequence quality (e.g. 40% or more of the bases with Sanger PHRED quality ≤ 2) or bad BWA mapping quality (≤ 20), or were located in regions classified to have exceptionally high sequence depth (top 0.1%) by the 1000 Genomes project’s data. To offset bias introduced by PCR amplification of the sequencing library, only one of the duplicated pairs were used for subsequent analyses. Datasets were used to generate contact matrices at 100 Kb.Xx-xXRNA-Seq Paired-end reads were mapped using GEM mRNA Mapping Pipeline (v1.7) using last Gencode Annotation version 18, which followed specific steps: 1) Map reads to genome reference index (including chromosme 1 to X) with maximum mismatch ratio of 0.06 and base quality threshold of 26. 2) Map reads to a transcriptome index generated from previous reference index using same options as step 1. 3) Create de-novo trasncriptome index and map again the reads to this index using maximum mismatch ratio of 0.04, minimmum split size of 15 bp and maximmum of 500000 bp. 4) Map reads again to de-novo transcriptome index created using same options as step 1. 5) Merge mappings created and pair alignments using with base quality threshold of 26 and maximmum insertion size of 500000 bp. As a result BAM alignment files were obtained and used to generate strand-specific genome-wide normalized profiles using RSeQC3 software. Exon quantifications summarized by per-genes for expression level determination were obtained either as raw read counts and reads per kilobase per milion mapped reads (RPKM) using Flux Capacitor.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: ChIP-seq data files are peak results “TSV” files with genomic coordinates of peaks. HiC data files are whole genome “TSV” matrices. RNA-Seq data file is a “TSV” table which summarizes read counts per genes.', 0, 'Sample 10Xx-xXmRNA-seqXx-xXprocessed data file: rna-seq.readCounts.txt', 'Total RNA was extracted using Trizol.Xx-xXmRNA-Seq sample prep kit (ref. RS-930-1001, Illumina)', 'GSM1294043', 'Illumina HiSeq 2000', 'Oct 06 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02464030Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX395605', 'GSE53463', 'Breast cancer cell line', 'Public on Oct 06 2014', 'Dec 18 2013', 'NONE', '', 9606, 'RNA-Seq R1', 'SRA', 44172637, 140064372),
(1, 'cell line: MDA231Xx-xXcell type: breast cancer cell lineXx-xXtransgene: RBM47Xx-xXdoxycycline concentration: 0 ng/ml', 'Box 197, Cambridge Biomedical Campus', 'Cambridge', 'United Kingdom', 'MRC Cancer Unit', 'sv358@mrc-cu.cam.ac.uk', 'University of Cambridge', 'Vanharanta', 'Sakari,,Vanharanta', '', 'CB2 0XZ', 'Read alignment: Raw FASTQ reads are aligned to human genome (Build hg19) with STAR2.3.0 using default settings.Xx-xXExpression levels: Reads mapped to each transcript are counted with HTSeq v0.5.4 using default settings.Xx-xXSAM and HTSeqXx-xXGenome_build: GRCh37Xx-xXSupplementary_files_format_and_content: Normalized read counts mapped to each gene/transcript.', 0, 'Single-cell derived clone (WT6) of the MDA231 breast cancer cell line transduced with doxycycline-inducible wildtype RBM47.', 'PrepEase® RNA Spin Kit (USB).Xx-xXAfter quality control with Agilent BioAnalyzer, total RNA underwent polyA selection and TruSeq library preparation according to instruction provided by Illumina. Samples were barcoded and run on a HiSeq 2000 in a 75bp/75bp paired-end run.', 'GSM1300621', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02566877Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX404071', 'GSE53779', 'breast cancer cell line clone, WT RBM47, 0 ng/ml doxycycline', 'Public on Jun 11 2014', 'Jan 02 2014', 'NONE', '', 9606, 'WT6-dox-0', 'SRA', 44762267, 50205047),
(1, 'cell type: Human breast adenocarcinomaXx-xXcell line: MCF7Xx-xXpassages: 20-30', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Comma-delimited text file with normalized read counts.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1348543', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02689297Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX490454', 'GSE55922Xx-xXGSE55923', 'MCF7 breast cancer cells', 'Public on Jul 10 2014', 'Mar 14 2014', 'NONE', '', 9606, 'Control siRNA E2 rep1', 'SRA', 36277825, 42442664),
(1, 'cell line: MDA-MB-231 breast cancer cell lineXx-xXtreatment: control siRNA-48hr', '3181 S.W. Sam Jackson Park Rd', 'Portland', 'USA', 'MMG', 'searsr@ohsu.edu', 'OHSU', '', 'Rosalie,,Sears', '5034946885', '97239-3098', 'Samples were sequenced on an Illumina HiSeq 2000 and assembled into fastq files using the CASAVA software at the OHSU Massively Parallel Sequencing Shared Resource.Xx-xXSequences were aligned to the human hg19 genome using Bowtie version 1.0.0 and seting requiring unique alignments with no more than 3 mismatches.Xx-xXTrim settings for Bowtie were selected to discard the first 4 bases, align to the next 44, and discard remaining bases from each sequence.Xx-xXAligned sequences that fell in exonic regions of RefSeq gene models were counted and normalized for sequencing depth.Xx-xXGenes that showed low normalized sequence counts for all samples were discarded from further analysis.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Tab-delimited .txt file containing normalized sequence counts for gene models before filtering out low expression genes.', 0, '', 'RNA was extracted from cells by Trozol reagent.Xx-xXTotal RNA was purified and reverse transcribed with oligo dT. Second strand DNA was synthesized and ligated to proprietary indexed adapters. Ligated material was amplified using Illumina genomic primers and sequenced on an Illumina GAII sequencer.', 'GSM1399410', 'Illumina HiSeq 2000', 'May 28 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02800679Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX554352', 'GSE58008', 'MDA-MB-231 breast cancer cell line', 'Public on May 28 2014', 'May 27 2014', 'NONE', '', 9606, 'siNT', 'SRA', 13669021, 15124612),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401736', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821061Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559801', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401736/suppl/GSM1401736_SL7920_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401736/suppl/GSM1401736_SL7920_gene.fpkm_tracking.gz', 9606, '09-06-A089A', 'SRA', 69574177, 86204960),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401744', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821064Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559809', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401744/suppl/GSM1401744_SL7932_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401744/suppl/GSM1401744_SL7932_gene.fpkm_tracking.gz', 9606, '04-08-A183c', 'SRA', 48206770, 86581994);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401748', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821074Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559813', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401748/suppl/GSM1401748_SL8213_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401748/suppl/GSM1401748_SL8213_gene.fpkm_tracking.gz', 9606, '03-12-A222a', 'SRA', 41760778, 67603352),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401681', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821126Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559746', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401681/suppl/GSM1401681_SL10643_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401681/suppl/GSM1401681_SL10643_gene.fpkm_tracking.gz', 9606, '01-03-A373a', 'SRA', 38883783, 60509961),
(1, 'tissue: ER+ Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401702', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821146Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559767', 'GSE58135', 'ER+ Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401702/suppl/GSM1401702_SL10807_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401702/suppl/GSM1401702_SL10807_gene.fpkm_tracking.gz', 9606, '07-11-A250a', 'SRA', 40687150, 64803681),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401656', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820996Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559721', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401656/suppl/GSM1401656_SL6570_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401656/suppl/GSM1401656_SL6570_gene.fpkm_tracking.gz', 9606, 'MDA-MB-436', 'SRA', 116529802, 156169253),
(1, 'tissue: Triple Negative Breast Cancer Primary Tumor', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401723', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02821115Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559788', 'GSE58135', 'Triple Negative Breast Cancer Primary Tumor', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401723/suppl/GSM1401723_SL7726_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401723/suppl/GSM1401723_SL7726_gene.fpkm_tracking.gz', 9606, '03-11-A307c', 'SRA', 62190245, 81397258),
(1, 'tissue: Breast Cancer Cell Line', '2000 Circle of Hope, Room 4363', 'Salt Lake City', 'USA', 'Department of Oncological Sciences', 'kt.varley@hci.utah.edu', 'Huntsman Cancer Institute University of Utah', 'Varley Lab', 'Katherine,E,Varley', '', '84112', 'TopHat v1.4.1 was used to align 50 million read pairs in the fastq files using with the options –r100 --mate-std-dev 75. GENCODE V9  was used as the transcriptome-indexXx-xXCufflinks 1.3.0 was used to calculate gene expression values (Fragments Per Kilobase of transcript Per Million reads, FPKMs) for each GENCODE V9 transcript using the –u option. It generated a gene.fpkm file listing the gene expression for each gene in each sample.Xx-xXChimeraScan 0.4.5a was used to align reads and identify fusion transcripts in each of the sequencing libraries using default parameters. It generated a bedpe file listing the fusions detected in each sample.Xx-xXThe ChimeraScan runconfig.xml file generated from running with default parameters included the following variables:   <num_processors>12</num_processors>    <quals>sanger</quals>    <keep_tmp>True</keep_tmp>    <bowtie_path />    <bowtie_args>--best --strata</bowtie_args>    <discord_bowtie_args>--best</discord_bowtie_args>    <multihits>100</multihits>    <mismatches>2</mismatches>    <discord_mismatches>3</discord_mismatches>    <segment_length>25</segment_length>    <trim5>0</trim5>    <trim3>0</trim3>    <min_fragment_length>0</min_fragment_length>    <max_fragment_length>1000</max_fragment_length>    <library_type>fr-unstranded</library_type>    <isize_mean>200</isize_mean>    <isize_stdev>40</isize_stdev>    <homology_mismatches>2</homology_mismatches>    <anchor_min>4</anchor_min>    <anchor_length>8</anchor_length>    <anchor_mismatches>0</anchor_mismatches>    <filter_unique_frags>2.0</filter_unique_frags>    <filter_isize_prob>0.01</filter_isize_prob>    <filter_isoform_fraction>0.01</filter_isoform_fraction>Xx-xXGenome_build: hg19', 0, '', 'The frozen breast tissue specimens were weighed, transferred to a 15 mL conical tube containing ceramic beads, and RLT Buffer  (Qiagen) plus 1% BME was added so that the tube contained 35 uL of buffer for each milligram of tissue.  The conical tubes containing tissue, ceramic beads and buffer were then shaken in a MP Biomedicals FastPrep machine until the tissue was visibly homogenized (90 seconds at 6.5 meters per second).  The homogenized tissue was stored at -80°C.  Total RNA was extracted from 5 million cultured cells or 350 uL of tissue homogenate (equivalent to 10 mg of tissue) using the Norgen Animal Tissue  RNA Purification Kit (Norgen Biotek Corporation). Cell lysate was treated with Proteinase K before it was applied to the column and on-column DNAse  treatment was performed according to the manufacturer’s instructions. Total RNA was eluted from the columns and quantified using the Qubit RNA Assay Kit and the Qubit 2.0 fluorometer (Invitrogen).Xx-xXRNA-seq libraries for each sample were constructed from 250 ng total RNA using the polyA selection and transposase-based non-stranded library construction (Tn-RNA-seq) described previously (Gertz et al. Transposase mediated construction of RNA-seq libraries.2012. Genome Res 22 (1):134-141). RNA-seq libraries were barcoded during PCR using Nextera barcoded primers according to the manufacturer (Epicentre). The RNA-seq libraries were quantified using the Qubit dsDNA HS Assay Kit and the Qubit 2.0  fluorometer (Invitrogen) and three barcoded libraries were pooled in equimolar quantities for sequencing. The pooled libraries were sequenced on an Illumina HiSeq 2000 sequencing machine using paired-end 50 bp reads and a 6 bp index  read, and we obtained at least 50 million read pairs from each library.', 'GSM1401660', 'Illumina HiSeq 2000', 'Jun 11 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02820991Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX559725', 'GSE58135', 'Breast Cancer Cell Line', 'Public on Jun 11 2014', 'Jun 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401660/suppl/GSM1401660_SL6574_chimeras.bedpe.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1401nnn/GSM1401660/suppl/GSM1401660_SL6574_gene.fpkm_tracking.gz', 9606, 'HCC-70', 'SRA', 72699600, 127603180),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancerXx-xXtreatment condition: 3h, ZNA, 30 uM', '825 Northeast 13th Street', 'Oklahoma City', 'USA', 'Immunobiology and Cancer', '', 'Oklahoma Medical Research Foundation', 'Bryan Welm Lab', 'Bryan,,Welm', '405-271-2050', '73104', 'reads were aligned to hg19 plus extended splice junctions using novoalign (2.08.01). Up to 50 alignments were reported and adapter sequences were trimmed by novoalign.Xx-xXsplice junction alignments were converted back to genomic space using the USeq (8.6.4) SamTranscriptomeParser application (default settings).Xx-xXthe final alignment files were used in the USeq (8.6.4) DefinedRegionDifferentialSeq application to generate counts per gene, variance stabilized counts, FPKM values and to determine differential expressionXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited files contain raw read counts, FPKM values, log2ratio (DESeq) and -10*log10(adjusted p-value) (DESeq)', 0, '', 'RNA was isolated from cells using a Qiagen Rneasy kit.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols.', 'GSM1431875', 'Illumina HiSeq 2000', 'Jul 10 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN02905572Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX648509', 'GSE59251', 'MCF-7 breast cancer cell line_3hZNA30uM-rep3', 'Public on Jul 10 2014', 'Jul 09 2014', 'NONE', '', 9606, '3hZNA30uM-rep3', 'SRA', 26282141, 28803208),
(1, 'cell line: MDA-MB-231Xx-xXcell type: Ductal breast carcinomaXx-xXpassage #: 3Xx-xXtreatment: DMSO for 2 hours', '5300 Chiron Way', 'Emeryville', 'USA', 'Oncology', 'claudiarubio@gmail.com', 'Novartis Institutes for Biomedical Research', '', 'Claudia,Ani,Rubio', '', '94608', 'Basecalls were performed with Casava v1.8.1 or 1.8.3Xx-xXSequence reads were trimmed for adapter and polyA sequences and mapped to the hg19 genome, as well as UCSC KnownGene canonical transcripts, using Bowtie v2 (no more than 2 mismatches, -a –m -16)Xx-xXAlignments were merged and uniquely mapped reads were counted to calculate the reads per kilobase per million (RPKM), or fragments per kilobase million (FPKM; INK128 data).  Genomic positions with non-uniquely aligned reads as well as positions that overlapped with more than one KnownGene canonical transcript were excluded from the RPKM calculation.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: Excel files containing RPKM (Silvestrol) or FPKM (INK128) data for each sample.', 0, 'RNAseq of total RNA from cells treated with DMSO for 2 hours', 'Extraction protocol was carried out as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.Xx-xXLibrary construction was performed as described by Ingolia et al. Nat Protoc 2012, 7:1534–50.', 'GSM1503444', 'Illumina HiSeq 2000', 'Oct 01 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03067412Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX699969', 'GSE61375', 'Breast cancer cells', 'Public on Sep 13 2014', 'Sep 12 2014', 'NONE', '', 9606, 'totRNA_dmso', 'SRA', 31702431, 437089422),
(1, 'cell line: MCF7Xx-xXtreatment: untreatedXx-xXsample type: Pleural effusionXx-xXpassages: 14-17', 'Geert Grooteplein 28', 'Nijmegen', 'USA', 'Department of Molecular Biology (274)', 'h.stunnenberg@ncmls.ru.nl', 'Radboud University', '', 'H.G.,,Stunnenberg', '-3610513', '6525 GA', 'ChIP-seq reads were mapped to hg19 using Genomatix Mapping Station.Xx-xXChIP-seq data were summarised by counting the fraction of reads in 200 bp bins forming the last 20% of the annotated gene region, background corrected by subtracting an average over manually selected inactive regions and normalised by the total number of reads and using the geometric median normalisation of Anders and Huber (2010).Xx-xXRNA-seq reads were mapped to Ensembl v68 transcriptome complemented by pre-mRNA sequences using bowtie.Xx-xXRNA-seq expression estimates (means and variances) were obtained using BitSeq.Xx-xXRNA-seq data were normalised using the geometric median method of Anders and Huber (2010).Xx-xXGenome_build: hg19Xx-xXtsv, summary of PolII activity at the last 20% of each gene for PolII, summary of gene expression (mean and standard deviation) for each gene for RNA-seq; wig, reads with unique PolII ChIP-seq alignments as fractions in 200 bp bins over the gene; wig, reads with unique alignments to pre-mRNA as fractions in 200 bp bins over the pre-mRNA transcript', 0, '', 'For DNA-Seq: lysates were clarified from sonicated nuclei and histone-DNA complexes were isolated with antibody. For RNA-Seq: total RNA was isolated using TRIzol according to the manufacturer\'s recommendations, followed by DNaseI treatment. 100–250 ng total RNA was subjected rRNA depletion with the Ribo-Zero Magnetic Gold Kit and processed further for strand specific RNAseq.Xx-xXDNA and RNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1533414', 'Illumina HiSeq 2000', 'Mar 04 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03145653Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX745845', 'GSE62789', 'Breast cancer cells', 'Public on Mar 04 2015', 'Oct 28 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533414/suppl/GSM1533414_MCF7_mRNA_t0000.txt.gz', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1533nnn/GSM1533414/suppl/GSM1533414_MCF7_premRNA_t0000min.wig.txt.gz', 9606, 'MCF7_untreated_RNASeq', 'SRA', 33445500, 71429918),
(1, 'subtype: basalXx-xXcell line: MDAMB468Xx-xXagent: PKC412', 'Lazarettgasse 14', 'Vienna', 'Austria', '', 'snijman@cemm.oeaw.ac.at', 'Research Center for Molecular Medicine', 'Nijman', 'Sebastian,M,Nijman', '', '1090', 'Illumina Casava 1.82 software used for base callingXx-xXElimination of low quality reads with Triagetools. Alignement with Tophat 2 with refSeq annotation.Xx-xXExpression estimates by read counting on refSeq gene modelsXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: gene-level expression measured in FPKM', 0, '', 'Cells were put on ice, medium was aspirated, washed with ice-cold PBS 2x followed by Trizol RNA extraction.Xx-xXIllumina Sequencing Protocol', 'GSM1556180', 'Illumina HiSeq 2000', 'Dec 03 2014', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03246798Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX790667', 'GSE63721', 'Breast cancer cell line (basal)', 'Public on Dec 02 2014', 'Dec 01 2014', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1556nnn/GSM1556180/suppl/GSM1556180_MDAMB468_PKC412_R2.csv.gz', '', 9606, 'MDAMB468_PKC412_R2', 'SRA', 33036710, 36670023),
(1, 'cell line: MCF10AXx-xXcell type: Human normal breast epithelial cellsXx-xXpassages: 5', 'Robert-Koch-Straße 40', 'Göttingen', 'Germany', 'Department of General, Visceral and Pediatric Surgery', 'sjohnsen@alumni.mayo.edu', 'University Medical Center Göttingen', '', 'Steven,A,Johnsen', '', '37075', 'The reads were mapped to the human reference transcriptome (UCSC HG19) using Bowtie2 2.1.0 (Langmead and Salzberg, 2012).Xx-xXRead counts for each sample and each gene were aggregated using a custom Ruby script.Xx-xXDESeq (version 1.14.0) was used for the measuring differential expression (Anders and Huber, 2010).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized count values for each Sample.', 0, '', 'Trizol mediated RNA isolation method was used for harvesting RNA.Xx-xX500 ng RNA was used for library preparation and libraries were made using the kit TruSeq RNA Sample Preparation Kit (Illumina, Cat. N°RS–122–2002)', 'GSM1874954', 'Illumina HiSeq 2000', 'Oct 06 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04046656Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1224641', 'GSE72932Xx-xXGSE72933', 'MCF10A normal breast epithelial cells breast cancer cells', 'Public on Oct 06 2016', 'Sep 11 2015', 'NONE', '', 9606, 'BRD4 siRNA rep1', 'SRA', 31709408, 36000000),
(1, 'cell line: CAL120Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'CAL120_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897283', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293296Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122162', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897283/suppl/GSM1897283_CAL120_FPKM.txt.gz', '', 9606, 'CAL120', 'SRA', 36310259, 45400476),
(1, 'cell line: MDAMB175VIIXx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'MDAMB175VII_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897325', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293338Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122144', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897325/suppl/GSM1897325_MDAMB175VII_FPKM.txt.gz', '', 9606, 'MDAMB175VII', 'SRA', 31014008, 39561566),
(1, 'cell line: 184A1Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, '184A1_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897275', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293288Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122214', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897275/suppl/GSM1897275_184A1_FPKM.txt.gz', '', 9606, '184A1', 'SRA', 24781011, 28135800),
(1, 'cell line: HCC1954Xx-xXcell type: breast cancer cell line', '550 First Avenue', 'New York', 'USA', '', 'benjamin.neel@nyumc.org', 'NYU Perlmutter Cancer Centre', 'Neel Lab', 'Benjamin,G.,Neel', '', '10016', 'Genomic alignment was performed with STAR  (v2.3.0) (Dobin et al., 2013), using  default parameters, except that –out SAMstrandField was set to intronMotif.Xx-xXGene expression levels were estimated with Cufflinks (v.2.2.1) (Trapnell et al., 2010), using default parameters and the Gencode V19 GTF file.Xx-xXGenome_build: hg19', 0, 'HCC1954_rnaseq', 'extract protocolXx-xXRNA (1 ug) from each sample was reverse transcribed into cDNA by using the Illumina TruSeq Stranded mRNA kit. Libraries were sized on an Agilent Bioanalyzer, and their concentrations were validated by qPCR. Six different libraries were normalized to 10nM and pooled, 13pM of pooled libraries were loaded onto an Illumina cBot for cluster generation, and the flow cell was subjected to 50-cycles of paired-end sequencing on an Illumina HighSeq 2000.', 'GSM1897304', 'Illumina HiSeq 2000', 'Jan 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1293317Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04122183', 'GSE73526', 'breast cancer cell line', 'Public on Jan 14 2016', 'Sep 28 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1897nnn/GSM1897304/suppl/GSM1897304_HCC1954_FPKM.txt.gz', '', 9606, 'HCC1954', 'SRA', 45656729, 49806567),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: UNC0638Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908562', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166662Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337016', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'UNC0638_3 (MCF7)', 'SRA', 32158515, 44597320),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+UNC0638Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908563', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166663Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337017', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+UNC0638_3 (MCF7)', 'SRA', 34975954, 50125930),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: S2101Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908560', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166660Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337014', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'S2101_3 (MCF7)', 'SRA', 19069798, 26015897),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+S2101Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908561', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166661Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337015', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+S2101_3 (MCF7)', 'SRA', 50299136, 69104409);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DepsiXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908566', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166666Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337020', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'Depsi_3 (MCF7)', 'SRA', 29873807, 39992568),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+DepsiXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908567', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166667Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337021', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+Depsi_3 (MCF7)', 'SRA', 39005072, 55081639),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: GSK343Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908564', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166664Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337018', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'GSK343_3 (MCF7)', 'SRA', 38438360, 53488059),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+GSK343Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908565', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166665Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337019', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+GSK343_3 (MCF7)', 'SRA', 50906385, 69854408),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: CntrlXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908548', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'Reanalyzed by: GSM1915702Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166680Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337002', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'Cntrl_2 (MCF7)', 'SRA', 17291104, 26777975),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DACXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908549', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166687Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337003', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC_2 (MCF7)', 'SRA', 12889544, 19371255),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: GSK343Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908544', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166676Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336998', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'GSK343 (MCF7)', 'SRA', 22233718, 31924553),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DepsiXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908546', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166678Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337000', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'Depsi (MCF7)', 'SRA', 19959088, 28287021),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+DepsiXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908547', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166679Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337001', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+Depsi (MCF7)', 'SRA', 24120171, 35496139),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: S2101Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908540', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166683Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336994', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'S2101 (MCF7)', 'SRA', 31540321, 42191901),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+S2101Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908541', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166684Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336995', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+S2101 (MCF7)', 'SRA', 38054394, 56189157),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: UNC0638Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908542', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166685Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336996', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'UNC0638 (MCF7)', 'SRA', 23945105, 37039109),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+UNC0638Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908553', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166670Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337007', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+UNC0638_2 (MCF7)', 'SRA', 55562418, 77751044),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: UNC0638Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908552', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166669Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337006', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'UNC0638_2 (MCF7)', 'SRA', 31921565, 48726130),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+S2101Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908551', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166668Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337005', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+S2101_2 (MCF7)', 'SRA', 36719089, 64800900),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: S2101Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908550', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166688Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337004', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'S2101_2 (MCF7)', 'SRA', 28129111, 38588424),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+DepsiXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908557', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166674Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337011', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+Depsi_2 (MCF7)', 'SRA', 35328338, 50954845),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DepsiXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908556', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166673Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337010', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'Depsi_2 (MCF7)', 'SRA', 39370592, 52858973),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+GSK343Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908555', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166672Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337009', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+GSK343_2 (MCF7)', 'SRA', 53644951, 75085570),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: GSK343Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908554', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166671Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337008', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'GSK343_2 (MCF7)', 'SRA', 26735133, 36120680),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DACXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908559', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166659Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337013', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC_3 (MCF7)', 'SRA', 53797913, 75098389),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: CntrlXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908558', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'Reanalyzed by: GSM1915703Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166675Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1337012', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'Cntrl_3 (MCF7)', 'SRA', 53051043, 75384199),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DACXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908539', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166682Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336993', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC (MCF7)', 'SRA', 29907469, 39658085),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: CntrlXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908538', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'Reanalyzed by: GSM1915701Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166681Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336992', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'Cntrl (MCF7)', 'SRA', 14019604, 18612786),
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+GSK343Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908545', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166677Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336999', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+GSK343 (MCF7)', 'SRA', 23093099, 32568485);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: mammary gland, breastXx-xXcell line: breast cancer cell line MCF7Xx-xXer status: positiveXx-xXtreatment: DAC+UNC0638Xx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first 30 columns are the RPKMs of the individual samples. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control samples', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1908543', 'Illumina HiSeq 2500', 'Dec 31 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04166686Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1336997', 'GSE74036', 'MCF7 breast cancer cell line', 'Public on Dec 31 2016', 'Oct 14 2015', 'NONE', '', 9606, 'DAC+UNC0638 (MCF7)', 'SRA', 30209642, 47455343),
(1, 'cell line: MCF-7Xx-xXtransfection: siNTXx-xXtreatment: 3h Estrogen', 'Forskningsparken - Gaustadalleen 21', 'Oslo', 'Norway', 'Norwegian Centre for Molecular Medicine (NCMM)', 'elisa.fiorito@ncmm.uio.no', 'University of Oslo (UiO)', 'Breast Cancer group', 'Elisa,,Fiorito', '', '349', 'The paired-end RNAseq data were aligned by using tophat.Xx-xXThe gene expression values (FPKM) were calculated by using cufflinks.Xx-xXThe differential expression values and plots were generated by using Cuffmerge and CummeRbund.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: txt files containing fpkm values for genes', 0, '', 'Total RNA was isolated with the total RNA isolation kit (RNeasy, Qiagen) according to the manufacturer’s protocol. RNA yield was assessed spectrophotometrically (NanoDrop 2000). Quantification and quality control for RNA-seq libraries were performed by Agilent Bioanalyzer (RNA nano 6000, Agilent).Xx-xXThe libraries were prepared at Norwegian Sequencing Center (NSC, Oslo). The instructions from TruSeq RNA-seq strand-specific sample protocol were followed.', 'GSM2257525', 'Illumina HiSeq 2000', 'Dec 10 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05504533Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1994471', 'GSE85088Xx-xXGSE85108', 'Luminal breast cancer cell line', 'Public on Dec 10 2016', 'Aug 02 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2257nnn/GSM2257525/suppl/GSM2257525_Genes_fpkm_siNT_E3h_cufflinks.txt.gz', '', 9606, 'siNT_E3h_RNAseq', 'SRA', 63294229, 73736717),
(1, 'cell line: MCF-7Xx-xXtransfection: siCTCFXx-xXtreatment: 3h Estrogen', 'Forskningsparken - Gaustadalleen 21', 'Oslo', 'Norway', 'Norwegian Centre for Molecular Medicine (NCMM)', 'elisa.fiorito@ncmm.uio.no', 'University of Oslo (UiO)', 'Breast Cancer group', 'Elisa,,Fiorito', '', '349', 'The paired-end RNAseq data were aligned by using tophat.Xx-xXThe gene expression values (FPKM) were calculated by using cufflinks.Xx-xXThe differential expression values and plots were generated by using Cuffmerge and CummeRbund.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: txt files containing fpkm values for genes', 0, '', 'Total RNA was isolated with the total RNA isolation kit (RNeasy, Qiagen) according to the manufacturer’s protocol. RNA yield was assessed spectrophotometrically (NanoDrop 2000). Quantification and quality control for RNA-seq libraries were performed by Agilent Bioanalyzer (RNA nano 6000, Agilent).Xx-xXThe libraries were prepared at Norwegian Sequencing Center (NSC, Oslo). The instructions from TruSeq RNA-seq strand-specific sample protocol were followed.', 'GSM2257526', 'Illumina HiSeq 2000', 'Dec 10 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05504532Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1994472', 'GSE85088Xx-xXGSE85108', 'Luminal breast cancer cell line', 'Public on Dec 10 2016', 'Aug 02 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2257nnn/GSM2257526/suppl/GSM2257526_Genes_fpkm_siCTCF_E3h_cufflinks.txt.gz', '', 9606, 'siCTCF_E3h_RNAseq', 'SRA', 50199845, 58291032),
(1, 'cell line: MCF-7Xx-xXtransfection: siCTCFXx-xXtreatment: 0h Estrogen', 'Forskningsparken - Gaustadalleen 21', 'Oslo', 'Norway', 'Norwegian Centre for Molecular Medicine (NCMM)', 'elisa.fiorito@ncmm.uio.no', 'University of Oslo (UiO)', 'Breast Cancer group', 'Elisa,,Fiorito', '', '349', 'The paired-end RNAseq data were aligned by using tophat.Xx-xXThe gene expression values (FPKM) were calculated by using cufflinks.Xx-xXThe differential expression values and plots were generated by using Cuffmerge and CummeRbund.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: txt files containing fpkm values for genes', 0, '', 'Total RNA was isolated with the total RNA isolation kit (RNeasy, Qiagen) according to the manufacturer’s protocol. RNA yield was assessed spectrophotometrically (NanoDrop 2000). Quantification and quality control for RNA-seq libraries were performed by Agilent Bioanalyzer (RNA nano 6000, Agilent).Xx-xXThe libraries were prepared at Norwegian Sequencing Center (NSC, Oslo). The instructions from TruSeq RNA-seq strand-specific sample protocol were followed.', 'GSM2257524', 'Illumina HiSeq 2000', 'Dec 10 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05504530Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1994470', 'GSE85088Xx-xXGSE85108', 'Luminal breast cancer cell line', 'Public on Dec 10 2016', 'Aug 02 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2257nnn/GSM2257524/suppl/GSM2257524_Genes_fpkm_siCTCF_E0h_cufflinks.txt.gz', '', 9606, 'siCTCF_E0h_RNAseq', 'SRA', 59569654, 69438997),
(1, 'cell line: MCF-7Xx-xXtransfection: siNTXx-xXtreatment: 0h Estrogen', 'Forskningsparken - Gaustadalleen 21', 'Oslo', 'Norway', 'Norwegian Centre for Molecular Medicine (NCMM)', 'elisa.fiorito@ncmm.uio.no', 'University of Oslo (UiO)', 'Breast Cancer group', 'Elisa,,Fiorito', '', '349', 'The paired-end RNAseq data were aligned by using tophat.Xx-xXThe gene expression values (FPKM) were calculated by using cufflinks.Xx-xXThe differential expression values and plots were generated by using Cuffmerge and CummeRbund.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: txt files containing fpkm values for genes', 0, '', 'Total RNA was isolated with the total RNA isolation kit (RNeasy, Qiagen) according to the manufacturer’s protocol. RNA yield was assessed spectrophotometrically (NanoDrop 2000). Quantification and quality control for RNA-seq libraries were performed by Agilent Bioanalyzer (RNA nano 6000, Agilent).Xx-xXThe libraries were prepared at Norwegian Sequencing Center (NSC, Oslo). The instructions from TruSeq RNA-seq strand-specific sample protocol were followed.', 'GSM2257523', 'Illumina HiSeq 2000', 'Dec 10 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05504531Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1994469', 'GSE85088Xx-xXGSE85108', 'Luminal breast cancer cell line', 'Public on Dec 10 2016', 'Aug 02 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2257nnn/GSM2257523/suppl/GSM2257523_Genes_fpkm_siNT_E0h_cufflinks.txt.gz', '', 9606, 'siNT_E0h_RNAseq', 'SRA', 63845037, 74540365),
(1, 'diagnosis: metastatic breast cancerXx-xXcell type: CD11b+ myelomonocytic cells', 'Albert-Gockel 1', 'Fribourg', 'Switzerland', 'Département de Médecine 3', 'curzio.ruegg@unifr.ch', 'Université de Fribourg', 'Pathology', 'Rüegg,,Curzio', '', '1700', 'Sequencing data was processed using the Illumina Pipeline Software version 1.82.Xx-xXPurity-filtered reads were adapters and quality trimmed with Cutadapt (v. 1.2.1, Martin 2011), and filtered for low complexity with Prinseq (v. 0.20.3, Schmieder and Edwards 2011).Xx-xXReads were aligned against Homo sapiens.GRCh38.76 genome using TopHat2 (v. 2.0.9, Kim et al. 2013).Xx-xXThe number of read counts per gene locus was summarized with htseq-count (v. 0.5.4p3, Anders et al. 2014) using Homo sapiens.GRCh38.76 gene annotation.Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: Tab-delimited text files include gene counts for each Sample.', 0, '', 'Qiagen RNeasy Plus Mini Kit was used following manufacturer\'s instructionsXx-xXRNA-seq libraries were prepared using the Illumina TruSeq Stranded mRNA protocol', 'GSM1726618', 'Illumina HiSeq 2500', 'Feb 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03801580Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1077250', 'GSE70404', 'metastatic breast cancer patient', 'Public on Feb 09 2017', 'Jun 30 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1726nnn/GSM1726618/suppl/GSM1726618_66.processed_counts.txt.gz', '', 9606, 'Cancer Patient 66', 'SRA', 23431861, 52932439),
(1, 'diagnosis: metastatic breast cancerXx-xXcell type: CD11b+ myelomonocytic cells', 'Albert-Gockel 1', 'Fribourg', 'Switzerland', 'Département de Médecine 3', 'curzio.ruegg@unifr.ch', 'Université de Fribourg', 'Pathology', 'Rüegg,,Curzio', '', '1700', 'Sequencing data was processed using the Illumina Pipeline Software version 1.82.Xx-xXPurity-filtered reads were adapters and quality trimmed with Cutadapt (v. 1.2.1, Martin 2011), and filtered for low complexity with Prinseq (v. 0.20.3, Schmieder and Edwards 2011).Xx-xXReads were aligned against Homo sapiens.GRCh38.76 genome using TopHat2 (v. 2.0.9, Kim et al. 2013).Xx-xXThe number of read counts per gene locus was summarized with htseq-count (v. 0.5.4p3, Anders et al. 2014) using Homo sapiens.GRCh38.76 gene annotation.Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: Tab-delimited text files include gene counts for each Sample.', 0, '', 'Qiagen RNeasy Plus Mini Kit was used following manufacturer\'s instructionsXx-xXRNA-seq libraries were prepared using the Illumina TruSeq Stranded mRNA protocol', 'GSM1726615', 'Illumina HiSeq 2500', 'Feb 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03801577Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1077247', 'GSE70404', 'metastatic breast cancer patient', 'Public on Feb 09 2017', 'Jun 30 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1726nnn/GSM1726615/suppl/GSM1726615_51.processed_counts.txt.gz', '', 9606, 'Cancer Patient 51', 'SRA', 22412952, 47815398),
(1, 'diagnosis: metastatic breast cancerXx-xXcell type: CD11b+ myelomonocytic cells', 'Albert-Gockel 1', 'Fribourg', 'Switzerland', 'Département de Médecine 3', 'curzio.ruegg@unifr.ch', 'Université de Fribourg', 'Pathology', 'Rüegg,,Curzio', '', '1700', 'Sequencing data was processed using the Illumina Pipeline Software version 1.82.Xx-xXPurity-filtered reads were adapters and quality trimmed with Cutadapt (v. 1.2.1, Martin 2011), and filtered for low complexity with Prinseq (v. 0.20.3, Schmieder and Edwards 2011).Xx-xXReads were aligned against Homo sapiens.GRCh38.76 genome using TopHat2 (v. 2.0.9, Kim et al. 2013).Xx-xXThe number of read counts per gene locus was summarized with htseq-count (v. 0.5.4p3, Anders et al. 2014) using Homo sapiens.GRCh38.76 gene annotation.Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: Tab-delimited text files include gene counts for each Sample.', 0, '', 'Qiagen RNeasy Plus Mini Kit was used following manufacturer\'s instructionsXx-xXRNA-seq libraries were prepared using the Illumina TruSeq Stranded mRNA protocol', 'GSM1726613', 'Illumina HiSeq 2500', 'Feb 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03801575Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1077245', 'GSE70404', 'metastatic breast cancer patient', 'Public on Feb 09 2017', 'Jun 30 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1726nnn/GSM1726613/suppl/GSM1726613_37.processed_counts.txt.gz', '', 9606, 'Cancer Patient 37', 'SRA', 23086105, 50786701),
(1, 'diagnosis: metastatic breast cancerXx-xXcell type: CD11b+ myelomonocytic cells', 'Albert-Gockel 1', 'Fribourg', 'Switzerland', 'Département de Médecine 3', 'curzio.ruegg@unifr.ch', 'Université de Fribourg', 'Pathology', 'Rüegg,,Curzio', '', '1700', 'Sequencing data was processed using the Illumina Pipeline Software version 1.82.Xx-xXPurity-filtered reads were adapters and quality trimmed with Cutadapt (v. 1.2.1, Martin 2011), and filtered for low complexity with Prinseq (v. 0.20.3, Schmieder and Edwards 2011).Xx-xXReads were aligned against Homo sapiens.GRCh38.76 genome using TopHat2 (v. 2.0.9, Kim et al. 2013).Xx-xXThe number of read counts per gene locus was summarized with htseq-count (v. 0.5.4p3, Anders et al. 2014) using Homo sapiens.GRCh38.76 gene annotation.Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: Tab-delimited text files include gene counts for each Sample.', 0, '', 'Qiagen RNeasy Plus Mini Kit was used following manufacturer\'s instructionsXx-xXRNA-seq libraries were prepared using the Illumina TruSeq Stranded mRNA protocol', 'GSM1726616', 'Illumina HiSeq 2500', 'Feb 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03801578Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1077248', 'GSE70404', 'metastatic breast cancer patient', 'Public on Feb 09 2017', 'Jun 30 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1726nnn/GSM1726616/suppl/GSM1726616_52.processed_counts.txt.gz', '', 9606, 'Cancer Patient 52', 'SRA', 24089278, 52257065),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cell lineXx-xXtissue: mammary gland, breastXx-xXer status: positiveXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control samples from GSE73966 for YB5 and GSE74036 for MCF7, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first sheet is the YB5 data. First 3 columns are the RPKM values from the control samples from GSE73966 and the next 3 columns are from the RPKM values from 1µM DAC. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control sample. The second sheet contains the MCF7 data in the same setup using control samples from GSE74036 and MCF7 cells treated with 100nM DAC.', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1915702', 'Illumina HiSeq 2500', 'Jan 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'Reanalysis of: GSM1908548Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04202262Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1363844', 'GSE74251', 'MCF7 breast cancer cell line', 'Public on Jan 31 2017', 'Oct 21 2015', 'NONE', '', 9606, 'MCF7_Cntrl_2', 'SRA', 17291104, 26777975),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cell lineXx-xXtissue: mammary gland, breastXx-xXer status: positiveXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control samples from GSE73966 for YB5 and GSE74036 for MCF7, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first sheet is the YB5 data. First 3 columns are the RPKM values from the control samples from GSE73966 and the next 3 columns are from the RPKM values from 1µM DAC. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control sample. The second sheet contains the MCF7 data in the same setup using control samples from GSE74036 and MCF7 cells treated with 100nM DAC.', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1915700', 'Illumina HiSeq 2500', 'Jan 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04202260Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1363842', 'GSE74251', 'MCF7 breast cancer cell line', 'Public on Jan 31 2017', 'Oct 21 2015', 'NONE', '', 9606, 'MCF7_100nM_DAC_3', 'SRA', 30216901, 42798000),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cell lineXx-xXtissue: mammary gland, breastXx-xXer status: positiveXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control samples from GSE73966 for YB5 and GSE74036 for MCF7, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first sheet is the YB5 data. First 3 columns are the RPKM values from the control samples from GSE73966 and the next 3 columns are from the RPKM values from 1µM DAC. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control sample. The second sheet contains the MCF7 data in the same setup using control samples from GSE74036 and MCF7 cells treated with 100nM DAC.', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1915701', 'Illumina HiSeq 2500', 'Jan 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'Reanalysis of: GSM1908538Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04202261Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1363843', 'GSE74251', 'MCF7 breast cancer cell line', 'Public on Jan 31 2017', 'Oct 21 2015', 'NONE', '', 9606, 'MCF7_Cntrl_1', 'SRA', 14019604, 18612786),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cell lineXx-xXtissue: mammary gland, breastXx-xXer status: positiveXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control samples from GSE73966 for YB5 and GSE74036 for MCF7, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first sheet is the YB5 data. First 3 columns are the RPKM values from the control samples from GSE73966 and the next 3 columns are from the RPKM values from 1µM DAC. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control sample. The second sheet contains the MCF7 data in the same setup using control samples from GSE74036 and MCF7 cells treated with 100nM DAC.', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1915699', 'Illumina HiSeq 2500', 'Jan 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04202259Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1363841', 'GSE74251', 'MCF7 breast cancer cell line', 'Public on Jan 31 2017', 'Oct 21 2015', 'NONE', '', 9606, 'MCF7_100nM_DAC_2', 'SRA', 26544316, 38087364),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cell lineXx-xXtissue: mammary gland, breastXx-xXer status: positiveXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control samples from GSE73966 for YB5 and GSE74036 for MCF7, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first sheet is the YB5 data. First 3 columns are the RPKM values from the control samples from GSE73966 and the next 3 columns are from the RPKM values from 1µM DAC. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control sample. The second sheet contains the MCF7 data in the same setup using control samples from GSE74036 and MCF7 cells treated with 100nM DAC.', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1915703', 'Illumina HiSeq 2500', 'Jan 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'Reanalysis of: GSM1908558Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04202263Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1363845', 'GSE74251', 'MCF7 breast cancer cell line', 'Public on Jan 31 2017', 'Oct 21 2015', 'NONE', '', 9606, 'MCF7_Cntrl_3', 'SRA', 53051043, 75384199),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer cell lineXx-xXtissue: mammary gland, breastXx-xXer status: positiveXx-xXgender: female', '3307 North Broad Street', 'Philadelphia', 'USA', 'Fels Institute for Cancer Research and Molecular Biology', '', 'Temple University', '', 'Takahiro,,Sato', '', '19140', 'Illumina HiSeq2500 single end sequencingXx-xXtophat2 --sensitive alignment of the reads in two fastq files from each sample to hg19, generating one BAM file per sampleXx-xXht-seq-count  with default settings to generate counts of each geneXx-xXedgeR to generate RPKM and fold change of each condition compared to control samples from GSE73966 for YB5 and GSE74036 for MCF7, only keep genes with at least one read in one of the samplesXx-xXexport fold change of each inhibitor vs. control into a text file, then import all the fold changes into one excel fileXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed files are in excel format. The first sheet is the YB5 data. First 3 columns are the RPKM values from the control samples from GSE73966 and the next 3 columns are from the RPKM values from 1µM DAC. The following columns are the fold change and FDR of the triplicates of each condition compared against the triplicates of control sample. The second sheet contains the MCF7 data in the same setup using control samples from GSE74036 and MCF7 cells treated with 100nM DAC.', 0, 'derived from metastatic site: pleural effusionXx-xXEach sample was run on two separate lanes on the HiSeq2500 generating two fastq files. The two files were merged for final analysis.', 'RNA was isolated using Rneasy Mini Kit (Qiagen)Xx-xXStrand-specific RNA libraries were generated from 1μg of RNA using TruSeq stranded total RNA with Ribo-Zero Gold (Illumina)', 'GSM1915698', 'Illumina HiSeq 2500', 'Jan 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04202258Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1363840', 'GSE74251', 'MCF7 breast cancer cell line', 'Public on Jan 31 2017', 'Oct 21 2015', 'NONE', '', 9606, 'MCF7_100nM_DAC_1', 'SRA', 24821025, 35224131),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098749', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578778Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660814', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZMAT4-126', 'SRA', 30089490, 33563687),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098743', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578870Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660808', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-HOXA11-120', 'SRA', 34039276, 37800398),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098737', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578864Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660802', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-CBFA2T2-114', 'SRA', 30135609, 33157680),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098715', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578842Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660780', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-IRX6-90', 'SRA', 27415631, 29864498),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098747', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578874Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660812', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-IRX6-124', 'SRA', 30967037, 34723834),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098642', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578790Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660707', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-YBX1-13', 'SRA', 28774478, 31770197),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098735', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578862Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660800', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZNF142-112', 'SRA', 31659635, 35057495),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098748', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578875Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660813', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZMAT4-125', 'SRA', 33929276, 37279656),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098712', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578839Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660777', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-PRRX1-87', 'SRA', 33554523, 37042968),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098681', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578770Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660746', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-PRRX1-54', 'SRA', 31439162, 34219997);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098683', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578772Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660748', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-FOXQ1-56', 'SRA', 28684090, 31180820),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098727', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578854Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660792', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-TRIM25-104', 'SRA', 34175257, 38154598),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098676', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578765Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660741', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-RPL7-49', 'SRA', 27453638, 29903677),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098719', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578846Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660784', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-TSHZ3-94', 'SRA', 29011542, 31644565),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098632', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578810Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660697', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-FOXD4L6-3', 'SRA', 31066173, 34751225),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098695', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578823Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660760', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-TRIM25-70', 'SRA', 31134771, 34464925),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098730', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578857Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660795', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-PAX5-107', 'SRA', 29382151, 32767715),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098750', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578779Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660815', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-TSHZ3-127', 'SRA', 29078725, 31986504),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098671', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578760Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660736', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-CTBP1-44', 'SRA', 31270354, 33913607),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098729', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578856Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660794', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-FOXD4L6-106', 'SRA', 33693487, 37352270),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098641', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578789Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660706', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-ZNF581-12', 'SRA', 35953274, 39819818),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098667', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578756Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660732', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-ZNF589-40', 'SRA', 32060391, 35323747),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098710', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578837Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660775', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-HOXA11-85', 'SRA', 33335284, 37549881),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098745', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578872Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660810', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-PRRX1-122', 'SRA', 32241390, 36192767),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098740', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578867Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660805', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-CNBP-117', 'SRA', 33376625, 36880188),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098744', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578871Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660809', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-PRRX1-121', 'SRA', 30984851, 34561182),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098666', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578755Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660731', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-ZNF589-39', 'SRA', 30339278, 32803992),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098728', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578855Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660793', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-FOXD4L6-105', 'SRA', 33460025, 36071575),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098677', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578766Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660742', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-RPL7-50', 'SRA', 31420580, 34204444),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098637', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578814Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660702', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-CNBP-8', 'SRA', 30727300, 34147992),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098640', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578788Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660705', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-ZNF581-11', 'SRA', 27802723, 31002113),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098638', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578815Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660703', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-CTBP1-9', 'SRA', 29252046, 32687546),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098746', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578873Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660811', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-IRX6-123', 'SRA', 36690922, 40379654);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098635', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578813Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660700', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-ZNF589-6', 'SRA', 33916811, 38514140),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098669', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578758Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660734', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-CNBP-42', 'SRA', 29263718, 31619171),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098736', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578863Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660801', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-CBFA2T2-113', 'SRA', 30339562, 34335101),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098636', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578807Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660701', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-CNBP-7', 'SRA', 30163229, 33760955),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098734', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578861Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660799', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZNF142-111', 'SRA', 29124778, 32519354),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098665', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578754Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660730', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-FOXD4L6-38', 'SRA', 27565300, 30214823),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098742', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578869Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660807', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-HOXA11-119', 'SRA', 33082496, 36761985),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098678', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578767Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660743', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-HOXA11-51', 'SRA', 28574012, 31077978),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098679', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578768Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660744', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-HOXA11-52', 'SRA', 31709617, 35288765),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098713', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578840Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660778', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-PRRX1-88', 'SRA', 35635723, 39604573),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098756', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578782Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660821', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-scramble-135', 'SRA', 31214947, 34811339),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098751', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578780Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660816', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-TSHZ3-128', 'SRA', 28113474, 31082538),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098720', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578847Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660785', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-NKX6-95', 'SRA', 32685798, 36749074),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098644', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578792Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660709', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-RPL7-15', 'SRA', 32324658, 35718694),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098651', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578799Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660716', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-FOXQ1-22', 'SRA', 31010801, 34635223),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098733', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578860Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660798', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZNF253-110', 'SRA', 26967052, 29774049),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098634', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578812Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660699', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-ZNF589-5', 'SRA', 32355727, 35932854),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098680', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578769Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660745', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-PRRX1-53', 'SRA', 33374865, 37015154),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098639', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578787Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660704', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-CTBP1-10', 'SRA', 29562492, 33050438),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098714', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578841Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660779', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-IRX6-89', 'SRA', 26501354, 30021406),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098702', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578829Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660767', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZNF142-77', 'SRA', 30942319, 34297500),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098643', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578791Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660708', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-YBX1-14', 'SRA', 33913234, 37950811),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098653', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578801Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660718', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-IRX6-24', 'SRA', 33709520, 37423446);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098647', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578795Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660712', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-HOXA11-18', 'SRA', 28564014, 31957353),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098694', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578822Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660759', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-TRIM25-69', 'SRA', 32234541, 35761764),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098732', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578859Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660797', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZNF253-109', 'SRA', 31683681, 35048805),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098739', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578866Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660804', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZNF589-116', 'SRA', 30400998, 33738953),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098754', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578785Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660819', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-mock-133', 'SRA', 31022167, 34826584),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098682', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578771Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660747', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-FOXQ1-55', 'SRA', 30881767, 33879304),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098631', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578809Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660696', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-TRIM25-2', 'SRA', 35883684, 39690172),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098672', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578761Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660737', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-ZNF581-45', 'SRA', 29876287, 32878311),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098709', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578836Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660774', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-CNBP-84', 'SRA', 36166866, 39929839),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098658', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578806Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660723', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-mock-31', 'SRA', 31200131, 34672991),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098663', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578752Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660728', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-TRIM25-36', 'SRA', 28180889, 31230496),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098670', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578759Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660735', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-CTBP1-43', 'SRA', 31203922, 33843395),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098674', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578763Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660739', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-YBX1-47', 'SRA', 31449846, 33942707),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098725', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578852Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660790', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-scramble-102', 'SRA', 32157917, 35854586),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098731', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578858Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660796', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-PAX5-108', 'SRA', 38720239, 42870702),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098722', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578849Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660787', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-mock-99', 'SRA', 29300330, 32239453),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098726', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578853Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660791', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-TRIM25-103', 'SRA', 32911756, 36327587),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098752', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578781Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660817', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-NKX6-129', 'SRA', 34125049, 38047613),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098684', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578773Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660749', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-IRX6-57', 'SRA', 32534849, 35872367),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098717', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578844Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660782', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZMAT4-92', 'SRA', 32722929, 36321617),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098630', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578808Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660695', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-TRIM25-1', 'SRA', 26604015, 29605672),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098656', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578804Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660721', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-BNC2-27', 'SRA', 29584982, 33226480),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098654', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578802Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660719', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-TSHZ3-25', 'SRA', 30042192, 33057345);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098721', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578848Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660786', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-NKX6-96', 'SRA', 25913515, 28584542),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098757', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578783Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660822', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-scramble-136', 'SRA', 26795973, 29836167),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098675', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578764Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660740', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-YBX1-48', 'SRA', 29169177, 31801073),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098699', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578826Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660764', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-PAX5-74', 'SRA', 31202933, 34522275),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098723', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578850Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660788', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-mock-100', 'SRA', 26581316, 29679332),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098707', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578834Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660772', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZNF589-82', 'SRA', 30614350, 33342238),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098708', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578835Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660773', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-CNBP-83', 'SRA', 29985652, 32911340),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098652', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578800Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660717', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-IRX6-23', 'SRA', 23772458, 26654384),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098716', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578843Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660781', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZMAT4-91', 'SRA', 27519315, 30587860),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098755', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578786Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660820', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-mock-134', 'SRA', 32810907, 36462196),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098649', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578797Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660714', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-PRRX1-20', 'SRA', 31450864, 35363755),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098704', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578831Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660769', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-CBFA2T2-79', 'SRA', 32806655, 36651627),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098664', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578753Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660729', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-FOXD4L6-37', 'SRA', 30183516, 32952572),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098673', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578762Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660738', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-ZNF581-46', 'SRA', 31115846, 33767925),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098711', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578838Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660776', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-HOXA11-86', 'SRA', 33437011, 37118933),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098703', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578830Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660768', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZNF142-78', 'SRA', 30703703, 34061797),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098706', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578833Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660771', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZNF589-81', 'SRA', 33635887, 37186192),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098738', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578865Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660803', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-ZNF589-115', 'SRA', 32604731, 36562879),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098660', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578749Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660725', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-scramble-33', 'SRA', 29108566, 32202085),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098686', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578775Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660751', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-TSHZ3-59', 'SRA', 31100926, 34183698),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098724', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578851Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660789', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-scramble-101', 'SRA', 29221541, 32031440),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098661', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578750Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660726', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-scramble-34', 'SRA', 32335653, 36115384),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098659', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578748Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660724', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-mock-32', 'SRA', 31441381, 35504060);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098668', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578757Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660733', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-CNBP-41', 'SRA', 29811369, 32374296),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098655', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578803Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660720', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-TSHZ3-26', 'SRA', 33097598, 36496089),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098697', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578824Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660762', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-FOXD4L6-72', 'SRA', 32316688, 36102581),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098696', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578820Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660761', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-FOXD4L6-71', 'SRA', 31797504, 35454513),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098691', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578818Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660756', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-mock-66', 'SRA', 32089892, 35762880),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098650', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578798Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660715', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-FOXQ1-21', 'SRA', 32971156, 36829608),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098718', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578845Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660783', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-TSHZ3-93', 'SRA', 30468328, 34632249),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098741', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578868Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660806', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-CNBP-118', 'SRA', 29357654, 32771645),
(1, 'strain: MCF7 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098753', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578784Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660818', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MCF7-NKX6-130', 'SRA', 31607301, 35362966),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098705', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578832Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660770', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-CBFA2T2-80', 'SRA', 31365503, 34772494),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098646', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578794Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660711', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-HOXA11-17', 'SRA', 33306484, 37100767),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098693', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578821Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660758', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-scramble-68', 'SRA', 31405094, 34465311),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098690', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578817Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660755', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-mock-65', 'SRA', 26383630, 28720713),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098657', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578805Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660722', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-BNC2-28', 'SRA', 34071349, 37895636),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098688', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578777Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660753', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-BNC2-61', 'SRA', 29852858, 33448126),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098692', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578819Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660757', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-scramble-67', 'SRA', 27678003, 30464872),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098701', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578828Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660766', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZNF253-76', 'SRA', 31704992, 34539894),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098648', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578796Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660713', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-PRRX1-19', 'SRA', 29921735, 32973227),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098645', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578793Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660710', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-RPL7-16', 'SRA', 36066231, 39919634),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098689', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578816Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660754', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-BNC2-62', 'SRA', 29315985, 31823264),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098698', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578825Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660763', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-PAX5-73', 'SRA', 26007147, 29393603),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098687', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578776Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660752', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-TSHZ3-60', 'SRA', 32046794, 35119162),
(1, 'strain: T47D cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098700', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578827Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660765', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'T47D-ZNF253-75', 'SRA', 30747285, 34491700);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098662', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578751Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660727', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-TRIM25-35', 'SRA', 31826546, 34665314),
(1, 'strain: MDAMB231 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098685', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578774Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660750', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'MDAMB231-IRX6-58', 'SRA', 32959748, 35944906),
(1, 'strain: BT549 cellsXx-xXtissue: breast', '1275 York Ave', 'New York', 'USA', 'Human Oncology and Pathogenesis Program', 'walshl@mskcc.org', 'Memorial Sloan Kettering Cancer Center', 'Timothy Chan', 'Logan,A,Walsh', '6468882783', '10065', 'Raw reads in the fastq format were aligned to Human Genome HG19 using the RNA-seq STAR aligner version 2.4.0d (http://www.ncbi.nlm.nih.gov/pubmed/23104886, http://www.ncbi.nlm.nih.gov/pubmed/26334920) as recommended by user manual downloaded along with the software. STAR aligner was chosen for mapping accuracy and speed (http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2722.html).  Mapped reads for each sample were counted for each gene in annotation files in GTF format (gencode.v19.annotation.gtf available for download from GENECODE website (http://www.gencodegenes.org/releases/19.html)) using the FeatureCounts read summarization program (http://www.ncbi.nlm.nih.gov/pubmed/?term=24227677) following the user guide (http://bioinf.wehi.edu.au/subread-package/SubreadUsersGuide.pdf). Individual count files were merged to generate the raw-counts matrix by an in-house R script, normalized to account for differences in library size and the variance was stabilized by fitting the dispersion to a negative-binomial distribution as implemented in the DESeq R package (http://bioconductor.org/packages/release/bioc/html/DESeq.html)(Anders and Huber, 2010).Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: txt file contains feature count matrix', 0, '', 'RNA was extracted from cells using an RNeasy kit (Qiagen)Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2098633', 'Illumina HiSeq 2500', 'Apr 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04578811Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1660698', 'GSE79586Xx-xXGSE79589', 'breast cancer cell line', 'Public on Apr 01 2017', 'Mar 24 2016', 'NONE', '', 9606, 'BT549-FOXD4L6-4', 'SRA', 33260876, 36768551),
(1, 'cell line: SK-BR-3Xx-xXtreatment: CDK12 siRNA-1', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126065', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867908Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711862', 'GSE80409', 'Breast cancer_SK-BR-3_CDK12si', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126065/suppl/GSM2126065_SA557.counts.txt.gz', '', 9606, 'SK-BR-3_CDK12si_replicate_3', 'SRA', 90241879, 110528022),
(1, 'cell line: SK-BR-3Xx-xXtreatment: scrambled siRNA', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126067', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867910Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711864', 'GSE80409', 'Breast cancer_SK-BR-3_scrambledsi_control', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126067/suppl/GSM2126067_SA554.counts.txt.gz', '', 9606, 'SK-BR-3_scrambledsi_replicate_2', 'SRA', 78514294, 95485182),
(1, 'cell line: SK-BR-3Xx-xXtreatment: scrambled siRNA', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126066', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867909Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711863', 'GSE80409', 'Breast cancer_SK-BR-3_scrambledsi_control', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126066/suppl/GSM2126066_SA518.counts.txt.gz', '', 9606, 'SK-BR-3_scrambledsi_replicate_1', 'SRA', 91577401, 107642011),
(1, 'cell line: 184-hTertXx-xXtreatment: CDK12 siRNA-1', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126070', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867913Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711867', 'GSE80409', 'Breast cancer_184-hTert_CDK12si', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126070/suppl/GSM2126070_SA562.counts.txt.gz', '', 9606, '184-hTert_CDK12si_replicate_2', 'SRA', 81465967, 98577843),
(1, 'cell line: SK-BR-3Xx-xXtreatment: CDK12 siRNA-1', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126064', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867907Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711861', 'GSE80409', 'Breast cancer_SK-BR-3_CDK12si', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126064/suppl/GSM2126064_SA556.counts.txt.gz', '', 9606, 'SK-BR-3_CDK12si_replicate_2', 'SRA', 73957209, 90546803),
(1, 'cell line: 184-hTertXx-xXtreatment: scrambled siRNA', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126072', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867915Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711869', 'GSE80409', 'Breast cancer_184-hTert_scrambledsi_control', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126072/suppl/GSM2126072_SA520.counts.txt.gz', '', 9606, '184-hTert_scrambledsi_replicate_1', 'SRA', 113671447, 133282650),
(1, 'cell line: 184-hTertXx-xXtreatment: CDK12 siRNA-1', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126071', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867914Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711868', 'GSE80409', 'Breast cancer_184-hTert_CDK12si', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126071/suppl/GSM2126071_SA563.counts.txt.gz', '', 9606, '184-hTert_CDK12si_replicate_3', 'SRA', 86265931, 104678110),
(1, 'cell line: SK-BR-3Xx-xXtreatment: scrambled siRNA', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126068', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867911Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711865', 'GSE80409', 'Breast cancer_SK-BR-3_scrambledsi_control', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126068/suppl/GSM2126068_SA555.counts.txt.gz', '', 9606, 'SK-BR-3_scrambledsi_replicate_3', 'SRA', 91466433, 112624947),
(1, 'cell line: SK-BR-3Xx-xXtreatment: CDK12 siRNA-1', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126063', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867906Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711860', 'GSE80409', 'Breast cancer_SK-BR-3_CDK12si', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126063/suppl/GSM2126063_SA519.counts.txt.gz', '', 9606, 'SK-BR-3_CDK12si_replicate_1', 'SRA', 75962765, 91106340),
(1, 'cell line: 184-hTertXx-xXtreatment: CDK12 siRNA-1', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126069', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867912Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711866', 'GSE80409', 'Breast cancer_184-hTert_CDK12si', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126069/suppl/GSM2126069_SA521.counts.txt.gz', '', 9606, '184-hTert_CDK12si_replicate_1', 'SRA', 87741854, 103742944),
(1, 'cell line: 184-hTertXx-xXtreatment: scrambled siRNA', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126073', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867916Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711870', 'GSE80409', 'Breast cancer_184-hTert_scrambledsi_control', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126073/suppl/GSM2126073_SA560.counts.txt.gz', '', 9606, '184-hTert_scrambledsi_replicate_2', 'SRA', 78460681, 95893295),
(1, 'cell line: 184-hTertXx-xXtreatment: scrambled siRNA', '675 W 10th Ave', 'Vancouver', 'Canada', '', '', 'University of British Columbia', '', 'Gregg,,Morin', '', 'V5Z 1L3', 'gsnap v2012-07-20 software used for alignmentXx-xXhtseq-count v0.6.1 software used to cound reads overlapping genesXx-xXDESeq v1.6.3 used for differential expression analysisXx-xXMISO v0.4.9 used to detect differential splicing events between treated and control samplesXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited file reporting counts', 0, '', 'SK-BR-3 and 184-hTERT cells treated with CDK12 siRNA or scrambled siRNA were harvested in Trizol (Invitrogen) and total RNA was isolated using the Rneasy kit (Qiagen).Xx-xXLibrary construction was performed on total RNA at the Michael Smith Genome Sciences Centre using the ssRNA-seq pipeline for mRNA libraries', 'GSM2126074', 'Illumina HiSeq 2000', 'Mar 17 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04867917Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1711871', 'GSE80409', 'Breast cancer_184-hTert_scrambledsi_control', 'Public on Mar 16 2017', 'Apr 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2126nnn/GSM2126074/suppl/GSM2126074_SA561.counts.txt.gz', '', 9606, '184-hTert_scrambledsi_replicate_3', 'SRA', 70070253, 84693732),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancer (Basal type)Xx-xXtreatment: 20μM Etoposide for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to mm9 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2252626', 'Illumina HiSeq 2000', 'Feb 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05441356Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1978327', 'GSE84863', 'Breast cancer MDA-MB231', 'Public on Feb 13 2017', 'Jul 26 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2252nnn/GSM2252626/suppl/GSM2252626_Sample_MDA231_Etoposide_Genes_ReadCount.txt.gz', '', 9606, 'MDA-MB231_Etoposide', 'SRA', 35137053, 38433887),
(1, 'cell type: Breast cancer (Basal type)Xx-xXcell line: MDA-MB231Xx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment: 40microg/ml MMS for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to mm9 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '40μg/ml MMS for 8 hours', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2284968', 'Illumina HiSeq 2000', 'Feb 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05591764Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2034933', 'GSE84863', 'Breast cancer MDA-MB231', 'Public on Feb 13 2017', 'Aug 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2284nnn/GSM2284968/suppl/GSM2284968_Sample_MDA231_MMS_8hr_Genes_ReadCount.txt.gz', '', 9606, 'MDA-MB231_MMS_8hr', 'SRA', 31538937, 34120265),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancer (Basal type)Xx-xXtreatment: 0.2μM Paclitaxel for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to mm9 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2252627', 'Illumina HiSeq 2000', 'Feb 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05441355Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1978328', 'GSE84863', 'Breast cancer MDA-MB231', 'Public on Feb 13 2017', 'Jul 26 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2252nnn/GSM2252627/suppl/GSM2252627_Sample_MDA231_Paclitaxel_Genes_ReadCount.txt.gz', '', 9606, 'MDA-MB231_Paclitaxel', 'SRA', 33771967, 36945938),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancer (Basal type)Xx-xXtreatment: 1μM Doxorubicin for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to mm9 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2252625', 'Illumina HiSeq 2000', 'Feb 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05441357Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1978326', 'GSE84863', 'Breast cancer MDA-MB231', 'Public on Feb 13 2017', 'Jul 26 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2252nnn/GSM2252625/suppl/GSM2252625_Sample_MDA231_Doxorubicin_Genes_ReadCount.txt.gz', '', 9606, 'MDA-MB231_Doxorubicin', 'SRA', 29431156, 32723303),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancer (Basal type)Xx-xXtreatment: DMSO', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to mm9 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2252624', 'Illumina HiSeq 2000', 'Feb 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05441358Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1978325', 'GSE84863', 'Breast cancer MDA-MB231', 'Public on Feb 13 2017', 'Jul 26 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2252nnn/GSM2252624/suppl/GSM2252624_Sample_MDA231_NT_Genes_ReadCount.txt.gz', '', 9606, 'MDA-MB231_Control', 'SRA', 30953179, 33726493),
(1, 'cell type: Breast cancer (Basal type)Xx-xXcell line: MDA-MB231Xx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment: DMSO', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to mm9 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, 'no treatment', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2284967', 'Illumina HiSeq 2000', 'Feb 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05591765Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2034932', 'GSE84863', 'Breast cancer MDA-MB231', 'Public on Feb 13 2017', 'Aug 18 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2284nnn/GSM2284967/suppl/GSM2284967_Sample_MDA231_Ctrl_8hr_Genes_ReadCount.txt.gz', '', 9606, 'MDA-MB231_Ctrl_8hr', 'SRA', 20040667, 21468231),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shMEN1Xx-xXtreatment: vehicle', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300036', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721621Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071015', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_Veh_ShMEN1_1', 'SRA', 38876597, 47311925),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shCtrlXx-xXtreatment: vehicle', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300033', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721624Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071012', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_Veh_ShCtrl_2', 'SRA', 32028331, 38317538),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shCtrlXx-xXtreatment: vehicle', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300032', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721613Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071011', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_Veh_ShCtrl_1', 'SRA', 62702152, 76388091),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shMEN1Xx-xXtreatment: vehicle', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300037', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721620Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071016', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_Veh_ShMEN1_2', 'SRA', 29811046, 36024041),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shCtrlXx-xXtreatment: 10nM estradiol', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300035', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721622Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071014', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_E2_ShCtrl_2', 'SRA', 45048203, 54468514),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shMEN1Xx-xXtreatment: 10nM estradiol', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300038', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721619Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071017', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_E2_ShMEN1_1', 'SRA', 34571648, 42099806),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shCtrlXx-xXtreatment: 10nM estradiol', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300034', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721623Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071013', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_E2_ShCtrl_1', 'SRA', 34371122, 40892911),
(1, 'cell line: MCF7Xx-xXinfection: lentivirally infected with constructs against shMEN1Xx-xXtreatment: 10nM estradiol', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_ShMEN1.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300039', 'Illumina HiSeq 2000', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721618Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071018', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_E2_ShMEN1_2', 'SRA', 45998811, 54166729),
(1, 'cell line: MCF-7Xx-xXtissue: breast cancerXx-xXgenotype/variation: mNT suppressed', '1155 Union Circle #305220', 'Denton', 'USA', '', '', 'University of North Texas', '', 'Rajeev,K.,Azad', '', '76210', 'Used Tophat and Bowtie for read alignment onto human reference genome and Cufflinks for obtaining the FPKMs and assessment of significance of fold changeXx-xXGenome_build: NCBI build 37.2Xx-xXSupplementary_files_format_and_content: Excel file; FPKMs for empty vector control and mNT suppressed breast cancer cell lines MCF-7', 0, '', 'Cells were grown to 80% confluence, medium was discarded and cells were washed with 1x PBS.  Cells were collected directly from plates and RNA was extracted using the Qiagen RNeasy Mini Kit (Cat.# 74104) according to manufacturers instructions.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2335759', 'Illumina HiSeq 2000', 'Mar 25 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05862248Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2210969', 'GSE87626', 'breast cancer', 'Public on Mar 25 2017', 'Oct 05 2016', 'NONE', '', 9606, 'mNT(-) Rep 3', 'SRA', 9324063, 10600499),
(1, 'cell line: MCF-7Xx-xXtissue: breast cancerXx-xXgenotype/variation: Control', '1155 Union Circle #305220', 'Denton', 'USA', '', '', 'University of North Texas', '', 'Rajeev,K.,Azad', '', '76210', 'Used Tophat and Bowtie for read alignment onto human reference genome and Cufflinks for obtaining the FPKMs and assessment of significance of fold changeXx-xXGenome_build: NCBI build 37.2Xx-xXSupplementary_files_format_and_content: Excel file; FPKMs for empty vector control and mNT suppressed breast cancer cell lines MCF-7', 0, '', 'Cells were grown to 80% confluence, medium was discarded and cells were washed with 1x PBS.  Cells were collected directly from plates and RNA was extracted using the Qiagen RNeasy Mini Kit (Cat.# 74104) according to manufacturers instructions.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2335755', 'Illumina HiSeq 2000', 'Mar 25 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05862252Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2210965', 'GSE87626', 'breast cancer', 'Public on Mar 25 2017', 'Oct 05 2016', 'NONE', '', 9606, 'EV Rep 2', 'SRA', 24043742, 27402210),
(1, 'cell line: MCF-7Xx-xXtissue: breast cancerXx-xXgenotype/variation: Control', '1155 Union Circle #305220', 'Denton', 'USA', '', '', 'University of North Texas', '', 'Rajeev,K.,Azad', '', '76210', 'Used Tophat and Bowtie for read alignment onto human reference genome and Cufflinks for obtaining the FPKMs and assessment of significance of fold changeXx-xXGenome_build: NCBI build 37.2Xx-xXSupplementary_files_format_and_content: Excel file; FPKMs for empty vector control and mNT suppressed breast cancer cell lines MCF-7', 0, '', 'Cells were grown to 80% confluence, medium was discarded and cells were washed with 1x PBS.  Cells were collected directly from plates and RNA was extracted using the Qiagen RNeasy Mini Kit (Cat.# 74104) according to manufacturers instructions.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2335756', 'Illumina HiSeq 2000', 'Mar 25 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05862251Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2210966', 'GSE87626', 'breast cancer', 'Public on Mar 25 2017', 'Oct 05 2016', 'NONE', '', 9606, 'EV Rep 3', 'SRA', 15772860, 17961877),
(1, 'cell line: MCF-7Xx-xXtissue: breast cancerXx-xXgenotype/variation: Control', '1155 Union Circle #305220', 'Denton', 'USA', '', '', 'University of North Texas', '', 'Rajeev,K.,Azad', '', '76210', 'Used Tophat and Bowtie for read alignment onto human reference genome and Cufflinks for obtaining the FPKMs and assessment of significance of fold changeXx-xXGenome_build: NCBI build 37.2Xx-xXSupplementary_files_format_and_content: Excel file; FPKMs for empty vector control and mNT suppressed breast cancer cell lines MCF-7', 0, '', 'Cells were grown to 80% confluence, medium was discarded and cells were washed with 1x PBS.  Cells were collected directly from plates and RNA was extracted using the Qiagen RNeasy Mini Kit (Cat.# 74104) according to manufacturers instructions.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2335754', 'Illumina HiSeq 2000', 'Mar 25 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05862247Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2210964', 'GSE87626', 'breast cancer', 'Public on Mar 25 2017', 'Oct 05 2016', 'NONE', '', 9606, 'EV Rep 1', 'SRA', 15717753, 17920510),
(1, 'cell line: MCF-7Xx-xXtissue: breast cancerXx-xXgenotype/variation: mNT suppressed', '1155 Union Circle #305220', 'Denton', 'USA', '', '', 'University of North Texas', '', 'Rajeev,K.,Azad', '', '76210', 'Used Tophat and Bowtie for read alignment onto human reference genome and Cufflinks for obtaining the FPKMs and assessment of significance of fold changeXx-xXGenome_build: NCBI build 37.2Xx-xXSupplementary_files_format_and_content: Excel file; FPKMs for empty vector control and mNT suppressed breast cancer cell lines MCF-7', 0, '', 'Cells were grown to 80% confluence, medium was discarded and cells were washed with 1x PBS.  Cells were collected directly from plates and RNA was extracted using the Qiagen RNeasy Mini Kit (Cat.# 74104) according to manufacturers instructions.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2335757', 'Illumina HiSeq 2000', 'Mar 25 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05862250Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2210967', 'GSE87626', 'breast cancer', 'Public on Mar 25 2017', 'Oct 05 2016', 'NONE', '', 9606, 'mNT(-) Rep 1', 'SRA', 21456824, 24443413),
(1, 'cell line: MCF-7Xx-xXtissue: breast cancerXx-xXgenotype/variation: mNT suppressed', '1155 Union Circle #305220', 'Denton', 'USA', '', '', 'University of North Texas', '', 'Rajeev,K.,Azad', '', '76210', 'Used Tophat and Bowtie for read alignment onto human reference genome and Cufflinks for obtaining the FPKMs and assessment of significance of fold changeXx-xXGenome_build: NCBI build 37.2Xx-xXSupplementary_files_format_and_content: Excel file; FPKMs for empty vector control and mNT suppressed breast cancer cell lines MCF-7', 0, '', 'Cells were grown to 80% confluence, medium was discarded and cells were washed with 1x PBS.  Cells were collected directly from plates and RNA was extracted using the Qiagen RNeasy Mini Kit (Cat.# 74104) according to manufacturers instructions.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2335758', 'Illumina HiSeq 2000', 'Mar 25 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05862249Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2210968', 'GSE87626', 'breast cancer', 'Public on Mar 25 2017', 'Oct 05 2016', 'NONE', '', 9606, 'mNT(-) Rep 2', 'SRA', 16470384, 19161290);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MCF7Xx-xXcell type: breast cancer', '101 College Street', 'Toronto', 'Canada', '', 'Hansen.He@uhnresearch.ca', 'University of Toronto', 'He lab', 'Housheng,Hansen,He', '', 'M5G 1L7', 'GRO-seq reads were mapped to the human genome (hg19) using Bowtie with default parameters.   RNA-seq reads were aligned to human genome (hg19) with tophat (v2.1.1). ChIP-seq and Dnase-seq reads were aligned to human genome (hg19) with Bowtie.Xx-xXMACS2 was used for ChIP-seq and Dnase-seq peak calling with the parameter \"–SPMR\" on to produce bedGraph files, which were then converted to bigwiggle filesXx-xXHtseq (v0.5.4p3) was used to get gene level read counts for GRO-seq data and DESeq2 (v1.8.2) was used for differential gene analysis; GFOLD (v1.1.3) was used to generate gene level read count and carry out differential expression analysis.  Xx-xXbedGraph files for GRO-seq were generated using genomeCoverageBed function in the BEDTools(v2.20.1) suite with total signal of 10 million, replicates were then combined before converted to bigwiggle files.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: bigwiggle files were converted from bedGraph with bedGraphToBigWig function in BLAT suite. For GRO-seq and RNA-seq data, score equals to reads per 20 million; for ChIP-seq and Dnase-seq data, score equals to reads per million.', 0, 'RNA-seq S phase', 'GRO-seq: Nuclear run-on RNA was further purified with TURBO DNA-freeTM kit (Life Technologies,  AM1907) to remove residue DNA contamination. RNA-seq: Total RNA was extracted from synchronized MCF-7 cells with TRIZOL REAGENT (Life Technologies, 15596026) and fragmentized with sonication. ChIP-seq: Chromatin from synchronized MCF-7 cells was digested with Nuclease micrococcal from Staphylococcus aureus (Sigma Aldrich, N3755-50UN) and dialyzed with Slide-A-Lyzer Dialysis Cassettes (Thermo Fisher Scientific, PI66380). Dynabeads Protein A (Life Technologies, 10002D) and G (Life Technologies, 10004D) were used for immunoprecipitation with antibodies against H3K4me2 (EMD Millipore, 07-030) and H3K27ac (Abcam, ab4729).Xx-xXGRO-seq and RNA-seq libraries were constructed with Encore Complete RNA-Seq DR Multiplex System 1-8 (Nugen Technologies lnc, 0333-32). ChIP-seq libraries were constructed with ThruPLEX DNA-seq kit (Rubicon) and sequenced to 50bp with Illumina HiSeq machine.', 'GSM2476256', 'Illumina HiSeq 2000', 'Mar 16 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06294280Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2537030', 'GSE94479', 'MCF7 breast cancer cell line', 'Public on Mar 16 2017', 'Feb 03 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2476nnn/GSM2476256/suppl/GSM2476256_RNA_S.bw', '', 9606, 'MCF7 RNA-seq S phase', 'SRA', 15155199, 35556497),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer', '101 College Street', 'Toronto', 'Canada', '', 'Hansen.He@uhnresearch.ca', 'University of Toronto', 'He lab', 'Housheng,Hansen,He', '', 'M5G 1L7', 'GRO-seq reads were mapped to the human genome (hg19) using Bowtie with default parameters.   RNA-seq reads were aligned to human genome (hg19) with tophat (v2.1.1). ChIP-seq and Dnase-seq reads were aligned to human genome (hg19) with Bowtie.Xx-xXMACS2 was used for ChIP-seq and Dnase-seq peak calling with the parameter \"–SPMR\" on to produce bedGraph files, which were then converted to bigwiggle filesXx-xXHtseq (v0.5.4p3) was used to get gene level read counts for GRO-seq data and DESeq2 (v1.8.2) was used for differential gene analysis; GFOLD (v1.1.3) was used to generate gene level read count and carry out differential expression analysis.  Xx-xXbedGraph files for GRO-seq were generated using genomeCoverageBed function in the BEDTools(v2.20.1) suite with total signal of 10 million, replicates were then combined before converted to bigwiggle files.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: bigwiggle files were converted from bedGraph with bedGraphToBigWig function in BLAT suite. For GRO-seq and RNA-seq data, score equals to reads per 20 million; for ChIP-seq and Dnase-seq data, score equals to reads per million.', 0, 'RNA-seq M phase', 'GRO-seq: Nuclear run-on RNA was further purified with TURBO DNA-freeTM kit (Life Technologies,  AM1907) to remove residue DNA contamination. RNA-seq: Total RNA was extracted from synchronized MCF-7 cells with TRIZOL REAGENT (Life Technologies, 15596026) and fragmentized with sonication. ChIP-seq: Chromatin from synchronized MCF-7 cells was digested with Nuclease micrococcal from Staphylococcus aureus (Sigma Aldrich, N3755-50UN) and dialyzed with Slide-A-Lyzer Dialysis Cassettes (Thermo Fisher Scientific, PI66380). Dynabeads Protein A (Life Technologies, 10002D) and G (Life Technologies, 10004D) were used for immunoprecipitation with antibodies against H3K4me2 (EMD Millipore, 07-030) and H3K27ac (Abcam, ab4729).Xx-xXGRO-seq and RNA-seq libraries were constructed with Encore Complete RNA-Seq DR Multiplex System 1-8 (Nugen Technologies lnc, 0333-32). ChIP-seq libraries were constructed with ThruPLEX DNA-seq kit (Rubicon) and sequenced to 50bp with Illumina HiSeq machine.', 'GSM2476257', 'Illumina HiSeq 2000', 'Mar 16 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06294279Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2537031', 'GSE94479', 'MCF7 breast cancer cell line', 'Public on Mar 16 2017', 'Feb 03 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2476nnn/GSM2476257/suppl/GSM2476257_RNA_M.bw', '', 9606, 'MCF7 RNA-seq M phase', 'SRA', 26319965, 51139876),
(1, 'cell line: MCF7Xx-xXcell type: breast cancer', '101 College Street', 'Toronto', 'Canada', '', 'Hansen.He@uhnresearch.ca', 'University of Toronto', 'He lab', 'Housheng,Hansen,He', '', 'M5G 1L7', 'GRO-seq reads were mapped to the human genome (hg19) using Bowtie with default parameters.   RNA-seq reads were aligned to human genome (hg19) with tophat (v2.1.1). ChIP-seq and Dnase-seq reads were aligned to human genome (hg19) with Bowtie.Xx-xXMACS2 was used for ChIP-seq and Dnase-seq peak calling with the parameter \"–SPMR\" on to produce bedGraph files, which were then converted to bigwiggle filesXx-xXHtseq (v0.5.4p3) was used to get gene level read counts for GRO-seq data and DESeq2 (v1.8.2) was used for differential gene analysis; GFOLD (v1.1.3) was used to generate gene level read count and carry out differential expression analysis.  Xx-xXbedGraph files for GRO-seq were generated using genomeCoverageBed function in the BEDTools(v2.20.1) suite with total signal of 10 million, replicates were then combined before converted to bigwiggle files.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: bigwiggle files were converted from bedGraph with bedGraphToBigWig function in BLAT suite. For GRO-seq and RNA-seq data, score equals to reads per 20 million; for ChIP-seq and Dnase-seq data, score equals to reads per million.', 0, 'RNA-seq G0/G1', 'GRO-seq: Nuclear run-on RNA was further purified with TURBO DNA-freeTM kit (Life Technologies,  AM1907) to remove residue DNA contamination. RNA-seq: Total RNA was extracted from synchronized MCF-7 cells with TRIZOL REAGENT (Life Technologies, 15596026) and fragmentized with sonication. ChIP-seq: Chromatin from synchronized MCF-7 cells was digested with Nuclease micrococcal from Staphylococcus aureus (Sigma Aldrich, N3755-50UN) and dialyzed with Slide-A-Lyzer Dialysis Cassettes (Thermo Fisher Scientific, PI66380). Dynabeads Protein A (Life Technologies, 10002D) and G (Life Technologies, 10004D) were used for immunoprecipitation with antibodies against H3K4me2 (EMD Millipore, 07-030) and H3K27ac (Abcam, ab4729).Xx-xXGRO-seq and RNA-seq libraries were constructed with Encore Complete RNA-Seq DR Multiplex System 1-8 (Nugen Technologies lnc, 0333-32). ChIP-seq libraries were constructed with ThruPLEX DNA-seq kit (Rubicon) and sequenced to 50bp with Illumina HiSeq machine.', 'GSM2476255', 'Illumina HiSeq 2000', 'Mar 16 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06294281Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2537028', 'GSE94479', 'MCF7 breast cancer cell line', 'Public on Mar 16 2017', 'Feb 03 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2476nnn/GSM2476255/suppl/GSM2476255_RNA_G0G1.bw', '', 9606, 'MCF7 RNA-seq G0/G1', 'SRA', 15414442, 39889994),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32076Xx-xXLZ32076_NoIndex_L005Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576202', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718022Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734418', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3904', 'SRA', 49334554, 111483885),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37138Xx-xXLZML_37138_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576189', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718035Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734405', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '49', 'SRA', 43969555, 102407437),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32176Xx-xXLZ32176_NoIndex_L005Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576206', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718018Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734422', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4175', 'SRA', 50053368, 111897829),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ39532Xx-xXLZ39532_NoIndex_L007Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576221', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718039Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734437', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, 'MC1', 'SRA', 49032451, 102119372),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32070Xx-xXLZ32070_NoIndex_L002Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576198', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718026Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734414', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3611', 'SRA', 48672709, 110700343),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32174Xx-xXLZ32174_NoIndex_L003Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576191', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718033Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734407', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '2665', 'SRA', 43147660, 105678517),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32179Xx-xXLZ32179_NoIndex_L008Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576214', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718046Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734430', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '5156', 'SRA', 57774167, 110596903),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37250Xx-xXLZML_37250_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576219', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718041Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734435', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '8563', 'SRA', 38102189, 94322467),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32181Xx-xXLZ32181_NoIndex_L002Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576216', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718044Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734432', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '5998', 'SRA', 48954986, 101811267),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32081Xx-xXLZ32081_NoIndex_L007Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576205', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718019Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734421', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4013', 'SRA', 51605140, 115234475),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32467Xx-xXLZ32467_NoIndex_L007Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576200', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718024Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734416', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3807', 'SRA', 39934825, 104545838),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37143Xx-xXLZML_37143_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576220', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718040Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734436', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '9161', 'SRA', 41771511, 97127098),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32084Xx-xXLZ32084_NoIndex_L008Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576210', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718014Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734426', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4664', 'SRA', 51771898, 114266840),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32422Xx-xXLZ32422_NoIndex_L006Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576211', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718013Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734427', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4888', 'SRA', 41672476, 108270621),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32420Xx-xXLZ32420_NoIndex_L005Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576209', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718015Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734425', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4272', 'SRA', 42750400, 104016437),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37139Xx-xXLZML_37139_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576196', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718028Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734412', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3469', 'SRA', 47202084, 102465477),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37388Xx-xXLZML_37388_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576217', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718043Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734433', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '6257', 'SRA', 38034034, 101515299),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32182Xx-xXLZ32182_NoIndex_L003Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576199', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718025Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734415', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3613', 'SRA', 39909472, 99059505),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32072Xx-xXLZ32072_NoIndex_L003Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576201', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718023Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734417', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3887', 'SRA', 52827886, 107573743),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32078Xx-xXLZ32078_NoIndex_L006Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576204', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718020Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734420', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3963', 'SRA', 48006041, 106600830),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32073Xx-xXLZ32073_NoIndex_L004Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576203', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718021Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734419', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3936', 'SRA', 48240176, 104565616),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32093Xx-xXLZ32093_NoIndex_L001Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576197', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718027Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734413', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3561', 'SRA', 41597836, 102628316),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37312Xx-xXLZML_37312_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576188', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718036Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734404', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '47', 'SRA', 37803008, 97895695),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32423Xx-xXLZ32423_NoIndex_L005Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576222', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718038Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734438', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, 'MDA-IBC3', 'SRA', 39912261, 97534246),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ39533Xx-xXLZ39533_NoIndex_L008Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576218', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718042Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734434', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '7441', 'SRA', 52869967, 107370096),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32468Xx-xXLZ32468_NoIndex_L008Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576212', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718012Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734428', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4913', 'SRA', 31531415, 101931501),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32178Xx-xXLZ32178_NoIndex_L007Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576213', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718047Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734429', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '5097', 'SRA', 48062219, 112852191),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32419Xx-xXLZ32419_NoIndex_L004Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576194', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718030Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734410', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3204', 'SRA', 41865404, 103030005);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32180Xx-xXLZ32180_NoIndex_L001Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576215', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718045Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734431', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '5471', 'SRA', 44713717, 100467386),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32094Xx-xXLZ32094_NoIndex_L002Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576207', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718017Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734423', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4189', 'SRA', 38836096, 106992396),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32175Xx-xXLZ32175_NoIndex_L004Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576192', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718032Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734408', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3107', 'SRA', 42862716, 100799700),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32067Xx-xXLZ32067_NoIndex_L001Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576193', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718031Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734409', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3143', 'SRA', 43908456, 105731737),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37140Xx-xXLZML_37140_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576195', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718029Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734411', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '3277', 'SRA', 44798179, 103506225),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ37246Xx-xXLZML_37246_NoIndexXx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576187', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718037Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734403', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '2', 'SRA', 38045456, 98977407),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32177Xx-xXLZ32177_NoIndex_L006Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576208', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718016Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734424', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '4195', 'SRA', 38707066, 106694193),
(1, 'tissue: breast cancer xenograft', '1 Baylor Plz', 'Houston', 'USA', 'Molecular and Human Genetics', 'eschen@bcm.edu', 'Baylor College of Medicine', '', 'Edward,S.,Chen', '7137985101', '77030', 'Xenome was used to qualify gene grouping by species.Xx-xXSTAR, with a hg19 reference, was used for alignment of human sequences.Xx-xXR\'s Rsubread (default settings) package was used to estimate gene counts.Xx-xXR\'s edgeR:::rpkm was then used to quantify gene expression.Xx-xXGenome_build: hg19 (GRCh37)Xx-xXSupplementary_files_format_and_content: xeno.exp.txt: Tab-delimited text file includes RPKM values.', 0, 'LZ32290Xx-xXLZ32290_NoIndex_L001Xx-xXHuman/MouseXx-xXPatient-derived breast cancer xenograft (PDX).Xx-xXProcessed data file: xeno.exp.txt', 'Total RNA was extracted from PDX samples and 10ng was used to generate and amplify whole transcriptome cDNA (NuGen Ovation v2, NuGen Technologies).Xx-xXIllumina TruSeq DNA-Seq library was quantified with the Kapa qPCR Library Quantitation Kit (Kapa).', 'GSM2576190', 'Illumina HiSeq 2000', 'Apr 14 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06718034Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2734406', 'GSE97726', 'Human breast cancer xenograft', 'Public on Apr 13 2017', 'Apr 12 2017', 'NONE', '', 9606, '2147', 'SRA', 68363426, 123956104),
(1, 'treatment: ControlXx-xXtissue: ZR75', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, 'ZR75 cells treated with vehicle+E2', 'Total RNA samples about 1-2 mg were used for sequencing library preparation according to Illumina TruSeq Total RNA Sample Preparation Guide (Illumina Cat. #: RS-122-2201). Each library was bar-coded and then pooled for cluster generation and sequencing run with 100bp single-end (SE) sequencing protocol.', 'GSM1963023', 'Illumina HiSeq 2000', 'Aug 18 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04315695Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1462165', 'GSE75664', 'ZR75 breast cancer cells', 'Public on Aug 18 2017', 'Dec 03 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1963nnn/GSM1963023/suppl/GSM1963023_Sample_ConZR75_1_Genes_ReadCount.txt.gz', '', 9606, 'ZR75-control-1', 'SRA', 28682468, 31747866),
(1, 'treatment: ControlXx-xXtissue: ZR75', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, 'ZR75 cells treated with vehicle+E2', 'Total RNA samples about 1-2 mg were used for sequencing library preparation according to Illumina TruSeq Total RNA Sample Preparation Guide (Illumina Cat. #: RS-122-2201). Each library was bar-coded and then pooled for cluster generation and sequencing run with 100bp single-end (SE) sequencing protocol.', 'GSM1963024', 'Illumina HiSeq 2000', 'Aug 18 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04315696Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1462166', 'GSE75664', 'ZR75 breast cancer cells', 'Public on Aug 18 2017', 'Dec 03 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1963nnn/GSM1963024/suppl/GSM1963024_Sample_ConZR75_2_Genes_ReadCount.txt.gz', '', 9606, 'ZR75-control-2', 'SRA', 26590303, 29543732),
(1, 'treatment: ESR1 Coregulator Binding Site InhibitorsXx-xXtissue: ZR75', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, 'ZR75 cells treated with ERX+E2', 'Total RNA samples about 1-2 mg were used for sequencing library preparation according to Illumina TruSeq Total RNA Sample Preparation Guide (Illumina Cat. #: RS-122-2201). Each library was bar-coded and then pooled for cluster generation and sequencing run with 100bp single-end (SE) sequencing protocol.', 'GSM1963025', 'Illumina HiSeq 2000', 'Aug 18 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04315697Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1462167', 'GSE75664', 'ZR75 breast cancer cells', 'Public on Aug 18 2017', 'Dec 03 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1963nnn/GSM1963025/suppl/GSM1963025_Sample_ERX-11_1_Genes_ReadCount.txt.gz', '', 9606, 'ZR75-ERX-11-1', 'SRA', 23251430, 31659364),
(1, 'treatment: ESR1 Coregulator Binding Site InhibitorsXx-xXtissue: ZR75', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, 'ZR75 cells treated with ERX+E2', 'Total RNA samples about 1-2 mg were used for sequencing library preparation according to Illumina TruSeq Total RNA Sample Preparation Guide (Illumina Cat. #: RS-122-2201). Each library was bar-coded and then pooled for cluster generation and sequencing run with 100bp single-end (SE) sequencing protocol.', 'GSM1963026', 'Illumina HiSeq 2000', 'Aug 18 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04315698Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1462168', 'GSE75664', 'ZR75 breast cancer cells', 'Public on Aug 18 2017', 'Dec 03 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1963nnn/GSM1963026/suppl/GSM1963026_Sample_ERX-11_2_Genes_ReadCount.txt.gz', '', 9606, 'ZR75-ERX-11-2', 'SRA', 29850641, 33035338),
(1, 'cell line: MDA-MB-231Xx-xXgenotype: Control cellsXx-xXchip antibody: none', 'Yonsei-ro 50-1', 'Seoul', 'South Korea', '', 'kimhp@yuhs.ac', 'Yonsei University', '', 'Hyoung-Pyo,,Kim', '82-2-2228-1842', '120-752', 'Reads were quality trimmed and filtered using the NGS QC toolkit v2.3 to remove reads containing low-quality bases.Xx-xXAfter preprocessing, the RNA sequenced reads were mapped to hg19 genome using RSEM with bowtie2 v2.0.0-beta7.Xx-xXChIP sequence reads were mapped to T.vaginalis genome using bowtie2 v2.0.0-beta7.Xx-xXFPKM (fragments per kilobase of exon per million fragments mapped) for RNA-Seq were calculated using RSEM.Xx-xXgenome build: hg19Xx-xXprocessed data files format and content: RNA-Seq wig files were generated using RSEM.', 0, '', 'Total RNA was isolated using RiboEx column (GeneAll Biotechnology).Xx-xXRNA-seq and ChIP-seq libraries were prepared for sequencing using standard Illumina protocols', 'GSM2323834', 'Illumina HiSeq 2000', 'Sep 22 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05792868Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2182192', 'GSE87169', 'Breast cancer cell line', 'Public on Sep 22 2017', 'Sep 21 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2323nnn/GSM2323834/suppl/GSM2323834_RNA-Seq_control.wig.gz', '', 9606, 'RNA-Seq control', 'SRA', 12176349, 14059888),
(1, 'cell line: MDA-MB-231Xx-xXgenotype: SATB1-depleted cellsXx-xXchip antibody: none', 'Yonsei-ro 50-1', 'Seoul', 'South Korea', '', 'kimhp@yuhs.ac', 'Yonsei University', '', 'Hyoung-Pyo,,Kim', '82-2-2228-1842', '120-752', 'Reads were quality trimmed and filtered using the NGS QC toolkit v2.3 to remove reads containing low-quality bases.Xx-xXAfter preprocessing, the RNA sequenced reads were mapped to hg19 genome using RSEM with bowtie2 v2.0.0-beta7.Xx-xXChIP sequence reads were mapped to T.vaginalis genome using bowtie2 v2.0.0-beta7.Xx-xXFPKM (fragments per kilobase of exon per million fragments mapped) for RNA-Seq were calculated using RSEM.Xx-xXgenome build: hg19Xx-xXprocessed data files format and content: RNA-Seq wig files were generated using RSEM.', 0, '', 'Total RNA was isolated using RiboEx column (GeneAll Biotechnology).Xx-xXRNA-seq and ChIP-seq libraries were prepared for sequencing using standard Illumina protocols', 'GSM2323835', 'Illumina HiSeq 2000', 'Sep 22 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05792867Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2182193', 'GSE87169', 'Breast cancer cell line', 'Public on Sep 22 2017', 'Sep 21 2016', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2323nnn/GSM2323835/suppl/GSM2323835_RNA-Seq_shSATB1.wig.gz', '', 9606, 'RNA-Seq shSATB1', 'SRA', 13326032, 15496328),
(1, 'cell line: MCF7Xx-xXtissue: breast cancerXx-xXseeding density: 0.6 million per 6cm tissue culture dishXx-xXtreatment: 24h hypoxia', '14 Medical Drive, #12-01', 'Singapore', 'Singapore', '', '', 'Cancer Science Institute of Singapore', '', 'Jia,,Li', '', '117599', 'RNA-seq aligned to hg19 genome and UCSC reference transcript using STAR (version 2.4.0).Xx-xXBigwig generated after normalization.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: Bigwig', 0, '', 'Total RNA was extracted from cells using the RNeasy Mini kit (Qiagen).Xx-xXBGI RNA-Seq library construction (90bp paired-end sequencing)', 'GSM2561535', 'Illumina HiSeq 2000', 'Jun 27 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06675649Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2700446', 'GSE97317', 'Human breast cancer', 'Public on Jun 27 2017', 'Apr 03 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2561nnn/GSM2561535/suppl/GSM2561535_hypoxia_24h.bw', '', 9606, 'Hypoxia 24h', 'SRA', 104165227, 119402180),
(1, 'subtype: Basal-likeXx-xXspecies: Human', '145 Gwanggyo-ro, Yeongtong-gu,', 'Suwon-si,', 'South Korea', '', 'jasonsjkim@snu.ac.kr', 'Nano-Bio Medicine Research Center', '', 'Seong-Jin,,Kim', '', '443-270', 'An Illumina platform (Illumina) was used to analyze transcriptomes with 90 bp paired-end library.Xx-xXIllumina Hiseq software used for basecalling.Xx-xXSamples were paired-end sequenced with the Illumina HiSeq 2000.Xx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: text files include FPKM values for each Sample.', 0, '', 'Total RNAs of each cell were isolated using the TRIzol reagent for RNA sequencing following manufacturer’s instructions. The total RNAs were treated with DNase I, purified with miRNeasy Mini Kit and subsequently quality checked using an Agilent 2100 Bioanalyzer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2695488', 'Illumina HiSeq 2000', 'Aug 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07327797Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2987428', 'GSE100878', 'Breast cancer cell lines', 'Public on Aug 09 2017', 'Jul 06 2017', 'NONE', '', 9606, 'Hs578T', 'SRA', 19879435, 21529485),
(1, 'subtype: LuminalXx-xXspecies: Human', '145 Gwanggyo-ro, Yeongtong-gu,', 'Suwon-si,', 'South Korea', '', 'jasonsjkim@snu.ac.kr', 'Nano-Bio Medicine Research Center', '', 'Seong-Jin,,Kim', '', '443-270', 'An Illumina platform (Illumina) was used to analyze transcriptomes with 90 bp paired-end library.Xx-xXIllumina Hiseq software used for basecalling.Xx-xXSamples were paired-end sequenced with the Illumina HiSeq 2000.Xx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: text files include FPKM values for each Sample.', 0, '', 'Total RNAs of each cell were isolated using the TRIzol reagent for RNA sequencing following manufacturer’s instructions. The total RNAs were treated with DNase I, purified with miRNeasy Mini Kit and subsequently quality checked using an Agilent 2100 Bioanalyzer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2695489', 'Illumina HiSeq 2000', 'Aug 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07327796Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2987429', 'GSE100878', 'Breast cancer cell lines', 'Public on Aug 09 2017', 'Jul 06 2017', 'NONE', '', 9606, 'MCF-7', 'SRA', 24410508, 27681356),
(1, 'subtype: Basal-likeXx-xXspecies: Human', '145 Gwanggyo-ro, Yeongtong-gu,', 'Suwon-si,', 'South Korea', '', 'jasonsjkim@snu.ac.kr', 'Nano-Bio Medicine Research Center', '', 'Seong-Jin,,Kim', '', '443-270', 'An Illumina platform (Illumina) was used to analyze transcriptomes with 90 bp paired-end library.Xx-xXIllumina Hiseq software used for basecalling.Xx-xXSamples were paired-end sequenced with the Illumina HiSeq 2000.Xx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: text files include FPKM values for each Sample.', 0, '', 'Total RNAs of each cell were isolated using the TRIzol reagent for RNA sequencing following manufacturer’s instructions. The total RNAs were treated with DNase I, purified with miRNeasy Mini Kit and subsequently quality checked using an Agilent 2100 Bioanalyzer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2695490', 'Illumina HiSeq 2000', 'Aug 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07327795Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2987430', 'GSE100878', 'Breast cancer cell lines', 'Public on Aug 09 2017', 'Jul 06 2017', 'NONE', '', 9606, 'MDA-MB-231', 'SRA', 33851771, 37826821),
(1, 'subtype: Basal-likeXx-xXspecies: Human', '145 Gwanggyo-ro, Yeongtong-gu,', 'Suwon-si,', 'South Korea', '', 'jasonsjkim@snu.ac.kr', 'Nano-Bio Medicine Research Center', '', 'Seong-Jin,,Kim', '', '443-270', 'An Illumina platform (Illumina) was used to analyze transcriptomes with 90 bp paired-end library.Xx-xXIllumina Hiseq software used for basecalling.Xx-xXSamples were paired-end sequenced with the Illumina HiSeq 2000.Xx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: text files include FPKM values for each Sample.', 0, '', 'Total RNAs of each cell were isolated using the TRIzol reagent for RNA sequencing following manufacturer’s instructions. The total RNAs were treated with DNase I, purified with miRNeasy Mini Kit and subsequently quality checked using an Agilent 2100 Bioanalyzer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2695491', 'Illumina HiSeq 2000', 'Aug 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07327794Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2987431', 'GSE100878', 'Breast cancer cell lines', 'Public on Aug 09 2017', 'Jul 06 2017', 'NONE', '', 9606, 'MDA-MB-435', 'SRA', 30839764, 34322509),
(1, 'subtype: LuminalXx-xXspecies: Human', '145 Gwanggyo-ro, Yeongtong-gu,', 'Suwon-si,', 'South Korea', '', 'jasonsjkim@snu.ac.kr', 'Nano-Bio Medicine Research Center', '', 'Seong-Jin,,Kim', '', '443-270', 'An Illumina platform (Illumina) was used to analyze transcriptomes with 90 bp paired-end library.Xx-xXIllumina Hiseq software used for basecalling.Xx-xXSamples were paired-end sequenced with the Illumina HiSeq 2000.Xx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: text files include FPKM values for each Sample.', 0, '', 'Total RNAs of each cell were isolated using the TRIzol reagent for RNA sequencing following manufacturer’s instructions. The total RNAs were treated with DNase I, purified with miRNeasy Mini Kit and subsequently quality checked using an Agilent 2100 Bioanalyzer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2695492', 'Illumina HiSeq 2000', 'Aug 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07327793Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2987432', 'GSE100878', 'Breast cancer cell lines', 'Public on Aug 09 2017', 'Jul 06 2017', 'NONE', '', 9606, 'T47D', 'SRA', 30187916, 33451270),
(1, 'subtype: LuminalXx-xXspecies: Human', '145 Gwanggyo-ro, Yeongtong-gu,', 'Suwon-si,', 'South Korea', '', 'jasonsjkim@snu.ac.kr', 'Nano-Bio Medicine Research Center', '', 'Seong-Jin,,Kim', '', '443-270', 'An Illumina platform (Illumina) was used to analyze transcriptomes with 90 bp paired-end library.Xx-xXIllumina Hiseq software used for basecalling.Xx-xXSamples were paired-end sequenced with the Illumina HiSeq 2000.Xx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: text files include FPKM values for each Sample.', 0, '', 'Total RNAs of each cell were isolated using the TRIzol reagent for RNA sequencing following manufacturer’s instructions. The total RNAs were treated with DNase I, purified with miRNeasy Mini Kit and subsequently quality checked using an Agilent 2100 Bioanalyzer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2695493', 'Illumina HiSeq 2000', 'Aug 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07327792Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2987433', 'GSE100878', 'Breast cancer cell lines', 'Public on Aug 09 2017', 'Jul 06 2017', 'NONE', '', 9606, 'ZR-75B', 'SRA', 35379328, 39346415),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: heavy polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871734', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118187Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436519', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871734/suppl/GSM2871734_1.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamR_NS_HD', 'SRA', 53884961, 60265309),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: light polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871735', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118186Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436520', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871735/suppl/GSM2871735_2.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamR_NS_LD', 'SRA', 55512098, 68469631),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: total mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871736', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118185Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436521', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871736/suppl/GSM2871736_3.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamR_NS_T', 'SRA', 52576169, 60690807),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: heavy polysome mRNAXx-xXgenotype: eIF4E-silencedXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871737', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118184Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436522', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871737/suppl/GSM2871737_4.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamR_4E_HD', 'SRA', 49827802, 56697725),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: light polysome mRNAXx-xXgenotype: eIF4E-silencedXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871738', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118183Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436523', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871738/suppl/GSM2871738_5.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamR_4E_LD', 'SRA', 45583306, 54770196),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: total mRNAXx-xXgenotype: eIF4E-silencedXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871739', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118182Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436524', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871739/suppl/GSM2871739_6.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamR_4E_T', 'SRA', 52376302, 62035520),
(1, 'tissue: tamoxifen sensitive MCF7 breast cancer cellsXx-xXextraction: heavy polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871740', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118181Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436525', 'GSE107590', 'MCF7 tamoxifen-sensitive breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871740/suppl/GSM2871740_7.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamS_NS_HD', 'SRA', 41032625, 51069297),
(1, 'tissue: tamoxifen sensitive MCF7 breast cancer cellsXx-xXextraction: light polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871741', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118180Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436526', 'GSE107590', 'MCF7 tamoxifen-sensitive breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871741/suppl/GSM2871741_8.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamS_NS_LD', 'SRA', 43719755, 52485361),
(1, 'tissue: tamoxifen sensitive MCF7 breast cancer cellsXx-xXextraction: total mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 1', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871742', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118179Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436527', 'GSE107590', 'MCF7 tamoxifen-sensitive breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871742/suppl/GSM2871742_9.counts.rRNArem.dups.txt.gz', '', 9606, 'R1_TamS_NS_T', 'SRA', 43459894, 53769217),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: heavy polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871743', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118178Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436528', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871743/suppl/GSM2871743_10.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamR_NS_HD', 'SRA', 50053580, 58456975),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: light polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871744', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118195Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436529', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871744/suppl/GSM2871744_11.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamR_NS_LD', 'SRA', 45068890, 53999487);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: total mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871745', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118194Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436530', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871745/suppl/GSM2871745_12.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamR_NS_T', 'SRA', 53748573, 64412614),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: heavy polysome mRNAXx-xXgenotype: eIF4E-silencedXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871746', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118193Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436531', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871746/suppl/GSM2871746_13.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamR_4E_HD', 'SRA', 52651007, 60886636),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: light polysome mRNAXx-xXgenotype: eIF4E-silencedXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871747', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118192Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436532', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871747/suppl/GSM2871747_14.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamR_4E_LD', 'SRA', 48409579, 56697233),
(1, 'tissue: selected tamoxifen resistant MCF7 breast cancer cellsXx-xXextraction: total mRNAXx-xXgenotype: eIF4E-silencedXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871748', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118191Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436533', 'GSE107590', 'MCF7 tamoxifen-resistant breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871748/suppl/GSM2871748_15.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamR_4E_T', 'SRA', 51786530, 60543304),
(1, 'tissue: tamoxifen sensitive MCF7 breast cancer cellsXx-xXextraction: heavy polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871749', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118190Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436534', 'GSE107590', 'MCF7 tamoxifen-sensitive breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871749/suppl/GSM2871749_16.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamS_NS_HD', 'SRA', 71629979, 80319677),
(1, 'tissue: tamoxifen sensitive MCF7 breast cancer cellsXx-xXextraction: light polysome mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871750', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118189Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436535', 'GSE107590', 'MCF7 tamoxifen-sensitive breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871750/suppl/GSM2871750_17.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamS_NS_LD', 'SRA', 52758142, 70998744),
(1, 'tissue: tamoxifen sensitive MCF7 breast cancer cellsXx-xXextraction: total mRNAXx-xXgenotype: non-silencingXx-xXreplicate: replicate 2', '550 First Ave, MSB 238', 'New York', 'USA', 'Microbiology', 'robert.schneider@nyumc.org', 'NYU School of Medicine', '', 'Robert,J,Schneider', '-6480', '10016', 'Sequenced reads were trimmed using trimmomatic v0.33Xx-xXReads were aligned using tophat v2.0.9/bowtie v1.0.0.0 to UCSC hg19 genome using UCSC hg19 transcriptome annotationsXx-xXAligned reads were quantified using htseq version 0.6.1p1Xx-xXDifferential analysis was performed using voom normalization / Limma linear modelingXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include count values for each gene of each sample', 0, '', 'Lysates were either purified (total mRNA) or subjected to polysome profiling.  Polysome profiling samples pertaining to 2-3 ribsomes (light mRNA) were purified or >4 ribosomes were purified (heavy mRNA).  All samples were purified using the RNeasy Mini Kit (Qiagen) as per manufacture instructions.Xx-xXRNA libraries were prepared for sequencing using poly A enrichment standard Illumina protocols', 'GSM2871751', 'Illumina HiSeq 2000', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08118188Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3436536', 'GSE107590', 'MCF7 tamoxifen-sensitive breast cancer cells', 'Public on Dec 21 2017', 'Dec 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2871nnn/GSM2871751/suppl/GSM2871751_18.counts.rRNArem.dups.txt.gz', '', 9606, 'R2_TamS_NS_T', 'SRA', 44955300, 52770054),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: cellular RNAXx-xXtreatment: no treatment', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443399', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198257Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458027', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_cell_ctrl_1', 'SRA', 849162, 5322969),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: cellular RNAXx-xXtreatment: no treatment', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443400', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198256Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458028', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_cell_ctrl_2', 'SRA', 1048238, 6497007),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: cellular RNAXx-xXtreatment: 0.1% Triton X-100 and micrococcal nuclease', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443401', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198255Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458029', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_cell_triton_1', 'SRA', 312912, 7172633),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: cellular RNAXx-xXtreatment: 0.1% Triton X-100 and micrococcal nuclease', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443402', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198254Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458030', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_cell_triton_2', 'SRA', 477804, 8959563),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: cellular RNAXx-xXtreatment: micrococcal nuclease only', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443403', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198253Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458031', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_cell_mnase_1', 'SRA', 721897, 6538985),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: cellular RNAXx-xXtreatment: micrococcal nuclease only', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443404', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198252Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458032', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_cell_mnase_2', 'SRA', 812156, 7173197),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: exosome RNAXx-xXtreatment: no treatment', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443405', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198251Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458033', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_exo_ctrl_1', 'SRA', 767936, 6482630),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: exosome RNAXx-xXtreatment: no treatment', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443406', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198250Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458034', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_exo_ctrl_2', 'SRA', 799190, 6874814),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: exosome RNAXx-xXtreatment: 0.1% Triton X-100 and micrococcal nuclease', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443407', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198249Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458035', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_exo_triton_1', 'SRA', 633692, 5461558),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: exosome RNAXx-xXtreatment: 0.1% Triton X-100 and micrococcal nuclease', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443408', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198248Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458036', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_exo_triton_2', 'SRA', 906005, 7737565),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: exosome RNAXx-xXtreatment: micrococcal nuclease only', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443409', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198247Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458037', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_exo_mnase_1', 'SRA', 753657, 6208828),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-cultureXx-xXfraction: exosome RNAXx-xXtreatment: micrococcal nuclease only', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, '', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA or cellular RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB).  Fragmentation step was skipped for exosome RNA.', 'GSM2443410', 'Illumina HiSeq 2500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198246Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458038', 'GSE93071Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'cocx_exo_mnase_2', 'SRA', 807901, 6697108),
(1, 'tissue: CAT-SKL treated Triple negative breast cancer cell lineXx-xXcell type: triple negative breast cancer cell line MDA-MB-468Xx-xXdna digestion: not applicableXx-xXtreatment: anti-oxidant CAT-SKL', '4100 John R', 'Detroit', 'USA', 'Oncology', '', 'Karmanos Cancer Institute & Wayne State University', '', 'Aliccia,,Bollig-Fischer', '', '48201', 'PVUseq and RNAseq: Basecalls were performed using Illumina\'s CASAVA vesrsion 1.8.2Xx-xXPVUseq and RNAseq: quality of the sequenced reads were assessd using FastQC from Babraham BioinformaticsXx-xXPVUseq and RNAseq: reads were aligned to Human Genome Consortium\'s reference human genome (hg19) using Novoalign V2.08.03Xx-xXPVUseq: PVU-cognate sequences were determined using the PERL script obtained from Sun et al, Cell Rep 2013, 3:567-576. differential 5hmC levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXRNAseq: differential expression levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited, significant fold changes for PVU and RNA data from DESeq pipeline and correlation statistic computed using a min-max normalization and the distance to the first diagonal (x=y)', 0, '', 'DNA extraction for PVU-Seq: DNeasy Blood and tissue kit (Qiagen, Valencia, CA) was used to extract genomic DNA from the fresh frozen tissues and MDA-MB-468 cells. RNaseA treatment was done according to the manufacturer’s instructions [4µL of RNaseA (100mg/mL)] to obtain RNA free genomic DNA. Extraction of the RNA was done using the RNeasy plus kit (Qiagen, Valencia, CA) according to manufacturer\'s instructionsXx-xXThruPLEX-FD prep kit (Rubicon Genomics, Ann Arbor, MI) was used to prepare DNA libraries for PVU-Seq. Illumina TruSeq (Low Sample) RNA library preparation was used according to the manufacturer\'s instructions', 'GSM2784646', 'Illumina HiSeq 2500', 'Dec 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07645527Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3186133', 'GSE103850', 'CAT-SKL treated Triple negative breast cancer cell line', 'Public on Dec 13 2017', 'Sep 14 2017', 'NONE', '', 9606, 'CAT1_RNA', 'SRA', 26078027, 28750533),
(1, 'tissue: CAT-SKL treated Triple negative breast cancer cell lineXx-xXcell type: triple negative breast cancer cell line MDA-MB-468Xx-xXdna digestion: not applicableXx-xXtreatment: anti-oxidant CAT-SKL', '4100 John R', 'Detroit', 'USA', 'Oncology', '', 'Karmanos Cancer Institute & Wayne State University', '', 'Aliccia,,Bollig-Fischer', '', '48201', 'PVUseq and RNAseq: Basecalls were performed using Illumina\'s CASAVA vesrsion 1.8.2Xx-xXPVUseq and RNAseq: quality of the sequenced reads were assessd using FastQC from Babraham BioinformaticsXx-xXPVUseq and RNAseq: reads were aligned to Human Genome Consortium\'s reference human genome (hg19) using Novoalign V2.08.03Xx-xXPVUseq: PVU-cognate sequences were determined using the PERL script obtained from Sun et al, Cell Rep 2013, 3:567-576. differential 5hmC levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXRNAseq: differential expression levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited, significant fold changes for PVU and RNA data from DESeq pipeline and correlation statistic computed using a min-max normalization and the distance to the first diagonal (x=y)', 0, '', 'DNA extraction for PVU-Seq: DNeasy Blood and tissue kit (Qiagen, Valencia, CA) was used to extract genomic DNA from the fresh frozen tissues and MDA-MB-468 cells. RNaseA treatment was done according to the manufacturer’s instructions [4µL of RNaseA (100mg/mL)] to obtain RNA free genomic DNA. Extraction of the RNA was done using the RNeasy plus kit (Qiagen, Valencia, CA) according to manufacturer\'s instructionsXx-xXThruPLEX-FD prep kit (Rubicon Genomics, Ann Arbor, MI) was used to prepare DNA libraries for PVU-Seq. Illumina TruSeq (Low Sample) RNA library preparation was used according to the manufacturer\'s instructions', 'GSM2784647', 'Illumina HiSeq 2500', 'Dec 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07645526Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3186134', 'GSE103850', 'CAT-SKL treated Triple negative breast cancer cell line', 'Public on Dec 13 2017', 'Sep 14 2017', 'NONE', '', 9606, 'CAT2_RNA', 'SRA', 21505972, 23795758),
(1, 'tissue: CAT-SKL treated Triple negative breast cancer cell lineXx-xXcell type: triple negative breast cancer cell line MDA-MB-468Xx-xXdna digestion: not applicableXx-xXtreatment: anti-oxidant CAT-SKL', '4100 John R', 'Detroit', 'USA', 'Oncology', '', 'Karmanos Cancer Institute & Wayne State University', '', 'Aliccia,,Bollig-Fischer', '', '48201', 'PVUseq and RNAseq: Basecalls were performed using Illumina\'s CASAVA vesrsion 1.8.2Xx-xXPVUseq and RNAseq: quality of the sequenced reads were assessd using FastQC from Babraham BioinformaticsXx-xXPVUseq and RNAseq: reads were aligned to Human Genome Consortium\'s reference human genome (hg19) using Novoalign V2.08.03Xx-xXPVUseq: PVU-cognate sequences were determined using the PERL script obtained from Sun et al, Cell Rep 2013, 3:567-576. differential 5hmC levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXRNAseq: differential expression levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited, significant fold changes for PVU and RNA data from DESeq pipeline and correlation statistic computed using a min-max normalization and the distance to the first diagonal (x=y)', 0, '', 'DNA extraction for PVU-Seq: DNeasy Blood and tissue kit (Qiagen, Valencia, CA) was used to extract genomic DNA from the fresh frozen tissues and MDA-MB-468 cells. RNaseA treatment was done according to the manufacturer’s instructions [4µL of RNaseA (100mg/mL)] to obtain RNA free genomic DNA. Extraction of the RNA was done using the RNeasy plus kit (Qiagen, Valencia, CA) according to manufacturer\'s instructionsXx-xXThruPLEX-FD prep kit (Rubicon Genomics, Ann Arbor, MI) was used to prepare DNA libraries for PVU-Seq. Illumina TruSeq (Low Sample) RNA library preparation was used according to the manufacturer\'s instructions', 'GSM2784648', 'Illumina HiSeq 2500', 'Dec 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07645525Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3186135', 'GSE103850', 'CAT-SKL treated Triple negative breast cancer cell line', 'Public on Dec 13 2017', 'Sep 14 2017', 'NONE', '', 9606, 'CAT3_RNA', 'SRA', 30474854, 33596466),
(1, 'tissue: Vehicle treated Triple negative breast cancer cell lineXx-xXcell type: triple negative breast cancer cell line MDA-MB-468Xx-xXdna digestion: not applicableXx-xXtreatment: vehicle', '4100 John R', 'Detroit', 'USA', 'Oncology', '', 'Karmanos Cancer Institute & Wayne State University', '', 'Aliccia,,Bollig-Fischer', '', '48201', 'PVUseq and RNAseq: Basecalls were performed using Illumina\'s CASAVA vesrsion 1.8.2Xx-xXPVUseq and RNAseq: quality of the sequenced reads were assessd using FastQC from Babraham BioinformaticsXx-xXPVUseq and RNAseq: reads were aligned to Human Genome Consortium\'s reference human genome (hg19) using Novoalign V2.08.03Xx-xXPVUseq: PVU-cognate sequences were determined using the PERL script obtained from Sun et al, Cell Rep 2013, 3:567-576. differential 5hmC levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXRNAseq: differential expression levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited, significant fold changes for PVU and RNA data from DESeq pipeline and correlation statistic computed using a min-max normalization and the distance to the first diagonal (x=y)', 0, '', 'DNA extraction for PVU-Seq: DNeasy Blood and tissue kit (Qiagen, Valencia, CA) was used to extract genomic DNA from the fresh frozen tissues and MDA-MB-468 cells. RNaseA treatment was done according to the manufacturer’s instructions [4µL of RNaseA (100mg/mL)] to obtain RNA free genomic DNA. Extraction of the RNA was done using the RNeasy plus kit (Qiagen, Valencia, CA) according to manufacturer\'s instructionsXx-xXThruPLEX-FD prep kit (Rubicon Genomics, Ann Arbor, MI) was used to prepare DNA libraries for PVU-Seq. Illumina TruSeq (Low Sample) RNA library preparation was used according to the manufacturer\'s instructions', 'GSM2784649', 'Illumina HiSeq 2500', 'Dec 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07645522Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3186136', 'GSE103850', 'Vehicle treated Triple negative breast cancer cell line', 'Public on Dec 13 2017', 'Sep 14 2017', 'NONE', '', 9606, 'UT1_RNA', 'SRA', 21738227, 23747769),
(1, 'tissue: Vehicle treated Triple negative breast cancer cell lineXx-xXcell type: triple negative breast cancer cell line MDA-MB-468Xx-xXdna digestion: not applicableXx-xXtreatment: vehicle', '4100 John R', 'Detroit', 'USA', 'Oncology', '', 'Karmanos Cancer Institute & Wayne State University', '', 'Aliccia,,Bollig-Fischer', '', '48201', 'PVUseq and RNAseq: Basecalls were performed using Illumina\'s CASAVA vesrsion 1.8.2Xx-xXPVUseq and RNAseq: quality of the sequenced reads were assessd using FastQC from Babraham BioinformaticsXx-xXPVUseq and RNAseq: reads were aligned to Human Genome Consortium\'s reference human genome (hg19) using Novoalign V2.08.03Xx-xXPVUseq: PVU-cognate sequences were determined using the PERL script obtained from Sun et al, Cell Rep 2013, 3:567-576. differential 5hmC levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXRNAseq: differential expression levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited, significant fold changes for PVU and RNA data from DESeq pipeline and correlation statistic computed using a min-max normalization and the distance to the first diagonal (x=y)', 0, '', 'DNA extraction for PVU-Seq: DNeasy Blood and tissue kit (Qiagen, Valencia, CA) was used to extract genomic DNA from the fresh frozen tissues and MDA-MB-468 cells. RNaseA treatment was done according to the manufacturer’s instructions [4µL of RNaseA (100mg/mL)] to obtain RNA free genomic DNA. Extraction of the RNA was done using the RNeasy plus kit (Qiagen, Valencia, CA) according to manufacturer\'s instructionsXx-xXThruPLEX-FD prep kit (Rubicon Genomics, Ann Arbor, MI) was used to prepare DNA libraries for PVU-Seq. Illumina TruSeq (Low Sample) RNA library preparation was used according to the manufacturer\'s instructions', 'GSM2784650', 'Illumina HiSeq 2500', 'Dec 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07645523Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3186137', 'GSE103850', 'Vehicle treated Triple negative breast cancer cell line', 'Public on Dec 13 2017', 'Sep 14 2017', 'NONE', '', 9606, 'UT2_RNA', 'SRA', 32655200, 35658494);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: Vehicle treated Triple negative breast cancer cell lineXx-xXcell type: triple negative breast cancer cell line MDA-MB-468Xx-xXdna digestion: not applicableXx-xXtreatment: vehicle', '4100 John R', 'Detroit', 'USA', 'Oncology', '', 'Karmanos Cancer Institute & Wayne State University', '', 'Aliccia,,Bollig-Fischer', '', '48201', 'PVUseq and RNAseq: Basecalls were performed using Illumina\'s CASAVA vesrsion 1.8.2Xx-xXPVUseq and RNAseq: quality of the sequenced reads were assessd using FastQC from Babraham BioinformaticsXx-xXPVUseq and RNAseq: reads were aligned to Human Genome Consortium\'s reference human genome (hg19) using Novoalign V2.08.03Xx-xXPVUseq: PVU-cognate sequences were determined using the PERL script obtained from Sun et al, Cell Rep 2013, 3:567-576. differential 5hmC levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXRNAseq: differential expression levels were calculated in R using DESeq from Bioconductor; the UCSC transcriptome (hg19) was used as annotation referenceXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited, significant fold changes for PVU and RNA data from DESeq pipeline and correlation statistic computed using a min-max normalization and the distance to the first diagonal (x=y)', 0, '', 'DNA extraction for PVU-Seq: DNeasy Blood and tissue kit (Qiagen, Valencia, CA) was used to extract genomic DNA from the fresh frozen tissues and MDA-MB-468 cells. RNaseA treatment was done according to the manufacturer’s instructions [4µL of RNaseA (100mg/mL)] to obtain RNA free genomic DNA. Extraction of the RNA was done using the RNeasy plus kit (Qiagen, Valencia, CA) according to manufacturer\'s instructionsXx-xXThruPLEX-FD prep kit (Rubicon Genomics, Ann Arbor, MI) was used to prepare DNA libraries for PVU-Seq. Illumina TruSeq (Low Sample) RNA library preparation was used according to the manufacturer\'s instructions', 'GSM2784651', 'Illumina HiSeq 2500', 'Dec 13 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07645524Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3186138', 'GSE103850', 'Vehicle treated Triple negative breast cancer cell line', 'Public on Dec 13 2017', 'Sep 14 2017', 'NONE', '', 9606, 'UT3_RNA', 'SRA', 19320051, 21099288),
(1, 'cell line: MDA-MB-231 cellsXx-xXagent: inhibitor', 'No. 555 Zuchongzhi Road', 'Shanghai', 'China', '', 'cluo@simm.ac.cn', 'Shanghai Institute of Materia Medica', '', 'Cheng,,Luo', '-50806535', '201203', 'tophat2 with default parametersXx-xXsamtools to sort and convert the file formatXx-xXcufflinks with default parametersXx-xXcuffmerge with default parametersXx-xXcuffdiff with default parametersXx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: gene_exp.diff, generated by Cuffdiff, normalized expression values for each sample type', 0, 'MDA-MB-231 cells treated with inhibitor 100uM repeat 1, R1Xx-xXcpd1-100-1-GCCAAT_L006', '2100 Bioanalyzer for quality check, DNaseI (Takara, Japan) for digestion at 37 C and 30 minsXx-xX100ng purified RNA, NEBNext® UltraTM RNA Library Prep Kit for Illumina (NEB, USA)', 'GSM2843828', 'Illumina HiSeq 2500', 'Dec 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07987669Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3369834', 'GSE106605', 'breast cancer', 'Public on Dec 07 2017', 'Nov 07 2017', 'NONE', '', 9606, 'inhibitor 100uM repeat 1', 'SRA', 15856683, 18804247),
(1, 'cell line: MDA-MB-231 cellsXx-xXagent: inhibitor', 'No. 555 Zuchongzhi Road', 'Shanghai', 'China', '', 'cluo@simm.ac.cn', 'Shanghai Institute of Materia Medica', '', 'Cheng,,Luo', '-50806535', '201203', 'tophat2 with default parametersXx-xXsamtools to sort and convert the file formatXx-xXcufflinks with default parametersXx-xXcuffmerge with default parametersXx-xXcuffdiff with default parametersXx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: gene_exp.diff, generated by Cuffdiff, normalized expression values for each sample type', 0, 'MDA-MB-231 cells treated with inhibitor 100uM repeat 2, R1Xx-xXcpd1-100-2-GATCAG_L006', '2100 Bioanalyzer for quality check, DNaseI (Takara, Japan) for digestion at 37 C and 30 minsXx-xX100ng purified RNA, NEBNext® UltraTM RNA Library Prep Kit for Illumina (NEB, USA)', 'GSM2843829', 'Illumina HiSeq 2500', 'Dec 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07987668Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3369835', 'GSE106605', 'breast cancer', 'Public on Dec 07 2017', 'Nov 07 2017', 'NONE', '', 9606, 'inhibitor 100uM repeat 2', 'SRA', 16234748, 19297311),
(1, 'cell line: MDA-MB-231 cellsXx-xXagent: DMSO', 'No. 555 Zuchongzhi Road', 'Shanghai', 'China', '', 'cluo@simm.ac.cn', 'Shanghai Institute of Materia Medica', '', 'Cheng,,Luo', '-50806535', '201203', 'tophat2 with default parametersXx-xXsamtools to sort and convert the file formatXx-xXcufflinks with default parametersXx-xXcuffmerge with default parametersXx-xXcuffdiff with default parametersXx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: gene_exp.diff, generated by Cuffdiff, normalized expression values for each sample type', 0, 'MDA-MB-231 cells treated with DMSO repeat 1, R1Xx-xXcpd1-con-1-ATCACG_L006', '2100 Bioanalyzer for quality check, DNaseI (Takara, Japan) for digestion at 37 C and 30 minsXx-xX100ng purified RNA, NEBNext® UltraTM RNA Library Prep Kit for Illumina (NEB, USA)', 'GSM2843830', 'Illumina HiSeq 2500', 'Dec 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07987667Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3369836', 'GSE106605', 'breast cancer', 'Public on Dec 07 2017', 'Nov 07 2017', 'NONE', '', 9606, 'DMSO repeat 1', 'SRA', 12843577, 15195326),
(1, 'cell line: MDA-MB-231 cellsXx-xXagent: DMSO', 'No. 555 Zuchongzhi Road', 'Shanghai', 'China', '', 'cluo@simm.ac.cn', 'Shanghai Institute of Materia Medica', '', 'Cheng,,Luo', '-50806535', '201203', 'tophat2 with default parametersXx-xXsamtools to sort and convert the file formatXx-xXcufflinks with default parametersXx-xXcuffmerge with default parametersXx-xXcuffdiff with default parametersXx-xXGenome_build: mm9Xx-xXSupplementary_files_format_and_content: gene_exp.diff, generated by Cuffdiff, normalized expression values for each sample type', 0, 'MDA-MB-231 cells treated with DMSO repeat 2, R1Xx-xXcpd1-con-2-ACAGTG_L006', '2100 Bioanalyzer for quality check, DNaseI (Takara, Japan) for digestion at 37 C and 30 minsXx-xX100ng purified RNA, NEBNext® UltraTM RNA Library Prep Kit for Illumina (NEB, USA)', 'GSM2843831', 'Illumina HiSeq 2500', 'Dec 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07987666Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3369837', 'GSE106605', 'breast cancer', 'Public on Dec 07 2017', 'Nov 07 2017', 'NONE', '', 9606, 'DMSO repeat 2', 'SRA', 17910782, 21197580),
(1, 'tissue type: ER+/PRA+/PRB- T47D cell modelsXx-xXer/pr status: ER+/PRA+/PRB-Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRA_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125090', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866015Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709945', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRA_Vehicle_RNAseq', 'SRA', 9140378, 11190388),
(1, 'tissue type: ER+/PRA+/PRB- T47D cell modelsXx-xXer/pr status: ER+/PRA+/PRB-Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRA_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125091', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866016Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709946', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRA_E2_RNAseq', 'SRA', 8759490, 10688717),
(1, 'tissue type: ER+/PRA+/PRB- T47D cell modelsXx-xXer/pr status: ER+/PRA+/PRB-Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRA_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125092', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866017Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709947', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRA_R5020_RNAseq', 'SRA', 15430494, 18022725),
(1, 'tissue type: ER+/PRA+/PRB- T47D cell modelsXx-xXer/pr status: ER+/PRA+/PRB-Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRA_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125093', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866018Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709948', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRA_E2+R5020_RNAseq', 'SRA', 15308974, 17804227),
(1, 'tissue type: ER+/PRA-/PRB+ T47D cell modelsXx-xXer/pr status: ER+/PRA-/PRB+Xx-xXdrug treatment: VehicleXx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRB_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125094', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866019Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709949', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRB_Vehicle_RNAseq', 'SRA', 15022102, 18109376),
(1, 'tissue type: ER+/PRA-/PRB+ T47D cell modelsXx-xXer/pr status: ER+/PRA-/PRB+Xx-xXdrug treatment: 10 nM EstradiolXx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRB_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125095', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866020Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709950', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRB_E2_RNAseq', 'SRA', 15527600, 18612822),
(1, 'tissue type: ER+/PRA-/PRB+ T47D cell modelsXx-xXer/pr status: ER+/PRA-/PRB+Xx-xXdrug treatment: 10 nM R5020Xx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRB_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125096', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866021Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709951', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRB_R5020_RNAseq', 'SRA', 14700987, 17856632),
(1, 'tissue type: ER+/PRA-/PRB+ T47D cell modelsXx-xXer/pr status: ER+/PRA-/PRB+Xx-xXdrug treatment: 10nM Estradiol + 10 nM R5020Xx-xXhormone exposure time: 12 Hours', '929 E 57th St', 'Chicago', 'USA', 'Ben May Department for Cancer Research', 'hari_singhal@dfci.harvard.edu', 'University of Chicago', 'Geoffrey L Greene', 'Hari,,Singhal', '', '60637', 'The FASTQ files were filtered and good quality reads were groomed and subsequently aligned to HG19 genome build using TopHat.Xx-xXRelative expression of transcripts were calculated by subjecting TopHat output (BAM files) to Cufflinks package.Xx-xXCufflinks assemblies obtained from Cufflinks package were merged using cuffmerge. For every model system, cufflink assemblies of each of the treatments were merged with the cufflinks assemblies of the respective vehicle-treated samples.Xx-xXThe output from Cuffmerge along with the corresponding TopHat output (BAM files) were subjected to Cuffdiff to calculate transcript expression.Xx-xXGenes that were up or down regulated by at least two fold were used for downstream analyses.Xx-xXGenome_build: HG19Xx-xXSupplementary_files_format_and_content: CSV includes RPKM values for each of the treatment conditions (E2, R5020 or E2+R5020) versus control treatment. Union of all estrogen and progestin-regulated genes for the respective samples is provided.', 0, 'Cell models RNA-seqXx-xXprocessed data file: T47D-PRB_RNAseq.csv', 'Hormone-treated cells were homogenized and total RNA was extracted using Qiagen RNAeasy kitXx-xXPolyA tail selection was performed on 250 ng of total RNA. RNA-seq libraries were prepared using NEBNext kit.Xx-xXDetails of library preparation strategy are provided in the materials and methods section (RNA expression and RNA-sequencing) of the paper.', 'GSM2125097', 'Illumina HiSeq 2500', 'Dec 31 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04866022Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1709952', 'GSE80358Xx-xXGSE80620', 'Breast cancer cells', 'Public on Dec 31 2017', 'Apr 16 2016', 'NONE', '', 9606, 'T47D_PRB_E2+R5020_RNAseq', 'SRA', 15089608, 18058607),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: SUM44Xx-xXestrogen: YesXx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'SUM44 WT replicate 1', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670883', 'Illumina HiSeq 2500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07246998Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922246', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'SUM44_WT_repl1', 'SRA', 44817152, 61325942),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: SUM44Xx-xXestrogen: YesXx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'SUM44 WT replicate 2', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670884', 'Illumina HiSeq 2500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07246997Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922247', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'SUM44_WT_repl2', 'SRA', 47937616, 63208580),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: SUM44Xx-xXestrogen: YesXx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'SUM44 WT replicate 3', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670885', 'Illumina HiSeq 2500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07246996Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922248', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'SUM44_WT_repl3', 'SRA', 49789782, 67656346),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: SUM44Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: Y537S', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'SUM44 LTED replicate 1', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670886', 'Illumina HiSeq 2500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07246995Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922249', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'SUM44_LTED_repl1', 'SRA', 57432549, 84790861),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: SUM44Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: Y537S', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'SUM44 LTED replicate 2', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670887', 'Illumina HiSeq 2500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07246994Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922250', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'SUM44_LTED_repl2', 'SRA', 58734760, 80592924),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: SUM44Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: Y537S', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'SUM44 LTED replicate 3', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670888', 'Illumina HiSeq 2500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07246993Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922251', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'SUM44_LTED_repl3', 'SRA', 51560255, 74424952),
(1, 'cell type: patient-derived xenograftXx-xXlabelling: Not LabelledXx-xXer-status: Negative', '12801 E. 17th Avenue', 'Aurora', 'USA', 'Division of Medical Oncology', '', 'University of Colorado School of Medicine', '', 'Peter,,Kabos', '', '80045', 'After demultiplexing, the resulting reads were trimmed with cutadapt to remove 3\' adaptor sequences and low quality 3\' bases (Q<10)Xx-xXThe trimmed reads were then aligned to both the human (hg19/GRCh37) and mouse (mm10) genomes using Tophat2Xx-xXReads were then assigned to either the human or mouse genome using disambiguate and ambiguous reads were discarded.Xx-xXUnambiguous reads were assigned to features using Rsubread and normalized read counts were produced using the rlog function in DESeq2.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text file containing normalized (log2) counts for each sample', 0, '', 'RNA was isolated from tumor samples using trizol followed by RNA cleanup using RNEeasy MinElute Cleanup kit (Qiagen), and RNA concentration was measured in a Nanodrop 2000 (Thermo Scientific)Xx-xXThe Genomics and Microarray core facility at the University of Colorado AMC performed RNA quality control using an Agilent 2100 Bioanalyzer, and prepared RNA-seq libraries using the Illumina TruSeq Stranded mRNA LT Sample Prep Kit.', 'GSM2787863', 'Illumina HiSeq 2500', 'Feb 14 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07671930Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3198463', 'GSE104020', 'breast cancer brain metastasis', 'Public on Feb 14 2018', 'Sep 19 2017', 'NONE', '', 9606, 'E22-1 PDX-P0', 'SRA', 44486456, 55824403),
(1, 'cell type: patient-derived xenograftXx-xXlabelling: LabelledXx-xXer-status: Negative', '12801 E. 17th Avenue', 'Aurora', 'USA', 'Division of Medical Oncology', '', 'University of Colorado School of Medicine', '', 'Peter,,Kabos', '', '80045', 'After demultiplexing, the resulting reads were trimmed with cutadapt to remove 3\' adaptor sequences and low quality 3\' bases (Q<10)Xx-xXThe trimmed reads were then aligned to both the human (hg19/GRCh37) and mouse (mm10) genomes using Tophat2Xx-xXReads were then assigned to either the human or mouse genome using disambiguate and ambiguous reads were discarded.Xx-xXUnambiguous reads were assigned to features using Rsubread and normalized read counts were produced using the rlog function in DESeq2.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text file containing normalized (log2) counts for each sample', 0, '', 'RNA was isolated from tumor samples using trizol followed by RNA cleanup using RNEeasy MinElute Cleanup kit (Qiagen), and RNA concentration was measured in a Nanodrop 2000 (Thermo Scientific)Xx-xXThe Genomics and Microarray core facility at the University of Colorado AMC performed RNA quality control using an Agilent 2100 Bioanalyzer, and prepared RNA-seq libraries using the Illumina TruSeq Stranded mRNA LT Sample Prep Kit.', 'GSM2787864', 'Illumina HiSeq 2500', 'Feb 14 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07671935Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3198464', 'GSE104020', 'breast cancer brain metastasis', 'Public on Feb 14 2018', 'Sep 19 2017', 'NONE', '', 9606, 'E22-1 PDX-P0-i1', 'SRA', 44648771, 58308766),
(1, 'cell type: patient-derived xenograftXx-xXlabelling: Not LabelledXx-xXer-status: Negative', '12801 E. 17th Avenue', 'Aurora', 'USA', 'Division of Medical Oncology', '', 'University of Colorado School of Medicine', '', 'Peter,,Kabos', '', '80045', 'After demultiplexing, the resulting reads were trimmed with cutadapt to remove 3\' adaptor sequences and low quality 3\' bases (Q<10)Xx-xXThe trimmed reads were then aligned to both the human (hg19/GRCh37) and mouse (mm10) genomes using Tophat2Xx-xXReads were then assigned to either the human or mouse genome using disambiguate and ambiguous reads were discarded.Xx-xXUnambiguous reads were assigned to features using Rsubread and normalized read counts were produced using the rlog function in DESeq2.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text file containing normalized (log2) counts for each sample', 0, '', 'RNA was isolated from tumor samples using trizol followed by RNA cleanup using RNEeasy MinElute Cleanup kit (Qiagen), and RNA concentration was measured in a Nanodrop 2000 (Thermo Scientific)Xx-xXThe Genomics and Microarray core facility at the University of Colorado AMC performed RNA quality control using an Agilent 2100 Bioanalyzer, and prepared RNA-seq libraries using the Illumina TruSeq Stranded mRNA LT Sample Prep Kit.', 'GSM2787865', 'Illumina HiSeq 2500', 'Feb 14 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07671934Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3198465', 'GSE104020', 'breast cancer brain metastasis', 'Public on Feb 14 2018', 'Sep 19 2017', 'NONE', '', 9606, 'CSF-1 PDX-P3', 'SRA', 46760003, 60104525),
(1, 'cell type: Cell LineXx-xXlabelling: Not LabelledXx-xXer-status: Negative', '12801 E. 17th Avenue', 'Aurora', 'USA', 'Division of Medical Oncology', '', 'University of Colorado School of Medicine', '', 'Peter,,Kabos', '', '80045', 'After demultiplexing, the resulting reads were trimmed with cutadapt to remove 3\' adaptor sequences and low quality 3\' bases (Q<10)Xx-xXThe trimmed reads were then aligned to both the human (hg19/GRCh37) and mouse (mm10) genomes using Tophat2Xx-xXReads were then assigned to either the human or mouse genome using disambiguate and ambiguous reads were discarded.Xx-xXUnambiguous reads were assigned to features using Rsubread and normalized read counts were produced using the rlog function in DESeq2.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text file containing normalized (log2) counts for each sample', 0, '', 'RNA was isolated from tumor samples using trizol followed by RNA cleanup using RNEeasy MinElute Cleanup kit (Qiagen), and RNA concentration was measured in a Nanodrop 2000 (Thermo Scientific)Xx-xXThe Genomics and Microarray core facility at the University of Colorado AMC performed RNA quality control using an Agilent 2100 Bioanalyzer, and prepared RNA-seq libraries using the Illumina TruSeq Stranded mRNA LT Sample Prep Kit.', 'GSM2787866', 'Illumina HiSeq 2500', 'Feb 14 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07671933Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3198466', 'GSE104020', 'breast cancer brain metastasis', 'Public on Feb 14 2018', 'Sep 19 2017', 'NONE', '', 9606, 'F2-7 Cells', 'SRA', 48734118, 56161442),
(1, 'cell type: patient-derived xenograftXx-xXlabelling: Not LabelledXx-xXer-status: Negative', '12801 E. 17th Avenue', 'Aurora', 'USA', 'Division of Medical Oncology', '', 'University of Colorado School of Medicine', '', 'Peter,,Kabos', '', '80045', 'After demultiplexing, the resulting reads were trimmed with cutadapt to remove 3\' adaptor sequences and low quality 3\' bases (Q<10)Xx-xXThe trimmed reads were then aligned to both the human (hg19/GRCh37) and mouse (mm10) genomes using Tophat2Xx-xXReads were then assigned to either the human or mouse genome using disambiguate and ambiguous reads were discarded.Xx-xXUnambiguous reads were assigned to features using Rsubread and normalized read counts were produced using the rlog function in DESeq2.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text file containing normalized (log2) counts for each sample', 0, '', 'RNA was isolated from tumor samples using trizol followed by RNA cleanup using RNEeasy MinElute Cleanup kit (Qiagen), and RNA concentration was measured in a Nanodrop 2000 (Thermo Scientific)Xx-xXThe Genomics and Microarray core facility at the University of Colorado AMC performed RNA quality control using an Agilent 2100 Bioanalyzer, and prepared RNA-seq libraries using the Illumina TruSeq Stranded mRNA LT Sample Prep Kit.', 'GSM2787867', 'Illumina HiSeq 2500', 'Feb 14 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07671932Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3198467', 'GSE104020', 'breast cancer brain metastasis', 'Public on Feb 14 2018', 'Sep 19 2017', 'NONE', '', 9606, 'F2-7 PDX-P5', 'SRA', 41571564, 53507552),
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: PBS', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+PBS rep1Xx-xXX18740_CAF.RNA.PBS.1.8.29.17_S1_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839374', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975471Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360142', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+PBS rep1', 'SRA', 25945267, 32181841),
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: PBS', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+PBS rep2Xx-xXX18741_CAF.RNA.PBS.2.8.29.17_S2_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839375', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975470Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360143', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+PBS rep2', 'SRA', 29426482, 36262196),
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: MCF10A EV', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+MCF10A EV rep1Xx-xXX18742_CAF.RNA.vec.EV.1.8.29.17_S3_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839376', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975469Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360144', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+MCF10A EV rep1', 'SRA', 28406769, 35033347),
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: MCF10A EV', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+MCF10A EV rep2Xx-xXX18743_CAF.RNA.vec.EV.2.8.29.17_S4_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839377', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975468Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360145', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+MCF10A EV rep2', 'SRA', 27892382, 34380835),
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: MCF10A/miR-105 EV', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+MCF10A/miR-105 EV rep1Xx-xXX18744_CAF.RNA.105.EV.1.8.29.17_S5_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839378', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975467Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360146', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+MCF10A/miR-105 EV rep1', 'SRA', 26658179, 32350465),
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: MCF10A/miR-105 EV', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+MCF10A/miR-105 EV rep2Xx-xXX18745_CAF.RNA.105.EV.2.8.29.17_S6_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839379', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975466Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360147', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+MCF10A/miR-105 EV rep2', 'SRA', 30645505, 37375895);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: MDA-MB-231 EV', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+MDA-MB-231 EV rep1Xx-xXX18746_CAF.RNA.231.EV.1.8.29.17_S7_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839380', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975465Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360148', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+MDA-MB-231 EV rep1', 'SRA', 29028809, 40220051),
(1, 'cell type: Primary CAF from a human breast tumorXx-xXpassage: P5Xx-xXtreatment: MDA-MB-231 EV', '1500 E. Duarte Rd.', 'Duarte', 'USA', 'Molecular and Cellular Biology', 'xwu@coh.org', 'City of Hope National Medical Center', '', 'Xiwei,,Wu', '', '91010', 'Sequences were aligned to hg19 using Tophat2 with default settingsXx-xXGene expression level of Refseq coding genes were quantified using in-house R scriptsXx-xXCount data were normalized using \"edgeR\"Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: edgeR normalized counts', 0, 'CAF+MDA-MB-231 EV rep2Xx-xXX18747_CAF.RNA.231.EV.2.8.29.17_S8_L999_R1_001.fastq.bam.1', 'TRIZOL reagent (Invitrogen) was used to extract total RNA from cells following the manufacturer’s protocols.  RNA pellet was dissolved in RNase-free water, and subjected to library construction and deep sequencing.Xx-xXpoly(A) RNA-seq following Illumima\'s recommended protocol', 'GSM2839381', 'Illumina HiSeq 2500', 'Feb 11 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07975464Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3360149', 'GSE106503Xx-xXGSE106508', 'breast cancer associated fibroblasts (CAF)', 'Public on Feb 11 2018', 'Nov 03 2017', 'NONE', '', 9606, 'CAF+MDA-MB-231 EV rep2', 'SRA', 23224446, 34238789),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_0_1Xx-xXprocessed data column: C0', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977775', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466360Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647797', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C0', 'SRA', 21699302, 31190694),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_1_1Xx-xXprocessed data column: C1', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977776', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466359Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647798', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C1', 'SRA', 24329047, 30464411),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_2_1Xx-xXprocessed data column: C2', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977777', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466358Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647799', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C2', 'SRA', 21316424, 26582487),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_3_1Xx-xXprocessed data column: C3', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977778', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466357Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647800', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C3', 'SRA', 17470929, 21081615),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_4_1Xx-xXprocessed data column: C4', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977779', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466356Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647801', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C4', 'SRA', 20277368, 25110916),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_5_1Xx-xXprocessed data column: C5', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977780', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466355Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647802', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C5', 'SRA', 24071396, 29543145),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_6_1Xx-xXprocessed data column: C6', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977781', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466354Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647803', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C6', 'SRA', 25753668, 32072702),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_7_1Xx-xXprocessed data column: C7', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977782', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466353Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647804', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C7', 'SRA', 22654318, 27635811),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_8_1Xx-xXprocessed data column: C8', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977783', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466352Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647805', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C8', 'SRA', 20142841, 25355277),
(1, 'tissue: breast cancer tissueXx-xXcancer type: Invasive ductal carcinomaXx-xXcancer subtype: Luminal B type', '119 Dandae-ro', 'Cheonan', 'South Korea', '', 'kangk1204@gmail.com', 'Dankook University', '', 'Keunsoo,,Kang', '', '31119', 'Low quality portions of paired-end reads were trimmed using Trim galore (version 0.4.2; https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) with cutadapt (version 1.1.2)Xx-xXSequenced reads were mapped to the human genomes (hg38) using STAR (version 2.5.2b).Xx-xXThe abundances of known genes (GENCODE comprehensive gene annotation, version 27; https://www.gencodegenes.org/releases/current.html) were estimated using Cufflink (version 2.2.1)Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: Text file', 0, 'C_160727_9_1Xx-xXprocessed data column: C9', 'Poly(A) RNA was purified from 1 g total RNA of each sample, and cDNA was synthesized using SuperScript II (Invitrogen).Xx-xXSequencing libraries were prepared using the TruSeq RNA Library preparation kit (Illumina) and sequenced by using HiSeq 2500 (Illumina).', 'GSM2977784', 'Illumina HiSeq 2500', 'Feb 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08466351Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3647806', 'GSE110114', 'Breast cancer tissue', 'Public on Feb 06 2018', 'Feb 05 2018', 'NONE', '', 9606, 'C9', 'SRA', 21644538, 27156644),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620416', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379443Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893014', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_207', 'SRA', 116045, 227932),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620417', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379479Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893015', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_21', 'SRA', 162851, 374344),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620420', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379428Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893018', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_238', 'SRA', 146362, 285295),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620424', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379382Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893022', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_254', 'SRA', 1345740, 2130548),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620425', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379450Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893023', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_260', 'SRA', 109468, 150849),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620426', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379490Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893024', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_28', 'SRA', 188780, 306523),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620427', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379508Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893025', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_284', 'SRA', 120976, 405024);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620431', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379457Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893029', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_32', 'SRA', 337879, 670283),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620433', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379432Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893031', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_341', 'SRA', 197326, 340182),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620434', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379506Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893032', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_363', 'SRA', 373062, 784847),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620435', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379423Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893033', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_366', 'SRA', 117776, 183759),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620437', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379166Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893035', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_388', 'SRA', 116711, 218839),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620439', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379269Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893037', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_411', 'SRA', 783055, 1365484),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620440', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379278Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893038', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_426', 'SRA', 260382, 401118),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620442', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379242Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893040', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_434', 'SRA', 203294, 406968),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620443', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379179Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893041', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_446', 'SRA', 175656, 384946),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620445', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379180Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893043', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_451', 'SRA', 124547, 421073),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620447', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379340Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893045', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_480', 'SRA', 113475, 166745);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620449', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379324Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893047', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_498', 'SRA', 170424, 270766),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620450', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379228Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893048', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_505', 'SRA', 101624, 405862),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620452', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379305Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893050', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_515', 'SRA', 419257, 937337),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620453', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379215Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893051', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_525', 'SRA', 105097, 201536),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620454', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379299Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893052', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_526', 'SRA', 128927, 193302),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620456', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379264Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893054', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_549', 'SRA', 703632, 1387824),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620457', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379288Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893055', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_572', 'SRA', 352544, 497288),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620458', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379195Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893056', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_592', 'SRA', 423515, 695563),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620460', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379304Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893058', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_603', 'SRA', 100015, 313250),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620461', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379323Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893059', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_614', 'SRA', 350562, 561496),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620463', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379230Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893061', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_656', 'SRA', 126819, 206264);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620464', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379184Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893062', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_660', 'SRA', 301574, 549225),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620465', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379341Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893063', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_664', 'SRA', 227572, 444083),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620466', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379258Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893064', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_673', 'SRA', 144677, 253339),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620467', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379333Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893065', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_674', 'SRA', 131509, 271767),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620468', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379301Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893066', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_68', 'SRA', 1036238, 2574162),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620470', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379217Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893068', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_694', 'SRA', 408734, 862839),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620471', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379322Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893069', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_703', 'SRA', 206033, 405494),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620472', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379208Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893070', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_712', 'SRA', 416881, 692161),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620473', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379295Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893071', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_713', 'SRA', 232492, 464240),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620475', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379251Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893073', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_717', 'SRA', 165740, 218219),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620477', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379201Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893075', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_726', 'SRA', 170180, 294192);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620479', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379213Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893077', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_776', 'SRA', 148262, 228056),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620482', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379161Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893080', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_829', 'SRA', 385907, 628240),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620483', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379276Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893081', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_835', 'SRA', 215671, 383395),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620484', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379268Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893082', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_841', 'SRA', 103784, 160175),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620494', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379162Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893092', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_927', 'SRA', 213112, 330701),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620498', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379218Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893096', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_947', 'SRA', 114532, 227714),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620501', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379328Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893099', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_97', 'SRA', 664276, 935989),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620502', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379255Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893100', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_98', 'SRA', 251573, 384897),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_1Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_1.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620503', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379160Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893101', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_1_99', 'SRA', 496612, 763211),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620513', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379205Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893111', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_237', 'SRA', 106848, 185367),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620514', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379254Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893112', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_241', 'SRA', 380554, 560822);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620515', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379174Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893113', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_254', 'SRA', 122428, 190599),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620517', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379282Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893115', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_276', 'SRA', 142034, 233303),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620518', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379270Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893116', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_285', 'SRA', 168341, 247059),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620519', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379183Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893117', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_295', 'SRA', 354236, 482469),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620524', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379315Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893122', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_387', 'SRA', 222706, 323802),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620528', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379260Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893126', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_453', 'SRA', 176523, 301340),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620529', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379289Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893127', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_465', 'SRA', 143779, 307420),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620530', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379294Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893128', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_469', 'SRA', 157155, 223294),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620531', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379169Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893129', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_505', 'SRA', 175694, 267448),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620535', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379233Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893133', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_568', 'SRA', 380902, 837391),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620551', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379320Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893149', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_783', 'SRA', 274087, 532780);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620553', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379312Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893151', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_806', 'SRA', 103002, 179674),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620554', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379156Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893152', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_809', 'SRA', 797358, 1084151),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620555', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379466Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893153', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_822', 'SRA', 109199, 182755),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620558', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379437Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893156', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_898', 'SRA', 169981, 336332),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_2Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_2.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620560', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379519Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893158', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_2_925', 'SRA', 137760, 220860),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620562', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379451Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893160', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_1', 'SRA', 123870, 239677),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620563', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379410Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893161', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_104', 'SRA', 191190, 355804),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620567', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379436Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893165', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_161', 'SRA', 213509, 331128),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620569', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379489Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893167', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_168', 'SRA', 402328, 834760),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620570', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379392Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893168', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_188', 'SRA', 191411, 418925),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620571', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379411Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893169', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_200', 'SRA', 150967, 265853);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620572', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379515Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893170', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_201', 'SRA', 115758, 347718),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620574', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379498Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893172', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_232', 'SRA', 114403, 192814),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620576', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379369Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893174', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_253', 'SRA', 1027756, 2218870),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620577', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379433Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893175', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_257', 'SRA', 254387, 454147),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620581', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379350Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893179', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_341', 'SRA', 337515, 798293),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620582', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379523Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893180', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_349', 'SRA', 116335, 286976),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620585', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379343Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893183', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_435', 'SRA', 145160, 379216),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620586', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379460Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893184', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_454', 'SRA', 157438, 313611),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620587', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379417Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893185', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_491', 'SRA', 113974, 190406),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620589', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379344Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893187', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_504', 'SRA', 160643, 261580),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620592', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379497Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893190', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_527', 'SRA', 170325, 282305);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620593', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379517Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893191', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_541', 'SRA', 226401, 376112),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620595', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379501Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893193', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_573', 'SRA', 111743, 276503),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620596', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379352Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893194', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_576', 'SRA', 145751, 349045),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620597', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379469Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893195', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_590', 'SRA', 283276, 419600),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620599', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379521Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893197', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_638', 'SRA', 124353, 235978),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620602', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379467Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893200', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_693', 'SRA', 134216, 372884),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620603', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379407Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893201', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_694', 'SRA', 247527, 439988),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620606', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379518Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893204', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_744', 'SRA', 114414, 262268),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620607', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379399Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893205', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_803', 'SRA', 117650, 218356),
(1, 'cell type: mixture of U87 human glioma cells and MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_3Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_3.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620613', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379360Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893211', 'GSE66357', 'mixture of U87 human glioma cells and MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_3_875', 'SRA', 202513, 380721),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620621', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379214Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893219', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_128', 'SRA', 155503, 306936);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620623', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379253Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893221', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_151', 'SRA', 178580, 339665),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620624', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379297Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893222', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_161', 'SRA', 122919, 224748),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620626', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379332Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893224', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_190', 'SRA', 113244, 171768),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620629', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379199Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893227', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_203', 'SRA', 111187, 214057),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620631', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379190Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893229', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_23', 'SRA', 197592, 358292),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620635', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379308Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893233', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_258', 'SRA', 804654, 1321035),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620636', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379158Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893234', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_262', 'SRA', 982637, 1415373),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620637', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379263Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893235', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_276', 'SRA', 275231, 405398),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620638', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379327Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893236', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_28', 'SRA', 213783, 398897),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620640', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379325Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893238', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_285', 'SRA', 341865, 547690),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620642', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379206Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893240', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_308', 'SRA', 378977, 698429);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620643', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379176Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893241', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_328', 'SRA', 228464, 352421),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620645', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379196Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893243', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_35', 'SRA', 135171, 327586),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620646', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379365Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893244', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_352', 'SRA', 583470, 827172),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620647', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379424Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893245', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_363', 'SRA', 288618, 470826),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620649', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379513Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893247', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_379', 'SRA', 118240, 163402),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620650', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379393Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893248', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_381', 'SRA', 190820, 315492),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620651', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379386Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893249', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_384', 'SRA', 182783, 315149),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620654', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379356Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893252', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_407', 'SRA', 150989, 364144),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620656', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379363Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893254', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_43', 'SRA', 112730, 229155),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620657', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379504Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893255', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_436', 'SRA', 418850, 1078284),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620658', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379493Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893256', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_464', 'SRA', 121404, 194428);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620659', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379478Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893257', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_481', 'SRA', 323901, 507255),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620660', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379349Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893258', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_483', 'SRA', 135847, 223137),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620661', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379418Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893259', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_486', 'SRA', 161532, 236873),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620662', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379379Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893260', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_493', 'SRA', 159435, 243508),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620665', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379434Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893263', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_518', 'SRA', 548568, 904462),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620668', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379465Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893266', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_544', 'SRA', 308845, 460480),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620669', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379372Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893267', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_559', 'SRA', 104550, 171238),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620670', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379494Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893268', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_587', 'SRA', 128683, 247637),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620675', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379425Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893273', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_624', 'SRA', 111620, 310952),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620679', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379384Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893277', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_671', 'SRA', 255224, 444213),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620681', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379348Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893279', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_700', 'SRA', 470750, 639066);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620683', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379474Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893281', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_715', 'SRA', 172107, 270250),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620684', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379496Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893282', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_718', 'SRA', 915888, 1788475),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620688', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379402Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893286', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_75', 'SRA', 207931, 318608),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620690', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379421Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893288', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_76', 'SRA', 118405, 166102),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620691', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379420Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893289', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_761', 'SRA', 186457, 310231),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620692', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379488Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893290', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_763', 'SRA', 147615, 227800),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620694', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379401Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893292', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_765', 'SRA', 798519, 1532243),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620698', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379367Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893296', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_801', 'SRA', 152127, 297348),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620704', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379522Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893302', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_88', 'SRA', 188730, 300858),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620705', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379458Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893303', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_881', 'SRA', 147814, 199622),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620707', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379239Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893305', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_894', 'SRA', 126837, 231184);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620713', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379310Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893311', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_955', 'SRA', 176584, 282604),
(1, 'cell type: MCF10a human breast cancer cells', '3960 Broadway, Lasker 203AC', 'New York', 'USA', '', 'pas2182@columbia.edu', 'Columbia University', '', 'Peter,A,Sims', '', '10032', 'We collected the set of reads that uniquely mapped to the transcriptome and assigned an address comprised of its cell-identifying barcode, gene, UMI, and mapping position.  We then filtered the reads to identify unique molecules. Reads with identical addresses were collapsed to a single molecule.  In addition, reads with identical cell-identifying barcodes, genes, mapping positions, and with UMIs having a Hamming distance less than or equal to two were collapsed to a single molecule. Finally, because the mapping positions produced by STAR do not necessarily correspond to the beginning of a read, we further considered reads to originate from identical molecules if they had identical genes, cell-identifying barcodes, UMIs with a Hamming distance less than or equal to two, and a mapping position within five bases.Xx-xXTo identify barcodes that correspond to actual individual cells in our device, we filtered the observed cell-identifying barcodes by progressively downsampling the corresponding gene profiles to the same number of total reads and assessing the number of unique molecules detected from each cell-identifying barcode. After excluding cell-identifying barcodes having zero associated molecules, we found the distribution of associated unique molecules to be bimodal, with one small subpopulation having nearly as many unique molecules as reads at low read totals. We found the size of this subpopulation to be in excellent agreement with our device imaging data. We took these 598 profiles to represent the actual individual cells captured in our device with a barcoded bead.Xx-xXWe conducted more detailed analysis on 370 single-cell profiles with the highest coverage in our data set across all five lanes of the microfluidic device. Raw fastq data from read 2 of those 370 cells is provided here. Note that the UMI for each read appears in the comment line of each fastq entry.Xx-xXThe processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules. If two UMIs had a Hamming distance less than three, they were considered to be the same UMI.  If two reads with identical UMIs mapped to the transcriptome to within 6 bases of each other, they were considered identical molecules.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: The processed data files contain the number of molecules counted for each gene based on counting reads with HTSeq and filtering the UMIs to identify unique molecules.', 0, 'PS034_4Xx-xX3\'-end RNA-SeqXx-xXprocessed data file: PS034_R2_4.txt', 'RNA was extracted from individual cells in individual microfluidic chambers following cell lysis by Triton X-100 and freeze-thaw.Xx-xXmRNA from individual cells was reverse transcribed with a primer containing a cell-identifying barcode followed by oligo(dT). Following second strand synthesis using DNA Polymerase I and reagents from the MessageAmp II kit (Ambion), ds-cDNA from all barcoded individual cells was pre-amplified by in vitro transcription using T7 RNA polymerase in a pool. The pools of amplified RNA from each lane of the microfluidic device were individually reverse transcribed using barcoded random hexamers containing both a unique molecular identifier (random 8-base barcode) followed by a lane-identifying barcode (6-base barcode). Illumina adapters were inserted on either end of the library during the two previous reverse transcription steps and were used to then enrich the library by PCR. The pooled library was sequenced on an Illumina NextSeq 500.', 'GSM1620716', 'Illumina NextSeq 500', 'May 21 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03379262Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX893314', 'GSE66357', 'MCF10a human breast cancer cells', 'Public on May 21 2015', 'Feb 27 2015', 'NONE', '', 9606, 'PS034_R2_4_97', 'SRA', 147214, 270218),
(1, 'cell type: Luminal cellsXx-xXcell line: 21NT', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693047', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703666Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035156', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693047/suppl/GSM1693047_21NT.genes.fpkm_tracking.gz', '', 9606, '21NT_RNAseq', 'SRA', 28527766, 40995718),
(1, 'cell type: Basal-like cellsXx-xXcell line: CAL51', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693048', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703667Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035157', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693048/suppl/GSM1693048_CAL51.genes.fpkm_tracking.gz', '', 9606, 'CAL51_RNAseq', 'SRA', 30574993, 51806277),
(1, 'cell type: HeterofusionXx-xXcell line: CAL51-MCF7', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693049', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703668Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035158', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693049/suppl/GSM1693049_CAL51MCF7.genes.fpkm_tracking.gz', '', 9606, 'CAL51MCF7_RNAseq', 'SRA', 26051385, 47141885),
(1, 'cell type: Luminal cellsXx-xXcell line: MDA-MB-453', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693050', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703669Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035159', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693050/suppl/GSM1693050_MDA453.genes.fpkm_tracking.gz', '', 9606, 'MDA453_RNAseq', 'SRA', 33409002, 49838779),
(1, 'cell type: HeterofusionXx-xXcell line: SUM159PT-21NT', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693051', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703670Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035160', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693051/suppl/GSM1693051_SUM15921NT.genes.fpkm_tracking.gz', '', 9606, 'SUM15921NT_RNAseq', 'SRA', 33720240, 48189388),
(1, 'cell type: HeterofusionXx-xXcell line: SUM159CAL51', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693052', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703671Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035161', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693052/suppl/GSM1693052_SUM159CAL51.genes.fpkm_tracking.gz', '', 9606, 'SUM159CAL51_RNAseq', 'SRA', 33032026, 45953284),
(1, 'cell type: HeterofusionXx-xXcell line: SMU159PT-MDA-MB-231', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693053', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703672Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035162', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693053/suppl/GSM1693053_SUM159MDA231.genes.fpkm_tracking.gz', '', 9606, 'SUM159MDA231_RNAseq', 'SRA', 25427868, 39150055),
(1, 'cell type: HeterofusionXx-xXcell line: SUM159PT-MDA-MB-453', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693054', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703673Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035163', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693054/suppl/GSM1693054_SUM159MDA453.genes.fpkm_tracking.gz', '', 9606, 'SUM159MDA453_RNAseq', 'SRA', 39467624, 52548998),
(1, 'cell type: HeterofusionXx-xXcell line: SUM159PT-ZR75-1', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693055', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703680Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035164', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693055/suppl/GSM1693055_SUM159ZR75.genes.fpkm_tracking.gz', '', 9606, 'SUM159ZR75_RNAseq', 'SRA', 40388296, 56724662),
(1, 'cell type: Luminal cellsXx-xXcell line: ZR75-1', '450 Brookline Ave', 'Boston', 'USA', 'Medical Oncology', 'kornelia_polyak@dfci.harvard.edu', 'Dana-Farber Cancer Institute', 'Polyak', 'Kornelia,,Polyak', '617-632-2106', '2215', 'Basecalls performed using CASAVA version 1.7 or 1.8.Xx-xXRNA-Seq raw data was mapped by Tophat to hg19.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: text file of normalized count data (FPKM). the file consists of following columns (tracking_id, class_code,nearest_ref_id, gene_id, gene_short_name, tss_id,locus, length, coverage, FPKM, FPKM_conf_lo, FPKM_conf_hi, FPKM_status)', 0, '', 'polyA RNA were purified using the MACS mRNA isolation kit (Miltenyi Biotec).  PolyA RNA was fragmented by RNA fragmentation reagent (Ambion), and subjected to first strand synthesis by using SupeScript II (Invitrogen) and random hexamer oligos. The second strand was synthesized, and then subjected to end repair, A-tailing, ligation of Illumia paired end adaptor. Following PCR amplification, the final library was purified with Ampure XP beads.', 'GSM1693056', 'Illumina NextSeq 500', 'Jun 10 2015', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03703681Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1035165', 'GSE38548Xx-xXGSE69114', 'Breast cancer cells', 'Public on Jun 09 2015', 'May 21 2015', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM1693nnn/GSM1693056/suppl/GSM1693056_ZR75.genes.fpkm_tracking.gz', '', 9606, 'ZR75_RNAseq', 'SRA', 35403241, 53181185),
(1, 'cell type: T47DXx-xXtreatment: AKT inhibitor MK2206', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2131183', 'Illumina NextSeq 500', 'Jun 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04886743Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1721557', 'GSE80591Xx-xXGSE80594', 'T47D breast cancer cells', 'Public on Jun 14 2016', 'Apr 22 2016', 'NONE', '', 9606, 'MK24h_2_s1_treat', 'SRA', 42857310, 48262945),
(1, 'cell type: T47DXx-xXtreatment: AKT inhibitor MK2206', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2131184', 'Illumina NextSeq 500', 'Jun 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04886744Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1721559', 'GSE80591Xx-xXGSE80594', 'T47D breast cancer cells', 'Public on Jun 14 2016', 'Apr 22 2016', 'NONE', '', 9606, 'MK24h_2_s2_treat', 'SRA', 36171144, 40645415),
(1, 'cell type: T47DXx-xXtreatment: DMSO', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2131185', 'Illumina NextSeq 500', 'Jun 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04886745Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1721561', 'GSE80591Xx-xXGSE80594', 'T47D breast cancer cells', 'Public on Jun 14 2016', 'Apr 22 2016', 'NONE', '', 9606, 'DMSO24h_1_s3_ctrl', 'SRA', 48867317, 54131532),
(1, 'cell type: T47DXx-xXtreatment: DMSO', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, '', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2131186', 'Illumina NextSeq 500', 'Jun 14 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04886746Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1721563', 'GSE80591Xx-xXGSE80594', 'T47D breast cancer cells', 'Public on Jun 14 2016', 'Apr 22 2016', 'NONE', '', 9606, 'DMSO24h_2_s4_ctrl', 'SRA', 34568674, 38017983),
(1, 'cell line name: Hs578TXx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179001', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187199Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808063', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'Hs5sh4plus', 'SRA', 34771001, 39324731),
(1, 'cell line name: BT549Xx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179002', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187200Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808064', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'BT5sh4plus', 'SRA', 31597261, 37932867),
(1, 'cell line name: Hs578TXx-xXdoxycycline added: minus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179003', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187201Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808065', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'Hs5sh4minus', 'SRA', 39748061, 44458824),
(1, 'cell line name: BT549Xx-xXdoxycycline added: minus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179004', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187202Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808066', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'BT5sh4minus', 'SRA', 37960206, 42499890),
(1, 'cell line name: HCC1954Xx-xXdoxycycline added: plus doxXx-xXgene target of shrna: Luciferase', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179005', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187203Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808067', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'HCC1954shLucplus', 'SRA', 34114255, 38731012),
(1, 'cell line name: HCC1954Xx-xXdoxycycline added: minus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179006', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187204Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808068', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'HCC1954sh4minus', 'SRA', 36157126, 41191572),
(1, 'cell line name: Cal51 CRISPR copy neutralXx-xXdoxycycline added: minus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179007', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187205Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808069', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51doubledeltash4minus', 'SRA', 30709669, 34929851),
(1, 'cell line name: Cal51 CRISPR copy neutralXx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179008', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187206Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808070', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51doubledeltash4plus', 'SRA', 35785487, 40777612),
(1, 'cell line name: Cal51Xx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179009', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187207Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808071', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51sh3plus', 'SRA', 44054547, 50452520),
(1, 'cell line name: Cal51Xx-xXdoxycycline added: minus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179010', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187208Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808072', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51sh4minus', 'SRA', 28989419, 32362509),
(1, 'cell line name: Cal51Xx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179011', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187209Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808073', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51sh4plus', 'SRA', 40385802, 46141750),
(1, 'cell line name: Cal51Xx-xXdoxycycline added: plus doxXx-xXgene target of shrna: Luciferase', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179012', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187210Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808074', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51shLucplus', 'SRA', 35011079, 39690244),
(1, 'cell line name: Cal51 CRISPR copy lossXx-xXdoxycycline added: minus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179013', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187211Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808075', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51singledeltash4minus', 'SRA', 48705948, 55022916),
(1, 'cell line name: Cal51 CRISPR copy lossXx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179014', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187212Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808076', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'C51singledeltash4plus', 'SRA', 47015336, 53948890),
(1, 'cell line name: HCC1954Xx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179015', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187213Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808077', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'HCC1954sh4plus', 'SRA', 29416807, 38407598),
(1, 'cell line name: HCC1954Xx-xXdoxycycline added: plus doxXx-xXgene target of shrna: SF3B1', '450 Brookline Ave. Smith 1010', 'Boston', 'USA', '', '', 'Dana-Farber Cancer Institute', 'Rameen Beroukhim', 'Brenton,,Paolella', '', '2215', 'Sequenced reads were trimmed for adaptor sequence, FASTQ files were aligned using TOPHAT v1.4 with parameters \"--mate-inner- dist 300 --mate-std-dev 500 --no-sort-bam --no-convert-bam -p 4\" and aligned to hg19Xx-xXReads Per Kilobase of exon per Megabase of library size (RPKM) were calculated using RNASeqMetrics 44Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include RPKM values for each Sample in .gct file format', 0, '', 'Qiangen RNA easy extraction kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2179016', 'Illumina NextSeq 500', 'Feb 07 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05187214Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1808078', 'GSE81978', 'breast cancer cell line', 'Public on Feb 07 2017', 'May 27 2016', 'NONE', '', 9606, 'HCC1954sh3plus', 'SRA', 23219222, 26622601);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MCF-7Xx-xXcell type: breast cancer cell lineXx-xXpassage: P140-P142Xx-xXreporter gene: Firefly Luciferase (MCF-7)', 'Alkek building N1130, 1 Baylor Plaza', 'Houston', 'USA', 'Breast Center', 'haiw@bcm.edu', 'Baylor College of Medicine', 'Xiang Zhang', 'Hai,,Wang', '', '77030', 'Illumina NextSeq 500 automatically used bcl2fastq2 (version 2.17) for basecalling. FASTQ files were downloaded from BaseSpace.Xx-xXRNA-seq NGS reads derived from xenograft materials contain a mixture of human and mouse reads were separated using Xenome (version 1.0.1).Xx-xXRNA-seq NGS reads were mapped using STAR RNA-seq aligner (version 2.4.1d).Xx-xXRSEM (version 1.2.28) was used to quantify gene and isoform expression.Xx-xXDESeq2 R package is used to normalize the gene/isoform expression matrix.Xx-xXGenome_build: Human: GRCh38.83Xx-xXGenome_build: Mouse: GRCm38.83Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.RData: Rdata object; RSEM read counts.Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.Normalized.RData: Rdata object; DESeq2 normalized gene/isoform expression matrix.', 0, 'Sample 1Xx-xXProcessed data file: BICA_RNAseq.RDataXx-xXProcessed data file: BICA_RNAseq.Normalized.RData', 'After homogenization, total RNA were extracted by Direct-zol RNA miniprep kit. The first and second strand cDNA were prepared by SuperScript III First-Strand Synthesis System and NEBNext mRNA Second Strand Synthesis Module from at least 200ng total RNA for each sample.Xx-xXIllumina Nextera XT DNA Sample Prep Kit was used with 1 ng of dsDNA for the construction of sequencing libraries.', 'GSM2227407', 'Illumina NextSeq 500', 'Jan 20 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05363387Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1901132', 'GSE84114', 'MCF-7 breast cancer cells', 'Public on Jan 20 2017', 'Jul 06 2016', 'NONE', '', 9606, 'MCF-7, rep1', 'SRA', 2866164, 9984579),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancer cell lineXx-xXpassage: P140-P142Xx-xXreporter gene: Firefly Luciferase (MCF-7)', 'Alkek building N1130, 1 Baylor Plaza', 'Houston', 'USA', 'Breast Center', 'haiw@bcm.edu', 'Baylor College of Medicine', 'Xiang Zhang', 'Hai,,Wang', '', '77030', 'Illumina NextSeq 500 automatically used bcl2fastq2 (version 2.17) for basecalling. FASTQ files were downloaded from BaseSpace.Xx-xXRNA-seq NGS reads derived from xenograft materials contain a mixture of human and mouse reads were separated using Xenome (version 1.0.1).Xx-xXRNA-seq NGS reads were mapped using STAR RNA-seq aligner (version 2.4.1d).Xx-xXRSEM (version 1.2.28) was used to quantify gene and isoform expression.Xx-xXDESeq2 R package is used to normalize the gene/isoform expression matrix.Xx-xXGenome_build: Human: GRCh38.83Xx-xXGenome_build: Mouse: GRCm38.83Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.RData: Rdata object; RSEM read counts.Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.Normalized.RData: Rdata object; DESeq2 normalized gene/isoform expression matrix.', 0, 'Sample 2Xx-xXProcessed data file: BICA_RNAseq.RDataXx-xXProcessed data file: BICA_RNAseq.Normalized.RData', 'After homogenization, total RNA were extracted by Direct-zol RNA miniprep kit. The first and second strand cDNA were prepared by SuperScript III First-Strand Synthesis System and NEBNext mRNA Second Strand Synthesis Module from at least 200ng total RNA for each sample.Xx-xXIllumina Nextera XT DNA Sample Prep Kit was used with 1 ng of dsDNA for the construction of sequencing libraries.', 'GSM2227408', 'Illumina NextSeq 500', 'Jan 20 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05363388Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1901133', 'GSE84114', 'MCF-7 breast cancer cells', 'Public on Jan 20 2017', 'Jul 06 2016', 'NONE', '', 9606, 'MCF-7, rep2', 'SRA', 1077390, 2694885),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancer cell lineXx-xXpassage: P140-P142Xx-xXreporter gene: Firefly Luciferase (MCF-7)', 'Alkek building N1130, 1 Baylor Plaza', 'Houston', 'USA', 'Breast Center', 'haiw@bcm.edu', 'Baylor College of Medicine', 'Xiang Zhang', 'Hai,,Wang', '', '77030', 'Illumina NextSeq 500 automatically used bcl2fastq2 (version 2.17) for basecalling. FASTQ files were downloaded from BaseSpace.Xx-xXRNA-seq NGS reads derived from xenograft materials contain a mixture of human and mouse reads were separated using Xenome (version 1.0.1).Xx-xXRNA-seq NGS reads were mapped using STAR RNA-seq aligner (version 2.4.1d).Xx-xXRSEM (version 1.2.28) was used to quantify gene and isoform expression.Xx-xXDESeq2 R package is used to normalize the gene/isoform expression matrix.Xx-xXGenome_build: Human: GRCh38.83Xx-xXGenome_build: Mouse: GRCm38.83Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.RData: Rdata object; RSEM read counts.Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.Normalized.RData: Rdata object; DESeq2 normalized gene/isoform expression matrix.', 0, 'Sample 3Xx-xXProcessed data file: BICA_RNAseq.RDataXx-xXProcessed data file: BICA_RNAseq.Normalized.RData', 'After homogenization, total RNA were extracted by Direct-zol RNA miniprep kit. The first and second strand cDNA were prepared by SuperScript III First-Strand Synthesis System and NEBNext mRNA Second Strand Synthesis Module from at least 200ng total RNA for each sample.Xx-xXIllumina Nextera XT DNA Sample Prep Kit was used with 1 ng of dsDNA for the construction of sequencing libraries.', 'GSM2227409', 'Illumina NextSeq 500', 'Jan 20 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05363389Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1901134', 'GSE84114', 'MCF-7 breast cancer cells', 'Public on Jan 20 2017', 'Jul 06 2016', 'NONE', '', 9606, 'MCF-7, rep3', 'SRA', 825033, 3105886),
(1, 'cell line: MCF-7Xx-xXcell type: breast cancer cell lineXx-xXpassage: P140-P142Xx-xXreporter gene: Firefly Luciferase (MCF-7)', 'Alkek building N1130, 1 Baylor Plaza', 'Houston', 'USA', 'Breast Center', 'haiw@bcm.edu', 'Baylor College of Medicine', 'Xiang Zhang', 'Hai,,Wang', '', '77030', 'Illumina NextSeq 500 automatically used bcl2fastq2 (version 2.17) for basecalling. FASTQ files were downloaded from BaseSpace.Xx-xXRNA-seq NGS reads derived from xenograft materials contain a mixture of human and mouse reads were separated using Xenome (version 1.0.1).Xx-xXRNA-seq NGS reads were mapped using STAR RNA-seq aligner (version 2.4.1d).Xx-xXRSEM (version 1.2.28) was used to quantify gene and isoform expression.Xx-xXDESeq2 R package is used to normalize the gene/isoform expression matrix.Xx-xXGenome_build: Human: GRCh38.83Xx-xXGenome_build: Mouse: GRCm38.83Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.RData: Rdata object; RSEM read counts.Xx-xXSupplementary_files_format_and_content: BICA_RNAseq.Normalized.RData: Rdata object; DESeq2 normalized gene/isoform expression matrix.', 0, 'Sample 4Xx-xXProcessed data file: BICA_RNAseq.RDataXx-xXProcessed data file: BICA_RNAseq.Normalized.RData', 'After homogenization, total RNA were extracted by Direct-zol RNA miniprep kit. The first and second strand cDNA were prepared by SuperScript III First-Strand Synthesis System and NEBNext mRNA Second Strand Synthesis Module from at least 200ng total RNA for each sample.Xx-xXIllumina Nextera XT DNA Sample Prep Kit was used with 1 ng of dsDNA for the construction of sequencing libraries.', 'GSM2227410', 'Illumina NextSeq 500', 'Jan 20 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05363390Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1901135', 'GSE84114', 'MCF-7 breast cancer cells', 'Public on Jan 20 2017', 'Jul 06 2016', 'NONE', '', 9606, 'MCF-7, rep4', 'SRA', 1095840, 3355705),
(1, 'cell line: MCF7Xx-xXtreatment: DMSO', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_MI2.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300040', 'Illumina NextSeq 500', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721617Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071019', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_DMSO_1', 'SRA', 59577160, 86172194),
(1, 'cell line: MCF7Xx-xXtreatment: DMSO', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_MI2.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300041', 'Illumina NextSeq 500', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721616Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071020', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_DMSO_2', 'SRA', 46949980, 68309833),
(1, 'cell line: MCF7Xx-xXtreatment: 1mM of the menin inhibitor MI-2', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_MI2.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300042', 'Illumina NextSeq 500', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721615Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071021', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_MI2_1', 'SRA', 56025347, 79025531),
(1, 'cell line: MCF7Xx-xXtreatment: 1mM of the menin inhibitor MI-2', '450 Brookline Avenue', 'Boston', 'USA', 'Medical Oncology', 'koendreijerink@yahoo.com', 'Dana-Farber Cancer Institute', '', 'Koen,M.A.,Dreijerink', '', '2215', 'Basecalls were performed with FastQC version 0.10.0.Xx-xXRNA-seq reads were aligned to the hg19 genome assembly using TophatXx-xXGenome_build: hg19', 0, 'Expression_transcripts_MI2.txt', 'RNA was extracted using RNeasyXx-xXLibraries were made from total RNA using the Illumina TruSeq Stranded mRNA Library Prep Kit', 'GSM2300043', 'Illumina NextSeq 500', 'Mar 09 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN05721614Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2071022', 'GSE86316', 'MCF-7 breast cancer cells', 'Public on Mar 09 2017', 'Aug 31 2016', 'NONE', '', 9606, 'MCF7_MI2_2', 'SRA', 40560830, 56927871),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-culture', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, 'exosome RNA', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB) without the fragmentation step.', 'GSM2443397', 'Illumina NextSeq 500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198242Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458024', 'GSE93070Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture exosome', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'RNA_Cocx_exoRNA_1', 'SRA', 841560, 7584396),
(1, 'cell line: MRC5 and 1833Xx-xXculture: co-culture', '421 Curie Blvd', 'Philadelphia', 'USA', 'Radiation Oncology', 'yuqiu@upenn.edu', 'University of Pennsylvania', 'Andy Minn Lab', 'Yu,,Qiu', '', '19104', 'Basecalls performed using Illumina CASAVA software.Xx-xXRaw reads were trimmed for adaptor sequence using cutadapt v1.9 (parameter -q 10 -m 30 -O 4)Xx-xXTrimmed reads were mapped to consensus rRNA sequences and RN7SL1 using bowtie2 v2.2.4 to remove rRNA reads and get more accurate count of 7SL RNA (parameter --fr --no-mixed --no-discordant)Xx-xXReads were then aligned with STAR v2.4.0k to GRCm38/mm10 reference genome (parameter --outFilterMultimapNmax 100 --outFilterMismatchNmax 999 --outFilterMismatchNoverLmax 0.06)Xx-xXNon primary alignment and QC failed reads were removed with samtools (parameter -F 772)Xx-xXAligned and filtered reads were counted against GENCODE annotation v21  and RepeatMasker features (UCSC genome browser) using Subread v1.4.6 (parameter -s 2 -minReadOverlap 10)Xx-xXGenome_build: GRCh38Xx-xXSupplementary_files_format_and_content: tab delimited txt file including raw read count for gene features and repeatmasker features.', 0, 'exosome RNA', 'Exosomes were isolated from conditioned media collected at 48-72 hours by serial high speed ultracentrifugation, or using 10% final concentration of polyethylene-glycol and low speed centrifugation. Exosome RNA was extracted with TRIzol.Xx-xXLibrary was prepared by using the NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB) without the fragmentation step.', 'GSM2443398', 'Illumina NextSeq 500', 'Jul 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06198241Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2458025', 'GSE93070Xx-xXGSE93078', 'MRC5 and 1833 (breast cancer) co-culture exosome', 'Public on Jul 01 2017', 'Jan 03 2017', 'NONE', '', 9606, 'RNA_Cocx_exoRNA_2', 'SRA', 602721, 6277005),
(1, 'cell line: SkBr3Xx-xXsubtype: HER2-overexpressing Breast CancerXx-xXmaterial platform: 2D Gel - 10 kPaXx-xXgrowth protocol: SkBr3 cells cultured on a soft 2D gel', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q1', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454730', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220497Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486984', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, '2D Soft - SkBr3', 'SRA', 39572961, 44835635),
(1, 'cell line: SkBr3Xx-xXsubtype: HER2-overexpressing Breast CancerXx-xXmaterial platform: 2D Gel - 33 kPaXx-xXgrowth protocol: SkBr3 cells cultured on a stiff 2D gel', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q2', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454731', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220508Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486985', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, '2D Stiff - SkBr3', 'SRA', 37215765, 44243706),
(1, 'cell line: MDA-MB-231Xx-xXsubtype: Triple Negative Breast CancerXx-xXmaterial platform: 2D Gel - 10 kPaXx-xXgrowth protocol: MDA-MB-231 cells cultured on a soft 2D gel', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q3', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454732', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220507Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486986', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, '2D Soft - 231', 'SRA', 39150841, 44887111),
(1, 'cell line: MDA-MB-231Xx-xXsubtype: Triple Negative Breast CancerXx-xXmaterial platform: 2D Gel - 33 kPaXx-xXgrowth protocol: MDA-MB-231 cells cultured on a stiff 2D gel', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q4', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454733', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220498Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486987', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, '2D Stiff - 231', 'SRA', 33898711, 55356719),
(1, 'cell line: MDA-MB-231Xx-xXsubtype: Triple Negative Breast CancerXx-xXmaterial platform: Mouse XenograftXx-xXgrowth protocol: MDA-MB-231 cells in vivo', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q5', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454734', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220506Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486988', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'Mouse - 231', 'SRA', 30843185, 44619113),
(1, 'cell line: SkBr3Xx-xXsubtype: HER2-overexpressing Breast CancerXx-xXmaterial platform: Mouse XenograftXx-xXgrowth protocol: SkBr3 cells in vivo', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q6', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454735', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220505Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486989', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'Mouse - SkBr3', 'SRA', 21033972, 47902941),
(1, 'cell line: MDA-MB-231Xx-xXsubtype: Triple Negative Breast CancerXx-xXmaterial platform: Spheroids dissociated onto TCPSXx-xXgrowth protocol: MDA-MB-231 cells grown into spheroids for 14 days, then replated onto TCPS', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q7', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454736', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220504Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486990', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'Spheroids-to-TCPS - 231', 'SRA', 34026683, 39233575),
(1, 'cell line: SkBr3Xx-xXsubtype: HER2-overexpressing Breast CancerXx-xXmaterial platform: Spheroids dissociated onto TCPSXx-xXgrowth protocol: SkBr3 cells grown into spheroids for 14 days, then replated onto TCPS', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q8', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454737', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220503Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486991', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'Spheroids-to-TCPS - SkBr3', 'SRA', 24547586, 27982374),
(1, 'cell line: MDA-MB-231Xx-xXsubtype: Triple Negative Breast CancerXx-xXmaterial platform: Spheroids in polyNIPAAMXx-xXgrowth protocol: MDA-MB-231 cells grown into spheroids for 14 days', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q9', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454738', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220502Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486992', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'Spheroids - 231', 'SRA', 48486652, 57218120),
(1, 'cell line: SkBr3Xx-xXsubtype: HER2-overexpressing Breast CancerXx-xXmaterial platform: Spheroids in polyNIPAAMXx-xXgrowth protocol: SkBr3 cells grown into spheroids for 14 days', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q10', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454739', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220501Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486993', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'Spheroids - SkBr3', 'SRA', 40158564, 44990599),
(1, 'cell line: MDA-MB-231Xx-xXsubtype: Triple Negative Breast CancerXx-xXmaterial platform: TCPSXx-xXgrowth protocol: MDA-MB-231 cells cultured on TCPS', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q11', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454740', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220500Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486994', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'TCPS - 231', 'SRA', 52359203, 61117407),
(1, 'cell line: SkBr3Xx-xXsubtype: HER2-overexpressing Breast CancerXx-xXmaterial platform: TCPSXx-xXgrowth protocol: SkBr3 cells cultured on TCPS', '686 N pleasant St, Goessmann 159', 'Amherst', 'USA', 'Chemical Engineering', '', 'University of Massachusetts Amherst', '', 'Alyssa,D,Schwartz', '', '1003', 'Reads were mapped to the reference genome using Tophat2 with a bowtie alignerXx-xXNormalized read counts were determined using cufflinksXx-xXDifferential expression was determined with CuffdiffXx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: fpkm_tracking file was generated using cufflinks to identify Fragments Per Kilobase of transcript per Million mapped reads', 0, 'Associated genes.fkpm_tracking header: q12', 'Total RNA was collected using  GenElute Mammalian total RNA miniprep kit (Sigma)Xx-xXLibraries were prepared following the protocol for the TRUseq LT kit', 'GSM2454741', 'Illumina NextSeq 500', 'Mar 01 2017', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN06220499Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2486995', 'GSE93562', 'Human Breast Cancer Cell Line', 'Public on Mar 01 2017', 'Jan 12 2017', 'NONE', '', 9606, 'TCPS - SkBr3', 'SRA', 41250129, 47046813),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: YesXx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 WT replicate 1', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670874', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247007Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922237', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_WT_repl1', 'SRA', 45151856, 57319531),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: YesXx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 WT replicate 2', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670875', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247006Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922238', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_WT_repl2', 'SRA', 43945756, 56922398),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: YesXx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 WT replicate 3', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670876', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247005Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922239', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_WT_repl3', 'SRA', 44422801, 57112748),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 LTED ESR1_WT replicate 1', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670877', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247004Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922240', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_LTED_ESR1_WT_repl1', 'SRA', 44674627, 56781911),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 LTED ESR1_WT replicate 2', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670878', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247003Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922241', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_LTED_ESR1_WT_repl2', 'SRA', 42937364, 56821642),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: none', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 LTED ESR1_WT replicate 3', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670879', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247002Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922242', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_LTED_ESR1_WT_repl3', 'SRA', 40452605, 56962874),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: Y537C', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 LTED ESR1_Y537C replicate 1', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670880', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247001Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922243', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_LTED_ESR1_Y537C_repl1', 'SRA', 45120861, 57460320),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: Y537C', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 LTED ESR1_Y537C replicate 2', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670881', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07247000Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922244', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_LTED_ESR1_Y537C_repl2', 'SRA', 44452765, 57080612),
(1, 'tissue: breast cancer cell lineXx-xXbiomaterial provider: ATCCXx-xXcell line: MCF7Xx-xXestrogen: No, LTED (long term estrogen deprived)Xx-xXesr1 mutation: Y537C', '123 Old Brompton Road', 'London', 'United Kingdom', 'Breast Cancer Research', 'gschuster@icr.ac.uk', 'Institute of Cancer Research', 'Endocrinology', 'Eugene,,Schuster', '4.42088E+11', 'SW7 3RP', 'Tophat (v2.1) and Cuffdiff (v2.2.1) using default parametersXx-xXGenome_build: Human Reference Genome (assembly hg19)Xx-xXSupplementary_files_format_and_content: Cuffdiff output files', 0, 'MCF7 LTED ESR1_Y537C replicate 3', 'RNA was extracted using the RNeasy kit (Qiagen)Xx-xXLibraries were created after Ribo-zero rRNA removal kit (Illumina) using NEBNext Ultra Directional RNA (NEB) kit', 'GSM2670882', 'Illumina NextSeq 500', 'Jan 12 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07246999Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2922245', 'GSE100075Xx-xXGSE100076', 'breast cancer cell line', 'Public on Jan 12 2018', 'Jun 15 2017', 'NONE', '', 9606, 'MCF7_LTED_ESR1_Y537C_repl3', 'SRA', 44393336, 56640246),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915403', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327211Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541122', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915403/suppl/GSM2915403_E-1_abundance.tsv.gz', '', 9606, 'DSP356_E-1', 'SRA', 19929690, 26728401),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915404', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327210Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541123', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915404/suppl/GSM2915404_E-2_abundance.tsv.gz', '', 9606, 'DSP356_E-2', 'SRA', 12966827, 16452261);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915405', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327209Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541124', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915405/suppl/GSM2915405_E-3_abundance.tsv.gz', '', 9606, 'DSP356_E-3', 'SRA', 16612576, 20802923),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915406', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327208Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541125', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915406/suppl/GSM2915406_ICI-1_abundance.tsv.gz', '', 9606, 'DSP356_ICI-1', 'SRA', 14462257, 18877904),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915407', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327207Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541126', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915407/suppl/GSM2915407_ICI-2_abundance.tsv.gz', '', 9606, 'DSP356_ICI-2', 'SRA', 8623625, 11192419),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915408', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327206Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541127', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915408/suppl/GSM2915408_ICI-3_abundance.tsv.gz', '', 9606, 'DSP356_ICI-3', 'SRA', 19065234, 24835720),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915409', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327205Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541128', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915409/suppl/GSM2915409_OHT-1_abundance.tsv.gz', '', 9606, 'DSP356_OHT-1', 'SRA', 14155423, 17635154),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915410', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327204Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541129', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915410/suppl/GSM2915410_OHT-2_abundance.tsv.gz', '', 9606, 'DSP356_OHT-2', 'SRA', 17427085, 22478249),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915411', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327203Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541130', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915411/suppl/GSM2915411_OHT-3_abundance.tsv.gz', '', 9606, 'DSP356_OHT-3', 'SRA', 16350999, 20953705),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915412', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327202Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541131', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915412/suppl/GSM2915412_V-1_abundance.tsv.gz', '', 9606, 'DSP356_V-1', 'SRA', 20528715, 26537050),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915413', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327201Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541132', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915413/suppl/GSM2915413_V-2_abundance.tsv.gz', '', 9606, 'DSP356_V-2', 'SRA', 18795871, 24360104),
(1, 'cell type: MCF-7 cellsXx-xXchip antibody: N/A', 'CP 6128 Succursale centre ville', 'MONTREAL', 'Canada', 'Biochemistry and Molecular Medicine', 'sylvie.mader@umontreal.ca', 'Université de Montréal', 'Molecular targeting in breast cancer', 'Sylvie,,Mader', '514 3437166', 'H3C 3J7', 'ChIP-seq reads were trimmed from the 3\' end to have a phred score of at least 30. Illumina sequencing adapters were removed from the reads, and all reads were required to have a length of at least 50 bp using Timmomatic. The trimmed reads were mapped onto the hg19 reference genome using the aligner BWA. Properly paired reads were identified as inward-facing reads that are on the same chromosome and separated by a distance less than 600 bp. The number of fragments overlapping each 50 bp window was counted for each library along each standard chromosome and excluding blacklisted regions 2. The starts of adjacent windows were separated by 50 bp.  To remove composition biases between libraries, reads were counted into 2 kbp bins over the genome for each library. Windows were filtered to retain only those with a 4-fold or greater increase in the average abundance above the scaled background estimate.Xx-xXGene expression was computed with Kallisto with default parameters (100 bootstraps) on the reference genome GRCh38 with the annotation of Ensembl v85 (with cdna and rna). Xx-xXGenome_build: hg19 for chipseq and GRCh38 for rnaseqXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the chiseq experiment include counts per 50bp window for each sample and calculated using the csaw bioconductor package  as described aboveXx-xXSupplementary_files_format_and_content: Tab-delimited text file for the rnaseq experiment include abundance estimate for each sample using kallisto as described above', 0, 'messenger RNA', 'Cell pellets were lysed with QIAzol and RNA was extracted per manufacturer\'s instructions.Xx-xXLibraries were prepared according to Roche\'s instructions. KAPA Stranded RNA-seq Library Preparation Kit.', 'GSM2915414', 'Illumina NextSeq 500', 'Jan 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL18573', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08327200Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3541133', 'GSE108883', 'breast cancer cells', 'Public on Jan 09 2018', 'Jan 08 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2915nnn/GSM2915414/suppl/GSM2915414_V-3_abundance.tsv.gz', '', 9606, 'DSP356_V-3', 'SRA', 15013237, 19102043),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: 1mM octyl-2HG', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130384', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883119Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717146', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M72HG1', 'SRA', 18527336, 20922633),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: 1mM octyl-2HG', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130385', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883120Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717147', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M72HG2', 'SRA', 19948684, 22539992),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: 1mM octyl-2HG', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130386', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883121Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717148', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M72HG3', 'SRA', 19499845, 22992708),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: 1mM octyl-2HG', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130387', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883122Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717149', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M72HG4', 'SRA', 17363170, 19852242),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: ADHFE1-overexpression', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130388', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883123Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717150', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7ADH1', 'SRA', 25410388, 27781521),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: ADHFE1-overexpression', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130389', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883124Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717151', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7ADH2', 'SRA', 19779767, 21390112),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: ADHFE1-overexpression', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130390', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883125Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717152', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7ADH4', 'SRA', 16692119, 18198342),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: 1mM Control compound', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130391', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883126Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717153', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7NC2', 'SRA', 11572957, 12846462),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: 1mM Control compound', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130392', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883127Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717154', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7NC3', 'SRA', 17404404, 20086103),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: 1mM Control compound', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130393', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883128Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717155', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7NC4', 'SRA', 16525818, 18911659),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: Untreated', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130394', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883129Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717156', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7UNT1', 'SRA', 14480994, 16299663),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: Untreated', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130395', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883102Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717157', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7UNT2', 'SRA', 14090811, 15322074),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: Untreated', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130396', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883103Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717158', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7UNT3', 'SRA', 16115354, 18005548),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: Control- Empty Vector expression', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130398', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883132Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717160', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7VC1', 'SRA', 17013318, 18403335),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: Control- Empty Vector expression', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130399', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883133Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717161', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7VC2', 'SRA', 19336546, 20827284),
(1, 'cell line: MCF7Xx-xXcell type: Breast cancer cellsXx-xXtreatment: Control- Empty Vector expression', '37 convent drive', 'Bethesda', 'USA', '', 'prachi.mishra@nih.gov', 'National Cancer Institute', 'Laboratory of Human Carcinogenesis', 'Prachi,,Mishra', '3014025667', '20892', 'read alignment: TopHatXx-xXSoftware used to count reads: HT-seqXx-xXGenome_build: Hg19Xx-xXSupplementary_files_format_and_content: text file with raw read counts', 0, 'RNA sequencing data', 'Cells were trypsinised, and RNA was harvested using Trizol reagent. Illumina HiSeq RNA Sample Prep Kit (Cat#FC-122-1001) was used with 1 ug of total RNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2130400', 'Illumina HiSeq 2000', 'Feb 16 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN04883134Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX1717162', 'GSE80537', 'Breast cancer cells', 'Public on Feb 16 2018', 'Apr 21 2016', 'NONE', '', 9606, 'M7VC4', 'SRA', 21173705, 23021315),
(1, 'cell line: MDA-MB-435Xx-xXtissue: BreastXx-xXage: NAXx-xXgenotype: ZCCHC24 CTRL', '650 Charles E. Young Drive South', 'Los Angeles', 'USA', 'Department of Microbiology, Immunology, & Molecular Genetics', '', 'University of California, Los Angeles', 'Xing Lab', 'Yu-Ting,,Tseng', '', '90095', 'Illumina Casava1.7 software used for basecalling.Xx-xXSequenced reads were mapped to Ensembl transcript (release 72) then hg19 whole genome using tophat v1.4.1 with parameters -a 8 -m 0 -I 300000 -p 8 -g 20 --library-type fr-unstranded --initial-read-mismatches 3 --segment-mismatches 2Xx-xXFragments Per Kilobase of exon per Million fragments mapped (FPKM) were calculated using cuffdiff v2.2.0 with uniquely and properly mapped pairs.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: an excel file includes FPKM values for each Sample', 0, 'ZCCHC24 CTRL', '1 ug of total RNA was used for  for RNA-seq library preparation using the TruSeq™ Stranded mRNA LT Sample Prep Kit (Illumina). 100x2 bp paired-end RNA-seq reads were generated on a HiSeq 2000 sequencer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1626701', 'Illumina HiSeq 2000', 'Mar 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03393357Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX906927', 'GSE66632', 'Breast cancer cell line', 'Public on Mar 06 2018', 'Mar 06 2015', 'NONE', '', 9606, 'ZCCHC24 CTRL, rep1', 'SRA', 40585928, 58956720),
(1, 'cell line: MDA-MB-435Xx-xXtissue: BreastXx-xXage: NAXx-xXgenotype: ZCCHC24 CTRL', '650 Charles E. Young Drive South', 'Los Angeles', 'USA', 'Department of Microbiology, Immunology, & Molecular Genetics', '', 'University of California, Los Angeles', 'Xing Lab', 'Yu-Ting,,Tseng', '', '90095', 'Illumina Casava1.7 software used for basecalling.Xx-xXSequenced reads were mapped to Ensembl transcript (release 72) then hg19 whole genome using tophat v1.4.1 with parameters -a 8 -m 0 -I 300000 -p 8 -g 20 --library-type fr-unstranded --initial-read-mismatches 3 --segment-mismatches 2Xx-xXFragments Per Kilobase of exon per Million fragments mapped (FPKM) were calculated using cuffdiff v2.2.0 with uniquely and properly mapped pairs.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: an excel file includes FPKM values for each Sample', 0, 'ZCCHC24 CTRL', '1 ug of total RNA was used for  for RNA-seq library preparation using the TruSeq™ Stranded mRNA LT Sample Prep Kit (Illumina). 100x2 bp paired-end RNA-seq reads were generated on a HiSeq 2000 sequencer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1626702', 'Illumina HiSeq 2000', 'Mar 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03393354Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX906928', 'GSE66632', 'Breast cancer cell line', 'Public on Mar 06 2018', 'Mar 06 2015', 'NONE', '', 9606, 'ZCCHC24 CTRL, rep2', 'SRA', 35130981, 52089160),
(1, 'cell line: MDA-MB-435Xx-xXtissue: BreastXx-xXage: NAXx-xXgenotype: ZCCHC24 CTRL', '650 Charles E. Young Drive South', 'Los Angeles', 'USA', 'Department of Microbiology, Immunology, & Molecular Genetics', '', 'University of California, Los Angeles', 'Xing Lab', 'Yu-Ting,,Tseng', '', '90095', 'Illumina Casava1.7 software used for basecalling.Xx-xXSequenced reads were mapped to Ensembl transcript (release 72) then hg19 whole genome using tophat v1.4.1 with parameters -a 8 -m 0 -I 300000 -p 8 -g 20 --library-type fr-unstranded --initial-read-mismatches 3 --segment-mismatches 2Xx-xXFragments Per Kilobase of exon per Million fragments mapped (FPKM) were calculated using cuffdiff v2.2.0 with uniquely and properly mapped pairs.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: an excel file includes FPKM values for each Sample', 0, 'ZCCHC24 CTRL', '1 ug of total RNA was used for  for RNA-seq library preparation using the TruSeq™ Stranded mRNA LT Sample Prep Kit (Illumina). 100x2 bp paired-end RNA-seq reads were generated on a HiSeq 2000 sequencer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1626703', 'Illumina HiSeq 2000', 'Mar 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03393353Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX906929', 'GSE66632', 'Breast cancer cell line', 'Public on Mar 06 2018', 'Mar 06 2015', 'NONE', '', 9606, 'ZCCHC24 CTRL, rep3', 'SRA', 41054869, 60882757);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: MDA-MB-435Xx-xXtissue: BreastXx-xXage: NAXx-xXgenotype: ZCCHC24 KD', '650 Charles E. Young Drive South', 'Los Angeles', 'USA', 'Department of Microbiology, Immunology, & Molecular Genetics', '', 'University of California, Los Angeles', 'Xing Lab', 'Yu-Ting,,Tseng', '', '90095', 'Illumina Casava1.7 software used for basecalling.Xx-xXSequenced reads were mapped to Ensembl transcript (release 72) then hg19 whole genome using tophat v1.4.1 with parameters -a 8 -m 0 -I 300000 -p 8 -g 20 --library-type fr-unstranded --initial-read-mismatches 3 --segment-mismatches 2Xx-xXFragments Per Kilobase of exon per Million fragments mapped (FPKM) were calculated using cuffdiff v2.2.0 with uniquely and properly mapped pairs.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: an excel file includes FPKM values for each Sample', 0, 'ZCCHC24 KD', '1 ug of total RNA was used for  for RNA-seq library preparation using the TruSeq™ Stranded mRNA LT Sample Prep Kit (Illumina). 100x2 bp paired-end RNA-seq reads were generated on a HiSeq 2000 sequencer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1626704', 'Illumina HiSeq 2000', 'Mar 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03393355Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX906930', 'GSE66632', 'Breast cancer cell line', 'Public on Mar 06 2018', 'Mar 06 2015', 'NONE', '', 9606, 'ZCCHC24 KD, rep1', 'SRA', 36695620, 55768531),
(1, 'cell line: MDA-MB-435Xx-xXtissue: BreastXx-xXage: NAXx-xXgenotype: ZCCHC24 KD', '650 Charles E. Young Drive South', 'Los Angeles', 'USA', 'Department of Microbiology, Immunology, & Molecular Genetics', '', 'University of California, Los Angeles', 'Xing Lab', 'Yu-Ting,,Tseng', '', '90095', 'Illumina Casava1.7 software used for basecalling.Xx-xXSequenced reads were mapped to Ensembl transcript (release 72) then hg19 whole genome using tophat v1.4.1 with parameters -a 8 -m 0 -I 300000 -p 8 -g 20 --library-type fr-unstranded --initial-read-mismatches 3 --segment-mismatches 2Xx-xXFragments Per Kilobase of exon per Million fragments mapped (FPKM) were calculated using cuffdiff v2.2.0 with uniquely and properly mapped pairs.Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: an excel file includes FPKM values for each Sample', 0, 'ZCCHC24 KD', '1 ug of total RNA was used for  for RNA-seq library preparation using the TruSeq™ Stranded mRNA LT Sample Prep Kit (Illumina). 100x2 bp paired-end RNA-seq reads were generated on a HiSeq 2000 sequencer.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM1626705', 'Illumina HiSeq 2000', 'Mar 06 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN03393356Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX906931', 'GSE66632', 'Breast cancer cell line', 'Public on Mar 06 2018', 'Mar 06 2015', 'NONE', '', 9606, 'ZCCHC24 KD, rep2', 'SRA', 39422406, 58678311),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: DMSO for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645630', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184316Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876766', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645630/suppl/GSM2645630_MDA231_Ctrl_8h_Genes_ReadCount.txt.gz', '', 9606, 'MDA231_Ctrl_8hr', 'SRA', 20040667, 21468231),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 40μM MMS for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645632', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184314Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876768', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645632/suppl/GSM2645632_MDA231_MMS_8h_Genes_ReadCount.txt.gz', '', 9606, 'MDA231_MMS_8hr', 'SRA', 31538937, 34120265),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 40μM MMS for 24 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645633', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184324Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876769', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645633/suppl/GSM2645633_MDA231_MMS_24h_Genes_ReadCount.txt.gz', '', 9606, 'MDA231_MMS_24hr', 'SRA', 20618606, 22085883),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 1μM Sorafenib for 24 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645635', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184325Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876771', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645635/suppl/GSM2645635_MDA231_Srfnb_24h_Genes_ReadCount.txt.gz', '', 9606, 'MDA231_Srfnb_24hr', 'SRA', 27874608, 30033157),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 1μM Sorafenib followed by 40μM MMS for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645636', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184312Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876772', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645636/suppl/GSM2645636_MDA231_MMS_Srfnb_8h_Genes_ReadCount.txt.gz', '', 9606, 'MDA231_MMS_Srfnb_8hr', 'SRA', 20205800, 21747218),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 1μM Sorafenib followed by 40μM MMS for 24 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645637', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184311Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876773', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645637/suppl/GSM2645637_MDA231_MMS_Srfnb_24h_Genes_ReadCount.txt.gz', '', 9606, 'MDA231_MMS_Srfnb_24hr', 'SRA', 31586715, 34065673),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: DMSO for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645638', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184318Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876774', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645638/suppl/GSM2645638_MDAMB231_Ctrl_8h_Genes_ReadCount.txt.gz', '', 9606, 'MDAMB231_Ctrl_8hr', 'SRA', 28833705, 31264021),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 40μM MMS for 24 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645641', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184323Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876777', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645641/suppl/GSM2645641_MDAMB231_MMS_24h_Genes_ReadCount.txt.gz', '', 9606, 'MDAMB231_MMS_24hr', 'SRA', 27671216, 29876263),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 1μM Sorafenib for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645642', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184322Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876778', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645642/suppl/GSM2645642_MDAMB231_Srfnb_8h_Genes_ReadCount.txt.gz', '', 9606, 'MDAMB231_Srfnb_8hr', 'SRA', 23700245, 25386601),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 1μM Sorafenib for 24 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645643', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184321Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876779', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645643/suppl/GSM2645643_MDAMB231_Srfnb_24h_Genes_ReadCount.txt.gz', '', 9606, 'MDAMB231_Srfnb_24hr', 'SRA', 21171647, 22718838),
(1, 'cell line: MDA-MB231Xx-xXcell type: Breast cancerXx-xXgrowth protocol: Cells were grown in DMEM supplemented with 10%FBSXx-xXtreatment prototcol: 1μM Sorafenib followed by 40μM MMS for 8 hours', '8403 Floyd Curl Drive', 'San Antonio', 'USA', 'Greehey CCRI', '', 'University of Texas Health Science Center at San Antonio', '', 'Hung-I,Harry,Chen', '', '78229', 'Illumina Casava v1.8.2 software used for base-callingXx-xXSequenced reads were trimmed for adaptor sequence, and masked for low-complexity or low-quality sequence, then mapped to hg19 whole genome using TopHat v2.0.8bXx-xXgene count measurements were obtained using HTSeq-countXx-xXGenome_build: UCSC hg19Xx-xXSupplementary_files_format_and_content: The processed data files are in tabular format.  The first column contains the gene symbols and the second column contain the read counts of the gene using htseq-count', 0, '', 'Total RNA was extracted from sub-confluent 10cm dishes using Qiagen RNeasy kitXx-xXRNA libraries were prepared for sequencing using Illumina TruSeq protocols', 'GSM2645644', 'Illumina HiSeq 2000', 'Apr 26 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07184320Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX2876780', 'GSE99536', 'Breast cancer MDA-MB231', 'Public on Apr 25 2018', 'Jun 01 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2645nnn/GSM2645644/suppl/GSM2645644_MDAMB231_MMS_Srfnb_8h_Genes_ReadCount.txt.gz', '', 9606, 'MDAMB231_MMS_Srfnb_8hr', 'SRA', 22193693, 23975433),
(1, 'cell line: MDA-MB-468Xx-xXtreatment: Control', '26-1, Muraoka-Higashi 2-chome', 'Fujisawa', 'Japan', '', '', 'TAKEDA pharaceutical comapny', '', 'Kenichi,,Iwai', '', '251-8555', 'Illumina mRNA-Seq reads were imported into TopHat v2.0.14 for alignment into BAM files.Xx-xXCufflinks was used to quantify gene expression FPKM values.Xx-xXGenome_build: hg19, GRCh37Xx-xXSupplementary_files_format_and_content: tab-delimited text files include FPKM values', 0, '', 'RNA was harvested using RNAeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2705901', 'Illumina HiSeq 2000', 'May 01 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07358573Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3010354', 'GSE101540', 'Breast cancer cell line', 'Public on May 01 2018', 'Jul 17 2017', 'NONE', '', 9606, 'MB468_0nM', 'SRA', 57956997, 63461129),
(1, 'cell line: MDA-MB-468Xx-xXtreatment: T-025 30nM', '26-1, Muraoka-Higashi 2-chome', 'Fujisawa', 'Japan', '', '', 'TAKEDA pharaceutical comapny', '', 'Kenichi,,Iwai', '', '251-8555', 'Illumina mRNA-Seq reads were imported into TopHat v2.0.14 for alignment into BAM files.Xx-xXCufflinks was used to quantify gene expression FPKM values.Xx-xXGenome_build: hg19, GRCh37Xx-xXSupplementary_files_format_and_content: tab-delimited text files include FPKM values', 0, '', 'RNA was harvested using RNAeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2705902', 'Illumina HiSeq 2000', 'May 01 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07358574Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3010355', 'GSE101540', 'Breast cancer cell line', 'Public on May 01 2018', 'Jul 17 2017', 'NONE', '', 9606, 'MB468_30nM', 'SRA', 55025515, 61150467),
(1, 'cell line: MDA-MB-468Xx-xXtreatment: T-025 100nM', '26-1, Muraoka-Higashi 2-chome', 'Fujisawa', 'Japan', '', '', 'TAKEDA pharaceutical comapny', '', 'Kenichi,,Iwai', '', '251-8555', 'Illumina mRNA-Seq reads were imported into TopHat v2.0.14 for alignment into BAM files.Xx-xXCufflinks was used to quantify gene expression FPKM values.Xx-xXGenome_build: hg19, GRCh37Xx-xXSupplementary_files_format_and_content: tab-delimited text files include FPKM values', 0, '', 'RNA was harvested using RNAeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2705903', 'Illumina HiSeq 2000', 'May 01 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07358577Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3010356', 'GSE101540', 'Breast cancer cell line', 'Public on May 01 2018', 'Jul 17 2017', 'NONE', '', 9606, 'MB468_100nM', 'SRA', 49943666, 57195077),
(1, 'cell line: MDA-MB-468Xx-xXtreatment: T-025 300nM', '26-1, Muraoka-Higashi 2-chome', 'Fujisawa', 'Japan', '', '', 'TAKEDA pharaceutical comapny', '', 'Kenichi,,Iwai', '', '251-8555', 'Illumina mRNA-Seq reads were imported into TopHat v2.0.14 for alignment into BAM files.Xx-xXCufflinks was used to quantify gene expression FPKM values.Xx-xXGenome_build: hg19, GRCh37Xx-xXSupplementary_files_format_and_content: tab-delimited text files include FPKM values', 0, '', 'RNA was harvested using RNAeasy kitXx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2705904', 'Illumina HiSeq 2000', 'May 01 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07358576Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3010357', 'GSE101540', 'Breast cancer cell line', 'Public on May 01 2018', 'Jul 17 2017', 'NONE', '', 9606, 'MB468_300nM', 'SRA', 47247756, 55937415),
(1, 'cell line: MCF-7Xx-xXtissue: mammary gland, breastXx-xXgenotype: knockdown control(iluc)', '22 Qixiangtai Road', 'Tianjin', 'China', '', 'zhyma@tmu.edu.cn', 'Tianjin Medical University', '', 'Zhenyi,,Ma', '86-22-83336533', '300070', 'Illumina Casava 1.8 sofware is used for basecallingXx-xXLow quality reads and adaptor contamination are removed by  ngsQCToolkit-2.3.3 software, then mapped to Homo_sapiens.GRCh38 genome by HISAT softwareXx-xXFPKM and expression difference analysis are performed and calculated by Cuffquant and the Cuffnorm softwareXx-xXGenome_build: Homo_sapiens.GRCh38Xx-xXSupplementary_files_format_and_content: tab-delimited text files include FPKM values', 0, '', 'Cells were collected, flash frozen on dry ice, and RNA was harvested using Trizol reagent. Thermo RevertAid First Strand cDNA Synthesis Kit (Cat#K1622) was used with 1 ug of total mRNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2788349', 'Illumina HiSeq 2500', 'Apr 27 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07674787Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3200841', 'GSE104050', 'breast cancer cell', 'Public on Apr 27 2018', 'Sep 20 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2788nnn/GSM2788349/suppl/GSM2788349_iluc.txt.gz', '', 9606, 'iluc', 'SRA', 17008052, 20297839),
(1, 'cell line: MCF-7Xx-xXtissue: mammary gland, breastXx-xXgenotype: tamoxifen resistant knockdown control(iluc)', '22 Qixiangtai Road', 'Tianjin', 'China', '', 'zhyma@tmu.edu.cn', 'Tianjin Medical University', '', 'Zhenyi,,Ma', '86-22-83336533', '300070', 'Illumina Casava 1.8 sofware is used for basecallingXx-xXLow quality reads and adaptor contamination are removed by  ngsQCToolkit-2.3.3 software, then mapped to Homo_sapiens.GRCh38 genome by HISAT softwareXx-xXFPKM and expression difference analysis are performed and calculated by Cuffquant and the Cuffnorm softwareXx-xXGenome_build: Homo_sapiens.GRCh38Xx-xXSupplementary_files_format_and_content: tab-delimited text files include FPKM values', 0, '', 'Cells were collected, flash frozen on dry ice, and RNA was harvested using Trizol reagent. Thermo RevertAid First Strand cDNA Synthesis Kit (Cat#K1622) was used with 1 ug of total mRNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2788350', 'Illumina HiSeq 2500', 'Apr 27 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07674768Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3200842', 'GSE104050', 'breast cancer cell', 'Public on Apr 27 2018', 'Sep 20 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2788nnn/GSM2788350/suppl/GSM2788350_T_iluc.txt.gz', '', 9606, 'T_iluc', 'SRA', 17637704, 21835091),
(1, 'cell line: MCF-7Xx-xXtissue: mammary gland, breastXx-xXgenotype: tamoxifen resistant knockdown NUPR1', '22 Qixiangtai Road', 'Tianjin', 'China', '', 'zhyma@tmu.edu.cn', 'Tianjin Medical University', '', 'Zhenyi,,Ma', '86-22-83336533', '300070', 'Illumina Casava 1.8 sofware is used for basecallingXx-xXLow quality reads and adaptor contamination are removed by  ngsQCToolkit-2.3.3 software, then mapped to Homo_sapiens.GRCh38 genome by HISAT softwareXx-xXFPKM and expression difference analysis are performed and calculated by Cuffquant and the Cuffnorm softwareXx-xXGenome_build: Homo_sapiens.GRCh38Xx-xXSupplementary_files_format_and_content: tab-delimited text files include FPKM values', 0, '', 'Cells were collected, flash frozen on dry ice, and RNA was harvested using Trizol reagent. Thermo RevertAid First Strand cDNA Synthesis Kit (Cat#K1622) was used with 1 ug of total mRNA for the construction of sequencing libraries.Xx-xXRNA libraries were prepared for sequencing using standard Illumina protocols', 'GSM2788351', 'Illumina HiSeq 2500', 'Apr 27 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN07674788Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3200843', 'GSE104050', 'breast cancer cell', 'Public on Apr 27 2018', 'Sep 20 2017', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM2788nnn/GSM2788351/suppl/GSM2788351_T_ip8.txt.gz', '', 9606, 'T_ip8', 'SRA', 18844648, 22874102),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: DMSO (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106217', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961954Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979816', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106217/suppl/GSM3106217_DMSO_6hr_1-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, 'DMSO (6hr) - 1', 'SRA', 97017971, 104402994),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: DMSO (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106218', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961953Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979817', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106218/suppl/GSM3106218_DMSO_6hr_2-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, 'DMSO (6hr) - 2', 'SRA', 94144517, 102473254),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: DMSO (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106219', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961952Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979818', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106219/suppl/GSM3106219_DMSO_24hr_1-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, 'DMSO (24hr) - 1', 'SRA', 90537576, 98007905),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: DMSO (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106220', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961951Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979819', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106220/suppl/GSM3106220_DMSO_24hr_2-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, 'DMSO (24hr) - 2', 'SRA', 81620186, 88576199),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 3 µM CC-671 (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106221', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961950Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979820', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106221/suppl/GSM3106221_3uM_6hr_1-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '3 µM CC-671 (6hr) - 1', 'SRA', 88585664, 97575278);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 3 µM CC-671 (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106222', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961949Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979821', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106222/suppl/GSM3106222_3uM_6hr_2-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '3 µM CC-671 (6hr) - 2', 'SRA', 91470254, 99900719),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 3 µM CC-671 (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106223', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961948Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979822', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106223/suppl/GSM3106223_3uM_6hr_3-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '3 µM CC-671 (6hr) - 3', 'SRA', 91513128, 99755018),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 3 µM CC-671 (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106224', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961947Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979823', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106224/suppl/GSM3106224_3uM_24hr_1-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '3 µM CC-671 (24hr) - 1', 'SRA', 85956838, 94532452),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 3 µM CC-671 (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106225', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961946Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979824', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106225/suppl/GSM3106225_3uM_24hr_2-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '3 µM CC-671 (24hr) - 2', 'SRA', 88867897, 97427328),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 3 µM CC-671 (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106226', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961945Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979825', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106226/suppl/GSM3106226_3uM_24hr_3-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '3 µM CC-671 (24hr) - 3', 'SRA', 84526835, 93364293),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 10 µM CC-671 (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106227', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961944Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979826', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106227/suppl/GSM3106227_10uM_6hr_1-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '10 µM CC-671 (6hr) - 1', 'SRA', 94750157, 105365758),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 10 µM CC-671 (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106228', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961943Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979827', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106228/suppl/GSM3106228_10uM_6hr_2-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '10 µM CC-671 (6hr) - 2', 'SRA', 89065234, 98254347),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 10 µM CC-671 (6hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106229', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961942Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979828', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106229/suppl/GSM3106229_10uM_6hr_3-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '10 µM CC-671 (6hr) - 3', 'SRA', 86622227, 96343379),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 10 µM CC-671 (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106230', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961941Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979829', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106230/suppl/GSM3106230_10uM_24hr_1-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '10 µM CC-671 (24hr) - 1', 'SRA', 93768989, 103216602),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 10 µM CC-671 (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106231', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961940Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979830', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106231/suppl/GSM3106231_10uM_24hr_2-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '10 µM CC-671 (24hr) - 2', 'SRA', 83909841, 93019997),
(1, 'cell type: Breast cancer cell lineXx-xXpassages: 10Xx-xXcell line: CAL51Xx-xXcytogenetics: Near-diploid karyotype with 28% tetraploidyXx-xXtreatment: 10 µM CC-671 (24hr)', '10300 Campus Point Dr', 'San Diego', 'USA', '', 'dzhu@celgene.com', 'Celgene', '', 'Dan,,Zhu', '8585316028', '92126-1204', 'HTSeq package was used to quantify gene expression level for each gene locus and annotation file used was \"gencode.v18\" (http://www.gencodegenes.org/).Xx-xXDifferential gene expression analysis as a result of treatment was carried out using R package “edgeR”,  genes with multiple test adjustment (“BH”) p.value < 0.05, absolute fold change >=2 are considered significant. A coverage filter (>=1 sample has >=50 read counts) was used to remove low coverage genes.Xx-xXTo identify pathways impacted by compound treatment, Gene Set Enrichment Analysis was carried out MSigDB canonical pathway collection C2.CP genesets (www.broadinstitute.org/gsea/msigdb ) was used, adjust.p.value < 0.05 was considered significant.Xx-xXGlobal splicing effect of the compound treatment was assessed by two measures. First, exon rate (bases mapped to coding region divided by bases aligned) was calculated for each sample, comparison of exon rate between treated and control was carried out by Kruskal.test. Second, PSI (percent of spliced in, Fig S4) of “exon skipping/inclusion” events was calculated. This was done via MISO software (http://genes.mit.edu/burgelab/miso/). PSI values from treated and control samples were compared.Xx-xXTo identify differentially splicing affected by compound treatment, \"DEXSeq\" R package was used. Expression of exon bin  was quantified using HTSeq, annotation database used was \"gencode.v18\" ,  differential exon usage identification was carried out using DEXSeq. Exons with adj.p (“BH”) < 0.05, and absolute fold change >=2 are considered significant. Significant hits were visualized using DEXSeq plotting utilities and genome browser. To prioritize hits of DEXSeq results, following criteria were used to : log2(fold change)>=1,  adj.p value < 0.0001,  exon bin size >= 100,  raw read counts >= 50  for at least one sample and gene expression is not significantly changed by compound treatment.Xx-xXTo identify pathways over-represented by genes whose splicing are changed by treatment, R package “GSEABase” (hypergeometric test) was used, adj.p.value (“BH”)  < 0.05, gene sets used was MSigDB canonical pathway collection C2.CP collection.Xx-xX‘Spliced Transcripts Alignment to a Reference (STAR)’ was used to align reads to human genome hg19 with default settingsXx-xXAll analyses were carried out using statistics software R (www.r-projects.org) unless stated otherwise.Xx-xXGenome_build: GRCh37.73Xx-xXSupplementary_files_format_and_content: Illumina', 0, '', 'RNeasy kit, Poly-A selection.Xx-xXStrand-specific RNA library construction were performed, followed by multiplexing indexed libraries and sequencing on the HiSeq 2500 with 2x100 bp read lengths', 'GSM3106232', 'Illumina HiSeq 2500', 'Apr 23 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN08961939Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX3979831', 'GSE113426', 'Triple negative breast cancer (TNBC) line CAL51', 'Public on Apr 23 2018', 'Apr 20 2018', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM3106nnn/GSM3106232/suppl/GSM3106232_10uM_24hr_3-sortedByName-MarkDup.htseq-count.txt.gz', '', 9606, '10 µM CC-671 (24hr) - 3', 'SRA', 86789095, 95577832),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T1Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127583', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015096Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024006', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T1 FF mRNA-seq (500ng)', 'SRA', 12971770, 16610231),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T1Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127584', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015095Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024007', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T1 FF mRNA-seq (100ng)', 'SRA', 29888795, 37517704),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T1Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127585', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015094Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024008', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T1 FFPE mRNA-seq (500ng)', 'SRA', 14204568, 29297131);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T1Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127586', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015093Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024009', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T1 FFPE mRNA-seq (100ng)', 'SRA', 15944328, 31553134),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: RiboZeroXx-xXstarting material (ng): 500Xx-xXpatient: T1Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127587', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015092Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024010', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T1 FFPE RiboZero (500ng)', 'SRA', 22877466, 115775406),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: RiboZeroXx-xXstarting material (ng): 500Xx-xXpatient: T1Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127588', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015091Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024011', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T1 FF RiboZero (500ng)', 'SRA', 5339322, 22324854),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T2Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127589', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015090Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024012', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T2 FF mRNA-seq (500ng)', 'SRA', 12753168, 14812088),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T2Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127590', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015089Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024013', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T2 FF mRNA-seq (100ng)', 'SRA', 29359450, 34585802),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T2Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127591', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015088Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024015', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T2 FFPE mRNA-seq (100ng)', 'SRA', 11783536, 27739170),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: RiboZeroXx-xXstarting material (ng): 500Xx-xXpatient: T2Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127592', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015087Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024016', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T2 FFPE RiboZero (500ng)', 'SRA', 28784829, 136526855),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: RiboZeroXx-xXstarting material (ng): 500Xx-xXpatient: T2Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127593', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015086Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024017', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T2 FF RiboZero (500ng)', 'SRA', 5639857, 19290362),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T3Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127594', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015085Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024018', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T3 FF mRNA-seq (500ng)', 'SRA', 14915013, 17850316),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T3Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127595', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015084Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024019', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T3 FF mRNA-seq (100ng)', 'SRA', 33714529, 40679089),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T3Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127596', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015083Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024020', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T3 FFPE mRNA-seq (500ng)', 'SRA', 8452657, 22178131),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T3Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127597', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015082Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024021', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T3 FFPE mRNA-seq (100ng)', 'SRA', 6767499, 25219363),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: RiboZeroXx-xXstarting material (ng): 500Xx-xXpatient: T3Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127598', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015081Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024022', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T3 FFPE RiboZero (500ng)', 'SRA', 5983611, 66786894),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFXx-xXprotocol: RiboZeroXx-xXstarting material (ng): 500Xx-xXpatient: T3Xx-xXarchival time: 4-5 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127599', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015121Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024023', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T3 FF RiboZero (500ng)', 'SRA', 7900824, 19630473),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T4Xx-xXarchival time: more than 10 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127600', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015120Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024024', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T4 FFPE mRNA-seq (500ng)', 'SRA', 6835287, 17807739),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T4Xx-xXarchival time: more than 10 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127601', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015119Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024025', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T4 FFPE mRNA-seq (100ng)', 'SRA', 3018844, 24000000),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T5Xx-xXarchival time: more than 10 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127602', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015118Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024026', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T5 FFPE mRNA-seq (500ng)', 'SRA', 11760240, 27221744);
INSERT INTO `sample_annotation` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `reads_aligned`, `reads_total`) VALUES
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T5Xx-xXarchival time: more than 10 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127603', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015117Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024027', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T5 FFPE mRNA-seq (100ng)', 'SRA', 4018893, 18531486),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 500Xx-xXpatient: T6Xx-xXarchival time: more than 10 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127604', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015116Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024028', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T6 FFPE mRNA-seq (500ng)', 'SRA', 5568972, 22341475),
(1, 'tissue: breast cancer tumor sampleXx-xXsample_type: FFPEXx-xXprotocol: mRNA-seqXx-xXstarting material (ng): 100Xx-xXpatient: T6Xx-xXarchival time: more than 10 years', 'Hertzl Street', 'Rehovot', 'Israel', '', '', 'Weizmann', '', 'Noa,,Bossel Ben-Moshe', '', '76100', 'Fragments were mapped to the human genome (hg19) using TopHat version V2.0.5Xx-xXOnly uniquely mapped fragments to the genome were considered and exonic and intronic signals were calculated using HTSeqXx-xXThe signals of the same sample from all lanes were summedXx-xXThe number of fragments obtained for each gene in each sample was normalized to the total number of fragments obtained from this sampleXx-xXThe minimum expression level threshold was set to 5 (log2 scale) to reduce noise (based on data distribution).Xx-xXGenome_build: hg19Xx-xXSupplementary_files_format_and_content: tab-delimited text files include normalized number of reads (log2 scale) for each sample', 0, '', 'FFPE tumor samples were sectioned to 5 micron slices and deparafinized at 90°C for 5 min. Total RNA was extracted using nucleic acid isolation kit (AllPrep® Qiagen) according to the protocol instructions, with the following modifications: DNAse treatment was at 37° for 20 minutes and the column was incubated for 1 min before each washing step with buffer RPE. Samples were eluted in 30 μL purified water. RNA from FF tumor samples was extracted using Trizol. RNA concentrations were determined by Qubit™ fluorometer (Thermofisher scientific) and the RNA quality was determined by measuring the RNA integrity number (RIN) using Agilent TapeStation.Xx-xXTotal RNA from FFPE and FF samples was processed using two protocols: 1. Truseq RNA Sample preparation kit v2 (illumina) (cat# RS-122-2002) for mRNA-seq libraries. Briefly, polyA fraction (mRNA) was purified from 500ng or 100ng of total RNA following by fragmentation and generation of double stranded cDNA. Then, end repair, A base addition, adapter ligation and PCR amplification steps were performed. 2. Truseq Stranded total RNA with Ribo-Zero Gold. Library Protocol: TruSeq® Stranded Total RNA (Illumina) (Cat # RS-122-2301, RS-122-2302) for ribosomal-depletion libraries. Briefly, After rRNA depletion from 500ng total RNA with rRNA  Ribo-Zero Gold removal mix, cDNA was performed followed by second strand synthesis with dUTP instead of dTTP. Then, A base addition, adapter ligation, UDG treatment and PCR amplification steps were performed', 'GSM3127605', 'Illumina HiSeq 2500', 'May 04 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09015115Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4024029', 'GSE113976', 'breast cancer tumor sample', 'Public on May 03 2018', 'May 02 2018', 'NONE', '', 9606, 'T6 FFPE mRNA-seq (100ng)', 'SRA', 1592563, 19238231),
(1, 'cell line: HMLEXx-xXcell type: breast cancer cell lineXx-xXcell status: TGFbeta induced EMT', '333 Cedar street', 'New Haven', 'USA', 'Genetics, Computer Science', 'david.vandijk@yale.edu', 'Yale University', 'Krishnaswamy lab', 'David,,van Dijk', '', '6510', 'ATAC-seq processing using Trimmomatic v0.36, bowtie2, MAPQ, samtools, MACS2, FIMOXx-xXInDrop single cell RNA-seq processing using SEQC v0.1.8rc3Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: ATAC-seq peaks; ATACseq_peaks_motifs_meta.txt with column header information; Count matrices; cb*.txt with indrop barcodes', 0, 'processed data file: HMLE_TGFb.tsv.gz', 'All cell dissociations were performed using TrypLE™ (Ambion, USA) reagent.Xx-xXscRNA-seq libraries were constructed following the InDrop protocol(Klein et al., 2015). ATAC-Seq was done following (Buenrostro et al., 2013).', 'GSM3141012', 'Illumina HiSeq 2500', 'May 15 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09205941Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4080313', 'GSE114397', 'HMLE breast cancer cell line', 'Public on May 15 2018', 'May 14 2018', 'NONE', '', 9606, 'InDrop_HMLE_TGFbeta_day8', 'SRA', 104686182, 263661624),
(1, 'cell line: HMLEXx-xXcell type: breast cancer cell lineXx-xXcell status: TGFbeta induced EMT', '333 Cedar street', 'New Haven', 'USA', 'Genetics, Computer Science', 'david.vandijk@yale.edu', 'Yale University', 'Krishnaswamy lab', 'David,,van Dijk', '', '6510', 'ATAC-seq processing using Trimmomatic v0.36, bowtie2, MAPQ, samtools, MACS2, FIMOXx-xXInDrop single cell RNA-seq processing using SEQC v0.1.8rc3Xx-xXGenome_build: hg38Xx-xXSupplementary_files_format_and_content: ATAC-seq peaks; ATACseq_peaks_motifs_meta.txt with column header information; Count matrices; cb*.txt with indrop barcodes', 0, 'processed data file: HMLE_TGFb.tsv.gz', 'All cell dissociations were performed using TrypLE™ (Ambion, USA) reagent.Xx-xXscRNA-seq libraries were constructed following the InDrop protocol(Klein et al., 2015). ATAC-Seq was done following (Buenrostro et al., 2013).', 'GSM3141013', 'Illumina HiSeq 2500', 'May 15 2018', 'cDNA', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL16791', 'BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN09205940Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX4080314', 'GSE114397', 'HMLE breast cancer cell line', 'Public on May 15 2018', 'May 14 2018', 'NONE', '', 9606, 'InDrop_HMLE_TGFbeta_day10', 'SRA', 99259752, 292870964);
