
-- --------------------------------------------------------

--
-- Table structure for table `newSamples`
--

CREATE TABLE `newSamples` (
  `Sample_channel_count` int(1) DEFAULT NULL,
  `Sample_characteristics_ch1` varchar(117) DEFAULT NULL,
  `Sample_contact_address` varchar(32) DEFAULT NULL,
  `Sample_contact_city` varchar(9) DEFAULT NULL,
  `Sample_contact_country` varchar(7) DEFAULT NULL,
  `Sample_contact_department` varchar(43) DEFAULT NULL,
  `Sample_contact_email` varchar(25) DEFAULT NULL,
  `Sample_contact_institute` varchar(43) DEFAULT NULL,
  `Sample_contact_laboratory` varchar(12) DEFAULT NULL,
  `Sample_contact_name` varchar(16) DEFAULT NULL,
  `Sample_contact_phone` varchar(13) DEFAULT NULL,
  `Sample_contact_zip-postal_code` int(5) DEFAULT NULL,
  `Sample_data_processing` varchar(827) DEFAULT NULL,
  `Sample_data_row_count` int(1) DEFAULT NULL,
  `Sample_description` varchar(89) DEFAULT NULL,
  `Sample_extract_protocol_ch1` varchar(791) DEFAULT NULL,
  `Sample_geo_accession` varchar(9) DEFAULT NULL,
  `Sample_instrument_model` varchar(19) DEFAULT NULL,
  `Sample_last_update_date` varchar(11) DEFAULT NULL,
  `Sample_library_selection` varchar(5) DEFAULT NULL,
  `Sample_library_source` varchar(14) DEFAULT NULL,
  `Sample_library_strategy` varchar(7) DEFAULT NULL,
  `Sample_molecule_ch1` varchar(9) DEFAULT NULL,
  `Sample_organism_ch1` varchar(12) DEFAULT NULL,
  `Sample_platform_id` varchar(8) DEFAULT NULL,
  `Sample_relation` varchar(147) DEFAULT NULL,
  `Sample_series_id` varchar(21) DEFAULT NULL,
  `Sample_source_name_ch1` varchar(25) DEFAULT NULL,
  `Sample_status` varchar(21) DEFAULT NULL,
  `Sample_submission_date` varchar(11) DEFAULT NULL,
  `Sample_supplementary_file_1` varchar(117) DEFAULT NULL,
  `Sample_supplementary_file_2` varchar(10) DEFAULT NULL,
  `Sample_taxid_ch1` int(4) DEFAULT NULL,
  `Sample_title` varchar(28) DEFAULT NULL,
  `Sample_type` varchar(3) DEFAULT NULL,
  `gene_accession` varchar(24) DEFAULT NULL,
  `gene_chromosome` varchar(8) DEFAULT NULL,
  `gene_entrezid` int(6) DEFAULT NULL,
  `gene_hgnc` varchar(10) DEFAULT NULL,
  `gene_name` varchar(47) DEFAULT NULL,
  `gene_refseqid` varchar(12) DEFAULT NULL,
  `gene_synonym` varchar(33) DEFAULT NULL,
  `genes` varchar(7) DEFAULT NULL,
  `reads_aligned` decimal(9,1) DEFAULT NULL,
  `reads_total` decimal(9,1) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8;

--
-- Dumping data for table `newSamples`
--

INSERT INTO `newSamples` (`Sample_channel_count`, `Sample_characteristics_ch1`, `Sample_contact_address`, `Sample_contact_city`, `Sample_contact_country`, `Sample_contact_department`, `Sample_contact_email`, `Sample_contact_institute`, `Sample_contact_laboratory`, `Sample_contact_name`, `Sample_contact_phone`, `Sample_contact_zip-postal_code`, `Sample_data_processing`, `Sample_data_row_count`, `Sample_description`, `Sample_extract_protocol_ch1`, `Sample_geo_accession`, `Sample_instrument_model`, `Sample_last_update_date`, `Sample_library_selection`, `Sample_library_source`, `Sample_library_strategy`, `Sample_molecule_ch1`, `Sample_organism_ch1`, `Sample_platform_id`, `Sample_relation`, `Sample_series_id`, `Sample_source_name_ch1`, `Sample_status`, `Sample_submission_date`, `Sample_supplementary_file_1`, `Sample_supplementary_file_2`, `Sample_taxid_ch1`, `Sample_title`, `Sample_type`, `gene_accession`, `gene_chromosome`, `gene_entrezid`, `gene_hgnc`, `gene_name`, `gene_refseqid`, `gene_synonym`, `genes`, `reads_aligned`, `reads_total`) VALUES
(1, 'cell line: HeLaXx-xXknockdown: ELAVL1/HuR siRNA1 (see Lebedeva et. Al 2011)Xx-xXtime post transfection: 120h', 'Robert Rössle Str. 10 (H. 87)', 'Berlin', 'Germany', 'Systems Biology of Gene Regulatory Elements', 'marvin.jens@mdc-berlin.de', 'Max Delbrueck Center for Molecular Medicine', 'Rajewsky lab', 'Marvin,,Jens', '+493094062989', 13092, 'No adapter removal was performed. Raw reads were converted to fastq and mapped with TopHat, supplying ENSEMBL splice junctions. Resulting alignments were analyzed by cuffdiff to estimate changes in RefSeq gene model expression.', 0, '120h post transfection with anti HuR siRNA1', 'Illumina paired end polyA standard protocol, fragment length ~250nt, stddev ~50nt.', 'GSM741172', 'Illumina HiSeq 2000', 'Mar 11 2015', 'other', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX083304Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00668441', 'GSE29943', 'HeLa ELAVL1/HuR siRNA1 5d', 'Public on Jun 30 2011', 'Jun 13 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM741nnn/GSM741172/suppl/GSM741172_siRNA1_5d.wig.gz', '', 9606, 'HeLa ELAVL1/HuR siRNA1 5d', 'SRA', '', '19q13.43', 1, 'HGNC:5', 'alpha-1-B glycoprotein', 'NM_130786', '', 'A1BG', '59390787.0', '76532846.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742939', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079945Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630508', 'GSE30017', 'brain', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742939/suppl/GSM742939_human_brain.single.bwtout.txt.gz', '', 9606, 'human_brain.single', 'SRA', 'AF271790', '10q11.23', 29974, 'HGNC:24086', 'APOBEC1 complementation factor', 'NM_014576', 'ACF, ASP, ACF64, ACF65, APOBEC1CF', 'A1CF', '279012.0', '281357.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742942', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079948Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630511', 'GSE30017', 'heart', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742942/suppl/GSM742942_human_heart.single.bwtout.txt.gz', '', 9606, 'human_heart.single', 'SRA', 'BX647329, X68728, M11313', '12p13.31', 2, 'HGNC:7', 'alpha-2-macroglobulin', 'NM_000014', 'FWP007, S863-7, CPAMD5', 'A2M', '334358.0', '337376.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742946', 'Illumina HiSeq 2000', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'Reanalyzed by: GSE81474Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079952Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630515', 'GSE30017', 'lymph node', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742946/suppl/GSM742946_human_lymph_node.single.bwtout.txt.gz', '', 9606, 'human_lymph_node.single', 'SRA', 'AK057908', '12p13.31', 144568, 'HGNC:23336', 'alpha-2-macroglobulin like 1', 'NM_144670', 'FLJ25179, p170', 'A2ML1', '336430.0', '339511.0'),
(1, 'cell line: HeLaXx-xXknockdown: mockXx-xXtime post transfection: 120h', 'Robert Rössle Str. 10 (H. 87)', 'Berlin', 'Germany', 'Systems Biology of Gene Regulatory Elements', 'marvin.jens@mdc-berlin.de', 'Max Delbrueck Center for Molecular Medicine', 'Rajewsky lab', 'Marvin,,Jens', '+493094062989', 13092, 'No adapter removal was performed. Raw reads were converted to fastq and mapped with TopHat, supplying ENSEMBL splice junctions. Resulting alignments were analyzed by cuffdiff to estimate changes in RefSeq gene model expression.', 0, 'reference sample collected 5 days after treatment with transfection reagents but no siRNA', 'Illumina paired end polyA standard protocol, fragment length ~250nt, stddev ~50nt.', 'GSM741170', 'Illumina HiSeq 2000', 'May 23 2013', 'other', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX083302Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00668439', 'GSE29943', 'HeLa mock knockdown 5d', 'Public on Jun 30 2011', 'Jun 13 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM741nnn/GSM741170/suppl/GSM741170_mock_5d.wig.gz', '', 9606, 'HeLa mock knockdown 5d', 'SRA', 'M24415', '12p13.31', 3, 'HGNC:8', 'alpha-2-macroglobulin pseudogene 1', 'NG_001067', '', 'A2MP1', '61676490.0', '77598048.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742947', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079953Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630516', 'GSE30017', 'ovary', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742947/suppl/GSM742947_human_ovary.single.bwtout.txt.gz', '', 9606, 'human_ovary.single', 'SRA', '', '1p35.1', 127550, 'HGNC:30005', 'alpha 1,3-galactosyltransferase 2', 'NM_001080438', 'IGBS3S, IGB3S', 'A3GALT2', '435207.0', '442289.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742943', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079949Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630512', 'GSE30017', 'kidney', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742943/suppl/GSM742943_human_kidney.single.bwtout.txt.gz', '', 9606, 'human_kidney.single', 'SRA', '', '22q13.2', 53947, 'HGNC:18149', 'alpha 1,4-galactosyltransferase (P blood group)', 'NM_017436', 'A14GALT, Gb3S, P(k)', 'A4GALT', '303697.0', '306984.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742944', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079950Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630513', 'GSE30017', 'liver', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742944/suppl/GSM742944_human_liver.single.bwtout.txt.gz', '', 9606, 'human_liver.single', 'SRA', 'AF141315', '3q22.3', 51146, 'HGNC:17968', 'alpha-1,4-N-acetylglucosaminyltransferase', 'NM_016161', 'alpha4GnT', 'A4GNT', '321948.0', '323971.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742951', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079957Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630520', 'GSE30017', 'thyroid', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742951/suppl/GSM742951_human_thyroid.single.bwtout.txt.gz', '', 9606, 'human_thyroid.single', 'SRA', 'AJ289841', '12q13.13', 8086, 'HGNC:13666', 'aladin WD repeat nucleoporin', '', '', 'AAAS', '508669.0', '514390.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742940', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079946Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630509', 'GSE30017', 'breast', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742940/suppl/GSM742940_human_breast.single.bwtout.txt.gz', '', 9606, 'human_breast.single', 'SRA', 'AK022451', '12q24.31', 65985, 'HGNC:21298', 'acetoacetyl-CoA synthetase', 'NM_023928', 'FLJ12389, SUR-5, ACSF1', 'AACS', '375149.0', '377731.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742950', 'Illumina HiSeq 2000', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'Reanalyzed by: GSE81474Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079956Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630519', 'GSE30017', 'testes', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742950/suppl/GSM742950_human_testes.single.bwtout.txt.gz', '', 9606, 'human_testes.single', 'SRA', '', '5q35.3', 729522, 'HGNC:18226', 'acetoacetyl-CoA synthetase pseudogene 1', 'NR_024035', '', 'AACSP1', '514548.0', '522107.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742941', 'Illumina HiSeq 2000', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'Reanalyzed by: GSE81474Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079947Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630510', 'GSE30017', 'colon', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742941/suppl/GSM742941_human_colon.single.bwtout.txt.gz', '', 9606, 'human_colon.single', 'SRA', 'L32179', '3q25.1', 13, 'HGNC:17', 'arylacetamide deacetylase', 'NM_001086', 'DAC, CES5A1', 'AADAC', '346632.0', '351059.0'),
(1, 'cell type: hTERT-immortalized retinal pigment epithelialXx-xXcell line: RPE-1 cellsXx-xXtransfection: scrambled siRNA', 'Ramat Aviv', 'Tel Aviv', 'Israel', 'Human Genetics', '', 'Tel Aviv University', '', 'Ran,,Elkon', '', 69494, 'Reads were aligned to the hg18 using bowtie. Wig files summarize read coverage at each genomic ccordinate (hg18)', 0, '3\'-Seq in control transfected RPE-1 cells (siC#2)Xx-xXhg18', 'Total RNA was extracted from either U2OS or RPE-1 cells.  mRNA was isolated, heat fragmented and the fragmented mRNA was then converted to single-stranded cDNA using SuperScript III reverse transcriptase and P7-t25-vn oligo-dT primer. This was followed by second-strand cDNA synthesis and end-repair using T4 DNA polymerase, Klenow DNA polymerase and T4 polynucleotide kinase. After purification, a P5-splinkerette was ligated and the splikerette-ligated cDNA was size-selected for approximately 220 bp long cDNA fragments using the E-Gel iBase Power System from Invitrogen. The final 3\'Seq libraries were then generated by PCR amplification of the linker-ligated cDNA with P5 and P7 primers. After a last purification step the 3\'Seq libraries were sequenced on the Illumina Genome Analyzer.', 'GSM678413', 'Illumina HiSeq 2000', 'May 23 2013', 'other', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX085424Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00690319', 'GSE27452Xx-xXGSE32303', 'RPE-1 cells', 'Public on Apr 15 2012', 'Feb 22 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM678nnn/GSM678413/suppl/GSM678413_RPE1_si.C2_SF_0.56.wig.gz', '', 9606, 'RPE1_siC2_3seq', 'SRA', 'BC065724', '3q25.1', 344752, 'HGNC:24427', 'arylacetamide deacetylase like 2', 'NM_207365', 'MGC72001', 'AADACL2', '23839890.0', '57768922.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742948', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079954Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630517', 'GSE30017', 'prostate', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742948/suppl/GSM742948_human_prostate.single.bwtout.txt.gz', '', 9606, 'human_prostate.single', 'SRA', '', '1p36.21', 126767, 'HGNC:32037', 'arylacetamide deacetylase like 3', 'NM_001103170', 'OTTHUMG00000001887', 'AADACL3', '435973.0', '439739.0'),
(1, 'cell line: HeLaXx-xXknockdown: ELAVL1/HuR  siRNA1 (see Lebedeva et. Al 2011)Xx-xXtime post transfection: 48h', 'Robert Rössle Str. 10 (H. 87)', 'Berlin', 'Germany', 'Systems Biology of Gene Regulatory Elements', 'marvin.jens@mdc-berlin.de', 'Max Delbrueck Center for Molecular Medicine', 'Rajewsky lab', 'Marvin,,Jens', '+493094062989', 13092, 'No adapter removal was performed. Raw reads were converted to fastq and mapped with TopHat, supplying ENSEMBL splice junctions. Resulting alignments were analyzed by cuffdiff to estimate changes in RefSeq gene model expression.', 0, '48h post transfection with anti HuR siRNA1', 'Illumina paired end polyA standard protocol, fragment length ~250nt, stddev ~50nt.', 'GSM741171', 'Illumina HiSeq 2000', 'May 23 2013', 'other', 'transcriptomic', 'RNA-Seq', 'total RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX083303Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00668440', 'GSE29943', 'HeLa ELAVL1/HuR siRNA1 2d', 'Public on Jun 30 2011', 'Jun 13 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM741nnn/GSM741171/suppl/GSM741171_siRNA1_2d.wig.gz', '', 9606, 'HeLa ELAVL1/HuR siRNA1 2d', 'SRA', '', '1p36.21', 343066, 'HGNC:32038', 'arylacetamide deacetylase like 4', 'NM_001013630', 'OTTHUMG00000001889', 'AADACL4', '63746708.0', '80188962.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742952', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079958Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630521', 'GSE30017', 'white blood cells', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742952/suppl/GSM742952_human_wbc.single.bwtout.txt.gz', '', 9606, 'human_wbc.single', 'SRA', '', '3q25.1', 201651, 'HGNC:50305', 'arylacetamide deacetylase pseudogene 1', '', '', 'AADACP1', '594779.0', '601463.0'),
(1, 'cell type: hTERT-immortalized retinal pigment epithelialXx-xXcell line: RPE-1 cellsXx-xXtransfection: scrambled siRNA', 'Ramat Aviv', 'Tel Aviv', 'Israel', 'Human Genetics', '', 'Tel Aviv University', '', 'Ran,,Elkon', '', 69494, 'Reads were aligned to the hg18 using bowtie. Wig files summarize read coverage at each genomic ccordinate (hg18)', 0, '3\'-Seq in control transfected RPE-1 cells (siC#1)Xx-xXhg18', 'Total RNA was extracted from either U2OS or RPE-1 cells.  mRNA was isolated, heat fragmented and the fragmented mRNA was then converted to single-stranded cDNA using SuperScript III reverse transcriptase and P7-t25-vn oligo-dT primer. This was followed by second-strand cDNA synthesis and end-repair using T4 DNA polymerase, Klenow DNA polymerase and T4 polynucleotide kinase. After purification, a P5-splinkerette was ligated and the splikerette-ligated cDNA was size-selected for approximately 220 bp long cDNA fragments using the E-Gel iBase Power System from Invitrogen. The final 3\'Seq libraries were then generated by PCR amplification of the linker-ligated cDNA with P5 and P7 primers. After a last purification step the 3\'Seq libraries were sequenced on the Illumina Genome Analyzer.', 'GSM678412', 'Illumina HiSeq 2000', 'May 23 2013', 'other', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX085423Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00690318', 'GSE27452Xx-xXGSE32303', 'RPE-1 cells', 'Public on Apr 15 2012', 'Feb 22 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM678nnn/GSM678412/suppl/GSM678412_RPE1_si.C1_SF_0.44.wig.gz', '', 9606, 'RPE1_siC1_3seq', 'SRA', 'AF097994', '4q33', 51166, 'HGNC:17929', 'aminoadipate aminotransferase', 'NM_016228', 'KATII, KAT2, KYAT2', 'AADAT', '24462998.0', '63511856.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742937', 'Illumina HiSeq 2000', 'Jun 01 2016', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'Reanalyzed by: GSE81474Xx-xXSRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079943Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630506', 'GSE30017', 'adipose', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742937/suppl/GSM742937_human_adipose.single.bwtout.txt.gz', '', 9606, 'human_adipose.single', 'SRA', 'BK000255', '9q22.33', 195827, 'HGNC:16881', 'AhpC/TSA antioxidant enzyme domain containing 1', 'NM_153698', '', 'AAED1', '396843.0', '401003.0'),
(1, 'library type: single-endXx-xXread length: 50', '77 Massachusetts Avenue, 68-223A', 'Cambridge', 'USA', 'Biology', '', 'MIT', '', 'Robert,K,Bradley', '', 2139, 'Alignment: The files human.nagnag.junctions.gff and mouse.nagnag.junctions.gff (see Series supplementary files) give the intron-proximal and intron-distal splice junctions of all NAGNAG splicing events analyzed in this study. The \'start\' field specifies the last base of the upstream exon and the \'end\' field specifies the first base of the downstream exon (upstream and downstream defined with respect to the gene). For each NAGNAG, we extracted the transcript sequence flanking the proximal and distal 3\' splice sites and used Bowtie version 0.12.7 to map reads to these two sequences. We required that short reads have at least 6 nt on either side of the splice junction (an \"overhang\" of 6 nt), and furthermore that there be no mismatches within the overhang region.Xx-xXHuman genome build: hg19Xx-xXMouse genome build: mm9', 0, '', 'Human: RNA-Seq libraries are from the Body Map 2.0 dataset distributed by Illumina, available at the ENA archive with accession number ERP000546.Xx-xXMouse: Tissues from a 10-week-old male were extracted immediately after death and stored in RNAlater per the manufacturerâs instructions (Ambion). Tissue was lysed in Trizol and RNA was extracted with Qiagen miRNeasy mini columns. Using 5ug of total RNA, we performed polyA selection and prepared strand-specific libraries for Illumina sequencing following the dUTP protocol and the SPRIworks Fragment library system (Beckman Coulter). We obtained final insert sizes of approximately 160 bp. We sequenced these libraries using the Illumina HiSeq 2000 and the GAIIx machines.', 'GSM742949', 'Illumina HiSeq 2000', 'May 23 2013', 'cDNA', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX079955Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00630518', 'GSE30017', 'skeletal muscle', 'Public on Jan 05 2012', 'Jun 16 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM742nnn/GSM742949/suppl/GSM742949_human_skeletal_muscle.single.bwtout.txt.gz', '', 9606, 'human_skeletal_muscle.single', 'SRA', 'AL136715', '15q23', 79719, 'HGNC:25662', 'alpha and gamma adaptin binding protein', 'NM_024666', 'FLJ11506, p34', 'AAGAB', '523196.0', '525659.0'),
(1, 'cell type: hTERT-immortalized retinal pigment epithelialXx-xXcell line: RPE-1 cellsXx-xXtransfection: PABPN1 - si#2', 'Ramat Aviv', 'Tel Aviv', 'Israel', 'Human Genetics', '', 'Tel Aviv University', '', 'Ran,,Elkon', '', 69494, 'Reads were aligned to the hg18 using bowtie. Wig files summarize read coverage at each genomic ccordinate (hg18)', 0, '3\'-Seq in RPE-1 cells transfected with PABPN1 - si#2Xx-xXhg18', 'Total RNA was extracted from either U2OS or RPE-1 cells.  mRNA was isolated, heat fragmented and the fragmented mRNA was then converted to single-stranded cDNA using SuperScript III reverse transcriptase and P7-t25-vn oligo-dT primer. This was followed by second-strand cDNA synthesis and end-repair using T4 DNA polymerase, Klenow DNA polymerase and T4 polynucleotide kinase. After purification, a P5-splinkerette was ligated and the splikerette-ligated cDNA was size-selected for approximately 220 bp long cDNA fragments using the E-Gel iBase Power System from Invitrogen. The final 3\'Seq libraries were then generated by PCR amplification of the linker-ligated cDNA with P5 and P7 primers. After a last purification step the 3\'Seq libraries were sequenced on the Illumina Genome Analyzer.', 'GSM678414', 'Illumina HiSeq 2000', 'May 23 2013', 'other', 'transcriptomic', 'RNA-Seq', 'polyA RNA', 'Homo sapiens', 'GPL11154', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX085425Xx-xXBioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN00690320', 'GSE27452Xx-xXGSE32303', 'RPE-1 cells', 'Public on Apr 15 2012', 'Feb 22 2011', 'ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM678nnn/GSM678414/suppl/GSM678414_RPE1_si.PABPN1_1_SF_0.41.wig.gz', '', 9606, 'RPE1_PABPN1kd2_3seq', 'SRA', 'AB028971', '2p13.3', 22848, 'HGNC:19679', 'AP2 associated kinase 1', 'NM_014911', 'KIAA1048, DKFZp686K16132', 'AAK1', '24110983.0', '65271695.0');
